<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<title>JabRef References output</title>
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<script type="text/javascript">
<!--
// QuickSearch script for JabRef HTML export 
// Version: 2.0
//
// Copyright (c) 2006-2007, Mark Schenk
//
// Permission to use, copy, modify, and distribute this software for any
// purpose with or without fee is hereby granted, provided that the above
// copyright notice and this permission notice appear in all copies.
//

// Some features:
// + optionally searches Abstracts and Reviews
// + allows RegExp searches
//   e.g. to search for entries between 1980 and 1989, type:  198[0-9]
//   e.g. for any entry ending with 'symmetry', type:  symmetry$
//   e.g. for all reftypes that are books: ^book$, or ^article$
//   e.g. for entries by either John or Doe, type john|doe
// + easy toggling of Abstract/Review

// Acknowledgements:
// + incorporates speed improvements suggested by Jean-Luc Falcone

// Search settings
var searchAbstract = true;
var searchReview = true;

// Speed optimisation introduced some esoteric problems with certain RegExp searches
// e.g. if the previous search is 200[-7] and the next search is 200[4-7] then the search doesn't work properly until the next 'keyup'
// hence the searchOpt can be turned off for RegExp adepts
var searchOpt = true;

if (window.addEventListener) {
	window.addEventListener("load",initSearch,false); }
else if (window.attachEvent) {
	window.attachEvent("onload", initSearch); }

function initSearch() {
	// basic object detection
	if(!document.getElementById || !document.getElementsByTagName) { return; }
	if (!document.getElementById('qstable')||!document.getElementById('qs')) { return; }

	// find QS table and appropriate rows
	searchTable = document.getElementById('qstable');
	var allRows = searchTable.getElementsByTagName('tbody')[0].getElementsByTagName('tr');

	// split all rows into entryRows and infoRows (e.g. abstract, review, bibtex)
	entryRows = new Array();
	infoRows = new Array();

	for (var i=0, k=0, j=0; i<allRows.length;i++) {
		if (allRows[i].className.indexOf('entry') != -1) {
			entryRows[j++] = allRows[i];
		} else {
			infoRows[k++] = allRows[i];		
		}
	}

	//number of entries and rows
	numRows = allRows.length;
	numEntries = entryRows.length;
	numInfo = infoRows.length;

	//find the query field
	qsfield = document.getElementById('qsfield');

	// previous search term; used for speed optimisation
	prevSearch = '';

	//find statistics location
	stats = document.getElementById('stat');
	setStatistics(-1);

	// shows the searchfield
	document.getElementById('qs').style.display = 'block';
	document.getElementById('qsfield').onkeyup = testEvent;
}

function quickSearch(tInput){

	 if (tInput.value.length == 0) {
		showAll();
		setStatistics(-1);
		qsfield.className = '';
		return;
	} else {
		// only search for valid RegExp
		try {
			var searchText = new RegExp(tInput.value,"i")
			closeAllInfo();
			qsfield.className = '';
		}
		catch(err) {
			prevSearch = tInput.value;
			qsfield.className = 'invalidsearch';
			return;
		}
	}
	
	// count number of hits
	var hits = 0;

	// start looping through all entry rows
	for (var i = 0; cRow = entryRows[i]; i++){

		// only show search the cells if it isn't already hidden OR if the search term is getting shorter, then search all
		// some further optimisation is possible: if the search string is getting shorter, and the row is already visible, skip it. Then be careful with hits!
		if(!searchOpt || cRow.className.indexOf('noshow')==-1 || tInput.value.length <= prevSearch.length){
			var found = false; 

			var inCells = cRow.getElementsByTagName('td');
			var numCols = inCells.length;
				
			for (var j=0; j<numCols; j++) {
				cCell = inCells[j];
				var t = cCell.innerText?cCell.innerText:getTextContent(cCell);
				if (t.search(searchText) != -1){ 
					found=true; 
					break;
				} 
			}

			// look for further hits in Abstract and Review
			if(!found) {
				var articleid = cRow.id;
				if(searchAbstract && (abs = document.getElementById('abs_'+articleid))) {
					if (getTextContent(abs).search(searchText) != -1){ found=true; } 
				}
				if(searchReview && (rev = document.getElementById('rev_'+articleid))) {
					if (getTextContent(rev).search(searchText) != -1){ found=true; } 
				}
			}
			
			if(found) {
				cRow.className = 'entry show';
				hits++;
			} else {
				cRow.className = 'entry noshow';
			}
		}
	}

	// update statistics
	setStatistics(hits)
	
	// set previous search value
	prevSearch = tInput.value;
}

function toggleInfo(articleid,info) {

	var entry = document.getElementById(articleid);
	var abs = document.getElementById('abs_'+articleid);
	var rev = document.getElementById('rev_'+articleid);
	var bib = document.getElementById('bib_'+articleid);
	
	if (abs && info == 'abstract') {
		if(abs.className.indexOf('abstract') != -1) {
		abs.className.indexOf('noshow') == -1?abs.className = 'abstract noshow':abs.className = 'abstract';
		}
	} else if (rev && info == 'review') {
		if(rev.className.indexOf('review') != -1) {
		rev.className.indexOf('noshow') == -1?rev.className = 'review noshow':rev.className = 'review';
		}
	} else if (bib && info == 'bibtex') {
		if(bib.className.indexOf('bibtex') != -1) {
		bib.className.indexOf('noshow') == -1?bib.className = 'bibtex noshow':bib.className = 'bibtex';
		}		
	} else { 
		return;
	}

	// check if one or the other is available
	var revshow = false;
	var absshow = false;
	var bibshow = false;
	(abs && abs.className.indexOf('noshow') == -1)? absshow = true: absshow = false;
	(rev && rev.className.indexOf('noshow') == -1)? revshow = true: revshow = false;	
	(bib && bib.className == 'bibtex')? bibshow = true: bibshow = false;
	
	// highlight original entry
	if(entry) {
		if (revshow || absshow || bibshow) {
		entry.className = 'entry highlight show';
		} else {
		entry.className = 'entry show';
		}		
	}
	
	// When there's a combination of abstract/review/bibtex showing, need to add class for correct styling
	if(absshow) {
		(revshow||bibshow)?abs.className = 'abstract nextshow':abs.className = 'abstract';
	} 
	if (revshow) {
		bibshow?rev.className = 'review nextshow': rev.className = 'review';
	}
	
}

function setStatistics (hits) {
	if(hits < 0) { hits=numEntries; }
	if(stats) { stats.firstChild.data = hits + '/' + numEntries}
}

function getTextContent(node) {
	// Function written by Arve Bersvendsen
	// http://www.virtuelvis.com
	
	if (node.nodeType == 3) {
	return node.nodeValue;
	} // text node
	if (node.nodeType == 1) { // element node
	var text = [];
	for (var chld = node.firstChild;chld;chld=chld.nextSibling) {
		text.push(getTextContent(chld));
	}
	return text.join("");
	} return ""; // some other node, won't contain text nodes.
}

function showAll(){
	// first close all abstracts, reviews, etc.
	closeAllInfo();

	for (var i = 0; i < numEntries; i++){
		entryRows[i].className = 'entry show'; 
	}
}

function closeAllInfo(){
	for (var i=0; i < numInfo; i++){
		if (infoRows[i].className.indexOf('noshow') ==-1) {
			infoRows[i].className = infoRows[i].className + ' noshow';
		}
	}
}

function testEvent(e){
	if (!e) var e = window.event;
	quickSearch(this);
}

function clearQS() {
	qsfield.value = '';
	quickSearch(qsfield);
}

function redoQS(){
	showAll();
	quickSearch(qsfield);
}

// Search Settings

function toggleQSettingsDialog() {

	var qssettings = document.getElementById('qssettings');

	if(qssettings.className.indexOf('active')==-1) {
		qssettings.className = 'active';

		var absCheckBox = document.getElementById('searchabs');
		var revCheckBox = document.getElementById('searchrev');
		if(searchAbstract == true) { absCheckBox.checked = 'checked'; }
		if(searchReview == true) { revCheckBox.checked = 'checked'; }

	} else {
		qssettings.className= '';
	}
}

function toggleQSetting(setting) {

	if(setting=='abs') { searchAbstract = !searchAbstract; }
	if(setting=='rev') { searchReview = !searchReview; }

	redoQS()
}
-->
</script>
<style type="text/css">
body { background-color: white; font-family: "Trebuchet MS", Arial, sans-serif; font-size: 12px; line-height: 1.2; padding: 1em; }

#qs { width: auto; border-style: solid; border-color: gray; border-width: 1px 1px 0px 1px; padding: 0.5em 0.5em; display:none; position:relative; }
#qs form { padding: 0px; margin: 0px; }
#qs form p { padding: 0px; margin: 0px; }

.invalidsearch { background-color: red; }

table { border: 1px gray solid; width: 100%; empty-cells: show; }
th, td { border: 1px gray solid; padding: 0.5em; vertical-align: top;  }
td { text-align: left; vertical-align: top; }
th { background-color: #EFEFEF; }

td a, td a:hover { color: navy; font-weight: bold; }

tr.noshow { display: none;}

tr.highlight td { background-color: #F1F1F1; border-top: 2px black solid; font-weight: bold; }
tr.abstract td, tr.review td, tr.bibtex td { background-color: #F1F1F1; border-bottom: 2px black solid; }
tr.nextshow td { border-bottom: 1px gray solid; }

tr.bibtex pre { width: 100%; overflow: auto;}

p.infolinks { margin: 0.5em 0em 0em 0em; padding: 0px; }

#qssettings { padding: 0.5em; position: absolute; top: 0.2em; right: 0.2em; border: 1px gray solid; background-color: white; display: block; }
#qssettings p { font-weight: bold; cursor: pointer; }
#qssettings ul { display: none; list-style-type: none; padding-left: 0; margin: 0; }
#qssettings.active ul { display: block; }
</style>
</head>
<body>

<div id="qs">
	<form action="">
	<p>QuickSearch: <input type="text" name="qsfield" id="qsfield" autocomplete="off" title="Allows plain text as well as RegExp searches" /><input type="button" onclick="clearQS()" value="clear" />&nbsp; Number of matching entries: <span id="stat">0</span>.</p>
	<div id="qssettings">
		<p onclick="toggleQSettingsDialog()">Search Settings</p>
		<ul>
		<li><label><input type="checkbox" id="searchabs" onclick="toggleQSetting('abs')"> search abstract</label></li>
		<li><label><input type="checkbox" id="searchrev" onclick="toggleQSetting('rev')"> search review</label></li>
		</ul>
	</div>
	</form>
</div>
<table id="qstable" border="1">
<thead><tr><th width="20%">Author</th><th width="30%">Title</th><th width="5%">Year</th><th width="30%">Journal/Proceedings</th><th width="10%">Reftype</th><th width="5%">DOI/URL</th></tr></thead>
<tbody><tr id="abe2007" class="entry">
	<td>Abe, N., Abe, H. &amp; Ito, Y.</td>
	<td>Dumbbell-shaped nanocircular RNAs for RNA interference. <p class="infolinks">[<a href="javascript:toggleInfo('abe2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>J Am Chem Soc &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja0754453">DOI</a><a href="http://dx.doi.org/10.1021/ja0754453">URL</a> &nbsp;</td>
</tr>
<tr id="bib_abe2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{abe2007,
  author = {Naoko Abe and Hiroshi Abe and Yoshihiro Ito},
  title = {Dumbbell-shaped nanocircular RNAs for RNA interference.},
  journal = {J Am Chem Soc},
  year = {2007},
  volume = {129},
  number = {49},
  pages = {15108--15109},
  url = {http://dx.doi.org/10.1021/ja0754453},
  doi = {http://dx.doi.org/10.1021/ja0754453}
}
</pre></td>
</tr>
<tr id="abels2005" class="entry">
	<td>Abels, J. A., Moreno-Herrero, F., van der Heijden, T., Dekker, C. &amp; Dekker, N. H.</td>
	<td>Single-molecule measurements of the persistence length of double-stranded RNA. <p class="infolinks">[<a href="javascript:toggleInfo('abels2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('abels2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biophys J &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.104.052811">DOI</a><a href="http://dx.doi.org/10.1529/biophysj.104.052811">URL</a> &nbsp;</td>
</tr>
<tr id="abs_abels2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Over the past few years, it has become increasingly apparent that double-stranded RNA (dsRNA) plays a far greater role in the life cycle of a cell than previously expected. Numerous proteins, including helicases, polymerases, and nucleases interact specifically with the double helix of dsRNA. To understand the detailed nature of these dsRNA-protein interactions, the (bio)chemical, electrostatic, and mechanical properties of dsRNA need to be fully characterized. We present measurements of the persistence length of dsRNA using two different single-molecule techniques: magnetic tweezers and atomic force microscopy. We deduce a mean persistence length for long dsRNA molecules of 63.8 +/- 0.7 nm from force-extension measurements with the magnetic tweezers. We present atomic force microscopy images of dsRNA and demonstrate a new method for analyzing these, which yields an independent, yet consistent value of 62 +/- 2 nm for the persistence length. The introduction of these single-molecule techniques for dsRNA analysis opens the way for real-time, quantitative analysis of dsRNA-protein interactions.</td>
</tr>
<tr id="bib_abels2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{abels2005,
  author = {J. A. Abels and F. Moreno-Herrero and T. van der Heijden and C. Dekker and N. H. Dekker},
  title = {Single-molecule measurements of the persistence length of double-stranded RNA.},
  journal = {Biophys J},
  year = {2005},
  volume = {88},
  number = {4},
  pages = {2737--2744},
  url = {http://dx.doi.org/10.1529/biophysj.104.052811},
  doi = {http://dx.doi.org/10.1529/biophysj.104.052811}
}
</pre></td>
</tr>
<tr id="abraham2008" class="entry">
	<td>Abraham, M., Dror, O. &amp; Nussinov, R.</td>
	<td>Analysis and Classification of RNA Tertiary Structures <p class="infolinks">[<a href="javascript:toggleInfo('abraham2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.853208">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_abraham2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{abraham2008,
  author = {Mira Abraham and Oranit Dror and Ruth Nussinov},
  title = {Analysis and Classification of RNA Tertiary Structures},
  journal = {RNA},
  year = {2008},
  volume = {14},
  pages = {2274-2289},
  doi = {http://dx.doi.org/10.1261/rna.853208}
}
</pre></td>
</tr>
<tr id="abramovitz1997" class="entry">
	<td>Abramovitz, D. L. &amp; Pyle, A. M.</td>
	<td>Remarkable morphological variability of a common RNA folding motif: the GNRATetraloop-receptor interaction <p class="infolinks">[<a href="javascript:toggleInfo('abramovitz1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('abramovitz1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1996.0810">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_abramovitz1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the most common RNA tertiary interactions involves the docking of GNRA hairpin loops into stem-loop structures on other regions of RNA. Domain 5 of the group II intron interacts with Domain 1 through such an interaction, which has been characterized thermodynamically and kinetically for the ai5g intron. Using this system, it was possible to test the morphological tolerances of the GNRA tetraloop involved in tertiary interactions. The data presented herein show that a GNRA tetraloop can still participate in tertiary interaction after being physically cut at any phosphodiester linkage within the loop. The "nicked tetraloop" can be expanded by many nucleotides in either direction and the covalently continuous loop can also be expanded without loss of interaction energy. In the context of the nicked tetraloop, the second nucleotide of the tetraloop sequence can be completely deleted without loss of function. By examining radical alterations in tetraloop structure, this study helps define the minimal sequence and structural requirements of a GNRA motif involved in long-range tertiary interaction. It shows that "tetraloop"-like structures capable of forming tertiary interactions can be imbedded in unexpected contexts, such as internal loops and apparently open structure within RNA. It demonstrates that pentaloops and hexaloops can form the same type of interaction, with almost equal affinity, as a tetraloop. Taken together, these data suggest a more generic term for the GNRA tetraloop-receptor interaction: It is proposed herein that the term "GNRA tetraloop" be replaced by "GNn/RA", where n represents a variable number of nucleotides and / indicates that the loop can be divided and interrupted by other sequences.</td>
</tr>
<tr id="bib_abramovitz1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{abramovitz1997,
  author = {Dana L. Abramovitz and Anna Marie Pyle},
  title = {Remarkable morphological variability of a common RNA folding motif: the GNRATetraloop-receptor interaction},
  journal = {Journal of Molecular Biology},
  year = {1997},
  volume = {266},
  number = {3},
  pages = {493-506},
  doi = {http://dx.doi.org/10.1006/jmbi.1996.0810}
}
</pre></td>
</tr>
<tr id="agris2007" class="entry">
	<td>Agris, P. F., Vendeix, F. A. P. &amp; Graham, W. D.</td>
	<td>tRNA's Wobble Decoding of the Genome: 40 Years of Modification <p class="infolinks">[<a href="javascript:toggleInfo('agris2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('agris2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2006.11.046">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_agris2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The genetic code is degenerate, in that 20 amino acids are encoded by 61 triplet codes. In 1966, Francis Crick hypothesized that the cell's limited number of tRNAs decoded the genome by recognizing more than one codon. The ambiguity of that recognition resided in the third base-pair, giving rise to the Wobble Hypothesis. Post-transcriptional modifications at tRNA's wobble position 34, especially modifications of uridine 34, enable wobble to occur. The Modified Wobble Hypothesis proposed in 1991 that specific modifications of a tRNA wobble nucleoside shape the anticodon architecture in such a manner that interactions were restricted to the complementary base plus a single wobble pairing for amino acids with twofold degenerate codons. However, chemically different modifications at position 34 would expand the ability of a tRNA to read three or even four of the fourfold degenerate codons. One foundation of Crick's Wobble Hypothesis was that a near-constant geometry of canonical base-pairing be maintained in forming all three base-pairs between the tRNA anticodon and mRNA codon on the ribosome. In accepting an aminoacyl-tRNA, the ribosome requires maintenance of a specific geometry for the anticodon-codon base-pairing. However, it is the post-transcriptional modifications at tRNA wobble position 34 and purine 37, 3'-adjacent to the anticodon, that pre-structure the anticodon domain to ensure the correct codon binding. The modifications create both the architecture and the stability needed for decoding through restraints on anticodon stereochemistry and conformational space, and through selective hydrogen bonding. A physicochemical understanding of modified nucleoside contributions to the tRNA anticodon domain architecture and its decoding of the genome has advanced RNA world evolutionary theory, the principles of RNA chemistry, and the application of this knowledge to the introduction of new amino acids to proteins.</td>
</tr>
<tr id="bib_agris2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{agris2007,
  author = {Paul F. Agris and Franck A. P. Vendeix and William D. Graham},
  title = {tRNA's Wobble Decoding of the Genome: 40 Years of Modification},
  journal = {Journal of Molecular Biology},
  year = {2007},
  volume = {366},
  pages = {1-13},
  doi = {http://dx.doi.org/10.1016/j.jmb.2006.11.046}
}
</pre></td>
</tr>
<tr id="albertini2006" class="entry">
	<td>Albertini, A. A. V., Wernimont, A. K., Muziol, T., Ravelli, R. B. G., Clapier, C. R., Schoehn, G., Weissenhorn, W. &amp; Ruigrok, R. W. H.</td>
	<td>Crystal structure of the rabies virus nucleoprotein-RNA complex. <p class="infolinks">[<a href="javascript:toggleInfo('albertini2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('albertini2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1125280">DOI</a><a href="http://dx.doi.org/10.1126/science.1125280">URL</a> &nbsp;</td>
</tr>
<tr id="abs_albertini2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Negative-strand RNA viruses condense their genome into a helical nucleoprotein-RNA complex, the nucleocapsid, which is packed into virions and serves as a template for the RNA-dependent RNA polymerase complex. The crystal structure of a recombinant rabies virus nucleoprotein-RNA complex, organized in an undecameric ring, has been determined at 3.5 angstrom resolution. Polymerization of the nucleoprotein is achieved by domain exchange between protomers, with flexible hinges allowing nucleocapsid formation. The two core domains of the nucleoprotein clamp around the RNA at their interface and shield it from the environment. RNA sequestering by nucleoproteins is likely a common mechanism used by negative-strand RNA viruses to protect their genomes from the innate immune response directed against viral RNA in human host cells at certain stages of an infectious cycle.</td>
</tr>
<tr id="bib_albertini2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{albertini2006,
  author = {Aurélie A V Albertini and Amy K Wernimont and Tadeusz Muziol and Raimond B G Ravelli and Cedric R Clapier and Guy Schoehn and Winfried Weissenhorn and Rob W H Ruigrok},
  title = {Crystal structure of the rabies virus nucleoprotein-RNA complex.},
  journal = {Science},
  year = {2006},
  volume = {313},
  number = {5785},
  pages = {360--363},
  url = {http://dx.doi.org/10.1126/science.1125280},
  doi = {http://dx.doi.org/10.1126/science.1125280}
}
</pre></td>
</tr>
<tr id="andreeva2004" class="entry">
	<td>Andreeva, A., Howorth, D., Brenner, S. E., Hubbard, T. J. P., Chothia, C. &amp; Murzin, A. G.</td>
	<td>SCOP database in 2004: refinements integrate structure and sequence family data. <p class="infolinks">[<a href="javascript:toggleInfo('andreeva2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('andreeva2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkh039">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_andreeva2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Structural Classification of Proteins (SCOP) database is a comprehensive ordering of all proteins of known structure, according to their evolutionary and structural relationships. Protein domains in SCOP are hierarchically classified into families, superfamilies, folds and classes. The continual accumulation of sequence and structural data allows more rigorous analysis and provides important information for understanding the protein world and its evolutionary repertoire. SCOP participates in a project that aims to rationalize and integrate the data on proteins held in several sequence and structure databases. As part of this project, starting with release 1.63, we have initiated a refinement of the SCOP classification, which introduces a number of changes mostly at the levels below superfamily. The pending SCOP reclassification will be carried out gradually through a number of future releases. In addition to the expanded set of static links to external resources, available at the level of domain entries, we have started modernization of the interface capabilities of SCOP allowing more dynamic links with other databases. SCOP can be accessed at http://scop.mrc-lmb.cam.ac.uk/scop.</td>
</tr>
<tr id="bib_andreeva2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{andreeva2004,
  author = {Antonina Andreeva and Dave Howorth and Steven E. Brenner and Tim J. P. Hubbard and Cyrus Chothia and Alexey G. Murzin},
  title = {SCOP database in 2004: refinements integrate structure and sequence family data.},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  pages = {D226-D229},
  doi = {http://dx.doi.org/10.1093/nar/gkh039}
}
</pre></td>
</tr>
<tr id="antypov2005" class="entry">
	<td>Antypov, D., Barbosa, M. C. &amp; Holm, C.</td>
	<td>Incorporation of excluded-volume correlations into Poisson-Boltzmann theory <p class="infolinks">[<a href="javascript:toggleInfo('antypov2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Physical Review E. &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_antypov2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{antypov2005,
  author = {Dmytro Antypov and Marcia C. Barbosa and Christian Holm},
  title = {Incorporation of excluded-volume correlations into Poisson-Boltzmann theory},
  journal = {Physical Review E.},
  year = {2005},
  volume = {71},
  number = {6},
  pages = {1-6}
}
</pre></td>
</tr>
<tr id="arnott1999" class="entry">
	<td>Arnott, S.</td>
	<td>Oxford Handbook of Nucleic Acid Structure <p class="infolinks">[<a href="javascript:toggleInfo('arnott1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_arnott1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{arnott1999,
  author = {Struther Arnott},
  title = {Oxford Handbook of Nucleic Acid Structure},
  publisher = {Oxford Science Publications},
  year = {1999},
  pages = {1-38}
}
</pre></td>
</tr>
<tr id="arnott1973" class="entry">
	<td>Arnott, S., Hukins, D. W. L., Dover, S. D., Fuller, W. &amp; Hodgson, A. R.</td>
	<td>Structures of Synthetic Polynucleotides in the A-RNA and A$'$-RNA Conformations: X-ray Diffraction Analyses of the Molecular Conformations of Polyadenylic acid $ot$ Polyuridylic acid and Polyinosinic acid $ot$ Polycytidylic acid <p class="infolinks">[<a href="javascript:toggleInfo('arnott1973','bibtex')">BibTeX</a>]</p></td>
	<td>1973</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_arnott1973" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{arnott1973,
  author = {Strutter Arnott and D. W. L. Hukins and S. D. Dover and W. Fuller and A. R. Hodgson},
  title = {Structures of Synthetic Polynucleotides in the A-RNA and A$'$-RNA Conformations: X-ray Diffraction Analyses of the Molecular Conformations of Polyadenylic acid $ot$ Polyuridylic acid and Polyinosinic acid $ot$ Polycytidylic acid},
  journal = {Journal of Molecular Biology},
  year = {1973},
  volume = {81},
  pages = {107-122}
}
</pre></td>
</tr>
<tr id="arnott1968" class="entry">
	<td>Arnott, S., W.Fuller, Hodgson, A. &amp; Prutton, I.</td>
	<td>Molecular Conformations and Structure Transitions of RNA Complementary Helices and their Possible Biological Significance <p class="infolinks">[<a href="javascript:toggleInfo('arnott1968','bibtex')">BibTeX</a>]</p></td>
	<td>1968</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_arnott1968" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{arnott1968,
  author = {S. Arnott and W.Fuller and A. Hodgson and I. Prutton},
  title = {Molecular Conformations and Structure Transitions of RNA Complementary Helices and their Possible Biological Significance},
  journal = {Nature},
  year = {1968},
  volume = {220},
  pages = {561-564}
}
</pre></td>
</tr>
<tr id="aufderheyde1990" class="entry">
	<td>AufderHeyde, T. P. E.</td>
	<td>Analyzing chemical data in more than two dimensions: A tutorial on factor and cluster analysis. <p class="infolinks">[<a href="javascript:toggleInfo('aufderheyde1990','bibtex')">BibTeX</a>]</p></td>
	<td>1990</td>
	<td>Journal of Chemical Education &nbsp;</td>
	<td>article</td>
	<td><a href="http://search.jce.divched.org:8081/JCEIndex/FMPro?-db=jceindex.fp5&amp;-lay=wwwform&amp;combo=auf%20der%20heyde&amp;-find=&amp;-format=detail.html&amp;-skip=0&amp;-max=1&amp;-token.2=0&amp;-token.3=10">URL</a> &nbsp;</td>
</tr>
<tr id="bib_aufderheyde1990" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{aufderheyde1990,
  author = {Thomas P. E. AufderHeyde},
  title = {Analyzing chemical data in more than two dimensions: A tutorial on factor and cluster analysis.},
  journal = {Journal of Chemical Education},
  year = {1990},
  volume = {67},
  pages = {461-469},
  url = {http://search.jce.divched.org:8081/JCEIndex/FMPro?-db=jceindex.fp5&amp;-lay=wwwform&amp;combo=auf%20der%20heyde&amp;-find=&amp;-format=detail.html&amp;-skip=0&amp;-max=1&amp;-token.2=0&amp;-token.3=10}
}
</pre></td>
</tr>
<tr id="babcock1994" class="entry">
	<td>Babcock, M. S., Pednault, E. P. D. &amp; Olson, W. K.</td>
	<td>Nucleic Acid Structure Analysis; Mathematics for Local Cartesian and Helical Structure Parameters that are Truly Comparable Between Structures <p class="infolinks">[<a href="javascript:toggleInfo('babcock1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('babcock1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1994.1213">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_babcock1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Analyzing nucleic acid structures in a comparable manner has become increasingly important as the number of solved structures has increased. This paper presents the concepts, mathematics, theorems, and proofs that form the basis of a new program to analyze three-dimensional DNA and RNA structures. The approach taken here provides numerical data in accordance with guidelines set at a 1988 EMBO workshop. Mathematical definitions are provided for all local structural parameters described in the guidelines. The definitions satisfy the guideline requirements while preserving the original physical intuition of the parameters. In particular, the rotational parameters are true rotations based on a simple physical model (net rotation at constant angular velocity), not Euler angles or angles between vectors and planes as is the case with other approaches. As a result, the mathematical definitions are symmetrical with the property that a 5ï¿½ tilt is the same as a 5ï¿½ roll and a 5ï¿½ twist, except that the rotations take place about different axes. In other approaches, a 5ï¿½ tilt can mean a different amount of net rotation than a 5ï¿½ roll or a 5ï¿½ twist. A second unique feature of the mathematics is that it explicitly incorporates the concept of a pivot point, which is the point about which a base in a base-pair rotates as it buckles, propeller twists, and opens. Pivot points enable one to model the physical motion of bases more accurately. As a result, they greatly reduce and/or eliminate the statistical correlations between rotational and translational parameters that arise as mathematically induced artifacts in other approaches. This paper, together with the statistical analysis in the companion paper for determining the locations of the pivot points, provides everything needed to understand the output of the program as it relates to individual structures.</td>
</tr>
<tr id="bib_babcock1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{babcock1994,
  author = {Marla S. Babcock and Edwin P. D. Pednault and Wilma K. Olson},
  title = {Nucleic Acid Structure Analysis; Mathematics for Local Cartesian and Helical Structure Parameters that are Truly Comparable Between Structures},
  journal = {Journal of Molecular Biology},
  year = {1994},
  volume = {237},
  pages = {125-156},
  doi = {http://dx.doi.org/10.1006/jmbi.1994.1213}
}
</pre></td>
</tr>
<tr id="bahar1998" class="entry">
	<td>Bahar, I. &amp; Jernigan, R. L.</td>
	<td>Vibrational dynamics of transfer RNAs: comparison of the free and synthetase-bound forms <p class="infolinks">[<a href="javascript:toggleInfo('bahar1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('bahar1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1998.1978">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_bahar1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The vibrational dynamics of transfer RNAs, both free, and complexed with the cognate synthetase, are analyzed using a model (Gaussian network model) which recently proved to satisfactorily describe the collective motions of folded proteins. The approach is similar to a normal mode analysis, with the major simplification that no residue specificity is taken into consideration, which permits us (i) to cast the problem into an analytical form applicable to biomolecular systems including about 103residues, and (ii) to acquire information on the essential dynamics of such large systems within computational times at least two orders of magnitude shorter than conventional simulations. On a local scale, the fluctuations calculated for yeast tRNAPhe and tRNAAsp in the free state, and for tRNAGlncomplexed with glutaminyl-tRNA synthetase (GlnRS) are in good agreement with the corresponding crystallographic B factors. On a global scale, a hinge-bending region comprising nucleotides U8 to C12 in the D arm, G20 to G22 in the D loop, and m7G46 to C48 in the variable loop (for tRNAPhe), is identified in the free tRNA, conforming with previous observations. The two regions subject to the largest amplitude anticorrelated fluctuations in the free form, i.e. the anticodon region and the acceptor arm are, at the same time, the regions that experience the most severe suppression in their flexibilities upon binding to synthetase, suggesting that their sampling of the conformational space facilitates their recognition by the synthetase. Likewise, examination of the global mode of motion of GlnRS in the complex indicates that residues 40 to 45, 260 to 270, 306 to 314, 320 to 327 and 478 to 485, all of which cluster near the ATP binding site, form a hinge-bending region controlling the cooperative motion, and thereby the catalytic function, of the enzyme. The distal B-barrel and the tRNA acceptor binding domain, on the other hand, are distinguished by their high mobilities in the global modes of motion, a feature typical of recognition sites, also observed for other proteins. Most of the conserved bases and residues of tRNA and GlnRS are severely constrained in the global motions of the molecules, suggesting their having a role in stabilizing and modulating the global motion.</td>
</tr>
<tr id="bib_bahar1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{bahar1998,
  author = {Ivet Bahar and Robert L. Jernigan},
  title = {Vibrational dynamics of transfer RNAs: comparison of the free and synthetase-bound forms},
  journal = {Journal of Molecular Biology},
  year = {1998},
  volume = {281},
  pages = {871-884},
  doi = {http://dx.doi.org/10.1006/jmbi.1998.1978}
}
</pre></td>
</tr>
<tr id="bahar2005" class="entry">
	<td>Bahar, I. &amp; Rader, A.</td>
	<td>Coarse-grained normal mode analysis in structural biology <p class="infolinks">[<a href="javascript:toggleInfo('bahar2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2005.08.007">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_bahar2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{bahar2005,
  author = {Ivet Bahar and AJ Rader},
  title = {Coarse-grained normal mode analysis in structural biology},
  journal = {Current Opinion in Structural Biology},
  year = {2005},
  volume = {15},
  pages = {586-592},
  doi = {http://dx.doi.org/10.1016/j.sbi.2005.08.007}
}
</pre></td>
</tr>
<tr id="sosnick2007" class="entry">
	<td>Baird, N. J., Fang, X., Srividya, N., Pan, T. &amp; Sosnick, T. R.</td>
	<td>Folding of a Universal Ribozyme: The Ribonuclease P RNA <p class="infolinks">[<a href="javascript:toggleInfo('sosnick2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sosnick2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Quarterly Reviews of Biophysics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1017/S0033583507004623">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sosnick2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Ribonuclease P is among the first ribozymes discovered, and is the only ubiquitously occurring ribozyme besides the ribosome. The bacterial RNase P RNA is catalytically active without its protein subunit and has been studied for over two decades as a model system for RNA catalysis, structure and folding. This review focuses on the thermodynamic, kinetic and structural frameworks derived from the folding studies of bacterial RNase P RNA.</td>
</tr>
<tr id="bib_sosnick2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sosnick2007,
  author = {Nathan J. Baird and Xing-Wang Fang and Narayanan Srividya and Tao Pan and Tobin R. Sosnick},
  title = {Folding of a Universal Ribozyme: The Ribonuclease P RNA},
  journal = {Quarterly Reviews of Biophysics},
  year = {2007},
  volume = {40},
  pages = {113-161},
  doi = {http://dx.doi.org/10.1017/S0033583507004623}
}
</pre></td>
</tr>
<tr id="baird2005" class="entry">
	<td>Baird, N. J., Westhof, E., Qin, H., Pan, T. &amp; Sosnick, T. R.</td>
	<td>Structure of a Folding Intermediate Reveals the Interplay Between Core and Peripheral Elements in RNA Folding <p class="infolinks">[<a href="javascript:toggleInfo('baird2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_baird2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{baird2005,
  author = {Nathan J. Baird and Eric Westhof and Hong Qin and Tao Pan and Tobin R. Sosnick},
  title = {Structure of a Folding Intermediate Reveals the Interplay Between Core and Peripheral Elements in RNA Folding},
  journal = {Journal of Molecular Biology},
  year = {2005},
  volume = {352},
  pages = {712-722}
}
</pre></td>
</tr>
<tr id="baker2001" class="entry">
	<td>Baker, N. A., Sept, D., Joseph, S., Holst, M. J. &amp; McCammon, J. A.</td>
	<td>Electrostatics of nanosystems: Application to microtubules and the ribosome <p class="infolinks">[<a href="javascript:toggleInfo('baker2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Proceedings of the National Academy of Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.181342398">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_baker2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{baker2001,
  author = {Nathan A. Baker and David Sept and Simpson Joseph and Michael J. Holst and J. Andrew McCammon},
  title = {Electrostatics of nanosystems: Application to microtubules and the ribosome},
  journal = {Proceedings of the National Academy of Science},
  year = {2001},
  volume = {98},
  pages = {10037-10041},
  doi = {http://dx.doi.org/10.1073/pnas.181342398}
}
</pre></td>
</tr>
<tr id="balasubramanian2009" class="entry">
	<td>Balasubramanian, S., Xu, F. &amp; Olson, W. K.</td>
	<td>DNA sequence-directed organization of chromatin: structure-based computational analysis of nucleosome-binding sequences. <p class="infolinks">[<a href="javascript:toggleInfo('balasubramanian2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('balasubramanian2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.bpj.2008.11.040">DOI</a><a href="http://dx.doi.org/10.1016/j.bpj.2008.11.040">URL</a> &nbsp;</td>
</tr>
<tr id="abs_balasubramanian2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The folding of DNA on the nucleosome core particle governs many fundamental issues in eukaryotic molecular biology. In this study, an updated set of sequence-dependent empirical "energy" functions, derived from the structures of other protein-bound DNA molecules, is used to investigate the extent to which the architecture of nucleosomal DNA is dictated by its underlying sequence. The potentials are used to estimate the cost of deforming a collection of sequences known to bind or resist uptake in nucleosomes along various left-handed superhelical pathways and to deduce the features of sequence contributing to a particular structural form. The deformation scores reflect the choice of template, the deviations of structural parameters at each step of the nucleosome-bound DNA from their intrinsic values, and the sequence-dependent "deformability" of a given dimer. The correspondence between the computed scores and binding propensities points to a subtle interplay between DNA sequence and nucleosomal folding, e.g., sequences with periodically spaced pyrimidine-purine steps deform at low cost along a kinked template whereas sequences that resist deformation prefer a smoother spatial pathway. Successful prediction of the known settings of some of the best-resolved nucleosome-positioning sequences, however, requires a template with "kink-and-slide" steps like those found in high-resolution nucleosome structures.</td>
</tr>
<tr id="bib_balasubramanian2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{balasubramanian2009,
  author = {Sreekala Balasubramanian and Fei Xu and Wilma K Olson},
  title = {DNA sequence-directed organization of chromatin: structure-based computational analysis of nucleosome-binding sequences.},
  journal = {Biophysical Journal},
  year = {2009},
  volume = {96},
  number = {6},
  pages = {2245--2260},
  url = {http://dx.doi.org/10.1016/j.bpj.2008.11.040},
  doi = {http://dx.doi.org/10.1016/j.bpj.2008.11.040}
}
</pre></td>
</tr>
<tr id="ban2000" class="entry">
	<td>Ban, N., Nissen, P., Hansen, J., Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The Complete Atomic Structure of the Large Ribosomal Subunit at 2.4 &Aring; Resolution <p class="infolinks">[<a href="javascript:toggleInfo('ban2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('ban2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.289.5481.905">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_ban2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The large ribosomal subunit catalyzes peptide bond formation and binds initiation, termination, and elongation factors. We have determined the crystal structure of the large ribosomal subunit from Haloarcula marismortui at 2.4 angstrom resolution, and it includes 2833 of the subunit's 3045 nucleotides and 27 of its 31 proteins. The domains of its RNAs all have irregular shapes and fit together in the ribosome like the pieces of a three-dimensional jigsaw puzzle to form a large, monolithic structure. Proteins are abundant everywhere on its surface except in the active site where peptide bond formation occurs and where it contacts the small subunit. Most of the proteins stabilize the structure by interacting with several RNA domains, often using idiosyncratically folded extensions that reach into the subunit's interior.</td>
</tr>
<tr id="bib_ban2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ban2000,
  author = {Nenad Ban and Poul Nissen and Jeffrey Hansen and Peter B. Moore and Thomas A. Steitz},
  title = {The Complete Atomic Structure of the Large Ribosomal Subunit at 2.4 &Aring; Resolution},
  journal = {Science},
  year = {2000},
  volume = {289},
  number = {5481},
  pages = {905-920},
  doi = {http://dx.doi.org/10.1126/science.289.5481.905}
}
</pre></td>
</tr>
<tr id="banerjee2007" class="entry">
	<td>Banerjee, A., Misra, M., Pai, D., Shih, L., Woodley, R., Lu, X., Srinivasan, A. R., Olson, W. K., Dave, R. N. &amp; Venanzi, C. A.</td>
	<td>Feature Extraction Using Molecular Planes for Fuzzy Relational Clustering of a Flexible Dopamine Reuptake Inhibitor <p class="infolinks">[<a href="javascript:toggleInfo('banerjee2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('banerjee2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Chemical Information and Modeling &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci7001632">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_banerjee2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Six rigid-body parameters (Shift, Slide, Rise, Tilt, Roll, Twist) are commonly used to describe the relative<p>displacement and orientation of successive base pairs in a nucleic acid structure. The present work adapts<p>this approach to describe the relative displacement and orientation of any two planes in an arbitrary molecules<p>specifically, planes which contain important pharmacophore elements. Relevant code from the 3DNA software<p>package (Nucleic Acids Res. 2003, 31, 5108-5121) was generalized to treat molecular fragments other than<p>DNA bases as input for the calculation of the corresponding rigid-body (or “planes”) parameters. These<p>parameters were used to construct feature vectors for a fuzzy relational clustering study of over 700<p>conformations of a flexible analogue of the dopamine reuptake inhibitor, GBR 12909. Several cluster validity<p>measures were used to determine the optimal number of clusters. Translational (Shift, Slide, Rise) rather<p>than rotational (Tilt, Roll, Twist) features dominate clustering based on planes that are relatively far apart,<p>whereas both types of features are important to clustering when the pair of planes are close by. This approach<p>was able to classify the data set of molecular conformations into groups and to identify representative<p>conformers for use as template conformers in future Comparative Molecular Field Analysis studies of GBR<p>12909 analogues. The advantage of using the planes parameters, rather than the combination of atomic<p>coordinates and angles between molecular planes used in our previous fuzzy relational clustering of the<p>same data set (J. Chem. Inf. Model. 2005, 45, 610-623), is that the present clustering results are independent<p>of molecular superposition and the technique is able to identify clusters in the molecule considered as a<p>whole. This approach is easily generalizable to any two planes in any molecule.</td>
</tr>
<tr id="bib_banerjee2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{banerjee2007,
  author = {Amit Banerjee and Milind Misra and Deepa Pai and Liang-Yu Shih and Rohan Woodley and Xiang-Jun Lu and A. R. Srinivasan and Wilma K. Olson and Rajesh N. Dave and Carol A. Venanzi},
  title = {Feature Extraction Using Molecular Planes for Fuzzy Relational Clustering of a Flexible Dopamine Reuptake Inhibitor},
  journal = {Journal of Chemical Information and Modeling},
  year = {2007},
  volume = {47},
  pages = {2216-2227},
  doi = {http://dx.doi.org/10.1021/ci7001632}
}
</pre></td>
</tr>
<tr id="bansal1995" class="entry">
	<td>Bansal, M., Bhattacharyya, D. &amp; Ravi, B.</td>
	<td>NUPARM and NUCGEN: software for analysis and generation of sequence dependent nucleic acid structures <p class="infolinks">[<a href="javascript:toggleInfo('bansal1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('bansal1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Computer Applications in the Biosciences: CABIOS &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_bansal1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Software packages NUPARM and NUCGEN, are described, which can be used to understand sequence directed structural variations in nucleic acids, by analysis and generation of non-uniform structures. A set of local inter basepair parameters (viz. tilt, roll, twist, shift, slide and rise) have been defined, which use geometry and coordinates of two successive basepairs only and can be used to generate polymeric structures with varying geometries for each of the 16 possible dinucleotide steps. Intra basepair parameters, propeller, buckle, opening and the C6...C8 distance can also be varied, if required, while the sugar phosphate backbone atoms are fixed in some standard conformation in each of the nucleotides. NUPARM can be used to analyse both DNA and RNA structures, with single as well as double stranded helices. The NUCGEN software generates double helical models with the backbone fixed in B-form DNA, but with appropriate modifications in the input data, it can also generate A-form DNA and RNA duplex structures.</td>
</tr>
<tr id="bib_bansal1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{bansal1995,
  author = {M. Bansal and D. Bhattacharyya and B. Ravi},
  title = {NUPARM and NUCGEN: software for analysis and generation of sequence dependent nucleic acid structures},
  journal = {Computer Applications in the Biosciences: CABIOS},
  year = {1995},
  volume = {11},
  pages = {281-287}
}
</pre></td>
</tr>
<tr id="batey1999" class="entry">
	<td>Batey, R. T., Rambo, R. P. &amp; Doudna, J. A.</td>
	<td>Tertiary Motifs in RNA Structure and Folding <p class="infolinks">[<a href="javascript:toggleInfo('batey1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Angewandte Chemie International Edition &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/(SICI)1521-3773(19990816)38:16<2326::AID-ANIE2326>3.0.CO;2-3">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_batey1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{batey1999,
  author = {Robert T. Batey and Robert P. Rambo and Jennifer A. Doudna},
  title = {Tertiary Motifs in RNA Structure and Folding},
  journal = {Angewandte Chemie International Edition},
  year = {1999},
  volume = {38},
  number = {16},
  pages = {2326-2343},
  doi = {http://dx.doi.org/10.1002/(SICI)1521-3773(19990816)38:16<2326::AID-ANIE2326>3.0.CO;2-3}
}
</pre></td>
</tr>
<tr id="berman1992" class="entry">
	<td>Berman, H. M., Olson, W. K., Beveridge, D. L., Westbrook, J., Gelbin, A., Demeny, T., Hsieh, S. H., Srinivasan, A. R. &amp; Schneider, B.</td>
	<td>The nucleic acid database. A comprehensive relational database of three-dimensional structures of nucleic acids. <p class="infolinks">[<a href="javascript:toggleInfo('berman1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('berman1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=1384741">URL</a> &nbsp;</td>
</tr>
<tr id="abs_berman1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Although nucleic acid structures are contained within the Protein Data Bank (PDB) (6) and the Cambridge Structural Database (7), no single database is devoted entirely to nucleic acid structures. Towards that end we have initiated the Nucleic Acid Database Project, whose goals are to continually develop computational tools for the extraction of information about nucleic acid structures and to distribute both the software and the data. This report describes our progress towards achieving these goals.</td>
</tr>
<tr id="bib_berman1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{berman1992,
  author = {H. M. Berman and W. K. Olson and D. L. Beveridge and J. Westbrook and A. Gelbin and T. Demeny and S. H. Hsieh and A. R. Srinivasan and B. Schneider},
  title = {The nucleic acid database. A comprehensive relational database of three-dimensional structures of nucleic acids.},
  journal = {Biophysical Journal},
  year = {1992},
  volume = {63},
  pages = {751-759},
  url = {http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=1384741}
}
</pre></td>
</tr>
<tr id="berman2000" class="entry">
	<td>Berman, H. M., Westbrook, J., Feng, Z., Gilliland, G., Bhat, T. N., Weissig, H., Shindyalov, I. N. &amp; Bourne, P. E.</td>
	<td>The Protein Data Bank <p class="infolinks">[<a href="javascript:toggleInfo('berman2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('berman2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/28.1.235">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_berman2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Protein Data Bank (PDB; http://www.rcsb.org/pdb/ ) is the single worldwide archive of structural data of biological macromolecules. This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</td>
</tr>
<tr id="bib_berman2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{berman2000,
  author = {Helen M. Berman and John Westbrook and Zukang Feng and Gary Gilliland and T. N. Bhat and Helge Weissig and Ilya N. Shindyalov and Philip E. Bourne},
  title = {The Protein Data Bank},
  journal = {Nucleic Acids Research},
  year = {2000},
  volume = {28},
  pages = {235-242},
  doi = {http://dx.doi.org/10.1093/nar/28.1.235}
}
</pre></td>
</tr>
<tr id="berninger2008" class="entry">
	<td>Berninger, P., Gaidatzis, D., van Nimwegen, E. &amp; Zavolan, M.</td>
	<td>Computational analysis of small RNA cloning data <p class="infolinks">[<a href="javascript:toggleInfo('berninger2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Methods &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.ymeth.2007.10.002">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_berninger2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{berninger2008,
  author = {Philipp Berninger and Dimos Gaidatzis and Erik van Nimwegen and Michaela Zavolan},
  title = {Computational analysis of small RNA cloning data},
  journal = {Methods},
  year = {2008},
  volume = {44},
  pages = {13-21},
  doi = {http://dx.doi.org/10.1016/j.ymeth.2007.10.002}
}
</pre></td>
</tr>
<tr id="Bloomfield2000" class="entry">
	<td>Bloomfield, V. A., Crothers, D. M. &amp; Jr., I. T.</td>
	<td>Nucleic Acids: Structures, Properties and Functions <p class="infolinks">[<a href="javascript:toggleInfo('Bloomfield2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Bloomfield2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_Bloomfield2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A comprehensive account of the structures and physical chemical properties of th nucleic acids, with emphasis on implications for biological function. The level of presentation assumes that the reader has knowledge of physical chemistry and molecular biology that would be obtained from introductory courses in these subjects.<p>1. Introduction<p>2. Bases, Nucleosides and Nucleotides<p>3. Chemical and Enzymatic Methods<p>4. Nucleic Acid Structures from Diffraction Methods<p>5. Structures and Dynaics by NMR<p>6. Electronic and Vibrational Spectroscopy<p>7. Theoretical Methods<p>8. Conformational Changes<p>9. Size and Shape of Nucleic Acids in Solution<p>10. Supercoiled DNA<p>11. Interaction of Nucleic Acids and Water and Ions<p>12. Interaction and Reaction with Drugs<p>13. Protein-Nucleic Acid Interactions<p>14. Higher Order Structure</td>
</tr>
<tr id="bib_Bloomfield2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{Bloomfield2000,
  author = {Victor A. Bloomfield and Donald M. Crothers and Ignacio Tinoco Jr.},
  title = {Nucleic Acids: Structures, Properties and Functions},
  publisher = {University Science Books},
  year = {2000}
}
</pre></td>
</tr>
<tr id="bon2008" class="entry">
	<td>Bon, M., Vernizzi, G. &amp; Orland, H.</td>
	<td>Topological Classification of RNA Structures <p class="infolinks">[<a href="javascript:toggleInfo('bon2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2008.04.033">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_bon2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{bon2008,
  author = {Micahel Bon and Graziano Vernizzi and Henri Orland},
  title = {Topological Classification of RNA Structures},
  journal = {Journal of Molecular Biology},
  year = {2008},
  volume = {379},
  pages = {900-911},
  doi = {http://dx.doi.org/10.1016/j.jmb.2008.04.033}
}
</pre></td>
</tr>
<tr id="borer1974" class="entry">
	<td>Borer, P. N., Dengler, B., Tinoco, J. I. &amp; Uhlenbeck, O. C.</td>
	<td>Stability of ribonucleic acid double-stranded helices <p class="infolinks">[<a href="javascript:toggleInfo('borer1974','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('borer1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/0022-2836(74)90357-X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_borer1974" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The hypochromicity, as a function of temperature for 19 oligoribonucleotides capable of forming perfectly base-paired double helices, is used to extract thermodynamic parameters of helix formation. The data are analyzed by an all or none model of helix melting which permits assignment of DG, DH, and DS of formation to each of the ten possible Watson-Crick base-paired nearest-neighbor sequences. Helix stability is found to have a striking dependence on sequence, and formulae are provided to predict the Tm of any RNA double helix of known sequence.</td>
</tr>
<tr id="bib_borer1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{borer1974,
  author = {Philip N. Borer and Barbara Dengler and Jr. Ignacio Tinoco and Olke C. Uhlenbeck},
  title = {Stability of ribonucleic acid double-stranded helices},
  journal = {Journal of Molecular Biology},
  year = {1974},
  volume = {86},
  pages = {843-853},
  doi = {http://dx.doi.org/10.1016/0022-2836(74)90357-X}
}
</pre></td>
</tr>
<tr id="burkard1999" class="entry">
	<td>Burkard, M. E., Turner, D. H. &amp; Jr., I. T.</td>
	<td>The RNA World <p class="infolinks">[<a href="javascript:toggleInfo('burkard1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('burkard1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td> &nbsp;</td>
	<td>inbook</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_burkard1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This chapter describes the effects of noncovalent interactions on RNA structure and evolution. The building blocks of RNA are well suited for taking advantage of relatively strong noncovalent interactions such as stacking and hydrogen-bonding to form ordered structures. These ordered structures are able to protect RNA from chemical degradation and to allow the specific binding and catalysis required for further evolution.</td>
</tr>
<tr id="bib_burkard1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{burkard1999,
  author = {Mark E. Burkard and Douglas H. Turner and Ignacion Tinoco Jr.},
  title = {The RNA World},
  publisher = {Cold Spring Harbor Laboratory Press},
  year = {1999},
  pages = {233-264},
  edition = {2nd}
}
</pre></td>
</tr>
<tr id="cannone2002" class="entry">
	<td>Cannone, J. J., Subramanian, S., Schnare, M. N., Collett, J. R., DSouza, L. M., Du, Y., Feng, B., Lin, N., Madabusi, L. V., Muller, K. M., Pande, N., Shang, Z., Yu, N. &amp; Gutell, R. R.</td>
	<td>The Comparative RNA Web (CRW) Site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs <p class="infolinks">[<a href="javascript:toggleInfo('cannone2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('cannone2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>BMC Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1186/1471-2105-3-2">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_cannone2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Background: Comparative analysis of RNA sequences is the basis for the detailed and accurate<p>predictions of RNA structure and the determination of phylogenetic relationships for organisms<p>that span the entire phylogenetic tree. Underlying these accomplishments are very large, wellorganized,<p>and processed collections of RNA sequences. This data, starting with the sequences<p>organized into a database management system and aligned to reveal their higher-order structure,<p>and patterns of conservation and variation for organisms that span the phylogenetic tree, has been<p>collected and analyzed. This type of information can be fundamental for and have an influence on<p>the study of phylogenetic relationships, RNA structure, and the melding of these two fields.</td>
</tr>
<tr id="bib_cannone2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{cannone2002,
  author = {Jamie J Cannone and Sankar Subramanian and Murray N Schnare and James R Collett and Lisa M DSouza and Yushi Du and Brian Feng and Nan Lin and Lakshmi V Madabusi and Kirsten M Muller and Nupur Pande and Zhidi Shang and Nan Yu and Robin R Gutell},
  title = {The Comparative RNA Web (CRW) Site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs},
  journal = {BMC Bioinformatics},
  year = {2002},
  volume = {3},
  pages = {1-31},
  doi = {http://dx.doi.org/10.1186/1471-2105-3-2}
}
</pre></td>
</tr>
<tr id="cao2005" class="entry">
	<td>Cao, S. &amp; Chen, S.</td>
	<td>Predicting RNA Folding Thermodynamics With a Reduced Chain Representation Model <p class="infolinks">[<a href="javascript:toggleInfo('cao2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('cao2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.2109105">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_cao2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Based on the virtual bond representation for the nucleotide backbone, we develop a reduced conformational model for RNA.<p>We use the experimentally measured atomic coordinates to model the helices and use the self-avoiding walks in a diamond<p>lattice to model the loop conformations. The atomic coordinates of the helices and the lattice representation for the loops are<p>matched at the loopï¿½helix junction, where steric viability is accounted for. Unlike the previous simplified lattice-based models,<p>the present virtual bond model can account for the atomic details of realistic three-dimensional RNA structures. Based on the<p>model, we develop a statistical mechanical theory for RNA folding energy landscapes and folding thermodynamics. Tests against<p>experiments show that the theory can give much more improved predictions for the native structures, the thermal denaturation<p>curves, and the equilibrium folding/unfolding pathways than the previous models. The application of the model to the P5abc<p>region of Tetrahymena group I ribozyme reveals the misfolded intermediates as well as the native-like intermediates in the<p>equilibrium folding process. Moreover, based on the free energy landscape analysis for each and every loop mutation, the model<p>predicts five lethal mutations that can completely alter the free energy landscape and the folding stability of the molecule.</td>
</tr>
<tr id="bib_cao2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{cao2005,
  author = {Song Cao and Shi-Jie Chen},
  title = {Predicting RNA Folding Thermodynamics With a Reduced Chain Representation Model},
  journal = {RNA},
  year = {2005},
  volume = {11},
  pages = {1884-1897},
  doi = {http://dx.doi.org/10.1261/rna.2109105}
}
</pre></td>
</tr>
<tr id="carbodorca2003" class="entry">
	<td>Carbo-Dorca, R.</td>
	<td>Applications of inward matrix products and matrix wave functions to Hï¿½ckel MO theory, Slater extended wave functions, spin extended functions, and Hartree method <p class="infolinks">[<a href="javascript:toggleInfo('carbodorca2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('carbodorca2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>International Journal of Quantum Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/qua.10464">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_carbodorca2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The operation of inward matrix product and the concept of extended wave functions are jointly invoked to set up a general theoretical framework to obtain a new look at old quantum chemical procedures like Hï¿½ckel method, Hartree theory, and some related questions. Among the discussion of several problems, the spin orbitals are presented as a matrix signature option, associated with an extended wave function in matrix form, common to both Hï¿½ckel and Hartree theories, as well as in the fundaments of Slater determinants construction</td>
</tr>
<tr id="bib_carbodorca2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{carbodorca2003,
  author = {Ramon Carbo-Dorca},
  title = {Applications of inward matrix products and matrix wave functions to Hï¿½ckel MO theory, Slater extended wave functions, spin extended functions, and Hartree method},
  journal = {International Journal of Quantum Chemistry},
  year = {2003},
  volume = {91},
  pages = {607-617},
  doi = {http://dx.doi.org/10.1002/qua.10464}
}
</pre></td>
</tr>
<tr id="carter2000" class="entry">
	<td>Carter, A. P., Clemons, W. M., Brodersen, D. E., Morgan-Warren, R. J., Wimberly, B. T. &amp; Ramakrishnan, V.</td>
	<td>Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics <p class="infolinks">[<a href="javascript:toggleInfo('carter2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('carter2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/35030019">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_carter2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The 30S ribosomal subunit has two primary functions in protein synthesis. It discriminates against aminoacyl transfer RNAs that do not match the codon of messenger RNA, thereby ensuring accuracy in translation of the genetic message in a process called decoding. Also, it works with the 50S subunit to move the tRNAs and associated mRNA by precisely one codon, in a process called translocation. Here we describe the functional implications of the high-resolution 30S crystal structure presented in the accompanying paper, and infer details of the interactions between the 30S subunit and its tRNA and mRNA ligands. We also describe the crystal structure of the 30S subunit complexed with the antibiotics paromomycin, streptomycin and spectinomycin, which interfere with decoding and translocation. This work reveals the structural basis for the action of these antibiotics, and leads to a model for the role of the universally conserved 16S RNA residues A1492 and A1493 in the decoding process.</td>
</tr>
<tr id="bib_carter2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{carter2000,
  author = {Andrew P. Carter and William M. Clemons and Ditlev E. Brodersen and Robert J. Morgan-Warren and Brian T. Wimberly and V. Ramakrishnan},
  title = {Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics},
  journal = {Nature},
  year = {2000},
  volume = {407},
  pages = {340-348},
  doi = {http://dx.doi.org/10.1038/35030019}
}
</pre></td>
</tr>
<tr id="chan2006" class="entry">
	<td>Chan, Y., Dresios, J. &amp; Wool, I. G.</td>
	<td>A Pathway for the Transmission of Allosteric Signals in the Ribosome through a Network of RNA Tertiary Interactions. <p class="infolinks">[<a href="javascript:toggleInfo('chan2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_chan2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chan2006,
  author = {Yuen-Ling Chan and John Dresios and Ira G. Wool},
  title = {A Pathway for the Transmission of Allosteric Signals in the Ribosome through a Network of RNA Tertiary Interactions.},
  journal = {Journal of Molecular Biology},
  year = {2006},
  volume = {355},
  pages = {1014-1025}
}
</pre></td>
</tr>
<tr id="chauhan2008" class="entry">
	<td>Chauhan, S. &amp; Woodson, S. A.</td>
	<td>Tertiary Interactions Determine the Accuracy of RNA Folding <p class="infolinks">[<a href="javascript:toggleInfo('chauhan2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja076166i">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_chauhan2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chauhan2008,
  author = {Seema Chauhan and Sarah A. Woodson},
  title = {Tertiary Interactions Determine the Accuracy of RNA Folding},
  journal = {Journal of the American Chemical Society},
  year = {2008},
  volume = {130},
  pages = {1296-1303},
  doi = {http://dx.doi.org/10.1021/ja076166i}
}
</pre></td>
</tr>
<tr id="chen2007" class="entry">
	<td>Chen, G., Kierzek, R., Yildirim, I., Krugh, T. R., Turner, D. H. &amp; Kennedy, S. D.</td>
	<td>Stacking Effects on Local Structure in RNA: Changes in the Structure of Tandem GA Pairs when Flanking GC Pairs Are Replaced by isoG-isoC Pairs <p class="infolinks">[<a href="javascript:toggleInfo('chen2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Physical Chemistry B &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp068732m">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_chen2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chen2007,
  author = {Gang Chen and Ryszard Kierzek and Ilyas Yildirim and Thomas R. Krugh and Douglas H. Turner and Scott D. Kennedy},
  title = {Stacking Effects on Local Structure in RNA: Changes in the Structure of Tandem GA Pairs when Flanking GC Pairs Are Replaced by isoG-isoC Pairs},
  journal = {Journal of Physical Chemistry B},
  year = {2007},
  volume = {111},
  pages = {6718-6727},
  doi = {http://dx.doi.org/10.1021/jp068732m}
}
</pre></td>
</tr>
<tr id="chen2000" class="entry">
	<td>Chen, S. &amp; Dill, K. A.</td>
	<td>RNA Folding Energy Landscapes <p class="infolinks">[<a href="javascript:toggleInfo('chen2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('chen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.">DOI</a><a href="http://www.pnas.org/cgi/content/abstract/97/2/646">URL</a> &nbsp;</td>
</tr>
<tr id="abs_chen2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Using a statistical mechanical treatment, we study RNA folding energy landscapes. We first validate the theory by showing that, for the RNA molecules we tested having only secondary structures, this treatment (i) predicts about the same native structures as the Zuker method, and (ii) qualitatively predicts the melting curve peaks and shoulders seen in experiments. We then predict thermodynamic folding intermediates. For one hairpin sequence, unfolding is a simple unzipping process. But for another sequence, unfolding is more complex. It involves multiple stable intermediates and a rezipping into a completely non-native conformation before unfolding. The principle that emerges, for which there is growing experimental support, is that although protein folding tends to involve highly cooperative two-state thermodynamic transitions, without detectable intermediates, the folding of RNA secondary structures may involve rugged landscapes, often with more complex intermediate states.</td>
</tr>
<tr id="bib_chen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chen2000,
  author = {Shi-Jie Chen and Ken A. Dill},
  title = {RNA Folding Energy Landscapes},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2000},
  volume = {97},
  pages = {646-651},
  url = {http://www.pnas.org/cgi/content/abstract/97/2/646},
  doi = {http://dx.doi.org/10.1073/pnas.}
}
</pre></td>
</tr>
<tr id="chen1998" class="entry">
	<td>Chen, S. &amp; Dill, K. A.</td>
	<td>Theory for the conformational changes of double-stranded chain molecules <p class="infolinks">[<a href="javascript:toggleInfo('chen1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('chen1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Chemical Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1063/1.477065">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_chen1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We develop statistical mechanical theory to predict the thermodynamic properties of chain molecules having noncovalent double-stranded conformations, as in RNA molecules and beta-sheets in proteins. Sequence dependence and excluded volume interactions are explicitly taken into account. We classify conformations by their polymer graphs and enumerate all the conformations corresponding to each graph by a recently developed matrix method [S-J. Chen and K. A. Dill, J. Chem. Phys. 103, 5802 (1995)]. All such graphs are summed by a recursive method. Tests against exact computer enumeration for short chains on a 2D lattice show that the density of states and partition function are given quite accurately. So far, we have explored two classes of conformations; hairpins, which model small beta-sheets, and RNA secondary structures. The main folding transition is predicted to be quite different for these two conformational classes: the hairpin transition is two-state while the RNA secondary structure transition is one-state for homopolymeric chains</td>
</tr>
<tr id="bib_chen1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chen1998,
  author = {Shi-Jie Chen and Ken A. Dill},
  title = {Theory for the conformational changes of double-stranded chain molecules},
  journal = {Journal of Chemical Physics},
  year = {1998},
  volume = {109},
  pages = {4602-4616},
  doi = {http://dx.doi.org/10.1063/1.477065}
}
</pre></td>
</tr>
<tr id="chen1995" class="entry">
	<td>Chen, S. &amp; Dill, K. A.</td>
	<td>Statistical thermodynamics of double-stranded polymer molecules <p class="infolinks">[<a href="javascript:toggleInfo('chen1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('chen1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Chemical Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1063/1.470461">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_chen1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We develop a matrix method to compute the configurational partition functions of chain molecules that have intramolecular double strands, including RNA and DNA and antiparallel beta-sheet regions in proteins. An accurate theory has not been available for these types of molecules due to the nature of their self-contacts: (i) biomolecules have many self-contacts, and (ii) contacts are sequence-specific. The present theory, which we call the firehose model, can treat such conformations, including specific loops, bulges, and stems. The method is based on the use of polymer graphs. Tests against exact enumeration results on the two-dimensional (2D) square lattice show that conformational entropies are predicted quite accurately. As an example, we compute the denaturation profile of a 2D chain designed to resemble an RNA structure.</td>
</tr>
<tr id="bib_chen1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chen1995,
  author = {Shi-Jie Chen and Ken A. Dill},
  title = {Statistical thermodynamics of double-stranded polymer molecules},
  journal = {Journal of Chemical Physics},
  year = {1995},
  volume = {103},
  pages = {5802-5813},
  doi = {http://dx.doi.org/10.1063/1.470461}
}
</pre></td>
</tr>
<tr id="chennubhotla2006" class="entry">
	<td>Chennubhotla, C. &amp; Bahar, I.</td>
	<td>Markov Propagation of Allosteric Effects in Biomolecular Systems: Application to GroEL-GroES <p class="infolinks">[<a href="javascript:toggleInfo('chennubhotla2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('chennubhotla2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Molecular Systems Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/msb4100075">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_chennubhotla2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce a novel approach for elucidating the potential pathways of allosteric communication in biomolecular systems. The methodology, based on Markov propagation of ï¿½information' across the structure, permits us to partition the network of interactions into soft clusters distinguished by their coherent stochastics. Probabilistic participation of residues in these clusters defines the communication patterns inherent to the network architecture. Application to bacterial chaperonin complex GroELï¿½GroES, an allostery-driven structure, identifies residues engaged in intra- and inter-subunit communication, including those acting as hubs and messengers. A number of residues are distinguished by their high potentials to transmit allosteric signals, including Pro33 and Thr90 at the nucleotide-binding site and Glu461 and Arg197 mediating inter- and intra-ring communication, respectively. We propose two most likely pathways of signal transmission, between nucleotide- and GroES-binding sites across the cis and trans rings, which involve several conserved residues. A striking observation is the opposite direction of information flow within cis and trans rings, consistent with negative inter-ring cooperativity. Comparison with collective modes deduced from normal mode analysis reveals the propensity of global hinge regions to act as messengers in the transmission of allosteric signals.</td>
</tr>
<tr id="bib_chennubhotla2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{chennubhotla2006,
  author = {Chakra Chennubhotla and Ivet Bahar},
  title = {Markov Propagation of Allosteric Effects in Biomolecular Systems: Application to GroEL-GroES},
  journal = {Molecular Systems Biology},
  year = {2006},
  volume = {2},
  pages = {1-13},
  doi = {http://dx.doi.org/10.1038/msb4100075}
}
</pre></td>
</tr>
<tr id="clery2008" class="entry">
	<td>Cléry, A., Blatter, M. &amp; Allain, F. H.</td>
	<td>RNA Recognition Motifs: Boring? Not Quite <p class="infolinks">[<a href="javascript:toggleInfo('clery2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('clery2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2008.04.002">DOI</a><a href="http://dx.doi.org/10.1016/j.sbi.2008.04.002">URL</a> &nbsp;</td>
</tr>
<tr id="abs_clery2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The RNA recognition motif (RRM) is one of the most abundant protein domains in eukaryotes. While the structure of this domain is well characterized by the packing of two alpha-helices on a four-stranded beta-sheet, the mode of protein and RNA recognition by RRMs is not clear owing to the high variability of these interactions. Here we report recent structural data on RRM-RNA and RRM-protein interactions showing the ability of this domain to modulate its binding affinity and specificity using each of its constitutive elements (beta-strands, loops, alpha-helices). The extreme structural versatility of the RRM interactions explains why RRM-containing proteins have so diverse biological functions.</td>
</tr>
<tr id="bib_clery2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{clery2008,
  author = {Antoine Cléry and Markus Blatter and Frédéric H-T Allain},
  title = {RNA Recognition Motifs: Boring? Not Quite},
  journal = {Current Opinion in Structural Biology},
  year = {2008},
  volume = {18},
  number = {3},
  pages = {290-298},
  url = {http://dx.doi.org/10.1016/j.sbi.2008.04.002},
  doi = {http://dx.doi.org/10.1016/j.sbi.2008.04.002}
}
</pre></td>
</tr>
<tr id="cohen2007" class="entry">
	<td>Cohen, M. H., Wasserman, A. &amp; Burke, K.</td>
	<td>Partition theory: a very simple illustration. <p class="infolinks">[<a href="javascript:toggleInfo('cohen2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('cohen2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>J Phys Chem A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp0743370">DOI</a><a href="http://dx.doi.org/10.1021/jp0743370">URL</a> &nbsp;</td>
</tr>
<tr id="abs_cohen2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We illustrate the main features of a recently proposed method based on ensemble density functional theory to divide rigorously a complex molecular system into its parts (J. Phys. Chem. A 2007, 111, 2229). The illustrative system is an analog of the hydrogen molecule for which analytic expressions for the densities of the parts (hydrogen "atoms") are found along with the "partition potential" that enters the theory. While previous formulations of chemical reactivity theory lead to zero, or undefined, values for the chemical hardness of the isolated parts, we demonstrate they can acquire a finite and positive hardness within the present formulation.</td>
</tr>
<tr id="bib_cohen2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{cohen2007,
  author = {Morrel H Cohen and Adam Wasserman and Kieron Burke},
  title = {Partition theory: a very simple illustration.},
  journal = {J Phys Chem A},
  year = {2007},
  volume = {111},
  number = {49},
  pages = {12447--12453},
  url = {http://dx.doi.org/10.1021/jp0743370},
  doi = {http://dx.doi.org/10.1021/jp0743370}
}
</pre></td>
</tr>
<tr id="cohen2009" class="entry">
	<td>Cohen, M. H., Wasserman, A., Car, R. &amp; Burke, K.</td>
	<td>Charge Transfer in Partition Theory (dagger). <p class="infolinks">[<a href="javascript:toggleInfo('cohen2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>J Phys Chem A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp807967e">DOI</a><a href="http://dx.doi.org/10.1021/jp807967e">URL</a> &nbsp;</td>
</tr>
<tr id="bib_cohen2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{cohen2009,
  author = {Morrel H Cohen and Adam Wasserman and Roberto Car and Kieron Burke},
  title = {Charge Transfer in Partition Theory (dagger).},
  journal = {J Phys Chem A},
  year = {2009},
  url = {http://dx.doi.org/10.1021/jp807967e},
  doi = {http://dx.doi.org/10.1021/jp807967e}
}
</pre></td>
</tr>
<tr id="collin2005" class="entry">
	<td>Collin, D., F.Ritort, Jarzynski, C., Smith, S. B., Jr., I. T. &amp; Bustamante, C.</td>
	<td>Verification of the Crooks fluctuation theorem and recovery of RNA folding free energies <p class="infolinks">[<a href="javascript:toggleInfo('collin2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('collin2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature04061">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_collin2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Atomic force microscopes and optical tweezers are widely used to probe the mechanical properties of individual molecules and molecular interactions, by exerting mechanical forces that induce transitions such as unfolding or dissociation. These transitions often occur under nonequilibrium conditions and are associated with hysteresis effects?features usually taken to preclude the extraction of equilibrium information from the experimental data. But fluctuation theorems1, 2, 3, 4, 5 allow us to relate the work along nonequilibrium trajectories to thermodynamic free-energy differences. They have been shown to be applicable to single-molecule force measurements6 and have already provided information on the folding free energy of a RNA hairpin7, 8. Here we show that the Crooks fluctuation theorem9 can be used to determine folding free energies for folding and unfolding processes occurring in weak as well as strong nonequilibrium regimes, thereby providing a test of its validity under such conditions. We use optical tweezers10 to measure repeatedly the mechanical work associated with the unfolding and refolding of a small RNA hairpin11 and an RNA three-helix junction12. The resultant work distributions are then analysed according to the theorem and allow us to determine the difference in folding free energy between an RNA molecule and a mutant differing only by one base pair, and the thermodynamic stabilizing effect of magnesium ions on the RNA structure.</td>
</tr>
<tr id="bib_collin2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{collin2005,
  author = {D. Collin and F.Ritort and C. Jarzynski and S. B. Smith and I. Tinoco Jr. and C. Bustamante},
  title = {Verification of the Crooks fluctuation theorem and recovery of RNA folding free energies},
  journal = {Nature},
  year = {2005},
  volume = {437},
  pages = {231-234},
  doi = {http://dx.doi.org/10.1038/nature04061}
}
</pre></td>
</tr>
<tr id="couch2006" class="entry">
	<td>Couch, G. S., Hendrix, D. K. &amp; Ferrin, T. E.</td>
	<td>Nucleic acid visualization with UCSF Chimera <p class="infolinks">[<a href="javascript:toggleInfo('couch2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('couch2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gnj031">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_couch2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: With the increase in the number of large, 3D, high-resolution nucleic acid structures, particularly of the 30S and 50S ribosomal subunits and the intact bacterial ribosome, advancements in the visualization of nucleic acid structural features are essential. Large molecular structures are complicated and detailed, and one goal of visualization software is to allow the user to simplify the display of some features and accent others. We describe an extension to the UCSF Chimera molecular visualization system for the purpose of displaying and highlighting nucleic acid characteristics, including a new representation of sugar pucker, several options for abstraction of base geometries that emphasize stacking and base pairing, and an adaptation of the ribbon backbone to accommodate the nucleic acid backbone. Molecules are displayed and manipulated interactively, allowing the user to change the representations as desired for small molecules, proteins and nucleic acids. This software is available as part of the UCSF Chimera molecular visualization system and thus is integrated with a suite of existing tools for molecular graphics.</td>
</tr>
<tr id="bib_couch2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{couch2006,
  author = {Gregory S. Couch and Donna K. Hendrix and Thomas E. Ferrin},
  title = {Nucleic acid visualization with UCSF Chimera},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {e29 1-5},
  doi = {http://dx.doi.org/10.1093/nar/gnj031}
}
</pre></td>
</tr>
<tr id="crick1966" class="entry">
	<td>Crick, F.</td>
	<td>Codon--anticodon pairing: the wobble hypothesis <p class="infolinks">[<a href="javascript:toggleInfo('crick1966','bibtex')">BibTeX</a>]</p></td>
	<td>1966</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_crick1966" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{crick1966,
  author = {Francis Crick},
  title = {Codon--anticodon pairing: the wobble hypothesis},
  journal = {Journal of Molecular Biology},
  year = {1966},
  volume = {19},
  pages = {548-555}
}
</pre></td>
</tr>
<tr id="crick1958" class="entry">
	<td>Crick, F.</td>
	<td>On protein synthesis <p class="infolinks">[<a href="javascript:toggleInfo('crick1958','bibtex')">BibTeX</a>]</p></td>
	<td>1958</td>
	<td>Symposia of the Society of Experimental Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_crick1958" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{crick1958,
  author = {Francis Crick},
  title = {On protein synthesis},
  journal = {Symposia of the Society of Experimental Biology},
  year = {1958},
  volume = {12},
  pages = {138-163}
}
</pre></td>
</tr>
<tr id="crooks1999" class="entry">
	<td>Crooks, G. E.</td>
	<td>Entropy production fluctuation theorem and the nonequilirium work relation for free-energy differences <p class="infolinks">[<a href="javascript:toggleInfo('crooks1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Physical Review E &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_crooks1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{crooks1999,
  author = {G. E. Crooks},
  title = {Entropy production fluctuation theorem and the nonequilirium work relation for free-energy differences},
  journal = {Physical Review E},
  year = {1999},
  volume = {60},
  pages = {2721-2726}
}
</pre></td>
</tr>
<tr id="crooks2004" class="entry">
	<td>Crooks, G. E. &amp; Brenner, S. E.</td>
	<td>Protein secondary structure: entropy, correlations and prediction. <p class="infolinks">[<a href="javascript:toggleInfo('crooks2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('crooks2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/bioinformatics/bth132">DOI</a><a href="http://dx.doi.org/10.1093/bioinformatics/bth132">URL</a> &nbsp;</td>
</tr>
<tr id="abs_crooks2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MOTIVATION: Is protein secondary structure primarily determined by local interactions between residues closely spaced along the amino acid backbone or by non-local tertiary interactions? To answer this question, we measure the entropy densities of primary and secondary structure sequences, and the local inter-sequence mutual information density. RESULTS: We find that the important inter-sequence interactions are short ranged, that correlations between neighboring amino acids are essentially uninformative and that only one-fourth of the total information needed to determine the secondary structure is available from local inter-sequence correlations. These observations support the view that the majority of most proteins fold via a cooperative process where secondary and tertiary structure form concurrently. Moreover, existing single-sequence secondary structure prediction algorithms are almost optimal, and we should not expect a dramatic improvement in prediction accuracy. AVAILABILITY: Both the data sets and analysis code are freely available from our Web site at http://compbio.berkeley.edu/</td>
</tr>
<tr id="bib_crooks2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{crooks2004,
  author = {Gavin E Crooks and Steven E Brenner},
  title = {Protein secondary structure: entropy, correlations and prediction.},
  journal = {Bioinformatics},
  year = {2004},
  volume = {20},
  number = {10},
  pages = {1603--1611},
  url = {http://dx.doi.org/10.1093/bioinformatics/bth132},
  doi = {http://dx.doi.org/10.1093/bioinformatics/bth132}
}
</pre></td>
</tr>
<tr id="czapla2008" class="entry">
	<td>Czapla, L., Swigon, D. &amp; Olson, W. K.</td>
	<td>Effects of the nucleoid protein HU on the structure, flexibility, and ring-closure properties of DNA deduced from Monte Carlo simulations. <p class="infolinks">[<a href="javascript:toggleInfo('czapla2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('czapla2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>J Mol Biol &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2008.05.088">DOI</a><a href="http://dx.doi.org/10.1016/j.jmb.2008.05.088">URL</a> &nbsp;</td>
</tr>
<tr id="abs_czapla2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The histone-like HU (heat unstable) protein plays a key role in the organization and regulation of the Escherichia coli genome. The nonspecific nature of HU binding to DNA complicates analysis of the mechanism by which the protein contributes to the looping of DNA. Conventional models of the looping of HU-bound duplexes attribute the changes in biophysical properties of DNA brought about by the random binding of protein to changes in the effective parameters of an ideal helical wormlike chain. Here, we introduce a novel Monte Carlo approach to study the effects of nonspecific HU binding on the configurational properties of DNA directly. We randomly decorated segments of an ideal double-helical DNA with HU molecules that induce the bends and other structural distortions of the double helix find in currently available X-ray structures. We find that the presence of HU at levels approximating those found in the cell reduces the persistence length by roughly threefold compared with that of naked DNA. The binding of protein has particularly striking effects on the cyclization properties of short duplexes, altering the dependence of ring closure on chain length in a way that cannot be mimicked by a simple wormlike model and accumulating at higher-than-expected levels on successfully closed chains. Moreover, the uptake of protein on small minicircles depends on chain length, taking advantage of the HU-induced deformations of DNA structure to facilitate ligation. Circular duplexes with bound HU show much greater propensity than protein-free DNA to exist as negatively supercoiled topoisomers, suggesting a potential role of HU in organizing the bacterial nucleoid. The local bending and undertwisting of DNA by HU, in combination with the number of bound proteins, provide a structural rationale for the condensation of DNA and the observed expression levels of reporter genes in vivo.</td>
</tr>
<tr id="bib_czapla2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{czapla2008,
  author = {Luke Czapla and David Swigon and Wilma K Olson},
  title = {Effects of the nucleoid protein HU on the structure, flexibility, and ring-closure properties of DNA deduced from Monte Carlo simulations.},
  journal = {J Mol Biol},
  year = {2008},
  volume = {382},
  number = {2},
  pages = {353--370},
  url = {http://dx.doi.org/10.1016/j.jmb.2008.05.088},
  doi = {http://dx.doi.org/10.1016/j.jmb.2008.05.088}
}
</pre></td>
</tr>
<tr id="czapla2006" class="entry">
	<td>Czapla, L., Swigon, D. &amp; Olson, W. K.</td>
	<td>Sequence-Dependent Effects in the Cyclization of Short DNA <p class="infolinks">[<a href="javascript:toggleInfo('czapla2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('czapla2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Chemical Theory and Computation &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ct060025+">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_czapla2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new, computationally efficient Monte Carlo approach has been developed to<p>estimate the ring-closure properties of short, realistically modeled DNA chains. The double helix<p>is treated at the level of base-pair steps using an elastic potential that accounts for the sequencedependent<p>variability in the intrinsic structure and elastic moduli of the base-pair steps, including<p>the known coupling of conformational variables. Rather than using traditional Metropolis-Monte<p>Carlo techniques to generate representative configurations, a Gaussian sampling method is<p>introduced to construct three-dimensional structures from linear combinations of the rigid-body<p>parameters defining the relative orientation and displacement of successive base pairs. The<p>computation of the J factor, the well-known ratio of the equilibrium constants for cyclization vs<p>bimolecular association of a linear molecule, takes into account restrictions on the displacement<p>and directions of the base pairs joined in ring closure, including the probability that the end-toend<p>vector is null and the terminal base pairs coincide. The increased sample sizes needed to<p>assess the likelihood that very short chains satisfy these criteria are attained using the<p>Alexandrowicz half-chain sampling enhancement technique in combination with selective linkage<p>of the two-half-chain segments. The method is used to investigate the cyclization properties of<p>arbitrary-length DNA with greatly enhanced sampling sizes, i.e., O(1014) configurations, and to<p>estimate J factors lower than 0.1 pM with high accuracy. The methodology has been checked<p>against classic theoretical predictions of the cyclization properties of an ideal, inextensible,<p>naturally straight, DNA elastic rod and then applied to investigate the extent to which one can<p>account for the unexpectedly large J factors of short DNA chains without the need to invoke<p>significant distortions of double helical structure. Several well-known structural features of DNAs<p>including the presence of intrinsic curvature, roll-twist coupling, or enhanced pyrimidine-purine<p>deformabilitysbring the computed J factors in line with the observed data. Moreover, periodically<p>distributed roll-twist coupling reduces the magnitude of oscillations in J, seen in plots of J vs<p>chain length, to the extent found experimentally.</td>
</tr>
<tr id="bib_czapla2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{czapla2006,
  author = {Luke Czapla and David Swigon and Wilma K. Olson},
  title = {Sequence-Dependent Effects in the Cyclization of Short DNA},
  journal = {Journal of Chemical Theory and Computation},
  year = {2006},
  volume = {2},
  pages = {685-695},
  doi = {http://dx.doi.org/10.1021/ct060025+}
}
</pre></td>
</tr>
<tr id="david-eden2008" class="entry">
	<td>David-Eden, H. &amp; Mandel-Gutfreund, Y.</td>
	<td>Revealing unique properties of the ribosome using a network based analysis <p class="infolinks">[<a href="javascript:toggleInfo('david-eden2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('david-eden2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkn433">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_david-eden2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The ribosome is a complex molecular machine that<p>offers many potential sites for functional interference,<p>therefore representing a major target for<p>antibacterial drugs. The growing number of highresolution<p>structures of ribosomes from different<p>organisms, in free form and in complex with various<p>ligands, provides unique data for structural and<p>comparative analyses of RNA structures. We<p>model the ribosome structure as a network, where<p>nucleotides are represented as nodes and intermolecular<p>interactions as edges. As shown previously<p>for proteins, we found that the major functional sites<p>of the ribosome exhibit significantly high centrality<p>measures. Specifically, we demonstrate that mutations<p>that strongly affect ribosome function and<p>assembly can be distinguished from mild mutations<p>based on their network properties. Furthermore, we<p>observed that closeness centrality of the rRNA<p>nucleotides is highly conserved in the bacteria, suggesting<p>the network representation as a comparative<p>tool for the ribosome analysis. Finally, we<p>suggest a global topology perspective to characterize<p>functional sites and to reveal the unique properties<p>of the ribosome.</td>
</tr>
<tr id="bib_david-eden2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{david-eden2008,
  author = {Hilda David-Eden and Yael Mandel-Gutfreund},
  title = {Revealing unique properties of the ribosome using a network based analysis},
  journal = {Nucleic Acids Research},
  year = {2008},
  volume = {36},
  pages = {4641-4652},
  doi = {http://dx.doi.org/10.1093/nar/gkn433}
}
</pre></td>
</tr>
<tr id="davis2007" class="entry">
	<td>Davis, J. H., Foster, T. R., Tonelli, M. &amp; Butcher, S. E.</td>
	<td>Role of Metal Ions in the Tetraloop-Receptor Complex as Analyzed by NMR. <p class="infolinks">[<a href="javascript:toggleInfo('davis2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('davis2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_davis2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Metal ions are critical for the proper folding of RNA, and the GAAA tetraloopï¿½receptor is necessary for the optimal folding and<p>function of many RNAs. We have used NMR to investigate the role of metal ions in the structure of the tetraloopï¿½receptor in<p>solution. The NMR data indicate native tertiary structure is formed under a wide range of ionic conditions. The lack of<p>conformational adaptation in response to very different ionic conditions argues against a structural role for divalent ions.<p>Nuclear Overhauser effects to cobalt hexammine and paramagnetic relaxation enhancement induced by manganese ions were<p>used to determine the NMR structures of the tetraloop receptor in association with metal ions, providing the first atomic-level<p>view of these interactions in the solution state. Five manganese and two cobalt hexammine ions could be localized to the RNA<p>surface. The locations of the associated metal ions are similar, but not identical to, those of previously determined crystal<p>structures. The sites of association are in general agreement with nonlinear Poissonï¿½Boltzmann calculations of the electrostatic<p>surface, emphasizing the general importance of diffusely associated ions in RNA tertiary structure.</td>
</tr>
<tr id="bib_davis2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{davis2007,
  author = {Jared H. Davis and Trenton R. Foster and Marco Tonelli and Samuel E. Butcher},
  title = {Role of Metal Ions in the Tetraloop-Receptor Complex as Analyzed by NMR.},
  journal = {RNA},
  year = {2007},
  volume = {13},
  pages = {76-86}
}
</pre></td>
</tr>
<tr id="dickerson1989" class="entry">
	<td>Dickerson, R. E. &amp; et al.</td>
	<td>Definitions and Nomenclature of Nucleic Acid Structure Components <p class="infolinks">[<a href="javascript:toggleInfo('dickerson1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('dickerson1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Nucleic Acid Research &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_dickerson1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We report here recommendations for the definitions and nomenclature of nucleic acid structure parameters. These recommendations result from discussions at an EMBO Workshop on DNA Curvature and Bending held at Churchill College, Cambridge, in September 1988.</td>
</tr>
<tr id="bib_dickerson1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{dickerson1989,
  author = {R. E. Dickerson and et al.},
  title = {Definitions and Nomenclature of Nucleic Acid Structure Components},
  journal = {Nucleic Acid Research},
  year = {1989},
  volume = {17},
  pages = {1797-1803},
  note = {It has additional info.}
}
</pre></td>
</tr>
<tr id="diekmann1989" class="entry">
	<td>Diekmann, S.</td>
	<td>Definitions and Nomenclature of Nucleic Acid Structure Parameters <p class="infolinks">[<a href="javascript:toggleInfo('diekmann1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('diekmann1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_diekmann1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: At an EMBO Workshop on DNA Curvature and Bending held at Churchill College, Cambridge, U.K. on 10 to 15 September 1998, two sessions were scheduled... The most widely used library of helix analysis programs, HELIB.</td>
</tr>
<tr id="bib_diekmann1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{diekmann1989,
  author = {Stephan Diekmann},
  title = {Definitions and Nomenclature of Nucleic Acid Structure Parameters},
  journal = {Journal of Molecular Biology},
  year = {1989},
  volume = {205},
  pages = {787-791}
}
</pre></td>
</tr>
<tr id="dima2005" class="entry">
	<td>Dima, R. I., Hyeon, C. &amp; Thirumalai, D.</td>
	<td>Extracting Stacking Interaction Parameters for RNA from the Data Set of Native Structures <p class="infolinks">[<a href="javascript:toggleInfo('dima2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('dima2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Molecular Biology  &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2004.12.012">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_dima2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A crucial step in the determination of the three-dimensional native structures of RNA is the prediction of their secondary structures, which are stable independent of the tertiary fold. Accurate prediction of the secondary structure requires context-dependent estimates of the interaction parameters. We have exploited the growing database of natively folded RNA structures in the Protein Data Bank (PDB) to obtain stacking interaction parameters using a knowledge-based approach. Remarkably, the calculated values of the resulting statistical potentials (SPs) are in excellent agreement with the parameters determined using measurements in small oligonucleotides. We validate the SPs by predicting 74% of the base-pairs in a dataset of structures using the ViennaRNA package. Interestingly, this number is similar to that obtained using the measured thermodynamic parameters. We also tested the efficacy of the SP in predicting secondary structure by using gapless threading, which we advocate as an alternative method for rapidly predicting RNA structures. For RNA molecules with less than 700 nucleotides, about 70% of the native base-pairs are correctly predicted. As a further validation of the SPs we calculated Z-scores, which measure the relative stability of the native state with respect to a manifold of higher free energy states. The computed Z-scores agree with estimates made using calorimetric measurements for a few RNA molecules. Structural analysis was used to rationalize the success and failures of SP and experimentally determined parameters. First, from the near perfect linear relationship between the number of native base-pairs and sequence length, we show that nearly 46% of nucleotides are not in stacks. Second, by analyzing the suboptimal structures that are generated in gapless threading we show that the SPs and experimentally determined parameters are most successful in predicting stacks that end in hairpins. These results show that further improvement in secondary structure prediction requires reliable estimates of interaction parameters for loops, bulges, and stacks that do not end in hairpins.</td>
</tr>
<tr id="bib_dima2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{dima2005,
  author = {Ruxandra I. Dima and Changbong Hyeon and D. Thirumalai},
  title = {Extracting Stacking Interaction Parameters for RNA from the Data Set of Native Structures},
  journal = {Journal of Molecular Biology },
  year = {2005},
  volume = {347},
  pages = {53-69},
  doi = {http://dx.doi.org/10.1016/j.jmb.2004.12.012}
}
</pre></td>
</tr>
<tr id="djelloul2008" class="entry">
	<td>Djelloul, M. &amp; Denise, A.</td>
	<td>Automated motif extraction and classification in RNA tertiary structures. <p class="infolinks">[<a href="javascript:toggleInfo('djelloul2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('djelloul2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.1061108">DOI</a><a href="http://dx.doi.org/10.1261/rna.1061108">URL</a> &nbsp;</td>
</tr>
<tr id="abs_djelloul2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We used a novel graph-based approach to extract RNA tertiary motifs. We cataloged them all and clustered them using an innovative graph similarity measure. We applied our method to three widely studied structures: Haloarcula marismortui 50S (H.m 50S), Escherichia coli 50S (E. coli 50S), and Thermus thermophilus 16S (T.th 16S) RNAs. We identified 10 known motifs without any prior knowledge of their shapes or positions. We additionally identified four putative new motifs.</td>
</tr>
<tr id="bib_djelloul2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{djelloul2008,
  author = {Mahassine Djelloul and Alain Denise},
  title = {Automated motif extraction and classification in RNA tertiary structures.},
  journal = {RNA},
  year = {2008},
  volume = {14},
  number = {12},
  pages = {2489--2497},
  url = {http://dx.doi.org/10.1261/rna.1061108},
  doi = {http://dx.doi.org/10.1261/rna.1061108}
}
</pre></td>
</tr>
<tr id="doudna1995" class="entry">
	<td>Doudna, J. A.</td>
	<td>Hammerhead Ribozyme Structure: U-turn for RNA Structural Biology <p class="infolinks">[<a href="javascript:toggleInfo('doudna1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0969-2126(01)00208-8">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_doudna1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{doudna1995,
  author = {Jennifer A. Doudna},
  title = {Hammerhead Ribozyme Structure: U-turn for RNA Structural Biology},
  journal = {Structure},
  year = {1995},
  volume = {3},
  pages = {747-750},
  doi = {http://dx.doi.org/10.1016/S0969-2126(01)00208-8}
}
</pre></td>
</tr>
<tr id="duarte1998" class="entry">
	<td>Duarte, C. M. &amp; Pyle, A. M.</td>
	<td>Stepping Through an RNA Structure: A Novel Approach to Conformational Analysis <p class="infolinks">[<a href="javascript:toggleInfo('duarte1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('duarte1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1998.2233">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_duarte1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Drawing from the growing database of complex three-dimensional RNA structures, a systematic method has been developed for classifying and analyzing the variety of conformations adopted by nucleic acids. This method is based on the development of a reduced representation for nucleic acid backbone conformation, simplifying the formidable eight-dimensional problem that has long complicated nucleic acid conformational analysis. Two pseudotorsion angles (n and t) have been defined, based on the selection of two appropriate pivot points along the RNA backbone, P and C4'. These pseudotorsions, together with a complete library of conventional torsion angles, can be calculated for any RNA structure or all-atom model using a new program called AMIGOS. Having computed n and t pseudotorsions for each position on an RNA molecule, they can be represented on a two-dimensional plot similar to the p-phi plots that have traditionally been used for protein conformational analysis. Like a Ramachandran plot, clusters of residues appear at discrete regions on an n-t plot. Nucleotides within these clusters share conformational properties, often belonging to the same type of structural motif such as A-platforms, sheared tandem purine-purine pairs and GNRA tetraloops. An n-t plot provides a two-dimensional representation of the conformational properties of an entire RNA molecule, facilitating rapid analysis of structural features. In addition to the utility of n-t plots for intuitive visualization of conformational space, the pseudotorsional convention described here should significantly simplify approaches to macromolecular modeling of RNA structure.</td>
</tr>
<tr id="bib_duarte1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{duarte1998,
  author = {Carlos M. Duarte and Anna Marie Pyle},
  title = {Stepping Through an RNA Structure: A Novel Approach to Conformational Analysis},
  journal = {Journal of Molecular Biology},
  year = {1998},
  volume = {284},
  pages = {1465-1478},
  doi = {http://dx.doi.org/10.1006/jmbi.1998.2233}
}
</pre></td>
</tr>
<tr id="duarte2003" class="entry">
	<td>Duarte, C. M., Wadley, L. M. &amp; Pyle, A. M.</td>
	<td>RNA Structure Comparison, Motif Search and Discovery Using a Reduced Representation of RNA Conformational Space <p class="infolinks">[<a href="javascript:toggleInfo('duarte2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('duarte2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg682">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_duarte2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Given the wealth of new RNA structures and the growing list of RNA functions in biology, it is of great interest to understand the repertoire of RNA folding motifs. The ability to identify new and known motifs within novel RNA structures, to compare tertiary structures with one another and to quantify the characteristics of a given RNA motif are major goals in the field of RNA research; however, there are few systematic ways to address these issues. Using a novel approach for visualizing and mathematically describing macromolecular structures, we have developed a means to quantitatively describe RNA molecules in order to rapidly analyze, compare and explore their features. This approach builds on the alternative , convention for describing RNA torsion angles and is executed using a new program called PRIMOS. Applying this methodology, we have successfully identified major regions of conformational change in the 50S and 30S ribosomal subunits, we have developed a means to search the database of RNA structures for the prevalence of known motifs and we have classified and identified new motifs. These applications illustrate the powerful capabilities of our new RNA structural convention, and they suggest future adaptations with important implications for bioinformatics and structural genomics.</td>
</tr>
<tr id="bib_duarte2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{duarte2003,
  author = {Carlos M. Duarte and Leven M. Wadley and Anna Marie Pyle},
  title = {RNA Structure Comparison, Motif Search and Discovery Using a Reduced Representation of RNA Conformational Space},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  number = {16},
  pages = {4755-4761},
  doi = {http://dx.doi.org/10.1093/nar/gkg682}
}
</pre></td>
</tr>
<tr id="eddy2004" class="entry">
	<td>Eddy, S. R.</td>
	<td>How do RNA folding algorithms work? <p class="infolinks">[<a href="javascript:toggleInfo('eddy2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('eddy2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nature Biotechnology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nbt1104-1457">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_eddy2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Programs such as MFOLD and ViennaRNA are widely used to predict RNA secondary structures. How do these algorithms work? Why can't they predict RNA pseudoknots? How accurate are they, and will they get better?</td>
</tr>
<tr id="bib_eddy2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{eddy2004,
  author = {Sean R. Eddy},
  title = {How do RNA folding algorithms work?},
  journal = {Nature Biotechnology},
  year = {2004},
  volume = {22},
  number = {11},
  pages = {1457-1458},
  doi = {http://dx.doi.org/10.1038/nbt1104-1457}
}
</pre></td>
</tr>
<tr id="eisen1998" class="entry">
	<td>Eisen, M. B., Spellman, P. T., Browndagger, P. O. &amp; Botstein, D.</td>
	<td>Cluster analysis and display of genome-wide expression patterns <p class="infolinks">[<a href="javascript:toggleInfo('eisen1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('eisen1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Proceedings of the National Academy of Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.pnas.org/cgi/content/full/95/25/14863">URL</a> &nbsp;</td>
</tr>
<tr id="abs_eisen1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A system of cluster analysis for genome-wide expression data from DNA microarray hybridization is described that uses standard statistical algorithms to arrange genes according to similarity in pattern of gene expression. The output is displayed graphically, conveying the clustering and the underlying expression data simultaneously in a form intuitive for biologists. We have found in the budding yeast Saccharomyces cerevisiae that clustering gene expression data groups together efficiently genes of known similar function, and we find a similar tendency in human data. Thus patterns seen in genome-wide expression experiments can be interpreted as indications of the status of cellular processes. Also, coexpression of genes of known function with poorly characterized or novel genes may provide a simple means of gaining leads to the functions of many genes for which information is not available currently.</td>
</tr>
<tr id="bib_eisen1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{eisen1998,
  author = {Michael B. Eisen and Paul T. Spellman and Patrick O. Browndagger and David Botstein},
  title = {Cluster analysis and display of genome-wide expression patterns},
  journal = {Proceedings of the National Academy of Science},
  year = {1998},
  volume = {95},
  pages = {14863-14868},
  url = {http://www.pnas.org/cgi/content/full/95/25/14863}
}
</pre></td>
</tr>
<tr id="endres2004" class="entry">
	<td>Endres, R. G., Cox, D. L. &amp; Singh, R. R. P.</td>
	<td>Colloquium: The Quest for High-conductance DNA <p class="infolinks">[<a href="javascript:toggleInfo('endres2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('endres2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Rev. Mod. Phys. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1103/RevModPhys.76.195">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_endres2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The DNA molecule, well known from biology for containing the genetic code of all living species, has recently caught the attention of chemists and physicists. A major reason for this interest is DNA's potential use in nanoelectronic devices, both as a template for assembling nanocircuits and as an element of such circuits. Without question, a truly conducting form of DNA would have a major impact on developments in nanotechnology. It has also been suggested that extended electronic states of DNA could play an important role in biology, e.g., through the processes of DNA damage sensing or repair or through long-range charge transfer. However, the electronic properties of DNA remain very controversial. Charge-transfer reactions and conductivity measurements show a large variety of possible electronic behavior, ranging from Anderson and band-gap insulators to effective molecular wires and induced superconductors. Indeed, understanding the conductance of a complicated polyelectrolytic aperiodic system is by itself a major scientific problem. In this Colloquium, the authors summarize the wide-ranging experimental and theoretical results and look for any consistencies between them. They also pose simple questions regarding the electronic states of DNA within the framework of generalized Hï¿½ckel and Slater-Koster theories. The Colloquium provides a quantitative overview of DNA's electronic states as obtained from density-functional theory, focusing on dependence on structure, on molecular stretching and twisting, and on water and counterions. While there is no clear theoretical basis for truly metallic DNA, situations are discussed in which very small energy gaps might arise in the overall DNA/water/counterion complex, leading to thermally activated conduction at room temperature.</td>
</tr>
<tr id="bib_endres2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{endres2004,
  author = {R. G. Endres and D. L. Cox and R. R. P. Singh},
  title = {Colloquium: The Quest for High-conductance DNA},
  journal = {Rev. Mod. Phys.},
  year = {2004},
  volume = {76},
  pages = {195-214},
  doi = {http://dx.doi.org/10.1103/RevModPhys.76.195}
}
</pre></td>
</tr>
<tr id="estrada2007" class="entry">
	<td>Estrada, E.</td>
	<td>Tight-Binding "Dihedral Orbitals" Approach to the Degree of Folding of Macromolecular Chains <p class="infolinks">[<a href="javascript:toggleInfo('estrada2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Physical Chemistry B. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp074595x">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_estrada2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{estrada2007,
  author = {Ernesto Estrada},
  title = {Tight-Binding "Dihedral Orbitals" Approach to the Degree of Folding of Macromolecular Chains},
  journal = {Journal of Physical Chemistry B.},
  year = {2007},
  volume = {111},
  pages = {13611-13618},
  doi = {http://dx.doi.org/10.1021/jp074595x}
}
</pre></td>
</tr>
<tr id="ferre1999" class="entry">
	<td>Ferre, A. R. &amp; Doudna, J. A.</td>
	<td>RNA Folds: Insights from Recent Crystal Structures <p class="infolinks">[<a href="javascript:toggleInfo('ferre1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('ferre1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Annual Review Biophysics Biomolecular Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/1056-8700/99/0610-0057">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_ferre1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Twenty-five years ago, the structure determinations of tRNAPhe demonstrated<p>that ribonucleic acid can adopt a compact, globular fold (27, 48). In the last<p>five years, the structures of three new RNA folds have been determined at nearatomic<p>resolution: the hammerhead ribozyme (45, 51), the P4ï¿½P6 domain of the<p>Tetrahymena group I intron (4), and the hepatitis delta virus (HDV) ribozyme<p>(18a). Side-by-side comparison of the four known RNA folds shows that they<p>all are constructed by packing together coaxially stacked helices. To different<p>extents in the four folds, the packing appears to be stabilized by nonhelical<p>or tertiary hydrogen bonding networks, coordination of metal ions, and by the<p>connectivity of the RNA backbone.<p>In this review, we first summarize the overall features of the four known<p>RNA folds, and then compare and contrast their size, complexity, and inferred<p>or experimentally verified modes of fold stabilization. We are witnessing rapid<p>growth of structural information on RNA and can expect many more structure<p>determinations of new RNA folds in the near future. The structural information<p>will be pivotal in elucidating the biochemical roles and capabilities of RNA in<p>biological systems.</td>
</tr>
<tr id="bib_ferre1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ferre1999,
  author = {Adrian R. Ferre and Jennifer A. Doudna},
  title = {RNA Folds: Insights from Recent Crystal Structures},
  journal = {Annual Review Biophysics Biomolecular Structure},
  year = {1999},
  volume = {28},
  pages = {57-73},
  doi = {http://dx.doi.org/1056-8700/99/0610-0057}
}
</pre></td>
</tr>
<tr id="flebig1993" class="entry">
	<td>Flebig, K. M. &amp; Dill, K. A.</td>
	<td>Protein Core Assembly Process <p class="infolinks">[<a href="javascript:toggleInfo('flebig1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('flebig1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>J. Chem. Phys. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1063/1.464068">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_flebig1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: How does a protein or HP (hydrophobic/polar) copolymer find its globally optimal (native) state without a globally exhaustive search? This is the Levinthal paradox. We consider three routes by which a copolymer might assemble a compact conformation with a maximum number of hydrophobic (HH) contacts: (i) the exhaustive search (ES) process, which assures the global optimum; (ii) a ``maximum entropy string'' (MES), a series of stepwise decisions each of which explores conformational space exhaustively for given prior contacts; and (iii) a ``T-local string,'' or ``hydrophobic zippers'' (HZ) process, which makes HH contacts opportunistically based on prior contacts. Using a two-dimensional HP short-chain lattice model, for which the partition function is exactly enumerable, we find that for many HP sequences, T-local strings lead to the globally optimal conformation, offering a resolution to the Levinthal paradox. The Journal of Chemical Physics is copyrighted by The American Institute of Physics.</td>
</tr>
<tr id="bib_flebig1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{flebig1993,
  author = {Klaus M. Flebig and Ken A. Dill},
  title = {Protein Core Assembly Process},
  journal = {J. Chem. Phys.},
  year = {1993},
  volume = {98},
  pages = {3475-3487},
  doi = {http://dx.doi.org/10.1063/1.464068}
}
</pre></td>
</tr>
<tr id="galindo2006" class="entry">
	<td>Galindo, J. F., Bermudez, C. I. &amp; Daza, E. E.</td>
	<td>tRNA structure from a graph and quantum theoretical perspective <p class="infolinks">[<a href="javascript:toggleInfo('galindo2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('galindo2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Theoretical Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jtbi.2005.10.017">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_galindo2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the objectives of theoretical biochemistry is to find a suitable representation of molecules allowing us to encode what we know about their structures, interactions and reactivity. Particularly, tRNA structure is involved in some processes like aminoacylation and genetic code translation, and for this reason these molecules represent a biochemical object of the utmost importance requiring characterization. We propose here two fundamental aspects for characterizing and modeling them. The first takes into consideration the connectivity patterns, i.e. the set of linkages between atoms or molecular fragments (a key tool for this purpose is the use of graph theory), and the second one requires the knowledge of some properties related to the interactions taking place within the molecule, at least in an approximate way, and perhaps of its reactivity in certain means. We used quantum mechanics to achieve this goal; specifically, we have used partial charges as a manifestation of the reply to structural changes. These charges were appropriately modified to be used as weighted factors for elements constituting the molecular graph. This new graph-tRNA context allow us to detect some structure-function relationships.</td>
</tr>
<tr id="bib_galindo2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{galindo2006,
  author = {Johan F. Galindo and Clara I. Bermudez and Edgar E. Daza},
  title = {tRNA structure from a graph and quantum theoretical perspective},
  journal = {Journal of Theoretical Biology},
  year = {2006},
  volume = {240},
  pages = {574-582},
  doi = {http://dx.doi.org/10.1016/j.jtbi.2005.10.017}
}
</pre></td>
</tr>
<tr id="gan2004" class="entry">
	<td>Gan, H. H., Fera, D., Julie, Z., Nahum, S., Michael, T., Uri, L., Namhee, K. &amp; Tamar, S.</td>
	<td>RAG: RNA-As-Graphs Database. Concepts, Analysis, and Features <p class="infolinks">[<a href="javascript:toggleInfo('gan2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gan2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="doi:10.1093/bioinformatics/bth084">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gan2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Motivation: Understanding RNA's structural diversity is vital for identifying novel RNA structures and pursuing RNA genomics initiatives. By classifying RNA secondary motifs based on correlations between conserved RNA secondary structures and functional properties, we offer an avenue for predicting novel motifs. Although several RNA databases exist, no comprehensive schemes are available for cataloguing the range and diversity of RNA's structural repertoire. Results: Our RNA-As-Graphs (RAG) database describes and ranks all mathematically possible (including existing and candidate) RNA secondary motifs on the basis of graphical enumeration techniques. We represent RNA secondary structures as two-dimensional graphs (networks), specifying the connectivity between RNA secondary structural elements such as loops, bulges, stems, and junctions. We archive RNA tree motifs as "tree graphs" and other RNAs, including pseudoknots, as general "dual graphs". All RNA motifs are catalogued by graph vertex number (a measure of sequence length) and ranked by topological complexity. The RAG inventory immediately suggests candidates for novel RNA motifs, either naturally occurring or synthetic, and thereby might stimulate the prediction and design of novel RNA motifs.</td>
</tr>
<tr id="bib_gan2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gan2004,
  author = {Hin Hark Gan and Daniela Fera and Zorn Julie and Shiffeldrim Nahum and Tang Michael and Laserson Uri and Kim Namhee and Schlick Tamar},
  title = {RAG: RNA-As-Graphs Database. Concepts, Analysis, and Features},
  journal = {Bioinformatics},
  year = {2004},
  volume = {20},
  number = {8},
  pages = {1285-1291},
  doi = {doi:10.1093/bioinformatics/bth084}
}
</pre></td>
</tr>
<tr id="gan2003" class="entry">
	<td>Gan, H. H., Pasquali, S. &amp; Schlick, T.</td>
	<td>Exploring the Repertoire of RNA Secondary Motifs Using Graph Theory; Implications for RNA Design <p class="infolinks">[<a href="javascript:toggleInfo('gan2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gan2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg365">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gan2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Understanding the structural repertoire of RNA is crucial for RNA genomics research. Yet current methods for finding novel RNAs are limited to small or known RNA families. To expand known RNA structural motifs, we develop a two-dimensional graphical representation approach for describing and estimating the size of RNA?s secondary structural repertoire, including naturally occurring and other possible RNA motifs. We employ tree graphs to describe RNA tree motifs and more general (dual) graphs to describe both RNA tree and pseudoknot motifs. Our estimates of RNA?s structural space are vastly smaller than the nucleotide sequence space, suggesting a new avenue for finding novel RNAs. Specifically our survey shows that known RNA trees and pseudoknots represent only a small subset of all possible motifs, implying that some of the ?missing? motifs may represent novel RNAs. To help pinpoint RNA-like motifs, we show that the motifs of existing functional RNAs are clustered in a narrow range of topological characteristics. We also illustrate the applications of our approach to the design of novel RNAs and automated comparison of RNA structures; we report several occurrences of RNA motifs within larger RNAs. Thus, our graph theory approach to RNA structures has implications for RNA genomics, structure analysis and design.</td>
</tr>
<tr id="bib_gan2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gan2003,
  author = {Hin Hark Gan and Samuela Pasquali and Tamar Schlick},
  title = {Exploring the Repertoire of RNA Secondary Motifs Using Graph Theory; Implications for RNA Design},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  number = {11},
  pages = {2926-2943},
  doi = {http://dx.doi.org/10.1093/nar/gkg365}
}
</pre></td>
</tr>
<tr id="garcia2007" class="entry">
	<td>Garcia, A. E. &amp; Paschek, D.</td>
	<td>Simulation of the Pressure and Temperature Folding/Unfolding Equilibrium of a Small RNA Hairpin <p class="infolinks">[<a href="javascript:toggleInfo('garcia2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('garcia2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja074191i">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_garcia2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We report molecular dynamics simulations of the equilibrium folding/unfolding thermodynamics of the RNA tetraloop in explicit solvent. A replica exchange molecular dynamics study of the r(CGUUGCCG) oligomer that forms a hairpin is performed for 226 ns per replica, using 52 replicas. We are able to show the unbiased folding of all replicas starting from extended conformations. The equilibrium pressure-temperature free energy of folding, DG(P,T), is calculated from the averaged energy, pressure, and specific volume change upon folding of the oligomer as a function of T at constant volume. We find that this oligomer is destabilized by increasing hydrostatic pressure, similar to the behavior of globular proteins</td>
</tr>
<tr id="bib_garcia2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{garcia2007,
  author = {Angel E. Garcia and Dietmar Paschek},
  title = {Simulation of the Pressure and Temperature Folding/Unfolding Equilibrium of a Small RNA Hairpin},
  journal = {Journal of the American Chemical Society},
  year = {2007},
  volume = {130},
  pages = {815-817},
  doi = {http://dx.doi.org/10.1021/ja074191i}
}
</pre></td>
</tr>
<tr id="gautheret1993" class="entry">
	<td>Gautheret, D., Major, F. &amp; Cedergren, R.</td>
	<td>Modeling the Three-dimensional Structure of RNA Using Discrete Nucleotide Conformational Sets <p class="infolinks">[<a href="javascript:toggleInfo('gautheret1993','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gautheret1993','bibtex')">BibTeX</a>]</p></td>
	<td>1993</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1993.1104">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gautheret1993" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The flexibility about seven torsion angles in nucleotides constitutes a severe obstacle to computer modeling of RNA. The computational feasibility of RNA conformational searches can be enhanced by assigning to each nucleotide a set of discrete conformations. In this work, four types of discrete conformational sets for the atomic representation of nucleotide structures were defined and evaluated. These sets, comprising between 10 and 30 conformations, were tested for their ability to reproduce known RNA structures and to generate structures responding to new specifications. Conformational searches were performed with the MC-SYM program, which allows for the generation of all structures satisfying a predetermined set of three-dimensional constraints in a given discrete space. Results with known hairpin loop structures show that root-mean-square deviations of about 1ï¿½5 ï¿½ for backbone atoms and about 2?0 ï¿½ for all atoms between the modeled and X-ray crystal structures can be expected. The conformational set that gives the most faithful representation of test structures is based on the classification of nucleotide conformations derived from a structural database. Representative conformations are selected from each class that adequately sample variations in backbone direction, sugar pucker and base orientation. With this conformational set, most of the important features of test hairpin structures are reproduced with fidelity, indicating that biologically useful models can be constructed from the combination of discrete nucleotide conformations and an algorithm that rapidly and systematically scans the pre-defined conformational space.</td>
</tr>
<tr id="bib_gautheret1993" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gautheret1993,
  author = {Daniel Gautheret and Franï¿½ois Major and Robert Cedergren},
  title = {Modeling the Three-dimensional Structure of RNA Using Discrete Nucleotide Conformational Sets},
  journal = {Journal of Molecular Biology},
  year = {1993},
  volume = {229},
  number = {4},
  pages = {1049-1064},
  doi = {http://dx.doi.org/10.1006/jmbi.1993.1104}
}
</pre></td>
</tr>
<tr id="ge2005" class="entry">
	<td>Ge, W., Schneider, B. &amp; Olson, W. K.</td>
	<td>Knowledge-Based Elastic Potentials for Docking Drugs or Proteins with Nucleic Acids <p class="infolinks">[<a href="javascript:toggleInfo('ge2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('ge2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.104.043612">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_ge2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Elastic ellipsoidal functions defined by the observed hydration patterns around the DNA bases provide a new basis for measuring the recognition of ligands in the grooves of double-helical structures. Here a set of knowledge-based potentials suitable for quantitative description of such behavior is extracted from the observed positions of water molecules and amino acid atoms that form hydrogen bonds with the nitrogenous bases in high resolution crystal structures. Energies based on the displacement of hydrogen-bonding sites on drugs in DNA-crystal complexes relative to the preferred locations of water binding around the heterocyclic bases are low, pointing to the reliability of the potentials and the apparent displacement of water molecules by drug atoms in these structures. The validity of the energy functions has been further examined in a series of sequence substitution studies based on the structures of DNA bound to polyamides that have been designed to recognize the minor-groove edges of Watson-Crick basepairs. The higher energies of binding to incorrect sequences superimposed (without conformational adjustment or displacement of polyamide ligands) on observed high resolution structures confirm the hypothesis that the drug subunits associate with specific DNA bases. The knowledge-based functions also account satisfactorily for the measured free energies of DNA-polyamide association in solution and the observed sites of polyamide binding on nucleosomal DNA. The computations are generally consistent with mechanisms by which minor-groove binding ligands are thought to recognize DNA basepairs. The calculations suggest that the asymmetric distributions of hydrogen-bond-forming atoms on the minor-groove edge of the basepairs may underlie ligand discrimination of Gï¿½C from Cï¿½G pairs, in addition to the commonly believed role of steric hindrance. The analysis of polyamide-bound nucleosomal structures reveals other discrepancies in the expected chemical design, including unexpected contacts to DNA and modified basepair targets of some ligands. The ellipsoidal potentials thus appear promising as a mathematical tool for the study of drug- and protein-DNA interactions and for gaining new insights into DNA-binding mechanisms.</td>
</tr>
<tr id="bib_ge2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ge2005,
  author = {Wei Ge and Bohdan Schneider and Wilma K. Olson},
  title = {Knowledge-Based Elastic Potentials for Docking Drugs or Proteins with Nucleic Acids},
  journal = {Biophysical Journal},
  year = {2005},
  volume = {88},
  pages = {1166-1190},
  doi = {http://dx.doi.org/10.1529/biophysj.104.043612}
}
</pre></td>
</tr>
<tr id="gerstein2003" class="entry">
	<td>Gerstein, M. &amp; Thornton, J. M.</td>
	<td>Sequences and Topology <p class="infolinks">[<a href="javascript:toggleInfo('gerstein2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gerstein2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(03)00080-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gerstein2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In what follows, we give an overview of the eight reviews in the Sequences and topology section, describing and summarizing their content. Basically, the reviews this year follow the overall progression in bioinformatics from genome sequence to functional genomics analysis to protein structure. We start with reviews by Karlin and Lancet on genome analysis, the former focusing on more global issues and the latter on what can be learned from specific families. Then, the review by Orengo discusses mapping from the genome to protein families and structure. Stolovitzky talks about analyzing expression information, functional genomics data that characterize the sequence products. The next group of reviews, by Eisenberg and Janin, discuss proteinï¿½protein interactions, the interactome, and the current excitement in interpreting this. Finally, we have reviews by Jackson and Westhead, and Nakamura on more detailed aspects of three-dimensional site analysis.</td>
</tr>
<tr id="bib_gerstein2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gerstein2003,
  author = {Mark Gerstein and Janet M. Thornton},
  title = {Sequences and Topology},
  journal = {Current Opinion in Structural Biology},
  year = {2003},
  volume = {13},
  pages = {341-343},
  doi = {http://dx.doi.org/10.1016/S0959-440X(03)00080-0}
}
</pre></td>
</tr>
<tr id="giegerich2004" class="entry">
	<td>Giegerich, R., Voï¿½, B. &amp; Rehmsmeier, M.</td>
	<td>Abstract shapes of RNA <p class="infolinks">[<a href="javascript:toggleInfo('giegerich2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('giegerich2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkh779">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_giegerich2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The function of a non-protein-coding RNA is often determined by its structure. Since experimental determination of RNA structure is time-consuming and expensive, its computational prediction is of great interest, and efficient solutions based on thermodynamic parameters are known. Frequently, however, the predicted minimum free energy structures are not the native ones, leading to the necessity of generating suboptimal solutions. While this can be accomplished by a number of programs, the user is often confronted with large outputs of similar structures, although he or she is interested in structures with more fundamental differences, or, in other words, with different abstract shapes. Here, we formalize the concept of abstract shapes and introduce their efficient computation. Each shape of an RNA molecule comprises a class of similar structures and has a representative structure of minimal free energy within the class. Shape analysis is implemented in the program RNAshapes. We applied RNAshapes to the prediction of optimal and suboptimal abstract shapes of several RNAs. For a given energy range, the number of shapes is considerably smaller than the number of structures, and in all cases, the native structures were among the top shape representatives. This demonstrates that the researcher can quickly focus on the structures of interest, without processing up to thousands of near-optimal solutions. We complement this study with a large-scale analysis of the growth behaviour of structure and shape spaces. RNAshapes is available for download and as an online version on the Bielefeld Bioinformatics Server.</td>
</tr>
<tr id="bib_giegerich2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{giegerich2004,
  author = {Robert Giegerich and Bjï¿½rn Voï¿½ and Marc Rehmsmeier},
  title = {Abstract shapes of RNA},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  pages = {4843-4851},
  doi = {http://dx.doi.org/10.1093/nar/gkh779}
}
</pre></td>
</tr>
<tr id="gnatt2001" class="entry">
	<td>Gnatt, A. L., Cramer, P., Fu, J., Bushnell, D. A. &amp; Kornberg, R. D.</td>
	<td>Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 &Aring; resolution. <p class="infolinks">[<a href="javascript:toggleInfo('gnatt2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gnatt2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1059495">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gnatt2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structure of RNA polymerase II in the act of transcription was determined at 3.3 ï¿½ resolution. Duplex DNA is seen entering the main cleft of the enzyme and unwinding before the active site. Nine base pairs of DNA-RNA hybrid extend from the active center at nearly right angles to the entering DNA, with the 3' end of the RNA in the nucleotide addition site. The 3' end is positioned above a pore, through which nucleotides may enter and through which RNA may be extruded during back-tracking. The 5'-most residue of the RNA is close to the point of entry to an exit groove. Changes in protein structure between the transcribing complex and free enzyme include closure of a clamp over the DNA and RNA and ordering of a series of "switches" at the base of the clamp to create a binding site complementary to the DNA-RNA hybrid. Protein-nucleic acid contacts help explain DNA and RNA strand separation, the specificity of RNA synthesis, "abortive cycling" during transcription initiation, and RNA and DNA translocation during transcription elongation.</td>
</tr>
<tr id="bib_gnatt2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gnatt2001,
  author = {Averell L. Gnatt and Patrick Cramer and Jianhua Fu and David A. Bushnell and Roger D. Kornberg},
  title = {Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 &Aring; resolution.},
  journal = {Science},
  year = {2001},
  volume = {292},
  pages = {1876-1882},
  doi = {http://dx.doi.org/10.1126/science.1059495}
}
</pre></td>
</tr>
<tr id="go1974" class="entry">
	<td>Go, N., Go, M. &amp; Scheraga, H.</td>
	<td>New Method for Calculating the Conformational Entropy of a Regular Helix <p class="infolinks">[<a href="javascript:toggleInfo('go1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>Macromolecules &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_go1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{go1974,
  author = {Nobuhiro Go and Mitiko Go and Harold Scheraga},
  title = {New Method for Calculating the Conformational Entropy of a Regular Helix},
  journal = {Macromolecules},
  year = {1974},
  volume = {7},
  pages = {137-139}
}
</pre></td>
</tr>
<tr id="gorodkin1999" class="entry">
	<td>Gorodkin, J., Staerfeldt, H. H., Lund, O. &amp; Brunak, S.</td>
	<td>MatrixPlot: visualizing sequence constraints <p class="infolinks">[<a href="javascript:toggleInfo('gorodkin1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/bioinformatics/15.9.769">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_gorodkin1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gorodkin1999,
  author = {J. Gorodkin and H. H. Staerfeldt and O. Lund and S. Brunak},
  title = {MatrixPlot: visualizing sequence constraints},
  journal = {Bioinformatics},
  year = {1999},
  volume = {15},
  pages = {769-770},
  doi = {http://dx.doi.org/10.1093/bioinformatics/15.9.769}
}
</pre></td>
</tr>
<tr id="takada1983" class="entry">
	<td>Guerrier-Takada, C., Gardiner, K., Marsh, T., Pace, N. &amp; Altman, S.</td>
	<td>The RNA moiety of ribonuclease P is the catalytic subunit of the enzyme. <p class="infolinks">[<a href="javascript:toggleInfo('takada1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Cell &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_takada1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{takada1983,
  author = {Cecilia Guerrier-Takada and Katheleen Gardiner and Terry Marsh and Norman Pace and Sidney Altman},
  title = {The RNA moiety of ribonuclease P is the catalytic subunit of the enzyme.},
  journal = {Cell},
  year = {1983},
  volume = {35},
  pages = {849-857}
}
</pre></td>
</tr>
<tr id="gutell1994" class="entry">
	<td>Gutell, R. R., Larsen, N. &amp; Woese, C. R.</td>
	<td>Lessons from an evolving rRNA: 16S and 23S rRNA structures from a comparative perspective. <p class="infolinks">[<a href="javascript:toggleInfo('gutell1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Microbiology and Molecular Biology Reviews &nbsp;</td>
	<td>article</td>
	<td><a href="http://mmbr.asm.org/cgi/content/abstract/58/1/10">URL</a> &nbsp;</td>
</tr>
<tr id="bib_gutell1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gutell1994,
  author = {R R Gutell and N Larsen and C R Woese},
  title = {Lessons from an evolving rRNA: 16S and 23S rRNA structures from a comparative perspective.},
  journal = {Microbiology and Molecular Biology Reviews},
  year = {1994},
  volume = {58},
  pages = {10-26},
  url = {http://mmbr.asm.org/cgi/content/abstract/58/1/10}
}
</pre></td>
</tr>
<tr id="gutell2002" class="entry">
	<td>Gutell, R. R., Lee, J. C. &amp; Cannone, J. J.</td>
	<td>The accuracy of ribosomal RNA comparative structure models <p class="infolinks">[<a href="javascript:toggleInfo('gutell2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('gutell2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(02)00339-1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_gutell2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The determination of the 16S and 23S rRNA secondary structure models was initiated shortly after the first complete 16S and 23S rRNA sequences were determined in the late 1970s. The structures that are common to all 16S rRNAs and all 23S rRNAs were determined using comparative methods from the analysis of thousands of rRNA sequences. Twenty-plus years later, the 16S and 23S rRNA comparative structure models have been evaluated against the recently determined high-resolution crystal structures of the 30S and 50S ribosomal subunits. Nearly all of the predicted covariation-based base pairs, including the regular base pairs and helices, and the irregular base pairs and tertiary interactions, were present in the 30S and 50S crystal structures.</td>
</tr>
<tr id="bib_gutell2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{gutell2002,
  author = {Robin R. Gutell and Jung C. Lee and Jamie J. Cannone},
  title = {The accuracy of ribosomal RNA comparative structure models},
  journal = {Current Opinion in Structural Biology},
  year = {2002},
  volume = {12},
  pages = {301-310},
  doi = {http://dx.doi.org/10.1016/S0959-440X(02)00339-1}
}
</pre></td>
</tr>
<tr id="hagerman1997" class="entry">
	<td>Hagerman, P. J.</td>
	<td>Flexibility of RNA. <p class="infolinks">[<a href="javascript:toggleInfo('hagerman1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hagerman1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Annu Rev Biophys Biomol Struct &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.biophys.26.1.139">DOI</a><a href="http://dx.doi.org/10.1146/annurev.biophys.26.1.139">URL</a> &nbsp;</td>
</tr>
<tr id="abs_hagerman1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the fundamental properties of the RNA helix is its intrinsic resistance to bend- or twist-deformations. Results of a variety of physical measurements point to a persistence length of 700-800 A for double-stranded RNA in the presence of magnesium cations, approximately 1.5-2.0-fold larger than the corresponding value for DNA. Although helix flexibility represents an important, quantifiable measure of the forces of interaction within the helix, it must also be considered in describing conformational variation of nonhelix elements (e.g. internal loops, branches), since the latter always reflect the properties of the flanking helices; that is, such elements are never completely rigid. For one important element of tertiary structure, namely, the core of yeast tRNAPhe, the above consideration has led to the conclusion that the core is not substantially more flexible than an equivalent length of pure helix.</td>
</tr>
<tr id="bib_hagerman1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hagerman1997,
  author = {P. J. Hagerman},
  title = {Flexibility of RNA.},
  journal = {Annu Rev Biophys Biomol Struct},
  year = {1997},
  volume = {26},
  pages = {139-156},
  url = {http://dx.doi.org/10.1146/annurev.biophys.26.1.139},
  doi = {http://dx.doi.org/10.1146/annurev.biophys.26.1.139}
}
</pre></td>
</tr>
<tr id="harary1972" class="entry">
	<td>Harary, F.</td>
	<td>Graph Theory <p class="infolinks">[<a href="javascript:toggleInfo('harary1972','bibtex')">BibTeX</a>]</p></td>
	<td>1972</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_harary1972" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{harary1972,
  author = {Frank Harary},
  title = {Graph Theory},
  publisher = {Addison-Wesley},
  year = {1972}
}
</pre></td>
</tr>
<tr id="harris1997" class="entry">
	<td>Harris, M. E., Kazantsev, A. V., Chen, J. L. &amp; Pace, N. R.</td>
	<td>Analysis of the tertiary structure of the ribonuclease P ribozyme-substrate complex by site-specific photoaffinity crosslinking <p class="infolinks">[<a href="javascript:toggleInfo('harris1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('harris1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.rnajournal.org/cgi/reprint/3/6/561.pdf">URL</a> &nbsp;</td>
</tr>
<tr id="abs_harris1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Bacterial ribonuclease P (RNase P), an endonuclease involved in tRNA maturation, is a ribonucleoprotein containing a catalytic RNA. The secondary structure of this ribozyme is well-established, and a low-resolution model of the three-dimensional structure of the ribozyme-substrate complex has been proposed based on site-specific crosslinking and phylogenetic comparative data [Harris ME et al., 1994 EMBO J 13:3953-3963]. However, several substructures of that model were poorly constrained by the available data. In the present analysis, additional constraints between elements within the Escherichia coli RNase P RNA-pre-tRNA complex were determined by intra- and intermolecular crosslinking experiments. Circularly permuted RNase P RNAs were used to position an azidophenacyl photoactive crosslinking agent specifically at strategic sites within the ribozyme-substrate complex. Crosslink sites were mapped by primer extension and confirmed by analysis of the mobility of the crosslinked RNA lariats on denaturing acrylamide gels relative to circular and linear RNA standards. Crosslinked species generally retained significant catalytic activity, indicating that the results reflect the native ribozyme structure. The crosslinking results support the general configuration of the structure model and predicate new positions and orientations for helices that were previously poorly constrained by the data set. The expanded library of crosslinking constraints was used, together with secondary and tertiary structure identified by phylogenetic sequence comparisons, to refine significantly the model of RNase P RNA with bound substrate pre-tRNA. The crosslinking results and data from chemical-modification and mutational studies are discussed in the context of the current structural perspective on this ribozyme.</td>
</tr>
<tr id="bib_harris1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{harris1997,
  author = {M. E. Harris and A. V. Kazantsev and J. L. Chen and N. R. Pace},
  title = {Analysis of the tertiary structure of the ribonuclease P ribozyme-substrate complex by site-specific photoaffinity crosslinking},
  journal = {RNA},
  year = {1997},
  volume = {3},
  pages = {561-576},
  url = {http://www.rnajournal.org/cgi/reprint/3/6/561.pdf}
}
</pre></td>
</tr>
<tr id="harris2006" class="entry">
	<td>Harris, S. A.</td>
	<td>Modelling the biomechanical properties of DNA using computer simulation. <p class="infolinks">[<a href="javascript:toggleInfo('harris2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('harris2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Philos Transact A Math Phys Eng Sci &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1098/rsta.2006.1906">DOI</a><a href="http://dx.doi.org/10.1098/rsta.2006.1906">URL</a> &nbsp;</td>
</tr>
<tr id="abs_harris2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Duplex DNA must remain stable when not in use to protect the genetic material. However, the two strands must be separated whenever genes are copied or expressed to expose the coding strand for synthesis of complementary RNA or DNA bases. Therefore, the double stranded structure must be relatively easy to take apart when required. These conflicting biological requirements have important implications for the mechanical properties of duplex DNA. Considerable insight into the forces required to denature DNA has been provided by nanomanipulation experiments, which measure the mechanical properties of single molecules in the laboratory. This paper describes recent computer simulation methods that have been developed to mimic nanomanipulation experiments and which, quite literally, 'destruction test' duplex DNA in silico. The method is verified by comparison with single molecule stretching experiments that measure the force required to unbind the two DNA strands. The model is then extended to investigate the thermodynamics of DNA bending and twisting. This is of biological importance as the DNA must be very tightly packaged to fit within the nucleus, and is therefore usually found in a highly twisted or supercoiled state (in bacteria) or wrapped tightly around histone proteins into a densely compacted structure (in animals). In particular, these simulations highlight the importance of thermal fluctuations and entropy in determining the biomechanical properties of DNA. This has implications for the action of DNA processing molecular motors, and also for nanotechnology. Biological machines are able to manipulate single molecules reliably on an energy scale comparable to that of thermal noise. The hope is that understanding the statistical mechanisms that a cell uses to achieve this will be invaluable for the future design of 'nanoengines' engineered to perform new technological functions at the nanoscale.</td>
</tr>
<tr id="bib_harris2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{harris2006,
  author = {Sarah Anne Harris},
  title = {Modelling the biomechanical properties of DNA using computer simulation.},
  journal = {Philos Transact A Math Phys Eng Sci},
  year = {2006},
  volume = {364},
  number = {1849},
  pages = {3319--3334},
  url = {http://dx.doi.org/10.1098/rsta.2006.1906},
  doi = {http://dx.doi.org/10.1098/rsta.2006.1906}
}
</pre></td>
</tr>
<tr id="harris2007" class="entry">
	<td>Harris, S. A. &amp; Laughton, C. A.</td>
	<td>A Simple Physical Description of DNA Dynamics: Quasi-Harmonic Analysis as a Route to the Configurational Entropy <p class="infolinks">[<a href="javascript:toggleInfo('harris2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('harris2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Physics Condensed Matter &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1088/0953-8984/19/7/076103">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_harris2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: It has become increasingly apparent that the dynamic as well as the structural<p>properties of biological macromolecules are important to their function.<p>However, information concerning molecular flexibility can be difficult to obtain<p>experimentally at the atomic level. Computer modelling techniques such<p>as molecular dynamics (MD) have therefore proved invaluable in advancing<p>our understanding of biomolecular flexibility. This paper describes how a<p>combination of atomistic MD simulations and quasi-harmonic analysis can<p>be used to describe the dynamics of duplex DNA, with a particular emphasis<p>on methods for calculating differences in configurational entropies. We<p>demonstrate that DNA possesses remarkably simple mechanical properties<p>relative to globular proteins, making it an ideal system for exploring<p>biomolecular flexibility in general. Our results also highlight the importance<p>of solvent viscosity in determining the dynamic behaviour of DNA in aqueous<p>solution</td>
</tr>
<tr id="bib_harris2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{harris2007,
  author = {S. A. Harris and C. A. Laughton},
  title = {A Simple Physical Description of DNA Dynamics: Quasi-Harmonic Analysis as a Route to the Configurational Entropy},
  journal = {Journal of Physics Condensed Matter},
  year = {2007},
  volume = {19},
  pages = {1-14},
  doi = {http://dx.doi.org/10.1088/0953-8984/19/7/076103}
}
</pre></td>
</tr>
<tr id="haspel2003" class="entry">
	<td>Haspel, N., Tsai, C., Wolfson, H. &amp; Nussinov, R.</td>
	<td>Reducing the computational complexity of protein folding via fragment folding and assembly <p class="infolinks">[<a href="javascript:toggleInfo('haspel2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('haspel2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Protein Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1110/ps.0232903">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_haspel2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Understanding, and ultimately predicting, how a 1-D protein chain reaches its native 3-D fold has been one of the most challenging problems during the last few decades. Data increasingly indicate that protein folding is a hierarchical process. Hence, the question arises as to whether we can use the hierarchical concept to reduce the practically intractable computational times. For such a scheme to work, the first step is to cut the protein sequence into fragments that form local minima on the polypeptide chain. The conformations of such fragments in solution are likely to be similar to those when the fragments are embedded in the native fold, although alternate conformations may be favored during the mutual stabilization in the combinatorial assembly process. Two elements are needed for such cutting: (1) a library of (clustered) fragments derived from known protein structures and (2) an assignment algorithm that selects optimal combinations to "cover" the protein sequence. The next two steps in hierarchical folding schemes, not addressed here, are the combinatorial assembly of the fragments and finally, optimization of the obtained conformations. Here, we address the first step in a hierarchical protein-folding scheme. The input is a target protein sequence and a library of fragments created by clustering building blocks that were generated by cutting all protein structures. The output is a set of cutout fragments. We briefly outline a graph theoretic algorithm that automatically assigns building blocks to the target sequence, and we describe a sample of the results we have obtained.</td>
</tr>
<tr id="bib_haspel2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{haspel2003,
  author = {Nurit Haspel and Chung-Jung Tsai and Haim Wolfson and Ruth Nussinov},
  title = {Reducing the computational complexity of protein folding via fragment folding and assembly},
  journal = {Protein Science},
  year = {2003},
  volume = {12},
  pages = {1177-1187},
  doi = {http://dx.doi.org/10.1110/ps.0232903}
}
</pre></td>
</tr>
<tr id="elhassan1996" class="entry">
	<td>Hassan, M. E. &amp; Calladine, C.</td>
	<td>Structural Mechanics of Bent DNA <p class="infolinks">[<a href="javascript:toggleInfo('elhassan1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('elhassan1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Endeavour &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/0160-9327(96)10016-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_elhassan1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The DNA molecule is a familiar object. It is often depicted in magazines and advertisements as a double helix, with the letters of the genetic code strung along the two spiral backbones and joined together in pairs. In such pictures the molecule is usually shown as straight; yet in the chromosomes of living organisms, DNA is curved and wound up into condensed packages. This article explains what is involved in such bending of DNA in the cell. It uses the ideas of structural mechanics--a tool of engineers--to show how the various components fit together when the molecule is bent.</td>
</tr>
<tr id="bib_elhassan1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{elhassan1996,
  author = {M.A. El Hassan and C.R. Calladine},
  title = {Structural Mechanics of Bent DNA},
  journal = {Endeavour},
  year = {1996},
  volume = {20},
  pages = {61-67},
  doi = {http://dx.doi.org/10.1016/0160-9327(96)10016-8}
}
</pre></td>
</tr>
<tr id="elhassan1998" class="entry">
	<td>Hassan, M. E. &amp; Calladine, C. R.</td>
	<td>Two Distinct Modes of Protein-induced Bending <p class="infolinks">[<a href="javascript:toggleInfo('elhassan1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('elhassan1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1998.1994">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_elhassan1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Crystallised "naked" DNA oligomers in the B form show significant conformational mobility, particularly at CA/TG and TA/TA steps: there is a range in Roll angle of some 15ï¿½ between consecutive base-pairs, and Slide and Twist are directly coupled to Roll. We call such motions "mode I". They are sufficient to enable DNA to curve gently around proteins such as histone octamers in the nucleosome particle. When DNA bends around other proteins, such as CAP and TBP, its distortion is much more severe. Although the DNA in close contact with these proteins includes the CA/TG and TA/TA steps, respectively, the mode I flexibility is not deployed: instead, a more severe "mode II" manoeuvre is observed in DNA/protein co-crystals. Mode II has several distinctive physical features. First, its range of Roll angle is much wider than for mode I. Second, the major-groove width remains more-or-less constant as Roll increases, whereas it decreases significantly as Roll increases in mode I; and this enables the major groove of the DNA to accommodate a protein moiety in its severely bent conformation. Third, the value of Slide remains more-or-less constant as Roll increases, whereas it decreases in mode I. In general, in both modes I and II, the major-groove width appears to be closely related to the Slide between base-pairs. In mode II there appears to be a definite "point pivot" on the major-groove side of the two base-pairs that constitute a dinucleotide step, formed either by the steric interlocking of propeller-twisted base-pairs or by a bifurcated hydrogen bond. Distortion of DNA in mode II seems to be an intrinsic property of the double-helical structure, since it occurs whether protein is bound on the major-groove side (e.g. CAP) or on the minor-groove side (e.g. TBP). Mode II distortion occurs in a wider range of steps than those that show the largest mode-I variation; nevertheless, "access" to mode II deformation appears to be gained via mode I distortion at particular steps CA/TG and TA/TA.</td>
</tr>
<tr id="bib_elhassan1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{elhassan1998,
  author = {M.A. El Hassan and C. R. Calladine},
  title = {Two Distinct Modes of Protein-induced Bending},
  journal = {Journal of Molecular Biology},
  year = {1998},
  volume = {282},
  pages = {331-343},
  doi = {http://dx.doi.org/10.1006/jmbi.1998.1994}
}
</pre></td>
</tr>
<tr id="elhassan1995" class="entry">
	<td>Hassan, M. E. &amp; Calladine, C. R.</td>
	<td>The Assessment of the Geometry of Dinucleotide Steps in Double-Helical DNA; a New Local Calculation Scheme <p class="infolinks">[<a href="javascript:toggleInfo('elhassan1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="doi:10.1006/jmbi.1995.0462">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_elhassan1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{elhassan1995,
  author = {M.A. El Hassan and C. R. Calladine},
  title = {The Assessment of the Geometry of Dinucleotide Steps in Double-Helical DNA; a New Local Calculation Scheme},
  journal = {Journal of Molecular Biology},
  year = {1995},
  volume = {251},
  number = {5},
  pages = {648-664},
  doi = {doi:10.1006/jmbi.1995.0462}
}
</pre></td>
</tr>
<tr id="helm2006" class="entry">
	<td>Helm, M.</td>
	<td>Post-transcriptional nucleotide modification and alternative folding of RNA. <p class="infolinks">[<a href="javascript:toggleInfo('helm2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('helm2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkj471">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_helm2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Alternative foldings are an inherent property of RNA and a ubiquitous problem in scientific investigations. To a living organism, alternative foldings can be a blessing or a problem, and so nature has found both, ways to harness this property and ways to avoid the drawbacks. A simple and effective method employed by nature to avoid unwanted folding is the modulation of conformation space through post-transcriptional base modification. Modified nucleotides occur in almost all classes of natural RNAs in great chemical diversity. There are about 100 different base modifications known, which may perform a plethora of functions. The presumably most ancient and simple nucleotide modifications, such as methylations and uridine isomerization, are able to perform structural tasks on the most basic level, namely by blocking or reinforcing single base-pairs or even single hydrogen bonds in RNA. In this paper, functional, genomic and structural evidence on cases of folding space alteration by post-transcriptional modifications in native RNA are reviewed.</td>
</tr>
<tr id="bib_helm2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{helm2006,
  author = {M. Helm},
  title = {Post-transcriptional nucleotide modification and alternative folding of RNA.},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {721-733},
  doi = {http://dx.doi.org/10.1093/nar/gkj471}
}
</pre></td>
</tr>
<tr id="hendrix2005" class="entry">
	<td>Hendrix, D. K., Brenner, S. E. &amp; Holbrook, S. R.</td>
	<td>RNA Structural Motifs : Building Blocks of a Modular Biomolecule <p class="infolinks">[<a href="javascript:toggleInfo('hendrix2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Quarterly Reviews of Biophysics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1017/S0033583506004215">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_hendrix2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hendrix2005,
  author = {Donna K. Hendrix and Steven E. Brenner and Stephen R. Holbrook},
  title = {RNA Structural Motifs : Building Blocks of a Modular Biomolecule},
  journal = {Quarterly Reviews of Biophysics},
  year = {2005},
  volume = {38},
  pages = {221},
  doi = {http://dx.doi.org/10.1017/S0033583506004215}
}
</pre></td>
</tr>
<tr id="hermann2000" class="entry">
	<td>Hermann, T. &amp; Patel, D. J.</td>
	<td>RNA bulges as architectural and recognition motifs <p class="infolinks">[<a href="javascript:toggleInfo('hermann2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hermann2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0969-2126(00)00110-6">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_hermann2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA bulges constitute versatile structural motifs in the assembly of RNA architectures. Three-dimensional structures of RNA molecules and their complexes reveal the role of bulges in RNA architectures and illustrate the molecular mechanisms by which they confer intramolecular interactions and intermolecular recognition.</td>
</tr>
<tr id="bib_hermann2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hermann2000,
  author = {Thomas Hermann and Dinshaw J. Patel},
  title = {RNA bulges as architectural and recognition motifs},
  journal = {Structure},
  year = {2000},
  volume = {8},
  number = {3},
  pages = {R47-R54},
  doi = {http://dx.doi.org/10.1016/S0969-2126(00)00110-6}
}
</pre></td>
</tr>
<tr id="hermann1999" class="entry">
	<td>Hermann, T. &amp; Patel, D. J.</td>
	<td>Stitching Together RNA Tertiary Architectures <p class="infolinks">[<a href="javascript:toggleInfo('hermann1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1999.3312">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_hermann1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hermann1999,
  author = {Thomas Hermann and Dinshaw J. Patel},
  title = {Stitching Together RNA Tertiary Architectures},
  journal = {Journal of Molecular Biology},
  year = {1999},
  volume = {294},
  pages = {829-849},
  doi = {http://dx.doi.org/10.1006/jmbi.1999.3312}
}
</pre></td>
</tr>
<tr id="hershkovitz2006" class="entry">
	<td>Hershkovitz, E., Sapiro, G., Tannenbaum, A. &amp; Williams, L. D.</td>
	<td>Statistical Analysis of RNA Backbone <p class="infolinks">[<a href="javascript:toggleInfo('hershkovitz2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hershkovitz2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Transactions on Computational Biology and Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1109/TCBB.2006.13">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_hershkovitz2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Local conformation is an important determinant of RNA catalysis and binding. The analysis of RNA conformation is particularly difficult due to the large number of degrees of freedom (torsion angles) per residue. Proteins, by comparison, have many fewer degrees of freedom per residue. In this work, we use and extend classical tools from statistics and signal processing to search for clusters in RNA conformational space. Results are reported both for scalar analysis, where each torsion angle is separately studied, and for vectorial analysis, where several angles are simultaneously clustered. Adapting techniques from vector quantization and clustering to the RNA structure, we find torsion angle clusters and RNA conformational motifs. We validate the technique using well-known conformational motifs, showing that the simultaneous study of the total torsion angle space leads to results consistent with known motifs reported in the literature and also to the finding of new ones.</td>
</tr>
<tr id="bib_hershkovitz2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hershkovitz2006,
  author = {Eli Hershkovitz and Guillermo Sapiro and Allen Tannenbaum and Loren Dean Williams},
  title = {Statistical Analysis of RNA Backbone},
  journal = {Transactions on Computational Biology and Bioinformatics},
  year = {2006},
  volume = {3},
  pages = {33-46},
  doi = {http://dx.doi.org/10.1109/TCBB.2006.13}
}
</pre></td>
</tr>
<tr id="hershkovitz2003" class="entry">
	<td>Hershkovitz, E., Tannenbaum, E., Howerton, S. B., Sheth, A., Tannenbaum, A. &amp; Williams, L. D.</td>
	<td>Automated Identification of RNA Conformational Motifs: Theory and Application to the HM LSU 23S rRNA <p class="infolinks">[<a href="javascript:toggleInfo('hershkovitz2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hershkovitz2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg835">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_hershkovitz2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We develop novel methods for recognizing and<p>cataloging conformational states of RNA, and for<p>discovering statistical rules governing those states.<p>We focus on the conformation of the large ribosomal<p>subunit from Haloarcula marismortui. The<p>two approaches described here involve torsion<p>matching and binning. Torsion matching is a<p>pattern-recognition code which ï¿½nds structural<p>repetitions. Binning is a classiï¿½cation technique<p>based on distributional models of the data. In comparing<p>the results of the two methods we have<p>tested the hypothesis that the conformation of a<p>very large complex RNA molecule can be described<p>accurately by a limited number of discrete conformational<p>states. We identify and eliminate extraneous<p>and redundant information without losing accuracy.<p>We conclude, as expected, that four of the torsion<p>angles contain the overwhelming bulk of the structural<p>information. That information is not signiï¿½-<p>cantly compromised by binning the continuous<p>torsional information into a limited number of<p>discrete values. The correspondence between<p>torsion matching and binning is 99% (per residue).<p>Binning, however, does have several advantages. In<p>particular, we demonstrate that the conformation of<p>a large complex RNA molecule can be represented<p>by a small alphabet. In addition, the binning method<p>lends itself to a natural graphical representation<p>using trees.</td>
</tr>
<tr id="bib_hershkovitz2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hershkovitz2003,
  author = {Eli Hershkovitz and Emmanuel Tannenbaum and Shelley B. Howerton and Ajay Sheth and Allen Tannenbaum and Loren Dean Williams},
  title = {Automated Identification of RNA Conformational Motifs: Theory and Application to the HM LSU 23S rRNA},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  pages = {6249-6257},
  doi = {http://dx.doi.org/10.1093/nar/gkg835}
}
</pre></td>
</tr>
<tr id="hobza2002" class="entry">
	<td>Hobza, P. &amp; Sponer, J.</td>
	<td>Toward True DNA Base-Stacking Energies: MP2, CCSD(T), and Complete Basis Set Calculations <p class="infolinks">[<a href="javascript:toggleInfo('hobza2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hobza2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja026759n">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_hobza2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Stacking energies in low-energy geometries of pyrimidine, uracil, cytosine, and guanine homodimers were determined by the MP2 and CCSD(T) calculations utilizing a wide range of split-valence, correlation-consistent, and bond-functions basis sets. Complete basis set MP2 (CBS MP2) stacking energies extrapolated using aug-cc-pVXZ (X = D, T, and for pyrimidine dimer Q) basis sets equal to -5.3, -12.3, and -11.2 kcal/mol for the first three dimers, respectively. Higher-order correlation corrections estimated as the difference between MP2 and CCSD(T) stacking energies amount to 2.0, 0.7, and 0.9 kcal/mol and lead to final estimates of the genuine stacking energies for the three dimers of -3.4, -11.6, and -10.4 kcal/mol. The CBS MP2 stacking-energy estimate for guanine dimer (-14.8 kcal/mol) was based on the 6-31G*(0.25) and aug-cc-pVDZ calculations. This simplified extrapolation can be routinely used with a meaningful accuracy around 1 kcal/mol for large aromatic stacking clusters. The final estimate of the guanine stacking energy after the CCSD(T) correction amounts to -12.9 kcal/mol. The MP2/6-31G*(0.25) method previously used as the standard level to calculate aromatic stacking in hundreds of geometries of nucleobase dimers systematically underestimates the base stacking by ca. 1.0-2.5 kcal/mol per stacked dimer, covering 75-90% of the intermolecular correlation stabilization. We suggest that this correction is to be considered in calibration of force fields and other cheaper computational methods. The quality of the MP2/6-31G*(0.25) predictions is nevertheless considerably better than suggested on the basis of monomer polarizability calculations. Fast and very accurate estimates of the MP2 aromatic stacking energies can be achieved using the RI-MP2 method. The CBS MP2 calculations and the CCSD(T) correction, when taken together, bring only marginal changes to the relative stability of H-bonded and stacked base pairs, with a slight shift of ca. 1 kcal/mol in favor of H-bonding. We suggest that the present values are very close to ultimate predictions of the strength of aromatic base stacking of DNA and RNA bases.</td>
</tr>
<tr id="bib_hobza2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hobza2002,
  author = {Pavel Hobza and Jiri Sponer},
  title = {Toward True DNA Base-Stacking Energies: MP2, CCSD(T), and Complete Basis Set Calculations},
  journal = {Journal of the American Chemical Society},
  year = {2002},
  volume = {124},
  pages = {11802-11808},
  doi = {http://dx.doi.org/10.1021/ja026759n}
}
</pre></td>
</tr>
<tr id="hofacker2006" class="entry">
	<td>Hofacker, I. L. &amp; F.Stadler, P.</td>
	<td>Memory efficient folding algorithms for circular RNA secondary structures. <p class="infolinks">[<a href="javascript:toggleInfo('hofacker2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hofacker2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_hofacker2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A small class of RNA molecules, in particular the tiny genomes of viroids, are circular. Yet most structure prediction algorithms handle only linear RNAs. The most straightforward approach is to compute circular structures from "internal" and "external" substructures separated by a base pair. This is incompatible, however, with the memory-saving approach of the Vienna RNA Package which builds a linear RNA structure from shorter (internal) structures only. RESULT: Here we describe how circular secondary structures can be obtained without additional memory requirements as a kind of "post-processing" of the linear structures. AVAILABILITY: The circular folding algorithm is implemented in the current version of the of RNAfold program of the Vienna RNA Package, which can be downloaded from http://www.tbi.univie.ac.at/RNA/</td>
</tr>
<tr id="bib_hofacker2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hofacker2006,
  author = {I. L. Hofacker and P. F.Stadler},
  title = {Memory efficient folding algorithms for circular RNA secondary structures.},
  journal = {Bioinformatics},
  year = {2006}
}
</pre></td>
</tr>
<tr id="hofacker1994" class="entry">
	<td>Hofacker, I. L., Fontana, W., Stadler, P. F., Bonhoeffer, L. S., Tacker, M. &amp; Schuster, P.</td>
	<td>Fast Folding and Comparison of RNA Secondary Structures <p class="infolinks">[<a href="javascript:toggleInfo('hofacker1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hofacker1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Monatshefte fur Chemie &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_hofacker1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Computer codes for computation and comparison of RNA secondary structures, the Vienna RNA package, are presented, that are based on dynamic programming algorithms and aim at predictions of structures with minimum free energies as well as at computations of the equilibrium partition functions and base pairing probabilities.<p>An efficient heuristic for the inverse folding problem of RNA is introduced. In addition we present compact and efficient programs for the comparison of RNA secondary structures based on tree editing and alignment.<p>All computer codes are written in ANSI C. They include implementations of modified algorithms on parallel computers with distributed memory. Performance analysis carried out on an Intel Hypercube shows that parallel computing becomes gradually more and more efficient the longer the sequences are.</td>
</tr>
<tr id="bib_hofacker1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hofacker1994,
  author = {Ivo L. Hofacker and Walter Fontana and Peter F. Stadler and L. Sebastian Bonhoeffer and Manfred Tacker and Peter Schuster},
  title = {Fast Folding and Comparison of RNA Secondary Structures},
  journal = {Monatshefte fur Chemie},
  year = {1994},
  volume = {125},
  pages = {167-188}
}
</pre></td>
</tr>
<tr id="holbrook2008" class="entry">
	<td>Holbrook, S. R.</td>
	<td>Structural Principles From large RNAs <p class="infolinks">[<a href="javascript:toggleInfo('holbrook2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('holbrook2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Annual Review in Biophysics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.biophys.36.040306.132755">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_holbrook2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Since the year 2000 a number of largeRNAthree-dimensional structures<p>have been determined by X-ray crystallography. Structures<p>composed of more than 100 nucleotide residues include the signal<p>recognition particle RNA, group I intron, the GlmS ribozyme,<p>RNAseP RNA, and ribosomal RNAs from Haloarcula morismortui,<p>Escherichia coli, Thermus thermophilus, and Deinococcus radiodurans.<p>These large RNAs are constructed from the same secondary and<p>tertiary structural motifs identified in smaller RNAs but appear to<p>have a larger organizational architecture. They are dominated by<p>long continuous interhelical base stacking, tend to segregate into<p>domains, and are planar in overall shape as opposed to their globular<p>protein counterparts. These findings have consequences in RNA<p>folding, intermolecular interaction, and packing, in addition to studies<p>of design and engineering and structure prediction.</td>
</tr>
<tr id="bib_holbrook2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{holbrook2008,
  author = {Stephen R. Holbrook},
  title = {Structural Principles From large RNAs},
  journal = {Annual Review in Biophysics},
  year = {2008},
  volume = {37},
  pages = {445-464},
  doi = {http://dx.doi.org/10.1146/annurev.biophys.36.040306.132755}
}
</pre></td>
</tr>
<tr id="holbrook2005" class="entry">
	<td>Holbrook, S. R.</td>
	<td>RNA Structure: The Long and the Short of it <p class="infolinks">[<a href="javascript:toggleInfo('holbrook2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('holbrook2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2005.04.005">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_holbrook2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The database of RNA structure has grown tremendously since the crystal structure analyses of ribosomal subunits in 2000ï¿½2001. During the past year, the trend toward determining the structure of large, complex biological RNAs has accelerated, with the analysis of three intact group I introns, A- and B-type ribonuclease P RNAs, a riboswitchï¿½substrate complex and other structures. The growing database of RNA structures, coupled with efforts directed at the standardization of nomenclature and classification of motifs, has resulted in the identification and characterization of numerous RNA secondary and tertiary structure motifs. Because a large proportion of RNA structure can now be shown to be composed of these recurring structural motifs, a view of RNA as a modular structure built from a combination of these building blocks and tertiary linkers is beginning to emerge. At the same time, however, more detailed analysis of water, metal, ligand and protein binding to RNA is revealing the effect of these moieties on folding and structure formation. The balance between the views of RNA structure either as strictly a construct of preformed building blocks linked in a limited number of ways or as a flexible polymer assuming a global fold influenced by its environment will be the focus of current and future RNA structural biology.</td>
</tr>
<tr id="bib_holbrook2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{holbrook2005,
  author = {Stephen R Holbrook},
  title = {RNA Structure: The Long and the Short of it},
  journal = {Current Opinion in Structural Biology},
  year = {2005},
  volume = {15},
  pages = {302-308},
  doi = {http://dx.doi.org/10.1016/j.sbi.2005.04.005}
}
</pre></td>
</tr>
<tr id="holbrook1997" class="entry">
	<td>Holbrook, S. R. &amp; Kim, S.</td>
	<td>RNA Crystallography <p class="infolinks">[<a href="javascript:toggleInfo('holbrook1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/(SICI)1097-0282(1997)44:1<3::AID-BIP2>3.0.CO;2-Z">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_holbrook1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{holbrook1997,
  author = {Stephen R. Holbrook and Sung-Hou Kim},
  title = {RNA Crystallography},
  journal = {Biopolymers},
  year = {1997},
  volume = {44},
  pages = {3-21},
  doi = {http://dx.doi.org/10.1002/(SICI)1097-0282(1997)44:1<3::AID-BIP2>3.0.CO;2-Z}
}
</pre></td>
</tr>
<tr id="hosoya1971" class="entry">
	<td>Hosoya, H.</td>
	<td>Topological Index: A Newly Proposed Quantity Characterizing theTopological Nature of Structural Isomers of Saturated Hydrocarbons <p class="infolinks">[<a href="javascript:toggleInfo('hosoya1971','bibtex')">BibTeX</a>]</p></td>
	<td>1971</td>
	<td>Bulletin of the Chemical Society Japan &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_hosoya1971" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hosoya1971,
  author = {H. Hosoya},
  title = {Topological Index: A Newly Proposed Quantity Characterizing theTopological Nature of Structural Isomers of Saturated Hydrocarbons},
  journal = {Bulletin of the Chemical Society Japan},
  year = {1971},
  volume = {44},
  pages = {2332-2339}
}
</pre></td>
</tr>
<tr id="huang2005" class="entry">
	<td>Huang, H., Nagaswamy, U. &amp; Fox, G. E.</td>
	<td>The Application of Cluster Analysis in the Intercomparison of Loop Structures in RNA <p class="infolinks">[<a href="javascript:toggleInfo('huang2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('huang2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.7238705">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_huang2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have developed a computational approach for the comparison and classification of RNA loop structures. Hairpin or interior loops identified in atomic resolution RNA structures were intercompared by conformational matching. The root-mean-square deviation (RMSD) values between all pairs of RNA fragments of interest, even if from different molecules, are calculated. Subsequently, cluster analysis is performed on the resulting matrix of RMSD distances using the unweighted pair group method with arithmetic mean (UPGMA). The cluster analysis objectively reveals groups of folds that resemble one another. To demonstrate the utility of the approach, a comprehensive analysis of all the terminal hairpin tetraloops that have been observed in 15 RNA structures that have been determined by X-ray crystallography was undertaken. The method found major clusters corresponding to the well-known GNRA and UNCG types. In addition, two tetraloops with the unusual primary sequence UMAC (M is A or C) were successfully assigned to the GNRA cluster. Larger loop structures were also examined and the clustering results confirmed the occurrence of variations of the GNRA and UNCG tetraloops in these loops and provided a systematic means for locating them. Nineteen examples of larger loops that closely resemble either the GNRA or UNCG tetraloop were found in the large ribosomal RNAs. When the clustering approach was extended to include all structures in the SCOR database, novel relationships were detected including one between the ANYA motif and a less common folding of the GAAA tetraloop sequence.</td>
</tr>
<tr id="bib_huang2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{huang2005,
  author = {Hung-Chung Huang and Uma Nagaswamy and George E. Fox},
  title = {The Application of Cluster Analysis in the Intercomparison of Loop Structures in RNA},
  journal = {RNA},
  year = {2005},
  volume = {11},
  pages = {412-423},
  doi = {http://dx.doi.org/10.1261/rna.7238705}
}
</pre></td>
</tr>
<tr id="hyeon2005" class="entry">
	<td>Hyeon, C. &amp; Thirumalai, D.</td>
	<td>Mechanical unfolding of RNA hairpins <p class="infolinks">[<a href="javascript:toggleInfo('hyeon2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Proceedings of the National Academy of Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0408314102">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_hyeon2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hyeon2005,
  author = {Changbong Hyeon and D. Thirumalai},
  title = {Mechanical unfolding of RNA hairpins},
  journal = {Proceedings of the National Academy of Science},
  year = {2005},
  volume = {102},
  number = {19},
  pages = {6789-6794},
  doi = {http://dx.doi.org/10.1073/pnas.0408314102}
}
</pre></td>
</tr>
<tr id="ihaka1996" class="entry">
	<td>Ihaka, R. &amp; Gentleman, R.</td>
	<td>R: a language for data analysis and graphics <p class="infolinks">[<a href="javascript:toggleInfo('ihaka1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Journal of Computational and Graphical Statistics &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_ihaka1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ihaka1996,
  author = {R. Ihaka and R. Gentleman},
  title = {R: a language for data analysis and graphics},
  journal = {Journal of Computational and Graphical Statistics},
  year = {1996},
  volume = {5},
  pages = {299-314}
}
</pre></td>
</tr>
<tr id="jain1999" class="entry">
	<td>Jain, A. K., Murthy, M. N. &amp; Flynn, P. J.</td>
	<td>Data Clustering: A Review <p class="infolinks">[<a href="javascript:toggleInfo('jain1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('jain1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>ACM Computing Surveys &nbsp;</td>
	<td>article</td>
	<td><a href="doi.acm.org/10.1145/331499.331504">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_jain1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Clustering is the unsupervised classification of patterns (observations, data items, or feature vectors) into groups (clusters). The clustering problem has been addressed in many contexts and by researchers in many disciplines; this reflects its broad appeal and usefulness as one of the steps in exploratory data analysis. However, clustering is a difficult problem combinatorially, and differences in assumptions and contexts in different communities has made the transfer of useful generic concepts and methodologies slow to occur. This paper presents an overview of pattern clustering methods from a statistical pattern recognition perspective, with a goal of providing useful advice and references to fundamental concepts accessible to the broad community of clustering practitioners. We present a taxonomy of clustering techniques, and identify cross-cutting themes and recent advances. We also describe some important applications of clustering algorithms such as image segmentation, object recognition, and information retrieval.</td>
</tr>
<tr id="bib_jain1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{jain1999,
  author = {A. K. Jain and M. N. Murthy and P. J. Flynn},
  title = {Data Clustering: A Review},
  journal = {ACM Computing Surveys},
  year = {1999},
  volume = {31},
  pages = {265},
  doi = {doi.acm.org/10.1145/331499.331504}
}
</pre></td>
</tr>
<tr id="jinek2009" class="entry">
	<td>Jinek, M. &amp; Doudna, J. A.</td>
	<td>A three-dimensional view of the molecular machinery of RNA interference. <p class="infolinks">[<a href="javascript:toggleInfo('jinek2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('jinek2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature07755">DOI</a><a href="http://dx.doi.org/10.1038/nature07755">URL</a> &nbsp;</td>
</tr>
<tr id="abs_jinek2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In eukaryotes, small non-coding RNAs regulate gene expression, helping to control cellular metabolism, growth and differentiation, to maintain genome integrity, and to combat viruses and mobile genetic elements. These pathways involve two specialized ribonucleases that control the production and function of small regulatory RNAs. The enzyme Dicer cleaves double-stranded RNA precursors, generating short interfering RNAs and microRNAs in the cytoplasm. These small RNAs are transferred to Argonaute proteins, which guide the sequence-specific silencing of messenger RNAs that contain complementary sequences by either enzymatically cleaving the mRNA or repressing its translation. The molecular structures of Dicer and the Argonaute proteins, free and bound to small RNAs, have offered exciting insights into the molecular mechanisms that are central to RNA silencing pathways.</td>
</tr>
<tr id="bib_jinek2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{jinek2009,
  author = {Martin Jinek and Jennifer A Doudna},
  title = {A three-dimensional view of the molecular machinery of RNA interference.},
  journal = {Nature},
  year = {2009},
  volume = {457},
  number = {7228},
  pages = {405--412},
  url = {http://dx.doi.org/10.1038/nature07755},
  doi = {http://dx.doi.org/10.1038/nature07755}
}
</pre></td>
</tr>
<tr id="katritch1997" class="entry">
	<td>Katritch, V., Olson, W. K., Pieranski, P., Dubochet, J. &amp; Stasiak, A.</td>
	<td>Properties of Ideal Composite Knots <p class="infolinks">[<a href="javascript:toggleInfo('katritch1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('katritch1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/40582">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_katritch1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The shortest tube of constant diameter that can form a given knot represents the 'ideal' form of the knot,. Ideal knots provide an irreducible representation of the knot, and they have some intriguing mathematical and physical features, including a direct correspondence with the time-averaged shapes of knotted DNA molecules in solution,. Here we describe the properties of ideal forms of composite knots ï¿½ knots obtained by the sequential tying of two or more independent knots (called factor knots) on the same string. We find that the writhe (related to the handedness of crossing points) of composite knots is the sum of that of the ideal forms of the factor knots. By comparing ideal composite knots with simulated configurations of knotted, thermally fluctuating DNA, we conclude that the additivity of writhe applies also to randomly distorted configurations of composite knots and their corresponding factor knots. We show that composite knots with several factor knots may possess distinct structural isomers that can be interconverted only by loosening the knot.</td>
</tr>
<tr id="bib_katritch1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{katritch1997,
  author = {Vsevolod Katritch and Wilma K. Olson and Piotr Pieranski and Jacques Dubochet and Andrzej Stasiak},
  title = {Properties of Ideal Composite Knots},
  journal = {Nature},
  year = {1997},
  volume = {388},
  pages = {148-151},
  doi = {http://dx.doi.org/10.1038/40582}
}
</pre></td>
</tr>
<tr id="kebbekus1995" class="entry">
	<td>Kebbekus, P., Draper, D. E. &amp; Hagerman, P.</td>
	<td>Persistence length of RNA. <p class="infolinks">[<a href="javascript:toggleInfo('kebbekus1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kebbekus1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Biochemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/bi00013a026">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_kebbekus1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A set of 10 double-stranded RNAs ranging in length from 92 to 317 base pairs has been synthesized, and their rotational diffusion times were measured by transient electric birefringence. A hydrodynamic analysis of the birefringence decay data yields a helix persistence length of 720 +/- 70 A, based on a helix with an effective hydrodynamic diameter of 26 A and a rise per base pair of 2.7 A in the presence of Mg2+ ions. Thus, duplex RNA is somewhat stiffer than DNA, for which the persistence length is 450-500 A. The measurements also suggest that (i) current hydrodynamic theory for the rotational decay times is applicable to RNA molecules that are longer than 100 base pairs and (ii) weak Mg(2+)-RNA association decreases the rise per base pair slightly, by no more than 0.1 A.</td>
</tr>
<tr id="bib_kebbekus1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kebbekus1995,
  author = {P. Kebbekus and D. E. Draper and P. Hagerman},
  title = {Persistence length of RNA.},
  journal = {Biochemistry},
  year = {1995},
  volume = {34},
  number = {13},
  pages = {4354-4357},
  doi = {http://dx.doi.org/10.1021/bi00013a026}
}
</pre></td>
</tr>
<tr id="keller2004" class="entry">
	<td>Keller, S. L.</td>
	<td>Sequential folding of a rigid wire into three-dimensional structures <p class="infolinks">[<a href="javascript:toggleInfo('keller2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('keller2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>American Journal of Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1119/1.1624117">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_keller2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Nitinol wire is used to encode a three-dimensional structure in a linear chain. The folding of the structure progresses sequentially along the rigid wire from one end using only short-range forces. Three different shapes are constructed, an overhand pseudoknot, a protein structure with alpha-helices, and a protein structure without alpha-helices. The overhand pseudoknot was designed to be noncommutative so that forward folding along the chain results in the correct structure whereas reverse folding does not. The manipulation of nitinol wire structures leads to simple empirical rules for folding for noncommutative pseudoknots. Suggestions for additional noncommutative pseudoknot structures that can be designed both from wire and from DNA or RNA are given. The folding also can be accomplished using ordinary stiff wire. The folding of the wire provides an intuitive way to visualize sequential folding freed from a lattice.</td>
</tr>
<tr id="bib_keller2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{keller2004,
  author = {Sarah L. Keller},
  title = {Sequential folding of a rigid wire into three-dimensional structures},
  journal = {American Journal of Physics},
  year = {2004},
  volume = {72},
  pages = {599-604},
  doi = {http://dx.doi.org/10.1119/1.1624117}
}
</pre></td>
</tr>
<tr id="kierzek2008" class="entry">
	<td>Kierzek, E., Kierzek, R., Moss, W. N., Christensen, S. M., Eickbush, T. H. &amp; Turner, D. H.</td>
	<td>Isoenergetic penta- and hexanucleotide microarray probing and chemical mapping provide a secondary structure model for an RNA element orchestrating R2 retrotransposon protein function <p class="infolinks">[<a href="javascript:toggleInfo('kierzek2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkm1085">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_kierzek2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kierzek2008,
  author = {Elzbieta Kierzek and Ryszard Kierzek and Walter N. Moss and Shawn M. Christensen and Thomas H. Eickbush and Douglas H. Turner},
  title = {Isoenergetic penta- and hexanucleotide microarray probing and chemical mapping provide a secondary structure model for an RNA element orchestrating R2 retrotransposon protein function},
  journal = {Nucleic Acids Research},
  year = {2008},
  volume = {36},
  pages = {1770-1782},
  doi = {http://dx.doi.org/10.1093/nar/gkm1085}
}
</pre></td>
</tr>
<tr id="kierzek2006" class="entry">
	<td>Kierzek, E., Kierzek, R., Turner, D. H. &amp; Catrina, I. E.</td>
	<td>Facilitating RNA Structure Prediction with Microarrays <p class="infolinks">[<a href="javascript:toggleInfo('kierzek2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Biochemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/bi051409+">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_kierzek2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kierzek2006,
  author = {Elzbieta Kierzek and Ryszard Kierzek and Douglas H. Turner and Irina E. Catrina},
  title = {Facilitating RNA Structure Prediction with Microarrays},
  journal = {Biochemistry},
  year = {2006},
  volume = {45},
  pages = {581-593},
  doi = {http://dx.doi.org/10.1021/bi051409+}
}
</pre></td>
</tr>
<tr id="kiliszek2009" class="entry">
	<td>Kiliszek, A., Kierzek, R., Krzyzosiak, W. J. &amp; Rypniewski, W.</td>
	<td>Structural insights into CUG repeats containing the 'stretched U-U wobble': implications for myotonic dystrophy. <p class="infolinks">[<a href="javascript:toggleInfo('kiliszek2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kiliszek2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Nucleic Acids Res &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkp350">DOI</a><a href="http://dx.doi.org/10.1093/nar/gkp350">URL</a> &nbsp;</td>
</tr>
<tr id="abs_kiliszek2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Tracks containing CUG repeats are abundant in human gene transcripts. Their biological role includes modulation of pre-mRNA splicing, mRNA transport and regulation of translation. Expanded forms of CUG runs are associated with pathogenesis of several neurodegenerative diseases, including myotonic dystrophy type 1. We have analysed two crystal structures of RNA duplexes containing the CUG repeats: G(CUG)(2)C and (CUG)(6). The first of the structures, analysed at 1.23 A resolution, is of an oligomer designed by us. The second model was obtained after 'detwinning' the 1.58 A X-ray data previously deposited in the PDB. The RNA duplexes are in the A-form in which all the C-G pairs form Watson-Crick interactions while all the uridine pairs can be described as U*U cis wobble having only one hydrogen bond between the bases. The residue, which accepts the H-bond, is inclined towards the minor groove. This previously unreported base pairing can be described as 'stretched U-U wobble'. The regular hydrogen-bonding pattern of interactions with the solvent, the electrostatic charge distribution and surface features indicate the ligand binding potential of the CUG tracks.</td>
</tr>
<tr id="bib_kiliszek2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kiliszek2009,
  author = {Agnieszka Kiliszek and Ryszard Kierzek and Wlodzimierz J Krzyzosiak and Wojciech Rypniewski},
  title = {Structural insights into CUG repeats containing the 'stretched U-U wobble': implications for myotonic dystrophy.},
  journal = {Nucleic Acids Res},
  year = {2009},
  volume = {37},
  number = {12},
  pages = {4149--4156},
  url = {http://dx.doi.org/10.1093/nar/gkp350},
  doi = {http://dx.doi.org/10.1093/nar/gkp350}
}
</pre></td>
</tr>
<tr id="kim1974" class="entry">
	<td>Kim, S. H.</td>
	<td>Three-Dimensional Tertiary Structure of Yeast Phenylalanine Transfer RNA <p class="infolinks">[<a href="javascript:toggleInfo('kim1974','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kim1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://links.jstor.org/sici?sici=0036-8075%2819740802%293%3A185%3A4149%3C435%3ATTSOYP%3E2.0.CO%3B2-N">URL</a> &nbsp;</td>
</tr>
<tr id="abs_kim1974" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The 3-angstrom electron density map of crystalline yeast phenylalanine transfer RNA has provided us with a complete three-dimensional model which defines the positions of all of the nucleotide residues in the molecule. The overall features of the molecule are virtually the same as those seen at a resolution of 4 angstroms except that many additional details of tertiary structure are now visualized. Ten types of hydrogen bonding are identified which define the specificity of tertiary interactions. The molecule is also stabilized by considerable stacking of the planar purines and pyrimidines. This tertiary structure explains, in a simple and direct fashion, chemical modification studies of transfer RNA. Since most of the tertiary interactions involve nucleotides which are common to all transfer RNA's, it is likely that this three-dimensional structure provides a basic pattern of folding which may help to clarify the three-dimensional structure of all transfer RNA's.</td>
</tr>
<tr id="bib_kim1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kim1974,
  author = {S. H. Kim},
  title = {Three-Dimensional Tertiary Structure of Yeast Phenylalanine Transfer RNA},
  journal = {Science},
  year = {1974},
  volume = {185},
  pages = {435},
  url = {http://links.jstor.org/sici?sici=0036-8075%2819740802%293%3A185%3A4149%3C435%3ATTSOYP%3E2.0.CO%3B2-N}
}
</pre></td>
</tr>
<tr id="klein2004a" class="entry">
	<td>Klein, D. J., Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The contribution of metal ions to the structural stability of the large ribosomal subunit. <p class="infolinks">[<a href="javascript:toggleInfo('klein2004a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('klein2004a','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.7390804">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_klein2004a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Both monovalent cations and magnesium ions are well known to be essential for the folding and stability of large RNA molecules that form complex and compact structures. In the atomic structure of the large ribosomal subunit from Haloarcula marismortui, we have identified 116 magnesium ions and 88 monovalent cations bound principally to rRNA. Although the rRNA structures to which these metal ions bind are highly idiosyncratic, a few common principles have emerged from the identities of the specific functional groups that coordinate them. The nonbridging oxygen of a phosphate group is the most common inner shell ligand of Mg++, and Mg++ ions having one or two such inner shell ligands are very common. Nonbridging phosphate oxygens and the heteroatoms of nucleotide bases are common outer shell ligands for Mg++ ions. Monovalent cations usually interact with nucleotide bases and protein groups, although some interactions with nonbridging phosphate oxygens are found. The most common monovalent cation binding site is the major groove side of G-U wobble pairs. Both divalent and monovalent cations stabilize the tertiary structure of 23S rRNA by mediating interactions between its structural domains. Bound metal ions are particularly abundant in the region surrounding the peptidyl transferase center, where stabilizing cationic tails of ribosomal proteins are notably absent. This may point to the importance of metal ions for the stabilization of specific RNA structures in the evolutionary period prior to the appearance of proteins, and hence many of these metal ion binding sites may be conserved across all phylogenetic kingdoms.</td>
</tr>
<tr id="bib_klein2004a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{klein2004a,
  author = {Daniel J Klein and Peter B Moore and Thomas A Steitz},
  title = {The contribution of metal ions to the structural stability of the large ribosomal subunit.},
  journal = {RNA},
  year = {2004},
  volume = {10},
  number = {9},
  pages = {1366--1379},
  doi = {http://dx.doi.org/10.1261/rna.7390804}
}
</pre></td>
</tr>
<tr id="klein2004" class="entry">
	<td>Klein, D. J., Palacios, J. L., Randic, M. &amp; Trinajstic, N.</td>
	<td>Random Walks and Chemical Graph Theory <p class="infolinks">[<a href="javascript:toggleInfo('klein2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('klein2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Chemical Information and Computer Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci040100e">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_klein2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Simple random walks probabilistically grown step by step on a graph are distinguished from walk enumerations and associated equipoise random walks. Substructure characteristics and graph invariants correspondingly defined for the two types of random walks are then also distinct, though there often are analogous relations. It is noted that the connectivity index as well as some resistance-distance-related invariants make natural appearances among the invariants defined from the simple random walks.</td>
</tr>
<tr id="bib_klein2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{klein2004,
  author = {Douglas J. Klein and Josï¿½ Luis Palacios and Milan Randic and Nenad Trinajstic},
  title = {Random Walks and Chemical Graph Theory},
  journal = {Journal of Chemical Information and Computer Science},
  year = {2004},
  volume = {44},
  pages = {1521-1525},
  doi = {http://dx.doi.org/10.1021/ci040100e}
}
</pre></td>
</tr>
<tr id="klein2001" class="entry">
	<td>Klein, D. J., Schmeing, T. M., Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The kink-turn: a new RNA secondary structure motif. <p class="infolinks">[<a href="javascript:toggleInfo('klein2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('klein2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>EMBO J &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/emboj/20.15.4214">DOI</a><a href="http://dx.doi.org/10.1093/emboj/20.15.4214">URL</a> &nbsp;</td>
</tr>
<tr id="abs_klein2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Analysis of the Haloarcula marismortui large ribosomal subunit has revealed a common RNA structure that we call the kink-turn, or K-turn. The six K-turns in H.marismortui 23S rRNA superimpose with an r.m.s.d. of 1.7 A. There are two K-turns in the structure of Thermus thermophilus 16S rRNA, and the structures of U4 snRNA and L30e mRNA fragments form K-turns. The structure has a kink in the phosphodiester backbone that causes a sharp turn in the RNA helix. Its asymmetric internal loop is flanked by C-G base pairs on one side and sheared G-A base pairs on the other, with an A-minor interaction between these two helical stems. A derived consensus secondary structure for the K-turn includes 10 consensus nucleotides out of 15, and predicts its presence in the 5'-UTR of L10 mRNA, helix 78 in Escherichia coli 23S rRNA and human RNase MRP. Five K-turns in 23S rRNA interact with nine proteins. While the observed K-turns interact with proteins of unrelated structures in different ways, they interact with L7Ae and two homologous proteins in the same way.</td>
</tr>
<tr id="bib_klein2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{klein2001,
  author = {D. J. Klein and T. M. Schmeing and P. B. Moore and T. A. Steitz},
  title = {The kink-turn: a new RNA secondary structure motif.},
  journal = {EMBO J},
  year = {2001},
  volume = {20},
  number = {15},
  pages = {4214--4221},
  url = {http://dx.doi.org/10.1093/emboj/20.15.4214},
  doi = {http://dx.doi.org/10.1093/emboj/20.15.4214}
}
</pre></td>
</tr>
<tr id="klosterman2004" class="entry">
	<td>Klosterman, P. S., Hendrix, D. K., Tamura, M., Holbrook, S. R. &amp; Brenner., S. E.</td>
	<td>Three-Dimensional Motifs from the SCOR, Structural Classification of RNA Database: Extruded Strands, Base Triples, Tetraloops and U-turns <p class="infolinks">[<a href="javascript:toggleInfo('klosterman2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('klosterman2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkh537">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_klosterman2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Release 2.0.1 of the Structural Classiï¿½cation of RNA (SCOR) database, http://scor.lbl.gov, contains a classiï¿½cation of the internal and hairpin loops in a comprehensive collection of 497 NMR and X-ray RNA structures. This report discusses ï¿½ndings of the classiï¿½cation that have not been reported previously. The SCOR database contains multiple examples of a newly described RNA motif, the extruded helical single strand. Internal loop base triples are classiï¿½ed in SCOR according to their three-dimensional context. These internal loop triples contain several examples of a frequently found motif, the minor groove AGC triple. SCOR also presents the predominant and alternate conformations of hairpin loops, as shown in the most well represented tetraloops, with consensus sequences GNRA, UNCG and ANYA. The ubiquity of the GNRA hairpin turn motif is illustrated by its presence in complex internal loops.</td>
</tr>
<tr id="bib_klosterman2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{klosterman2004,
  author = {Peter S. Klosterman and Donna K. Hendrix and Makio Tamura and Stephen R. Holbrook and Steven E. Brenner.},
  title = {Three-Dimensional Motifs from the SCOR, Structural Classification of RNA Database: Extruded Strands, Base Triples, Tetraloops and U-turns},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  number = {8},
  pages = {2342-2352},
  doi = {http://dx.doi.org/10.1093/nar/gkh537}
}
</pre></td>
</tr>
<tr id="klosterman2002" class="entry">
	<td>Klosterman, P. S., Tamura, M., Holbrook, S. R. &amp; Brenner, S. E.</td>
	<td>SCOR: a Structural Classification of RNA Database <p class="infolinks">[<a href="javascript:toggleInfo('klosterman2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('klosterman2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://scor.lbl.gov/">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_klosterman2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The Structural Classification of RNA (SCOR) database provides a survey of the three-dimensional motifs contained in 259 NMR and X-ray RNA structures. In one classification, the structures are grouped according to function. The RNA motifs, including internal and external loops, are also organized in a hierarchical classification. The 259 database entries contain 223 internal and 203 external loops; 52 entries consist of fully complementary duplexes. A classification of the well-characterized tertiary interactions found in the larger RNA structures is also included along with examples. The SCOR database is accessible at http://scor.lbl.gov.</td>
</tr>
<tr id="bib_klosterman2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{klosterman2002,
  author = {Peter S. Klosterman and Makio Tamura and Stephen R. Holbrook and Steven E. Brenner},
  title = {SCOR: a Structural Classification of RNA Database},
  journal = {Nucleic Acids Research},
  year = {2002},
  volume = {30},
  pages = {392-394},
  doi = {http://scor.lbl.gov/}
}
</pre></td>
</tr>
<tr id="koculi2004" class="entry">
	<td>Koculi, E., Lee, N., Thirumalai, D. &amp; Woodson, S. A.</td>
	<td>Folding of the Tetrahymena Ribozyme by Polyamines: Importance of Counterion Valence and Size <p class="infolinks">[<a href="javascript:toggleInfo('koculi2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('koculi2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2004.06.008">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_koculi2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Polyamines are abundant metabolites that directly influence gene expression. Although the role of polyamines in DNA condensation is well known, their role in RNA folding is less understood. Non-denaturing gel electrophoresis was used to monitor the equilibrium folding transitions of the Tetrahymena ribozyme in the presence of polyamines. All of the polyamines tested induce near-native structures that readily convert to the native conformation in Mg2+. The stability of the folded structure increases with the charge of the polyamine and decreases with the size of the polyamine. When the counterion excluded volume becomes large, the transition to the native state does not go to completion even under favorable folding conditions. Brownian dynamics simulations of a model polyelectrolyte suggest that the kinetics of counterion-mediated collapse and the dimensions of the collapsed RNA chains depend on the structure of the counterion. The results are consistent with delocalized condensation of polyamines around the RNA. However, the effective charge of the counterions is lowered by their excluded volume. The stability of the folded RNA is enhanced when the spacing between amino groups matches the distance between adjacent phosphate groups. These results show how changes in intracellular polyamine concentrations could alter RNA folding pathways.</td>
</tr>
<tr id="bib_koculi2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{koculi2004,
  author = {Eda Koculi and Nam-Kyung Lee and D. Thirumalai and Sarah A. Woodson},
  title = {Folding of the Tetrahymena Ribozyme by Polyamines: Importance of Counterion Valence and Size},
  journal = {Journal of Molecular Biology},
  year = {2004},
  volume = {341},
  pages = {27-36},
  doi = {http://dx.doi.org/10.1016/j.jmb.2004.06.008}
}
</pre></td>
</tr>
<tr id="kopeikin2005" class="entry">
	<td>Kopeikin, Z. &amp; Chen, S.</td>
	<td>Statistical thermodynamics for chain molecules with simple RNA tertiary contacts <p class="infolinks">[<a href="javascript:toggleInfo('kopeikin2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kopeikin2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>The Journal of Chemical Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1063/1.1857831">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_kopeikin2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A statistical thermodynamic model is developed for chain molecules with simple RNA tertiary contacts. The model, which accounts for the excluded volume effect and the nonadditivity in the free energy, enables reliable predictions for the conformational entropy and partition function for simple tertiary folds. Illustrative applications are made to conformational transitions involving simple tertiary contacts. The model can predict the interplay between the secondary and the tertiary interactions in the conformational changes. Though the present form of the theory is tested and validated in a two-dimensional lattice model, the methodology, which is developed based on a general graphical representation for chain conformations, is applicable to any off-lattice chain representations. Moreover, the analytical formulation of the method makes possible the systematic development of the theory for more complex tertiary structures</td>
</tr>
<tr id="bib_kopeikin2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kopeikin2005,
  author = {Zoia Kopeikin and Shi-Jie Chen },
  title = {Statistical thermodynamics for chain molecules with simple RNA tertiary contacts},
  journal = {The Journal of Chemical Physics},
  year = {2005},
  volume = {122},
  pages = {1-13},
  doi = {http://dx.doi.org/10.1063/1.1857831}
}
</pre></td>
</tr>
<tr id="kosikov1999" class="entry">
	<td>Kosikov, K. M., Gorin, A. A., Zhurkin, V. B. &amp; Olson, W. K.</td>
	<td>DNA stretching and compression: large-scale simulations of double helical structures <p class="infolinks">[<a href="javascript:toggleInfo('kosikov1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kosikov1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1999.2798">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_kosikov1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Computer-simulated elongation and compression of A- and B-DNA structures beyond the range of thermal fluctuations provide new insights into high energy "activated" forms of DNA implicated in biochemical processes, such as recombination and transcription. All-atom potential energy studies of regular poly(dG)ï¿½poly(dC) and poly(dA)ï¿½poly(dT) double helices, stretched from compressed states of 2.0 ï¿½ per base-pair step to highly extended forms of 7.0 ï¿½ per residue, uncover four different hyperfamilies of right-handed structures that differ in mutual base-pair orientation and sugar-phosphate backbone conformation. The optimized structures embrace all currently known right-handed forms of double-helical DNA identified in single crystals as well as non-canonical forms, such as the original "Watson-Crick" duplex with trans conformations about the P?O5? and C5??C4? backbone bonds. The lowest energy minima correspond to canonical A and B-form duplexes. The calculations further reveal a number of unusual helical conformations that are energetically disfavored under equilibrium conditions but become favored when DNA is highly stretched or compressed. The variation of potential energy versus stretching provides a detailed picture of dramatic conformational changes that accompany the transitions between various families of double-helical forms. In particular, the interchanges between extended canonical and non-canonical states are reminiscent of the cooperative transitions identified by direct stretching experiments. The large-scale, concerted changes in base-pair inclination, brought about by changes in backbone and glycosyl torsion angles, could easily give rise to the observed sharp increase in force required to stretch single DNA molecules more than 1.6?1.65 times their canonical extension. Our extended duplexes also help to tie together a number of previously known structural features of the RecA-DNA complex and offer a self-consistent stereochemical model for the single-stranded/duplex DNA recognition brought in register by recombination proteins. The compression of model duplexes, by contrast, yields non-canonical structures resembling the deformed steps in crystal complexes of DNA with the TATA-box binding protein (TBP). The crystalline TBP-bound DNA steps follow the calculated compression-elongation pattern of an unusual "vertical" duplex with base planes highly inclined with respect to the helical axis, exposed into the minor groove, and accordingly accessible for recognition. Significantly, the double helix can be stretched by a factor of two and compressed roughly in half before its computed internal energy rises sharply. The energy profiles show that DNA extension-compression is related not only to the variation of base-pair Rise but also to concerted changes of Twist, Roll, and Slide. We suggest that the high energy "activated" forms calculated here are critical for DNA processing, e.g. nucleo-protein recognition, DNA/RNA synthesis, and strand exchange.</td>
</tr>
<tr id="bib_kosikov1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kosikov1999,
  author = {Konstantin M. Kosikov and Andrey A. Gorin and Victor B. Zhurkin and Wilma K. Olson},
  title = {DNA stretching and compression: large-scale simulations of double helical structures},
  journal = {Journal of Molecular Biology},
  year = {1999},
  volume = {289},
  pages = {1301-1326},
  doi = {http://dx.doi.org/10.1006/jmbi.1999.2798}
}
</pre></td>
</tr>
<tr id="kraut1988" class="entry">
	<td>Kraut, J.</td>
	<td>How do Enzymes Work <p class="infolinks">[<a href="javascript:toggleInfo('kraut1988','bibtex')">BibTeX</a>]</p></td>
	<td>1988</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.jstor.org/cgi-bin/jstor/printpage/00368075/ap003609/00a00190/0.pdf?backcontext=page&amp;dowhat=Acrobat&amp;config=jstor&amp;userID=80061b12@rutgers.edu/01cc99334115c710d5b334e1c&amp;0.pdf">URL</a> &nbsp;</td>
</tr>
<tr id="bib_kraut1988" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kraut1988,
  author = {Joseph Kraut},
  title = {How do Enzymes Work},
  journal = {Science},
  year = {1988},
  volume = {242},
  pages = {533-540},
  url = {http://www.jstor.org/cgi-bin/jstor/printpage/00368075/ap003609/00a00190/0.pdf?backcontext=page&amp;dowhat=Acrobat&amp;config=jstor&amp;userID=80061b12@rutgers.edu/01cc99334115c710d5b334e1c&amp;0.pdf}
}
</pre></td>
</tr>
<tr id="krilov2004" class="entry">
	<td>Krilov, G. &amp; Randic, M.</td>
	<td>Quantitative Characterization of Protein Structure: Application to a Novel $ Fold <p class="infolinks">[<a href="javascript:toggleInfo('krilov2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>New Journal of Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1039/b405153j">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_krilov2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{krilov2004,
  author = {Goran Krilov and Milan Randic},
  title = {Quantitative Characterization of Protein Structure: Application to a Novel $ Fold},
  journal = {New Journal of Chemistry},
  year = {2004},
  volume = {28},
  pages = {1608-1614},
  doi = {http://dx.doi.org/10.1039/b405153j}
}
</pre></td>
</tr>
<tr id="kruger1982" class="entry">
	<td>Kruger, K., Grabowski, P. J., Zaug, A. J., Sands, J., Gottschling, D. E. &amp; Cech, T. R.</td>
	<td>Self-splicing RNA: Autoexcision and autocyclization of the ribosomal RNA intervening sequence of tetrahymena <p class="infolinks">[<a href="javascript:toggleInfo('kruger1982','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kruger1982','bibtex')">BibTeX</a>]</p></td>
	<td>1982</td>
	<td>Cell &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_kruger1982" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In the macronuclear rRNA genes of Tetrahymena thermophila, a 413 bp intervening sequence (IVS) interrupts the 26S rRNA-coding region. A restriction fragment of the rDNA containing the IVS and portions of the adjacent rRNA sequences (exons) was inserted downstream from the lac UV5 promoter in a recombinant plasmid. Transcription of this template by purified Escherichia coli RNA polymerase in vitro produced a shortened version of the prerRNA, which was then deproteinized. When incubated with monovalent and divalent cations and a guanosine factor, this RNA underwent splicing. The reactions that were characterized included the precise excision of the IVS, attachment of guanosine to the 5' end of the IVS, covalent cyclization of the IVS and ligation of the exons. We conclude that splicing activity is intrinsic to the structure of the RNA, and that enzymes, small nuclear RNAs and folding of the pre-rRNA into an RNP are unnecessary for these reactions. We propose that the IVS portion of the RNA has several enzyme-like properties that enable it to break and reform phosphodiester bonds. The finding of autocatalytic rearrangements of RNA molecules has implications for the mechanism and the evolution of other reactions that involve RNA.</td>
</tr>
<tr id="bib_kruger1982" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kruger1982,
  author = {Kelly Kruger and Paula J. Grabowski and Arthur J. Zaug and Julie Sands and Daniel E. Gottschling and Thomas R. Cech},
  title = {Self-splicing RNA: Autoexcision and autocyclization of the ribosomal RNA intervening sequence of tetrahymena},
  journal = {Cell},
  year = {1982},
  volume = {31},
  pages = {147-157}
}
</pre></td>
</tr>
<tr id="pinaki.2004" class="entry">
	<td>Kundu, S. &amp; Gupta-Bhaya, P.</td>
	<td>How a repulsive charge distribution becomes attractive and stabilized by a polarizable protein dielectric <p class="infolinks">[<a href="javascript:toggleInfo('pinaki.2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('pinaki.2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>TheoChem &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.theochem.2003.10.006">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_pinaki.2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Electrostatic energies (U) arising from sources of non-bonded interactions and their components from fixed charges (Uf) and induced dipoles (Ub) are calculated for the protein cytochrome P450 and its different substrate bound complexes. Uf is further broken up into charge?charge (Uc?c), charge?dipole (Uc?d) and dipole?dipole (Ud?d) interactions. It is found that charge?charge interaction is always repulsive and is much smaller in magnitude than the attractive interaction between permanent dipoles. Charge?dipole interaction is smaller in magnitude than both these interactions and is attractive in some proteins and repulsive in others. The energy changes on charging a protein are calculated and it is found that energy is lowered as a result of charging. Even though the charge-charge interaction in these proteins are repulsive, other interactions cause a lowering of energy on charging. The mutually repelling distribution of free charges is stabilized by the polarizable protein dielectric and become attractive. An effective screening constant for the charge?charge interactions in the protein dielectric turns out to be negative. The high sensitivity of these quantities to small structural changes on ligand binding reestablish the importance of electrostatics on protein dynamics.</td>
</tr>
<tr id="bib_pinaki.2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{pinaki.2004,
  author = {Sibsankar Kundu and Pinaki Gupta-Bhaya},
  title = {How a repulsive charge distribution becomes attractive and stabilized by a polarizable protein dielectric},
  journal = {TheoChem},
  year = {2004},
  volume = {668},
  number = {1},
  pages = {65-73},
  doi = {http://dx.doi.org/10.1016/j.theochem.2003.10.006}
}
</pre></td>
</tr>
<tr id="laederach2007" class="entry">
	<td>Laederach, A., Chan, J. M., Schwartzman, A., Willgohs, E. &amp; Altman, R. B.</td>
	<td>Coplanar and coaxial orientations of RNA bases and helices <p class="infolinks">[<a href="javascript:toggleInfo('laederach2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.381407">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_laederach2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{laederach2007,
  author = {Alain Laederach and Joseph M. Chan and Armin Schwartzman and Eric Willgohs and Russ B. Altman},
  title = {Coplanar and coaxial orientations of RNA bases and helices},
  journal = {RNA},
  year = {2007},
  volume = {13},
  pages = {643-650},
  doi = {http://dx.doi.org/10.1261/rna.381407}
}
</pre></td>
</tr>
<tr id="laederach2006" class="entry">
	<td>Laederach, A., Shcherbakova, I., Liang, M. P., Brenowitz, M. &amp; Altman, R. B.</td>
	<td>Local Kinetic Measures of Macromolecular Structure Reveal Partitioning among Multiple Parallel Pathways from the Earliest Steps in the Folding of a Large RNA Molecule <p class="infolinks">[<a href="javascript:toggleInfo('laederach2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('laederach2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2006.02.075">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_laederach2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: At the heart of the RNA folding problem is the number, structures, and relationships among the intermediates that populate the folding pathways of most large RNA molecules. Unique insight into the structural dynamics of these intermediates can be gleaned from the time-dependent changes in local probes of macromolecular conformation (e.g. reports on individual nucleotide solvent accessibility offered by hydroxyl radical (radical dotOH) footprinting). Local measures distributed around a macromolecule individually illuminate the ensemble of separate changes that constitute a folding reaction. Folding pathway reconstruction from a multitude of these individual measures is daunting due to the combinatorial explosion of possible kinetic models as the number of independent local measures increases. Fortunately, clustering of time progress curves sufficiently reduces the dimensionality of the data so as to make reconstruction computationally tractable. The most likely folding topology and intermediates can then be identified by exhaustively enumerating all possible kinetic models on a super-computer grid. The folding pathways and measures of the relative flux through them were determined for Mg2+ and Na+-mediated folding of the Tetrahymena thermophila group I intron using this combined experimental and computational approach. The flux during Mg2+-mediated folding is divided among numerous parallel pathways. In contrast, the flux during the Na+-mediated reaction is predominantly restricted through three pathways, one of which is without detectable passage through intermediates. Under both conditions, the folding reaction is highly parallel with no single pathway accounting for more than 50% of the molecular flux. This suggests that RNA folding is non-sequential under a variety of different experimental conditions even at the earliest stages of folding. This study provides a template for the systematic analysis of the time-evolution of RNA structure from ensembles of local measures that will illuminate the chemical and physical characteristics of each step in the process. The applicability of this analysis approach to other macromolecules is discussed.</td>
</tr>
<tr id="bib_laederach2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{laederach2006,
  author = {Alain Laederach and Inna Shcherbakova and Mike P. Liang and Michael Brenowitz and Russ B. Altman},
  title = {Local Kinetic Measures of Macromolecular Structure Reveal Partitioning among Multiple Parallel Pathways from the Earliest Steps in the Folding of a Large RNA Molecule},
  journal = {Journal of Molecular Biology},
  year = {2006},
  volume = {358},
  pages = {1179-1190},
  doi = {http://dx.doi.org/10.1016/j.jmb.2006.02.075}
}
</pre></td>
</tr>
<tr id="laing2009" class="entry">
	<td>Laing, C., Jung, S., Iqbal, A. &amp; Schlick, T.</td>
	<td>Tertiary Motifs Revealed in Analyses of Higher-Order RNA Junctions <p class="infolinks">[<a href="javascript:toggleInfo('laing2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('laing2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2009.07.089">DOI</a><a href="http://dx.doi.org/10.1016/j.jmb.2009.07.089">URL</a> &nbsp;</td>
</tr>
<tr id="abs_laing2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA junctions are secondary-structure elements formed when three or more helices come together. They are present in diverse RNA molecules with various fundamental functions in the cell. To better understand the intricate architecture of three-dimensional (3D) RNAs, we analyze currently solved 3D RNA junctions in terms of base-pair interactions and 3D configurations. First, we study base-pair interaction diagrams for solved RNA junctions with 5 to 10 helices and discuss common features. Second, we compare these higher-order junctions to those containing 3 or 4 helices and identify global motif patterns such as coaxial stacking and parallel and perpendicular helical configurations. These analyses show that higher-order junctions organize their helical components in parallel and helical configurations similar to lower-order junctions. Their sub-junctions also resemble local helical configurations found in three- and four-way junctions and are stabilized by similar long-range interaction preferences such as A-minor interactions. Furthermore, loop regions within junctions are high in adenine but low in cytosine, and in agreement with previous studies, we suggest that coaxial stacking between helices likely forms when the common single-stranded loop is small in size; however, other factors such as stacking interactions involving noncanonical base pairs and proteins can greatly determine or disrupt coaxial stacking. Finally, we introduce the ribo-base interactions: when combined with the along-groove packing motif, these ribo-base interactions form novel motifs involved in perpendicular helix-helix interactions. Overall, these analyses suggest recurrent tertiary motifs that stabilize junction architecture, pack helices, and help form helical configurations that occur as sub-elements of larger junction networks. The frequent occurrence of similar helical motifs suggest nature's finite and perhaps limited repertoire of RNA helical conformation preferences. More generally, studies of RNA junctions and tertiary building blocks can ultimately help in the difficult task of RNA 3D structure prediction.</td>
</tr>
<tr id="bib_laing2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{laing2009,
  author = {Christian Laing and Segun Jung and Abdul Iqbal and Tamar Schlick},
  title = {Tertiary Motifs Revealed in Analyses of Higher-Order RNA Junctions},
  journal = {Journal of Molecular Biology},
  year = {2009},
  volume = {393},
  number = {1},
  pages = {67-82},
  url = {http://dx.doi.org/10.1016/j.jmb.2009.07.089},
  doi = {http://dx.doi.org/10.1016/j.jmb.2009.07.089}
}
</pre></td>
</tr>
<tr id="laing2009a" class="entry">
	<td>Laing, C. &amp; Schlick, T.</td>
	<td>Analysis of Four-way Junctions in RNA Structures <p class="infolinks">[<a href="javascript:toggleInfo('laing2009a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('laing2009a','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2009.04.084">DOI</a><a href="http://dx.doi.org/10.1016/j.jmb.2009.04.084">URL</a> &nbsp;</td>
</tr>
<tr id="abs_laing2009a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA secondary structures can be divided into helical regions composed of canonical Watson-Crick and related base pairs, as well as single-stranded regions such as hairpin loops, internal loops, and junctions. These elements function as building blocks in the design of diverse RNA molecules with various fundamental functions in the cell. To better understand the intricate architecture of three-dimensional (3D) RNAs, we analyze existing RNA four-way junctions in terms of base-pair interactions and 3D configurations. Specifically, we identify nine broad junction families according to coaxial stacking patterns and helical configurations. We find that helices within junctions tend to arrange in roughly parallel and perpendicular patterns and stabilize their conformations using common tertiary motifs such as coaxial stacking, loop-helix interaction, and helix packing interaction. Our analysis also reveals a number of highly conserved base-pair interaction patterns and novel tertiary motifs such as A-minor-coaxial stacking combinations and sarcin/ricin motif variants. Such analyses of RNA building blocks can ultimately help in the difficult task of RNA 3D structure prediction.</td>
</tr>
<tr id="bib_laing2009a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{laing2009a,
  author = {Christian Laing and Tamar Schlick},
  title = {Analysis of Four-way Junctions in RNA Structures},
  journal = {Journal of Molecular Biology},
  year = {2009},
  volume = {390},
  number = {3},
  pages = {547--559},
  url = {http://dx.doi.org/10.1016/j.jmb.2009.04.084},
  doi = {http://dx.doi.org/10.1016/j.jmb.2009.04.084}
}
</pre></td>
</tr>
<tr id="landau2004" class="entry">
	<td>Landau, D. P., Tsai, S. &amp; Exler, M.</td>
	<td>A new approach to Monte Carlo simulations in statistical physics: Wang-Landau sampling <p class="infolinks">[<a href="javascript:toggleInfo('landau2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>American Journal of Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1119/1.1707017">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_landau2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{landau2004,
  author = {D. P. Landau and Shan-Ho Tsai and M. Exler},
  title = {A new approach to Monte Carlo simulations in statistical physics: Wang-Landau sampling},
  journal = {American Journal of Physics},
  year = {2004},
  volume = {72},
  pages = {1294-1302},
  doi = {http://dx.doi.org/10.1119/1.1707017}
}
</pre></td>
</tr>
<tr id="lankas2004" class="entry">
	<td>Lankas, F., Sponer, J., Langowski, J. &amp; III, T. E. C.</td>
	<td>DNA Deformability at the Base Pair Level <p class="infolinks">[<a href="javascript:toggleInfo('lankas2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lankas2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja0390449">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lankas2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A complete set of harmonic force constants describing the DNA deformation energetics at the base pair level was obtained using unrestrained atomic-resolution molecular dynamics simulations of selected duplex oligonucleotides and subsequent analysis of structural fluctuations from the simulated trajectories. The deformation was described by the six base pair conformational parameters (buckle, propeller, opening, shear, stretch, stagger). The results for 13 AT pairs and 11 GC pairs in different sequence contexts suggest that buckle and propeller are very flexible (more than roll in TA dinucleotide steps), while stretch is exceptionally stiff. Only stretch and opening stiffness were found to depend unambiguously on the base pair identity (AT vs GC). The relationship of the results to a simple plates-and-springs model of base-base interactions is discussed.</td>
</tr>
<tr id="bib_lankas2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lankas2004,
  author = {Filip Lankas and Jiri Sponer and Jorg Langowski and Thomas E. Cheatham III},
  title = {DNA Deformability at the Base Pair Level},
  journal = {Journal of the American Chemical Society},
  year = {2004},
  volume = {126},
  pages = {4124-4125},
  doi = {http://dx.doi.org/10.1021/ja0390449}
}
</pre></td>
</tr>
<tr id="laserson2005" class="entry">
	<td>Laserson, U., Gan, H. H. &amp; Schlick, T.</td>
	<td>Predicting Candidate Genomic Sequences that Correspond to Synthetic Functional RNA Motifs <p class="infolinks">[<a href="javascript:toggleInfo('laserson2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('laserson2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gki911">DOI</a><a href="http://dx.doi.org/10.1093/nar/gki911">URL</a> &nbsp;</td>
</tr>
<tr id="abs_laserson2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Riboswitches and RNA interference are important emerging mechanisms found in many organisms to control gene expression. To enhance our understanding of such RNA roles, finding small regulatory motifs in genomes presents a challenge on a wide scale. Many simple functional RNA motifs have been found by in vitro selection experiments, which produce synthetic target-binding aptamers as well as catalytic RNAs, including the hammerhead ribozyme. Motivated by the prediction of Piganeau and Schroeder [(2003) Chem. Biol., 10, 103-104] that synthetic RNAs may have natural counterparts, we develop and apply an efficient computational protocol for identifying aptamer-like motifs in genomes. We define motifs from the sequence and structural information of synthetic aptamers, search for sequences in genomes that will produce motif matches, and then evaluate the structural stability and statistical significance of the potential hits. Our application to aptamers for streptomycin, chloramphenicol, neomycin B and ATP identifies 37 candidate sequences (in coding and non-coding regions) that fold to the target aptamer structures in bacterial and archaeal genomes. Further energetic screening reveals that several candidates exhibit energetic properties and sequence conservation patterns that are characteristic of functional motifs. Besides providing candidates for experimental testing, our computational protocol offers an avenue for expanding natural RNA's functional repertoire.</td>
</tr>
<tr id="bib_laserson2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{laserson2005,
  author = {Uri Laserson and Hin Hark Gan and Tamar Schlick},
  title = {Predicting Candidate Genomic Sequences that Correspond to Synthetic Functional RNA Motifs},
  journal = {Nucleic Acids Research},
  year = {2005},
  volume = {33},
  number = {18},
  pages = {6057-6069},
  url = {http://dx.doi.org/10.1093/nar/gki911},
  doi = {http://dx.doi.org/10.1093/nar/gki911}
}
</pre></td>
</tr>
<tr id="lavery2009" class="entry">
	<td>Lavery, R., Moakher, M., Maddocks, J. H., Petkeviciute, D. &amp; Zakrzewska, K.</td>
	<td>Conformational analysis of nucleic acids revisited: Curves+ <p class="infolinks">[<a href="javascript:toggleInfo('lavery2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lavery2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Nucleic Acids Res &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkp608">DOI</a><a href="http://dx.doi.org/10.1093/nar/gkp608">URL</a> &nbsp;</td>
</tr>
<tr id="abs_lavery2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We describe Curves+, a new nucleic acid conformational analysis program which is applicable to a wide range of nucleic acid structures, including those with up to four strands and with either canonical or modified bases and backbones. The program is algorithmically simpler and computationally much faster than the earlier Curves approach, although it still provides both helical and backbone parameters, including a curvilinear axis and parameters relating the position of the bases to this axis. It additionally provides a full analysis of groove widths and depths. Curves+ can also be used to analyse molecular dynamics trajectories. With the help of the accompanying program Canal, it is possible to produce a variety of graphical output including parameter variations along a given structure and time series or histograms of parameter variations during dynamics.</td>
</tr>
<tr id="bib_lavery2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lavery2009,
  author = {R. Lavery and M. Moakher and J. H. Maddocks and D. Petkeviciute and K. Zakrzewska},
  title = {Conformational analysis of nucleic acids revisited: Curves+},
  journal = {Nucleic Acids Res},
  year = {2009},
  url = {http://dx.doi.org/10.1093/nar/gkp608},
  doi = {http://dx.doi.org/10.1093/nar/gkp608}
}
</pre></td>
</tr>
<tr id="lee2004" class="entry">
	<td>Lee, J. C. &amp; Gutell, R. R.</td>
	<td>Diversity of Base-pair Conformations and their Occurrence in rRNA Structure and RNA Structural Motifs <p class="infolinks">[<a href="javascript:toggleInfo('lee2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lee2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2004.09.072">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lee2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: In addition to the canonical base-pairs comprising the standard Watsonï¿½Crick (C:G and U:A) and wobble U:G conformations, an analysis of the base-pair types and conformations in the rRNAs in the high-resolution crystal structures of the Thermus thermophilus 30 S and Haloarcula marismortui 50 S ribosomal subunits has identified a wide variety of non-canonical base-pair types and conformations. However, the existing nomenclatures do not describe all of the observed non-canonical conformations or describe them with some ambiguity. Thus, a standardized system is required to classify all of these non-canonical conformations appropriately. Here, we propose a new, simple and systematic nomenclature that unambiguously classifies base-pair conformations occurring in base-pairs, base-triples and base-quadruples that are associated with secondary and tertiary interactions. This system is based on the topological arrangement of the two bases and glycosidic bonds in a given base-pair. Base-pairs in the internal positions of regular secondary structure helices usually form with canonical base-pair groups (C:G, U:A, and U:G) and canonical conformations (C:G WC, U:A WC, and U:G Wb). In contrast, non-helical base-pairs outside of regular structure helices usually have non-canonical base-pair groups and conformations. In addition, many non-helical base-pairs are involved in RNA motifs that form a defined set of non-canonical conformations. Thus, each rare non-canonical conformation may be functionally and structurally important. Finally, the topology-based isostericity of base-pair conformations can rationalize base-pair exchanges in the evolution of RNA molecules.</td>
</tr>
<tr id="bib_lee2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lee2004,
  author = {Jung C. Lee and Robin R. Gutell},
  title = {Diversity of Base-pair Conformations and their Occurrence in rRNA Structure and RNA Structural Motifs},
  journal = {Journal of Molecular Biology},
  year = {2004},
  volume = {344},
  pages = {1225-1249},
  doi = {http://dx.doi.org/10.1016/j.jmb.2004.09.072}
}
</pre></td>
</tr>
<tr id="lee2006" class="entry">
	<td>Lee, J. C., Gutell, R. R. &amp; Russell, R.</td>
	<td>The UAA/GAN Internal Loop Motif: A New RNA Structural Element that Forms a Cross-strand AAA Stack and Long-range Tertiary Interactions <p class="infolinks">[<a href="javascript:toggleInfo('lee2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2006.05.066">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_lee2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lee2006,
  author = {Jung C. Lee and Robin R. Gutell and Rick Russell},
  title = {The UAA/GAN Internal Loop Motif: A New RNA Structural Element that Forms a Cross-strand AAA Stack and Long-range Tertiary Interactions},
  journal = {Journal of Molecular Biology},
  year = {2006},
  volume = {360},
  pages = {978-988},
  doi = {http://dx.doi.org/10.1016/j.jmb.2006.05.066}
}
</pre></td>
</tr>
<tr id="cramer2007" class="entry">
	<td>Lehmann, E., Brueckner, F. &amp; Cramer, P.</td>
	<td>Molecular Basis of RNA-dependent RNA Polymerase II Activity <p class="infolinks">[<a href="javascript:toggleInfo('cramer2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('cramer2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature06290">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_cramer2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA polymerase (Pol) II catalyses DNA-dependent RNA synthesis during gene transcription. There is, however, evidence that Pol II also possesses RNA-dependent RNA polymerase (RdRP) activity. Pol II can use a homopolymeric RNA template1, can extend RNA by several nucleotides in the absence of DNA2, and has been implicated in the replication of the RNA genomes of hepatitis delta virus (HDV)3, 4 and plant viroids5. Here we show the intrinsic RdRP activity of Pol II with only pure polymerase, an RNA templateï¿½product scaffold and nucleoside triphosphates (NTPs). Crystallography reveals the templateï¿½product duplex in the site occupied by the DNAï¿½RNA hybrid during transcription. RdRP activity resides at the active site used during transcription, but it is slower and less processive than DNA-dependent activity. RdRP activity is also obtained with part of the HDV antigenome. The complex of transcription factor IIS (TFIIS) with Pol II can cleave one HDV strand, create a reactive stem-loop in the hybrid site, and extend the new RNA 3' end. Short RNA stem-loops with a 5' extension suffice for activity, but their growth to a critical length apparently impairs processivity. The RdRP activity of Pol II provides a missing link in molecular evolution, because it suggests that Pol II evolved from an ancient replicase that duplicated RNA genomes.</td>
</tr>
<tr id="bib_cramer2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{cramer2007,
  author = {Elisabeth Lehmann and Florian Brueckner and Patrick Cramer},
  title = {Molecular Basis of RNA-dependent RNA Polymerase II Activity},
  journal = {Nature},
  year = {2007},
  volume = {450},
  pages = {445-449},
  doi = {http://dx.doi.org/10.1038/nature06290}
}
</pre></td>
</tr>
<tr id="lemieux2006" class="entry">
	<td>Lemieux, S. &amp; Major, F.</td>
	<td>Automated Extraction and Classification of RNA Tertiary Structure Cyclic Motifs <p class="infolinks">[<a href="javascript:toggleInfo('lemieux2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lemieux2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkl120">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lemieux2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A minimum cycle basis of the tertiary structure<p>of a large ribosomal subunit (LSU) X-ray crystal<p>structure was analyzed. Most cycles are small, as<p>they are composed of 3- to 5 nt, and repeated across<p>the LSU tertiary structure. We used hierarchical<p>clustering to quantify and classify the 4 nt cycles.<p>One class is defined by the GNRA tetraloop motif.<p>The inspection of the GNRA class revealed peculiar<p>instances in sequence. First is the presence of<p>UA, CA, UC and CC base pairs that substitute the<p>usual sheared GA base pair. Second is the revelation<p>of GNR(Xn)A tetraloops, where Xn is bulged<p>out of the classical GNRA structure, and of GN/RA<p>formed by the two strands of interior-loops. We<p>were able to unambiguously characterize the cycle<p>classes using base stacking and base pairing<p>annotations. The cycles identified correspond to<p>small and cyclic motifs that compose most of the<p>LSU RNA tertiary structure and contribute to<p>its thermodynamic stability. Consequently, the<p>RNA minimum cycles could well be used as the<p>basic elements of RNA tertiary structure prediction<p>methods.</td>
</tr>
<tr id="bib_lemieux2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lemieux2006,
  author = {Sebastien Lemieux and Francois Major},
  title = {Automated Extraction and Classification of RNA Tertiary Structure Cyclic Motifs},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {2340-2346},
  doi = {http://dx.doi.org/10.1093/nar/gkl120}
}
</pre></td>
</tr>
<tr id="lemieux2002" class="entry">
	<td>Lemieux, S. &amp; Major, F.</td>
	<td>RNA Canonical and Non-Canonical Base Pairing Types: A Recognition Method and Complete Repertoire <p class="infolinks">[<a href="javascript:toggleInfo('lemieux2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lemieux2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkf540">DOI</a><a href="http://nar.oxfordjournals.org/cgi/content/abstract/30/19/4250">URL</a> &nbsp;</td>
</tr>
<tr id="abs_lemieux2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The problem of systematic and objective identification of canonical and non-canonical base pairs in RNA three-dimensional (3D) structures was studied. A probabilistic approach was applied, and an algorithm and its implementation in a computer program that detects and analyzes all the base pairs contained in RNA 3D structures were developed. The algorithm objectively distinguishes among canonical and non-canonical base pairing types formed by three, two and one hydrogen bonds (H-bonds), as well as those containing bifurcated and C-H.X...H-bonds. The nodes of a bipartite graph are used to encode the donor and acceptor atoms of a 3D structure. The capacities of the edges correspond to probabilities computed from the geometry of the donor and acceptor groups to form H-bonds. The maximum flow from donors to acceptors directly identifies base pairs and their types. A complete repertoire of base pairing types was built from the detected H-bonds of all X-ray crystal structures of a resolution of 3.0 A or better, including the large and small ribosomal subunits. The base pairing types are labeled using an extension of the nomenclature recently introduced by Leontis and Westhof. The probabilistic method was implemented in MC-Annotate, an RNA structure analysis computer program used to determine the base pairing parameters of the 3D modeling system MC-Sym.</td>
</tr>
<tr id="bib_lemieux2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lemieux2002,
  author = {Sébastien Lemieux and François Major},
  title = {RNA Canonical and Non-Canonical Base Pairing Types: A Recognition Method and Complete Repertoire},
  journal = {Nucleic Acids Research},
  year = {2002},
  volume = {30},
  number = {19},
  pages = {4250-4263},
  url = {http://nar.oxfordjournals.org/cgi/content/abstract/30/19/4250},
  doi = {http://dx.doi.org/10.1093/nar/gkf540}
}
</pre></td>
</tr>
<tr id="leontis2006" class="entry">
	<td>Leontis, N. B., Lescoute, A. &amp; Westhof, E.</td>
	<td>The Building Blocks and Motifs of RNA Architecture <p class="infolinks">[<a href="javascript:toggleInfo('leontis2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('leontis2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2006.05.009">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_leontis2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA motifs can be defined broadly as recurrent structural elements containing multiple intramolecular RNAï¿½RNA interactions, as observed in atomic-resolution RNA structures. They constitute the modular building blocks of RNA architecture, which is organized hierarchically. Recent work has focused on analyzing RNA backbone conformations to identify, define and search for new instances of recurrent motifs in X-ray structures. One current view asserts that recurrent RNA strand segments with characteristic backbone configurations qualify as independent motifs. Other considerations indicate that, to characterize modular motifs, one must take into account the larger structural context of such strand segments. This follows the biologically relevant motivation, which is to identify RNA structural characteristics that are subject to sequence constraints and that thus relate RNA architectures to sequences.</td>
</tr>
<tr id="bib_leontis2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{leontis2006,
  author = {Neocles B Leontis and Aurelie Lescoute and Eric Westhof},
  title = {The Building Blocks and Motifs of RNA Architecture},
  journal = {Current Opinion in Structural Biology},
  year = {2006},
  volume = {16},
  pages = {279-287},
  doi = {http://dx.doi.org/10.1016/j.sbi.2006.05.009}
}
</pre></td>
</tr>
<tr id="leontis2002" class="entry">
	<td>Leontis, N. B., Stombaugh, J. &amp; Westhof, E.</td>
	<td>The Non-Watson-Crick Base Pairs and their Associated Isostericity Matrices <p class="infolinks">[<a href="javascript:toggleInfo('leontis2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('leontis2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkf481">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_leontis2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA molecules exhibit complex structures in which a large fraction of the bases engage in non-Watsonï¿½Crick base pairing, forming motifs that mediate long-range RNAï¿½RNA interactions and create binding sites for proteins and small molecule ligands. The rapidly growing number of three-dimensional RNA structures at atomic resolution requires that databases contain the annotation of such base pairs. An unambiguous and descriptive nomenclature was proposed recently in which RNA base pairs were classified by the base edges participating in the interaction (Watsonï¿½Crick, Hoogsteen/CH or sugar edge) and the orientation of the glycosidic bonds relative to the hydrogen bonds (cis or trans). Twelve basic geometric families were identified and all 12 have been observed in crystal structures. For each base pairing family, we present here the 4 x 4 ï¿½isostericity matricesï¿½ summarizing the geometric relationships between the 16 pairwise combinations of the four standard bases, A, C, G and U. Whenever available, a representative example of each observed base pair from X-ray crystal structures (3.0 ï¿½ resolution or better) is provided or, otherwise, theoretically plausible models. This format makes apparent the recurrent geometric patterns that are observed and helps identify isosteric pairs that co-vary or interchange in sequences of homologous molecules while maintaining conserved three-dimensional motifs.</td>
</tr>
<tr id="bib_leontis2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{leontis2002,
  author = {Neocles B. Leontis and Jesse Stombaugh and Eric Westhof},
  title = {The Non-Watson-Crick Base Pairs and their Associated Isostericity Matrices},
  journal = {Nucleic Acids Research},
  year = {2002},
  volume = {30},
  pages = {3497-3531},
  doi = {http://dx.doi.org/10.1093/nar/gkf481}
}
</pre></td>
</tr>
<tr id="leontis2003" class="entry">
	<td>Leontis, N. B. &amp; Westhof, E.</td>
	<td>Analysis of RNA Motifs <p class="infolinks">[<a href="javascript:toggleInfo('leontis2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('leontis2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(03)00076-9">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_leontis2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA motifs are directed and ordered stacked arrays of non-Watson?Crick base pairs forming distinctive foldings of the phosphodiester backbones of the interacting RNA strands. They correspond to the ?loops? ? hairpin, internal and junction ? that intersperse the Watson?Crick two-dimensional helices as seen in two-dimensional representations of RNA structure. RNA motifs mediate the specific interactions that induce the compact folding of complex RNAs. RNA motifs also constitute specific protein or ligand binding sites. A given motif is characterized by all the sequences that fold into essentially identical three-dimensional structures with the same ordered array of isosteric non-Watson?Crick base pairs. It is therefore crucial, when analyzing a three-dimensional RNA structure in order to identify and compare motifs, to first classify its non-Watson?Crick base pairs geometrically.</td>
</tr>
<tr id="bib_leontis2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{leontis2003,
  author = {N. B. Leontis and E. Westhof},
  title = {Analysis of RNA Motifs},
  journal = {Current Opinion in Structural Biology},
  year = {2003},
  volume = {13},
  pages = {300-308},
  doi = {http://dx.doi.org/10.1016/S0959-440X(03)00076-9}
}
</pre></td>
</tr>
<tr id="leontis2002b" class="entry">
	<td>Leontis, N. B. &amp; Westhof, E.</td>
	<td>The Annotation of RNA Motifs <p class="infolinks">[<a href="javascript:toggleInfo('leontis2002b','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Comparative and Functional Genomics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/cfg.213">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_leontis2002b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{leontis2002b,
  author = {Neocles B. Leontis and Eric Westhof},
  title = {The Annotation of RNA Motifs},
  journal = {Comparative and Functional Genomics},
  year = {2002},
  volume = {3},
  pages = {518-524},
  doi = {http://dx.doi.org/10.1002/cfg.213}
}
</pre></td>
</tr>
<tr id="lescoute2005" class="entry">
	<td>Lescoute, A., Leontis, N. B., Massire, C. &amp; Westhof, E.</td>
	<td>Recurrent Structural RNA Motifs, Isostericity Matrices and Sequence Alignments <p class="infolinks">[<a href="javascript:toggleInfo('lescoute2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lescoute2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gki535">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lescoute2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The occurrences of two recurrent motifs in ribosomal RNA sequences, the Kink-turn and the C-loop, are examined in crystal structures and systematically compared with sequence alignments of rRNAs from the three kingdoms of life in order to identify the range of the structural and sequence variations. Isostericity Matrices are used to analyze structurally the sequence variations of the characteristic non-Watson?Crick base pairs for each motif. We show that Isostericity Matrices for non-Watson?Crick base pairs provide important tools for deriving the sequence signatures of recurrent motifs, for scoring and refining sequence alignments, and for determining whether motifs are conserved throughout evolution. The systematic use of Isostericity Matrices identifies the positions of the insertion or deletion of one or more nucleotides relative to the structurally characterized examples of motifs and, most importantly, specifies whether these changes result in new motifs. Thus, comparative analysis coupled with Isostericity Matrices allows one to produce and refine structural sequence alignments. The analysis, based on both sequence and structure, permits therefore the evaluation of the conservation of motifs across phylogeny and the derivation of rules of equivalence between structural motifs. The conservations observed in Isostericity Matrices form a predictive basis for identifying motifs in sequences.</td>
</tr>
<tr id="bib_lescoute2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lescoute2005,
  author = {Aurelie Lescoute and Neocles B. Leontis and Christian Massire and Eric Westhof},
  title = {Recurrent Structural RNA Motifs, Isostericity Matrices and Sequence Alignments},
  journal = {Nucleic Acids Research},
  year = {2005},
  volume = {33},
  number = {8},
  pages = {2395-2409},
  doi = {http://dx.doi.org/10.1093/nar/gki535}
}
</pre></td>
</tr>
<tr id="lescoute2006" class="entry">
	<td>Lescoute, A. &amp; Westhof, E.</td>
	<td>Topology of three-way junctions in folded RNAs <p class="infolinks">[<a href="javascript:toggleInfo('lescoute2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lescoute2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.2208106">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lescoute2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The three-way junctions contained in X-ray structures of folded RNAs have been compiled and analyzed. Three-way junctions with two helices approximately coaxially stacked can be divided into three main families depending on the relative lengths of the segments linking the three Watson-Crick helices. Each family has topological characteristics with some conservation in the non- Watson-Crick pairs within the linking segments as well as in the types of contacts between the segments and the helices. The most populated family presents tertiary interactions between two helices as well as extensive shallow/minor groove contacts between a linking segment and the third helix. On the basis of the lengths of the linking segments, some guidelines could be deduced for choosing a topology for a three-way junction on the basis of a secondary structure. Examples and prediction based on those rules are discussed.</td>
</tr>
<tr id="bib_lescoute2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lescoute2006,
  author = {Aurelie Lescoute and Eric Westhof},
  title = {Topology of three-way junctions in folded RNAs},
  journal = {RNA},
  year = {2006},
  volume = {12},
  pages = {83-93},
  doi = {http://dx.doi.org/10.1261/rna.2208106}
}
</pre></td>
</tr>
<tr id="lescoute2005b" class="entry">
	<td>Lescoute, A. &amp; Westhof, E.</td>
	<td>Riboswitch Structures: Purine Ligands Replace Tertiary Contacts <p class="infolinks">[<a href="javascript:toggleInfo('lescoute2005b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lescoute2005b','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Chemistry and Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.chembiol.2005.01.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lescoute2005b" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Two recent reports describe the fascinating crystal structures of the G-riboswitch complexed to three different purine ligands, hypoxanthine [1], guanine, and adenine [2].</td>
</tr>
<tr id="bib_lescoute2005b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lescoute2005b,
  author = {Aurï¿½lie Lescoute and Eric Westhof},
  title = {Riboswitch Structures: Purine Ligands Replace Tertiary Contacts},
  journal = {Chemistry and Biology},
  year = {2005},
  volume = {12},
  pages = {10-13},
  doi = {http://dx.doi.org/10.1016/j.chembiol.2005.01.002}
}
</pre></td>
</tr>
<tr id="levitt1983" class="entry">
	<td>Levitt, M.</td>
	<td>Computer Simulation of DNA Double-helix Dynamics <p class="infolinks">[<a href="javascript:toggleInfo('levitt1983','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('levitt1983','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td>Cold Spring Harb Symp Quant Biol. &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_levitt1983" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The static structure of DNA has been known for 30 years (Watson and Crick 1953). During the past 5 years, DNA has been shown to have a surprising degreee of conformational flexibility in that the number of base pairs per turn is not the same in solution and in fibers (Arnott and Hukins 1972; Wang 1979; Rhodes and Klug 1980), that the base and backbone atoms undergo angular motions of large amplitude (>25deg) on a time scale of nanoseconds (Early and Kearns 1979; Bolton and James; Hogan and Jardetsky), and that there are cooperative conformational transitions mediated by changing environment or binding of drug molecules (Sobell et al. 1997; Hogan et al.; Datagupta and Crothers 1981). Model building and computer calculation have considered the static deformation of the DNA double helix by kinking (Crick and Klug 1975; Sobell et al. 1977) or by smooth bending (Levitt 1978; Sussman and Trifonov 1978). Elegant mathematics has been used to analyze the dynamic behaviour of DNA by assuming that the molecule behaves like an isotropic elastic rod (Barkley and Zimm 1979).</td>
</tr>
<tr id="bib_levitt1983" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{levitt1983,
  author = {Michael Levitt},
  title = {Computer Simulation of DNA Double-helix Dynamics},
  journal = {Cold Spring Harb Symp Quant Biol.},
  year = {1983},
  volume = {47},
  pages = {251-262}
}
</pre></td>
</tr>
<tr id="li2006" class="entry">
	<td>Li, P. T. X., Bustamante, C. &amp; Jr., I. T.</td>
	<td>Unusual Mechanical Stability of a Minimal RNA Kissing Complex <p class="infolinks">[<a href="javascript:toggleInfo('li2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Proceedings of the National Academy of Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0607202103">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_li2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{li2006,
  author = {Pan T. X. Li and Carlos Bustamante and Ignacio Tinoco Jr.},
  title = {Unusual Mechanical Stability of a Minimal RNA Kissing Complex},
  journal = {Proceedings of the National Academy of Science},
  year = {2006},
  volume = {103},
  pages = {15847-15852},
  doi = {http://dx.doi.org/10.1073/pnas.0607202103}
}
</pre></td>
</tr>
<tr id="lim2003" class="entry">
	<td>Lim, T.</td>
	<td>Spring Constant Analogy for Estimating Stiffness of a Single Polyethylene Molecule <p class="infolinks">[<a href="javascript:toggleInfo('lim2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lim2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Journal of Mathematical Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1023/B:JOMC.0000004065.34221.47">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lim2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The stiffness of a simple planar polymeric chain is modeled using analogies of mechanical springs arranged in series and in parallel assemblies. The stiffness of chemical bonds is resolved into two perpendicular axes defined by the longitudinal and transverse axes in the molecular plane. Using Hooke's definition of spring stiffness, the molecular stiffness of polyethylene is obtained along the longitudinal and transverse directions. This paper demonstrates the use of physical analogies and mathematical approximations for obtaining an analytical form for the stiffness of a simple single-molecule.</td>
</tr>
<tr id="bib_lim2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lim2003,
  author = {Teik-Cheng Lim},
  title = {Spring Constant Analogy for Estimating Stiffness of a Single Polyethylene Molecule},
  journal = {Journal of Mathematical Chemistry},
  year = {2003},
  volume = {34},
  pages = {151-161},
  doi = {http://dx.doi.org/10.1023/B:JOMC.0000004065.34221.47}
}
</pre></td>
</tr>
<tr id="liphardt2001" class="entry">
	<td>Liphardt, J., Onoa, B., Smith, S., Jr., I. T. &amp; Bustamante, C.</td>
	<td>Reversible unfolding of single RNA molecules by mechanical force. <p class="infolinks">[<a href="javascript:toggleInfo('liphardt2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('liphardt2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1058498">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_liphardt2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Here we use mechanical force to induce the unfolding and refolding of single RNA molecules: a simple RNA hairpin, a molecule containing a three-helix junction, and the P5abc domain of the Tetrahymena thermophila ribozyme. All three molecules (P5abc only in the absence of Mg21) can be mechanically unfolded at equilibrium, and when kept at constant force within a critical force range, are bi-stable and hop between folded and unfolded states. We determine the force-dependent equilibrium constants for folding/unfolding these single RNA molecules and the positions of their transition states along the reaction coordinate.</td>
</tr>
<tr id="bib_liphardt2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{liphardt2001,
  author = {J. Liphardt and B. Onoa and S.B. Smith and I. Tinoco Jr. and C. Bustamante},
  title = {Reversible unfolding of single RNA molecules by mechanical force.},
  journal = {Science},
  year = {2001},
  volume = {292},
  pages = {733-737},
  doi = {http://dx.doi.org/10.1126/science.1058498}
}
</pre></td>
</tr>
<tr id="liu2005" class="entry">
	<td>Liu, F. &amp; Ou-Yang, Z.</td>
	<td>Monte Carlo Simulation for Single RNA Unfolding by Force <p class="infolinks">[<a href="javascript:toggleInfo('liu2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('liu2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biphysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.104.049239">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_liu2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Using polymer elastic theory and known RNA free energies, we construct a Monte Carlo algorithm to simulate the<p>single RNA folding and unfolding by mechanical force on the secondary structure level. For the constant force ensemble, we<p>simulate the force-extension curves of the P5ab, P5abcDA, and P5abc molecules in equilibrium. For the constant extension<p>ensemble, we focus on the mechanical behaviors of the RNA P5ab molecule, which include the unfolding force dependence on<p>the pulling speed, the force-hysteresis phenomenon, and the coincidence of stretching-relaxing force-curves in thermal<p>equilibrium. We particularly simulate the time traces of the end-to-end distance of the P5ab under the constant force in equilibrium,<p>which also have been recorded in the recent experiment. The reaction rate constants for the folding and unfolding are calculated.<p>Our results show that the agreement between the simulation and the experimental measurements is satisfactory.</td>
</tr>
<tr id="bib_liu2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{liu2005,
  author = {Fei Liu and Zhong-can Ou-Yang},
  title = {Monte Carlo Simulation for Single RNA Unfolding by Force},
  journal = {Biphysical Journal},
  year = {2005},
  volume = {88},
  pages = {76-84},
  doi = {http://dx.doi.org/10.1529/biophysj.104.049239}
}
</pre></td>
</tr>
<tr id="lu1997" class="entry">
	<td>Lu, X., Hassan, M. A. E. &amp; Hunter, C. A.</td>
	<td>Structure and Conformation of Helical Nucleic Acids: Analysis Program (SCHNAaP) <p class="infolinks">[<a href="javascript:toggleInfo('lu1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lu1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1997.1346">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lu1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a new versatile program, SCHNAaP, for the analysis of double-helical nucleic acid structures. The program uses mathematically rigorous and fully reversible procedures for calculating the structural parameters: the Imageambridge University Imagengineering Department Imageelix computation Imagecheme (CEHS) is used to determine the local helical parameters and an analogous procedure is used to determine the global helical parameters. These parameters form a complete set that conforms to the "Cambridge Accord" on definitions and nomenclature of nucleic acid structure parameters. In addition to the two standard Watson-Crick base-pairs, the program handles mismatched base-pairs and chemically modified bases. An analysis of the sugar-phosphate backbone conformation is included. Standardized base-stacking diagrams of each dinucleotide step with reference to the mid-step triad are generated. Structures are classified as one of the four polymorphic families, A/B,Z, W or R, although W- and R-DNA (two types of hypothetical structure) have yet to be observed experimentally.</td>
</tr>
<tr id="bib_lu1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lu1997,
  author = {Xiang-Jun Lu and M. A. El Hassan and C. A. Hunter},
  title = {Structure and Conformation of Helical Nucleic Acids: Analysis Program (SCHNAaP)},
  journal = {Journal of Molecular Biology},
  year = {1997},
  volume = {273},
  pages = {668-680},
  doi = {http://dx.doi.org/10.1006/jmbi.1997.1346}
}
</pre></td>
</tr>
<tr id="lu1997a" class="entry">
	<td>Lu, X., Hassan, M. A. E. &amp; Hunter, C. A.</td>
	<td>Structure and Conformation of Helical Nucleic Acids: Rebuilding Program (SCHNArP) <p class="infolinks">[<a href="javascript:toggleInfo('lu1997a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lu1997a','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1997.1345">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lu1997a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a program, SCHNArP, for rebuilding double-helical nucleic acid structures from a set of helical parameters. The parameter sets are based on mathematically reversible schemes that allow direct comparison of data from experimental X-ray crystal structures analyzed using the analysis program, SCHNAaP (see accompanying paper), and structures built using the rebuilding program, SCHNArP. The program uses either local CEHS helical parameters or global helical parameters. A number of standard parameter sets from the literature are included that allow comparison of oligomer and polymer structures generated using different models for sequence-dependent DNA bending. Exact atomic models are provided for the bases. Schematic models that trace the path of the backbone and use rectangular blocks for the bases can be generated.</td>
</tr>
<tr id="bib_lu1997a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lu1997a,
  author = {Xiang-Jun Lu and M. A. El Hassan and C. A. Hunter},
  title = {Structure and Conformation of Helical Nucleic Acids: Rebuilding Program (SCHNArP)},
  journal = {Journal of Molecular Biology},
  year = {1997},
  volume = {273},
  pages = {681-691},
  doi = {http://dx.doi.org/10.1006/jmbi.1997.1345}
}
</pre></td>
</tr>
<tr id="lu2003" class="entry">
	<td>Lu, X. &amp; Olson, W.</td>
	<td>3DNA: A Software Package for the Analysis, Rebuilding and Visualization of the Three-Dimensional Nucleic Acid Structures <p class="infolinks">[<a href="javascript:toggleInfo('lu2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lu2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg680">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lu2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a comprehensive software package, 3DNA, for the analysis, reconstruction and visualization of three-dimensional nucleic acid structures. Starting from a coordinate file in Protein Data Bank (PDB) format, 3DNA can handle antiparallel and parallel double helices, single-stranded structures, triplexes, quadruplexes and other complex tertiary folding motifs found in both DNA and RNA structures. The analysis routines identify and categorize all base interactions and classify the double helical character of appropriate base pair steps. The program makes use of a recently recommended reference frame for the description of nucleic acid base pair geometry and a rigorous matrix-based scheme to calculate local conformational parameters and rebuild the structure from these parameters. The rebuilding routines produce rectangular block representations of nucleic acids as well as full atomic models with the sugar?phosphate backbone and publication quality ?standardized? base stacking diagrams. Utilities are provided to locate the base pairs and helical regions in a structure and to reorient structures for effective visualization. Regular helical models based on X-ray diffraction measurements of various repeating sequences can also be generated within the program.</td>
</tr>
<tr id="bib_lu2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lu2003,
  author = {Xiang-Jun Lu and Wilma Olson},
  title = {3DNA: A Software Package for the Analysis, Rebuilding and Visualization of the Three-Dimensional Nucleic Acid Structures},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  number = {17},
  pages = {5108-5121},
  doi = {http://dx.doi.org/10.1093/nar/gkg680}
}
</pre></td>
</tr>
<tr id="lu1999" class="entry">
	<td>Lu, X. &amp; Olson, W. K.</td>
	<td>Resolving the discrepancies among nucleic acid conformational analyses <p class="infolinks">[<a href="javascript:toggleInfo('lu1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('lu1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1998.2390">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_lu1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Growing interest in understanding the relationship between the global folding of nucleic acids and the sequence-dependent structure of individual base-pair steps has stimulated the development of new mathematical methods to define the geometry of the constituent base-pairs. Several approaches, designed to meet guidelines set by the nucleic acid community, permit rigorous comparative analyses of different three-dimensional structures, as well as allow for reconstruction of chain molecules at the base-pair level. The different computer programs, however, yield inconsistent descriptions of chain conformation. Here we report our own implementation of seven algorithms used to determine base-pair and dimer step parameters. Aside from reproducing the results of individual programs, we uncover the reasons why the different algorithms come to conflicting structural interpretations. The choice of mathematics has only a limited effect on the computed parameters, even in highly deformed duplexes. The results are much more sensitive to the choice of reference frame. The disparate schemes yield very similar conformational descriptions if the calculations are based on a common reference frame. The current positioning of reference frames at the inner and outer edges of complementary bases exaggerates the rise at distorted dimer steps, and points to the need for a carefully defined conformational standard.</td>
</tr>
<tr id="bib_lu1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{lu1999,
  author = {Xiang-Jun Lu and Wilma K. Olson},
  title = {Resolving the discrepancies among nucleic acid conformational analyses},
  journal = {Journal of Molecular Biology},
  year = {1999},
  volume = {285},
  pages = {1563-1575},
  doi = {http://dx.doi.org/10.1006/jmbi.1998.2390}
}
</pre></td>
</tr>
<tr id="mathews2006.pdf" class="entry">
	<td>Lu, Z. J., Turner, D. H. &amp; Mathews, D. H.</td>
	<td>A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation <p class="infolinks">[<a href="javascript:toggleInfo('mathews2006.pdf','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mathews2006.pdf','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkl472">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mathews2006.pdf" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A complete set of nearest neighbor parameters to<p>predict the enthalpy change of RNA secondary<p>structure formation was derived. These parameters<p>can be used with available free energy nearest<p>neighbor parameters to extend the secondary structure<p>prediction of RNA sequences to temperatures<p>other than 37 degrees C. The parameters were tested by<p>predicting the secondary structures of sequences<p>with known secondary structure that are from<p>organisms with known optimal growth temperatures.<p>Compared with the previous set of enthalpy<p>nearest neighbor parameters, the sensitivity of base<p>pair prediction improved from 65.2 to 68.9% at<p>optimal growth temperatures ranging from 10 to<p>60 degrees C. Base pair probabilities were predicted with a<p>partition function and the positive predictive value<p>of structure prediction is 90.4% when considering<p>the base pairs in the lowest free energy structure<p>with pairing probability of 0.99 or above. Moreover, a<p>strong correlation is found between the predicted<p>melting temperatures of RNA sequences and the<p>optimal growth temperatures of the host organism.<p>This indicates that organisms that live at higher<p>temperatures have evolved RNA sequences with<p>higher melting temperatures.</td>
</tr>
<tr id="bib_mathews2006.pdf" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mathews2006.pdf,
  author = {Zhi John Lu and Douglas H. Turner and David H. Mathews},
  title = {A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {4912-4924},
  doi = {http://dx.doi.org/10.1093/nar/gkl472}
}
</pre></td>
</tr>
<tr id="macdougall2001" class="entry">
	<td>MacDougall, P. J. &amp; Henze, C. E.</td>
	<td>Identification of Molecular Reactive Sites With an Interactive Volume Rendering Tool <p class="infolinks">[<a href="javascript:toggleInfo('macdougall2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('macdougall2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Theoretical Chemistry Accounts &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1007/s002140000225">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_macdougall2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new molecular visualization tool is presented, EVolVis (an abbreviation of electronic volume visualizer). This software enables the user to interactively explore a molecule's electronic charge density for topological features. Using novel volume rendering techniques, the entire molecule can be probed without obstruction by opaque surfaces or preselection of specific orbitals or contours. Menus with sliding scales permit various ranges of the function of interest to be examined interactively. We have investigated the Laplacian of the electronic charge density calculated for several biomolecules that contain a variety of reactive sites: cisplatin, penamecillin and isomers of nitrogen bases from DNA. EVolVis is useful in identifying inner-shell features (cisplatin), polar, nonpolar and aromatic regions (penamecillin), hydrogen-bonding sites of varying strength (nitrogen bases and penamecillin) and other reactive sites (strained heterocycle in penamecillin).</td>
</tr>
<tr id="bib_macdougall2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{macdougall2001,
  author = {Preston J. MacDougall and Christopher E. Henze},
  title = {Identification of Molecular Reactive Sites With an Interactive Volume Rendering Tool},
  journal = {Theoretical Chemistry Accounts},
  year = {2001},
  volume = {105},
  pages = {345-353},
  doi = {http://dx.doi.org/10.1007/s002140000225}
}
</pre></td>
</tr>
<tr id="macrae2006" class="entry">
	<td>MacRae, I. J., Zhou, K., Li, F., Repic, A., Brooks, A. N., Cande, W. Z., Adams, P. D. &amp; Doudna, J. A.</td>
	<td>Structural Basis for Double-Stranded RNA Processing by Dicer <p class="infolinks">[<a href="javascript:toggleInfo('macrae2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('macrae2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1121638">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_macrae2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The specialized ribonuclease Dicer initiates RNA interference by cleaving double-stranded RNA (dsRNA) substrates into small fragments about 25 nucleotides in length. In the crystal structure of an intact Dicer enzyme, the PAZ domain, a module that binds the end of dsRNA, is separated from the two catalytic ribonuclease III (RNase III) domains by a flat, positively charged surface. The 65 angstrom distance between the PAZ and RNase III domains matches the length spanned by 25 base pairs of RNA. Thus, Dicer itself is a molecular ruler that recognizes dsRNA and cleaves a specified distance from the helical end.</td>
</tr>
<tr id="bib_macrae2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{macrae2006,
  author = {Ian J. MacRae and Kaihong Zhou and Fei Li and Adrian Repic and Angela N. Brooks and W. Zacheus Cande and Paul D. Adams and Jennifer A. Doudna},
  title = {Structural Basis for Double-Stranded RNA Processing by Dicer},
  journal = {Science},
  year = {2006},
  volume = {311},
  pages = {195-198},
  doi = {http://dx.doi.org/10.1126/science.1121638}
}
</pre></td>
</tr>
<tr id="malhotra1990" class="entry">
	<td>Malhotra, A., Tan, R. K. &amp; Harvey, S. C.</td>
	<td>Prediction of the three-dimensional structure of Escherichia coli 30S ribosomal subunit: a molecular mechanics approach. <p class="infolinks">[<a href="javascript:toggleInfo('malhotra1990','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('malhotra1990','bibtex')">BibTeX</a>]</p></td>
	<td>1990</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_malhotra1990" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce a computer-assisted procedure for folding large RNA chains into three-dimensional conformations consistent with their secondary structure and other known experimental constraints. The RNA chain is modeled using pseudoatoms at different levels of detail--from a single pseudoatom per helix to a single pseudoatom for each nucleotide. A stepwise procedure is used, starting with a simple representation of the macromolecule that is refined and then extrapolated into higher resolution for further refinement. The procedure is capable of folding different random-walk chains by using energy minimization, allowing generation of a range of conformations consistent with given experimental data. We use this procedure to generate several possible conformations of the 16S RNA in the 30S ribosomal subunit of Escherichia coli by using secondary structure and the neutron-scattering map of the 21 proteins in the small subunit. The RNA chain is modeled using a single pseudoatom per helix. RNA-RNA and RNA-protein crosslinks, reported in current literature, are included in our model. Footprinting data for different ribosomal proteins in the 16S RNA are also used. Several conformations of the 16S RNA are generated and compared to predict gross structural features of the 30S subunit as well as to identify regions of the 16S RNA that cannot be well-defined with current experimental data.</td>
</tr>
<tr id="bib_malhotra1990" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{malhotra1990,
  author = {A. Malhotra and R. K. Tan and S. C. Harvey},
  title = {Prediction of the three-dimensional structure of Escherichia coli 30S ribosomal subunit: a molecular mechanics approach.},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {1990},
  volume = {87},
  pages = {1950-1954}
}
</pre></td>
</tr>
<tr id="mamasakhlisov2007" class="entry">
	<td>Mamasakhlisov, Y. S., Hayryan, S., Morozov, V. F. &amp; Hu, C.</td>
	<td>RNA folding in the presence of counterions <p class="infolinks">[<a href="javascript:toggleInfo('mamasakhlisov2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mamasakhlisov2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Physical Review E. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1103/PhysRevE.75.061907">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mamasakhlisov2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We present a general thermodynamic picture of the folding of RNA-like heteropolymer based on the basic physical principles. The Hamiltonian of the model includes all characteristic interactions explicitly. A particular attention is paid to the electrostatic interactions whose role in the RNA folding is known to be crucial. In this paper we study RNA folding with the full Hamiltonian and describe the spin-glass behavior on the level of tertiary structure. We show that formation of the stable tertiary structure is possible in the random RNA-like molecule. By including into the model the nonspecific interactions of the RNA molecule with counterions, we derive the logarithmic dependencies of the melting and freezing temperatures on the ion concentration, which is consistent with experimental data [R. Shiman and D. E. Draper, J. Mol. Biol. 302, 79 (2000)]. We also infer that the large RNA folds slower than the hierarchical model predicts, which was observed in the experiments.</td>
</tr>
<tr id="bib_mamasakhlisov2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mamasakhlisov2007,
  author = {Yevgeni Sh. Mamasakhlisov and Shura Hayryan and V. F. Morozov and Chin-Kun Hu},
  title = {RNA folding in the presence of counterions},
  journal = {Physical Review E.},
  year = {2007},
  volume = {75},
  pages = {0619071-06190710},
  doi = {http://dx.doi.org/10.1103/PhysRevE.75.061907}
}
</pre></td>
</tr>
<tr id="mandal2005" class="entry">
	<td>Mandal, B., Banerjee, M. &amp; Mukherjee, A. K.</td>
	<td>Wiener and Hosoya indices of reciprocal graphs <p class="infolinks">[<a href="javascript:toggleInfo('mandal2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mandal2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Molecular Physics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1080/00268970500134714">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mandal2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Two types of topological indices, Wiener and Hosoya indices, of three classes of reciprocal<p>graphs (namely GLn , GKn and GCn ) have been shown to be expressed in terms of the number of<p>pendant vertices (n). The Wiener index is expressed in analytical form whereas the Hosoya<p>index is expressed either by recurrence relations or directly by matrix products for which a<p>facile computer program can be written. It has also been shown that the Hosoya index of GKn<p>can be obtained in analytical form.</td>
</tr>
<tr id="bib_mandal2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mandal2005,
  author = {B. Mandal and M. Banerjee and A. K. Mukherjee},
  title = {Wiener and Hosoya indices of reciprocal graphs},
  journal = {Molecular Physics},
  year = {2005},
  volume = {19},
  pages = {10},
  doi = {http://dx.doi.org/10.1080/00268970500134714}
}
</pre></td>
</tr>
<tr id="manning2003" class="entry">
	<td>Manning, G. S.</td>
	<td>Comments on Selected Aspects of Nucleic Acid Electrostatics <p class="infolinks">[<a href="javascript:toggleInfo('manning2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('manning2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.10361">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_manning2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent experimental, theoretical, and computational developments in the field of nucleic acid electrostatics have brought interesting concepts to the fore. The phosphate charge on the double helix apparently influences its structure. When the charge is neutralized asymmetrically, the resulting force imbalance drives bending toward the neutralized side. When the charge is uniformly neutralized, the force imbalance acts to buckle the helical axis, resulting in a compact tertiary conformation. Sharing of condensed counterions by single strands is a stabilizing factor for formation of the double helix. Sharing of condensed counterions by two double helices causes clustering of DNA and may be a factor in RNA folding. Support for these statements is reviewed.</td>
</tr>
<tr id="bib_manning2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{manning2003,
  author = {Gerald S. Manning},
  title = {Comments on Selected Aspects of Nucleic Acid Electrostatics},
  journal = {Biopolymers},
  year = {2003},
  volume = {69},
  pages = {137-143},
  doi = {http://dx.doi.org/10.1002/bip.10361}
}
</pre></td>
</tr>
<tr id="manning1977" class="entry">
	<td>Manning, G. S.</td>
	<td>Limiting laws and counterion condensation in polyelectrolyte solutions. IV. The approach to the limit and the extraordinary stability of the charge fraction. <p class="infolinks">[<a href="javascript:toggleInfo('manning1977','bibtex')">BibTeX</a>]</p></td>
	<td>1977</td>
	<td>Biophysical Chemistry &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_manning1977" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{manning1977,
  author = {G. S. Manning},
  title = {Limiting laws and counterion condensation in polyelectrolyte solutions. IV. The approach to the limit and the extraordinary stability of the charge fraction.},
  journal = {Biophysical Chemistry},
  year = {1977},
  volume = {7},
  pages = {95-102}
}
</pre></td>
</tr>
<tr id="maragakis2008" class="entry">
	<td>Maragakis, P., Ritort, F., Bustamante, C., Karplus, M. &amp; Crooks, G. E.</td>
	<td>Bayesian estimates of free energies from nonequilibrium work data in the presence of instrument noise. <p class="infolinks">[<a href="javascript:toggleInfo('maragakis2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Chemical Physics &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_maragakis2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{maragakis2008,
  author = {Paul Maragakis and Felix Ritort and Carlos Bustamante and Martin Karplus and Gavin E. Crooks},
  title = {Bayesian estimates of free energies from nonequilibrium work data in the presence of instrument noise.},
  journal = {Journal of Chemical Physics},
  year = {2008},
  volume = {129}
}
</pre></td>
</tr>
<tr id="margush1981" class="entry">
	<td>Margush, T. &amp; McMorris, F. R.</td>
	<td>Consensus n-Trees <p class="infolinks">[<a href="javascript:toggleInfo('margush1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>Bulletin of Mathematical Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_margush1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{margush1981,
  author = {T. Margush and F. R. McMorris},
  title = {Consensus n-Trees},
  journal = {Bulletin of Mathematical Biology},
  year = {1981},
  volume = {43},
  pages = {239-244}
}
</pre></td>
</tr>
<tr id="marky1982" class="entry">
	<td>Marky, L. A. &amp; Breslauer, K. J.</td>
	<td>Calorimetric determination of base-stacking enthalpies in double-helical DNA molecules <p class="infolinks">[<a href="javascript:toggleInfo('marky1982','bibtex')">BibTeX</a>]</p></td>
	<td>1982</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_marky1982" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{marky1982,
  author = {L. A. Marky and K. J. Breslauer},
  title = {Calorimetric determination of base-stacking enthalpies in double-helical DNA molecules},
  journal = {Biopolymers},
  year = {1982},
  volume = {11},
  pages = {2185-2194}
}
</pre></td>
</tr>
<tr id="mathews2004" class="entry">
	<td>Mathews, D. H., Disney, M. D., Childs, J. L., Schroeder, S. J., Zuker, M. &amp; Turner, D. H.</td>
	<td>Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure <p class="infolinks">[<a href="javascript:toggleInfo('mathews2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mathews2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Proceedings of the National Academy of Sciences &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0401799101">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mathews2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A dynamic programming algorithm for prediction of RNA secondary structure has been revised to accommodate folding constraints determined by chemical modification and to include free energy increments for coaxial stacking of helices when they are either adjacent or separated by a single mismatch. Furthermore, free energy parameters are revised to account for recent experimental results for terminal mismatches and hairpin, bulge, internal, and multibranch loops. To demonstrate the applicability of this method, in vivo modification was performed on 5S rRNA in both Escherichia coli and Candida albicans with 1-cyclohexyl-3-(2-morpholinoethyl) carbodiimide metho-p-toluene sulfonate, dimethyl sulfate, and kethoxal. The percentage of known base pairs in the predicted structure increased from 26.3% to 86.8% for the E. coli sequence by using modification constraints. For C. albicans, the accuracy remained 87.5% both with and without modification data. On average, for these sequences and a set of 14 sequences with known secondary structure and chemical modification data taken from the literature, accuracy improves from 67% to 76%. This enhancement primarily reflects improvement for three sequences that are predicted with <40% accuracy on the basis of energetics alone. For these sequences, inclusion of chemical modification constraints improves the average accuracy from 28% to 78%. For the 11 sequences with <6% pseudoknotted base pairs, structures predicted with constraints from chemical modification contain on average 84% of known canonical base pairs.</td>
</tr>
<tr id="bib_mathews2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mathews2004,
  author = {David H. Mathews and Matthew D. Disney and Jessica L. Childs and Susan J. Schroeder and Michael Zuker and Douglas H. Turner},
  title = {Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2004},
  volume = {101},
  number = {19},
  pages = {7287-7292},
  doi = {http://dx.doi.org/10.1073/pnas.0401799101}
}
</pre></td>
</tr>
<tr id="turner2006" class="entry">
	<td>Mathews, D. H. &amp; Turner, D. H.</td>
	<td>Prediction of RNA secondary structure by free energy minimization <p class="infolinks">[<a href="javascript:toggleInfo('turner2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2006.05.010">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_turner2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{turner2006,
  author = {David H Mathews and Douglas H Turner},
  title = {Prediction of RNA secondary structure by free energy minimization},
  journal = {Current Opinion in Structural Biology},
  year = {2006},
  volume = {16},
  pages = {27-278},
  doi = {http://dx.doi.org/10.1016/j.sbi.2006.05.010}
}
</pre></td>
</tr>
<tr id="matsumoto2006" class="entry">
	<td>Matsumoto, A. &amp; Olson, W. K.</td>
	<td>Effects of Sequence, Cyclization, and Superhelical Stress on the Internal Motions of DNA <p class="infolinks">[<a href="javascript:toggleInfo('matsumoto2006','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td> &nbsp;</td>
	<td>inbook</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_matsumoto2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{matsumoto2006,
  author = {Atsushi Matsumoto and Wilma K. Olson},
  title = {Effects of Sequence, Cyclization, and Superhelical Stress on the Internal Motions of DNA},
  year = {2005},
  pages = {187-211}
}
</pre></td>
</tr>
<tr id="matsumoto1_2005" class="entry">
	<td>Matsumoto, A., Tobias, I. &amp; Olson, W. K.</td>
	<td>Normal-Mode Analysis of Circular DNA at the Base-Pair Level. 1. Comparison of Computed Motions with the Predicted Behavior of an Ideal Elastic Rod <p class="infolinks">[<a href="javascript:toggleInfo('matsumoto1_2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Chemical Theory and Computation &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ct049950r">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_matsumoto1_2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{matsumoto1_2005,
  author = {Atsushi Matsumoto and Irwin Tobias and Wilma K. Olson},
  title = {Normal-Mode Analysis of Circular DNA at the Base-Pair Level. 1. Comparison of Computed Motions with the Predicted Behavior of an Ideal Elastic Rod},
  journal = {Journal of Chemical Theory and Computation},
  year = {2005},
  volume = {1},
  pages = {117-129},
  doi = {http://dx.doi.org/10.1021/ct049950r}
}
</pre></td>
</tr>
<tr id="matsumoto2_2005" class="entry">
	<td>Matsumoto, A., Tobias, I. &amp; Olson, W. K.</td>
	<td>Normal-Mode Analysis of Circular DNA at the Base-Pair Level. 2. Large-Scale Configurational Transformation of a Naturally Curved Molecule <p class="infolinks">[<a href="javascript:toggleInfo('matsumoto2_2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('matsumoto2_2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Chemical Theory and Computation &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ct049949s">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_matsumoto2_2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Fine structural and energetic details embedded in the DNA base sequence, such as intrinsic curvature, are important to the packaging and processing of the genetic material. Here we investigate the internal dynamics of a 200 bp closed circular molecule with natural curvature using a newly developed normal-mode treatment of DNA in terms of neighboring base-pair "step" parameters. The intrinsic curvature of the DNA is described by a 10 bp repeating pattern of bending distortions at successive base-pair steps. We vary the degree of intrinsic curvature and the superhelical stress on the molecule and consider the normal-mode fluctuations of both the circle and the stable figure-8 configuration under conditions where the energies of the two states are similar. To extract the properties due solely to curvature, we ignore other important features of the double helix, such as the extensibility of the chain, the anisotropy of local bending, and the coupling of step parameters. We compare the computed normal modes of the curved DNA model with the corresponding dynamical features of a covalently closed duplex of the same chain length constructed from naturally straight DNA and with the theoretically predicted dynamical properties of a naturally circular, inextensible elastic rod, i.e., an O-ring. The cyclic molecules with intrinsic curvature are found to be more deformable under superhelical stress than rings formed from naturally straight DNA. As superhelical stress is accumulated in the DNA, the frequency, i.e., energy, of the dominant bending mode decreases in value, and if the imposed stress is sufficiently large, a global configurational rearrangement of the circle to the figure-8 form takes place. We combine energy minimization with normal-mode calculations of the two states to decipher the configurational pathway between the two states. We also describe and make use of a general analytical treatment of the thermal fluctuations of an elastic rod to characterize the motions of the minicircle as a whole from knowledge of the full set of normal modes. The remarkable agreement between computed and theoretically predicted values of the average deviation and dispersion of the writhe of the circular configuration adds to the reliability in the computational approach. Application of the new formalism to the computed modes of the figure-8 provides insights into macromolecular motions which are beyond the scope of current theoretical treatments.</td>
</tr>
<tr id="bib_matsumoto2_2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{matsumoto2_2005,
  author = {Atsushi Matsumoto and Irwin Tobias and Wilma K. Olson},
  title = {Normal-Mode Analysis of Circular DNA at the Base-Pair Level. 2. Large-Scale Configurational Transformation of a Naturally Curved Molecule},
  journal = {Journal of Chemical Theory and Computation},
  year = {2005},
  volume = {1},
  pages = {130-142},
  doi = {http://dx.doi.org/10.1021/ct049949s}
}
</pre></td>
</tr>
<tr id="mccloskey.2005" class="entry">
	<td>McCloskey, J. A. &amp; Rozenski, J.</td>
	<td>The Small Subunit rRNA Modification Database <p class="infolinks">[<a href="javascript:toggleInfo('mccloskey.2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gki015">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_mccloskey.2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mccloskey.2005,
  author = {James A McCloskey and Jef Rozenski},
  title = {The Small Subunit rRNA Modification Database},
  journal = {Nucleic Acids Research},
  year = {2005},
  volume = {33},
  pages = {D135-D138},
  doi = {http://dx.doi.org/10.1093/nar/gki015}
}
</pre></td>
</tr>
<tr id="meiler2003" class="entry">
	<td>Meiler, J. &amp; Baker, D.</td>
	<td>Coupled prediction of protein secondary and tertiary structure <p class="infolinks">[<a href="javascript:toggleInfo('meiler2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('meiler2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_meiler2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The strong coupling between secondary and tertiary structure formation in protein folding is neglected in most structure prediction methods. In this work we investigate the extent to which nonlocal interactions in predicted tertiary structures can be used to improve secondary structure prediction. The architecture of a neural network for secondary structure prediction that utilizes multiple sequence alignments was extended to accept low-resolution nonlocal tertiary structure information as an additional input. By using this modified network, together with tertiary structure information from native structures, the Q3-prediction accuracy is increased by 7ï¿½10% on average and by up to 35% in individual cases for independent test data. By using tertiary structure information from models generated with the rosetta de novo tertiary structure prediction method, the Q3-prediction accuracy is improved by 4ï¿½5% on average for small and medium-sized single-domain proteins. Analysis of proteins with particularly large improvements in secondary structure prediction using tertiary structure information provides insight into the feedback from tertiary to secondary structure.</td>
</tr>
<tr id="bib_meiler2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{meiler2003,
  author = {Jens Meiler and David Baker},
  title = {Coupled prediction of protein secondary and tertiary structure},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2003},
  volume = {100},
  pages = {12105-12110}
}
</pre></td>
</tr>
<tr id="merritt1997" class="entry">
	<td>Merritt, E. A. &amp; Bacon, D. J.</td>
	<td>Raster3D: Photorealistic Molecular Graphics <p class="infolinks">[<a href="javascript:toggleInfo('merritt1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Methods in Enzymology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_merritt1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{merritt1997,
  author = {Ethan A. Merritt and David J. Bacon},
  title = {Raster3D: Photorealistic Molecular Graphics},
  journal = {Methods in Enzymology},
  year = {1997},
  volume = {277},
  pages = {505-524}
}
</pre></td>
</tr>
<tr id="mokdad2008" class="entry">
	<td>Mokdad, A. &amp; Frankel, A. D.</td>
	<td>ISFOLD: Structure Prediction of Base-pairs in Non-helical RNA Motifs From Isostericity Signatures in their Sequence Alignments <p class="infolinks">[<a href="javascript:toggleInfo('mokdad2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mokdad2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Biomolecular Structure and Dynamics &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.jbsdonline.com/c3031/c4251/ISFOLD-Structure-Prediction-of-Base-Pairs-in-Non-Helical-RNA-Motifs-from-Isostericity-Signatures-in-Their-Sequence-Alignments-p-467-472-p16422.html">URL</a> &nbsp;</td>
</tr>
<tr id="abs_mokdad2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The existence and identity of non-Watson-Crick base pairs (bps) within RNA bulges, internal loops, and hairpin loops cannot reliably be predicted by existing algorithms. We have developed the Isfold (Isosteric Folding) program as a tool to examine patterns of nucleotide substitutions from sequence alignments or mutation experiments and identify plausible bp interactions. We infer these interactions based on the observation that each non-Watson-Crick bp has a signature pattern of isosteric substitutions where mutations can be made that preserve the 3D structure. Isfold produces a dynamic representation of predicted bps within defined motifs in order of their probabilities. The software was developed under Windows XP, and is capable of running on PC and MAC with Matlab 7.1 (SP3) or higher. A PC stand-alone version that does not require Matlab also is available. This software and a user manual are freely available at www.ucsf.edu/frankel/isfold.</td>
</tr>
<tr id="bib_mokdad2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mokdad2008,
  author = {Ali Mokdad and Alan D Frankel},
  title = {ISFOLD: Structure Prediction of Base-pairs in Non-helical RNA Motifs From Isostericity Signatures in their Sequence Alignments},
  journal = {Journal of Biomolecular Structure and Dynamics},
  year = {2008},
  volume = {25},
  number = {5},
  pages = {467-472},
  url = {http://www.jbsdonline.com/c3031/c4251/ISFOLD-Structure-Prediction-of-Base-Pairs-in-Non-Helical-RNA-Motifs-from-Isostericity-Signatures-in-Their-Sequence-Alignments-p-467-472-p16422.html}
}
</pre></td>
</tr>
<tr id="mmoore2005" class="entry">
	<td>Moore, M. J.</td>
	<td>From Birth to Death: The Complex Lives of Eukaryotic mRNAs <p class="infolinks">[<a href="javascript:toggleInfo('mmoore2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mmoore2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1111443">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mmoore2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent work indicates that the posttranscriptional control of eukaryotic gene expression is much more elaborate and extensive than previously thought, with essentially every step of messenger RNA (mRNA) metabolism being subject to regulation in an mRNA-specific manner. Thus, a comprehensive understanding of eukaryotic gene expression requires an appreciation for how the lives of mRNAs are influenced by a wide array of diverse regulatory mechanisms.</td>
</tr>
<tr id="bib_mmoore2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mmoore2005,
  author = {Melissa J. Moore},
  title = {From Birth to Death: The Complex Lives of Eukaryotic mRNAs},
  journal = {Science},
  year = {2005},
  volume = {309},
  pages = {1514-1518},
  doi = {http://dx.doi.org/10.1126/science.1111443}
}
</pre></td>
</tr>
<tr id="moore2007" class="entry">
	<td>Moore, P. B.</td>
	<td>Let's call the whole thing off: some thoughts on the protein structure initiative. <p class="infolinks">[<a href="javascript:toggleInfo('moore2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.str.2007.10.006">DOI</a><a href="http://dx.doi.org/10.1016/j.str.2007.10.006">URL</a> &nbsp;</td>
</tr>
<tr id="bib_moore2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{moore2007,
  author = {Peter B Moore},
  title = {Let's call the whole thing off: some thoughts on the protein structure initiative.},
  journal = {Structure},
  year = {2007},
  volume = {15},
  number = {11},
  pages = {1350--1352},
  url = {http://dx.doi.org/10.1016/j.str.2007.10.006},
  doi = {http://dx.doi.org/10.1016/j.str.2007.10.006}
}
</pre></td>
</tr>
<tr id="moore1999" class="entry">
	<td>Moore, P. B.</td>
	<td>Structural Motifs in RNA <p class="infolinks">[<a href="javascript:toggleInfo('moore1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('moore1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Annual Review of Biochemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=5366477">URL</a> &nbsp;</td>
</tr>
<tr id="abs_moore1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An RNA motif is a discrete sequence or combination of base juxtapositions found in naturally occurring RNAs in unexpectedly high abundance. Because all the motifs examined so far have three-dimensional structures independent of the context in which they are embedded, they are important components of the kit of structural elements from which RNAs are constructed. This review discusses the structures of the motifs that have been identified so far and speculates on the importance of their role in determining RNA conformation and their evolutionary origin.</td>
</tr>
<tr id="bib_moore1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{moore1999,
  author = {P. B. Moore},
  title = {Structural Motifs in RNA},
  journal = {Annual Review of Biochemistry},
  year = {1999},
  volume = {68},
  pages = {287-300},
  url = {http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=5366477}
}
</pre></td>
</tr>
<tr id="moore1999a" class="entry">
	<td>Moore, P. B.</td>
	<td>The RNA World <p class="infolinks">[<a href="javascript:toggleInfo('moore1999a','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td> &nbsp;</td>
	<td>inbook</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_moore1999a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{moore1999a,
  author = {Peter B. Moore},
  title = {The RNA World},
  publisher = {Cold Spring Harbor Laboratory Press},
  year = {1999},
  pages = {381-401},
  edition = {2nd}
}
</pre></td>
</tr>
<tr id="moore2005" class="entry">
	<td>Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The ribosome revealed <p class="infolinks">[<a href="javascript:toggleInfo('moore2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Trends in Biochemical Sciences  &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.tibs.2005.04.006">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_moore2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{moore2005,
  author = {Peter B. Moore and Thomas A. Steitz},
  title = {The ribosome revealed},
  journal = {Trends in Biochemical Sciences },
  year = {2005},
  volume = {30},
  number = {6},
  pages = {281-283},
  doi = {http://dx.doi.org/10.1016/j.tibs.2005.04.006}
}
</pre></td>
</tr>
<tr id="moore2003" class="entry">
	<td>Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The Structural Basis of Large Ribosomal Subunit Function <p class="infolinks">[<a href="javascript:toggleInfo('moore2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('moore2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Annual Review of Biochemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.biochem.72.110601.135450">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_moore2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The ribosome crystal structures published in the past two years have revolutionized our understanding of ribonucleoprotein structure, and more specifi- cally, the structural basis of the peptide bonding forming activity of the ribosome. This review concentrates on the crystallographic developments that made it possible to solve these structures. It also discusses the information obtained from these structures about the three-dimensional architecture of the large ribosomal subunit, the mechanism by which it facilitates peptide bond formation, and the way antibiotics inhibit large subunit function. The work reviewed, taken as a whole, proves beyond doubt that the ribosome is an RNA enzyme, as had long been surmised on the basis of less conclusive evidence.</td>
</tr>
<tr id="bib_moore2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{moore2003,
  author = {Peter B. Moore and Thomas A. Steitz},
  title = {The Structural Basis of Large Ribosomal Subunit Function},
  journal = {Annual Review of Biochemistry},
  year = {2003},
  volume = {72},
  pages = {813-850},
  doi = {http://dx.doi.org/10.1146/annurev.biochem.72.110601.135450}
}
</pre></td>
</tr>
<tr id="mueller1997" class="entry">
	<td>Mueller, F. &amp; Brimacombe, R.</td>
	<td>A new model for the three-dimensional folding of Escherichia coli 16 S ribosomal RNA. I. fitting the RNA to a 3D electron microscopic map at 20 A <p class="infolinks">[<a href="javascript:toggleInfo('mueller1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mueller1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1997.1211">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mueller1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The map of the mass centres of the 21 proteins from the Escherichia coli 30 S ribosomal subunit, as determined by neutron scattering, was fitted to a cryoelectron microscopic (cryo-EM) model at a resolution of 20 ï¿½ of 70 S ribosomes in the pre-translocational state, carrying tRNA molecules at the A and P sites. The fit to the 30 S moiety of the 70 S particles was accomplished with the help of the well-known distribution of the ribosomal proteins in the head, body and side lobe regions of the 30 S subunit, as determined by immuno electron microscopy (IEM). Most of the protein mass centres were found to lie close to the surface (or even outside) of the cryo-EM contour of the 30 S subunit, supporting the idea that the ribosomal proteins are arranged peripherally around the rRNA. The ribosomal protein distribution was then compared with the corresponding model for the 16 S rRNA, fitted to the same EM contour (described in an accompanying paper), in order to analyse the mutual compatibility of the arrangement of proteins and rRNA in terms of the available RNA-protein interaction data. The information taken into account included the hydroxyl radical and base foot-printing data from Nollerï¿½s laboratory, and our own in situ cross-linking results. Proteins S1 and S14 were not considered, due to the lack of RNA-protein data. Among the 19 proteins analysed, 12 (namely S2, S4, S5, S7, S8, S9, S10, S11, S12, S15, S17 and S21) showed a fit to the rRNA model that varied from being excellent to at least acceptable. Of the remaining 7, S3 and S13 showed a rather poor fit, as did S18 (which is considered in combination with S6 in the foot-printing experiments). S16 was difficult to evaluate, as the foot-print data for this protein cover a large area of the rRNA. S19 and S20 showed a bad fit in terms of the neutron map, but their foot-print and cross-link sites were clustered into compact groups in the rRNA model in those regions of the 30 S subunit where these proteins have respectively been located by IEM studies.</td>
</tr>
<tr id="bib_mueller1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mueller1997,
  author = {Florian Mueller and Richard Brimacombe},
  title = {A new model for the three-dimensional folding of Escherichia coli 16 S ribosomal RNA. I. fitting the RNA to a 3D electron microscopic map at 20 A},
  journal = {Journal of Molecular Biology},
  year = {1997},
  volume = {271},
  pages = {524-544},
  doi = {http://dx.doi.org/10.1006/jmbi.1997.1211}
}
</pre></td>
</tr>
<tr id="mukhopadhyay2003" class="entry">
	<td>Mukhopadhyay, R., Emberly, E., Tang, C. &amp; Wingreen, N. S.</td>
	<td>Statistical mechanics of RNA folding: Importance of alphabet size <p class="infolinks">[<a href="javascript:toggleInfo('mukhopadhyay2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('mukhopadhyay2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Physical Review E. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1103/PhysRevE.68.041904">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_mukhopadhyay2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We construct a base-stacking model of RNA secondary-structure formation and use it to study the mapping from sequence to structure. There are strong, qualitative differences between two-letter and four- or six-letter alphabets. With only two kinds of bases, most sequences have many alternative folding configurations and are consequently thermally unstable. Stable ground states are found only for a small set of structures of high designability, i.e., total number of associated sequences. In contrast, sequences made from four bases, as found in nature, or six bases have far fewer competing folding configurations, resulting in a much greater average stability of the ground state.</td>
</tr>
<tr id="bib_mukhopadhyay2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{mukhopadhyay2003,
  author = {Ranjan Mukhopadhyay and Eldon Emberly and Chao Tang and Ned S. Wingreen},
  title = {Statistical mechanics of RNA folding: Importance of alphabet size},
  journal = {Physical Review E.},
  year = {2003},
  volume = {68},
  pages = {041904-1 041904-5},
  doi = {http://dx.doi.org/10.1103/PhysRevE.68.041904}
}
</pre></td>
</tr>
<tr id="murray2003" class="entry">
	<td>Murray, L. J. W., III, W. B. A., Richardson, D. C. &amp; Richardson, J. S.</td>
	<td>RNA Backbone is Rotameric <p class="infolinks">[<a href="javascript:toggleInfo('murray2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('murray2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Proceedings of the National Academy of Sciences of the United States of America &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.1835769100">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_murray2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Despite the importance of local structural detail to a mechanistic understanding of RNA catalysis and binding functions, RNA backbone conformation has been quite recalcitrant to analysis. There are too many variable torsion angles per residue, and their raw empirical distributions are poorly clustered. This study applies quality-filtering techniques (using resolution, crystallographic B factor, and all-atom steric clashes) to the backbone torsion angle distributions from an 8,636-residue RNA database. With noise levels greatly reduced, clear signal appears for the underlying angle preferences. Half-residue torsion angle distributions for ?-?-? and for ?-?-? are plotted and contoured in 3D; each shows about a dozen distinct peaks, which can then be combined in pairs to define complete RNA backbone conformers. Traditional nucleic acid residues are defined from phosphate to phosphate, but here we use a base-to-base (or sugar-to-sugar) division into ?suites? to parse the RNA backbone repeats, both because most backbone steric clashes are within suites and because the relationship of successive bases is both reliably determined and conformationally important. A suite conformer has seven variables, with sugar pucker specified at both ends. Potential suite conformers were omitted if not represented by at least a small cluster of convincing data points after application of quality filters. The final result is a small library of 42 RNA backbone conformers, which should provide valid conformations for nearly all RNA backbone encountered in experimental structures.</td>
</tr>
<tr id="bib_murray2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{murray2003,
  author = {Laura J. W. Murray and W. Bryan Arendall III and David C. Richardson and Jane S. Richardson},
  title = {RNA Backbone is Rotameric},
  journal = {Proceedings of the National Academy of Sciences of the United States of America},
  year = {2003},
  volume = {100},
  pages = {13904-13909},
  doi = {http://dx.doi.org/10.1073/pnas.1835769100}
}
</pre></td>
</tr>
<tr id="murray2005" class="entry">
	<td>Murray, L. J. W., Richardson, J. S., III, W. B. A. &amp; Richardson, D. C.</td>
	<td>RNA Backbone Rotamers - Finding your Way in Seven Dimensions <p class="infolinks">[<a href="javascript:toggleInfo('murray2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biochemical Society Transactions &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_murray2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{murray2005,
  author = {L. J. W. Murray and J. S. Richardson and W. B. Arendall III and D. C. Richardson},
  title = {RNA Backbone Rotamers - Finding your Way in Seven Dimensions},
  journal = {Biochemical Society Transactions},
  year = {2005},
  volume = {33},
  pages = {485-487}
}
</pre></td>
</tr>
<tr id="murthya1999" class="entry">
	<td>Murthya, V. L., Srinivasana, R., Draperb, D. E. &amp; Rose, G. D.</td>
	<td>A Complete Conformational Map for RNA <p class="infolinks">[<a href="javascript:toggleInfo('murthya1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('murthya1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1999.2958">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_murthya1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A simple stereochemical framework for understanding RNA structure has remained elusive to date. We present a comprehensive conformational map for two nucleoside-5?,3?-diphosphates and for a truncated dinucleotide derived from a grid search of all potential conformers using hard sphere steric exclusion criteria to define allowed conformers. The eight-dimensional conformational space is presented as a series of two-dimensional projections. These projections reveal several well-defined allowed and disallowed regions which correlate well with data obtained from X-ray crystallography of both large and small RNA molecules. Furthermore, the two-dimensional projections show that consecutive and ribose ring-proximal torsion angles are interdependent, while more distant torsion angles are not. Remarkably, using steric criteria alone, it is possible to generate a predictive conformational map for RNA.</td>
</tr>
<tr id="bib_murthya1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{murthya1999,
  author = {Venkatesh L. Murthya and Rajgopal Srinivasana and David E. Draperb and George D. Rose},
  title = {A Complete Conformational Map for RNA},
  journal = {Journal of Molecular Biology},
  year = {1999},
  volume = {291},
  number = {2},
  pages = {313-327},
  doi = {http://dx.doi.org/10.1006/jmbi.1999.2958}
}
</pre></td>
</tr>
<tr id="nasalean2009" class="entry">
	<td>Nasalean, L., Stombaugh, J., Zirbel, C. L. &amp; Leontis, N. B.</td>
	<td>RNA 3D Structural Motifs: Definition, Identification, Annotation, and Database Searching <p class="infolinks">[<a href="javascript:toggleInfo('nasalean2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('nasalean2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td> &nbsp;</td>
	<td>inbook</td>
	<td><a href="http://dx.doi.org/10.1007/978-3-540-70840-7">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_nasalean2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Structured RNA molecules resemble proteins in the hierarchical organization of their global structures, folding and broad range of functions. Structured RNAs are composed of recurrent modular motifs that play specific functional roles. Some motifs direct the folding of the RNA or stabilize the folded structure through tertiary interactions. Others bind ligands or proteins or catalyze chemical reactions. Therefore, it is desirable, starting from the RNA sequence, to be able to predict the locations of recurrent motifs in RNA molecules. Conversely, the potential occurrence of one or more known 3D RNA motifs may indicate that a genomic sequence codes for a structured RNA molecule. To identify known RNA structural motifs in new RNA sequences, precise structure-based definitions are needed that specify the core nucleotides of each motif and their conserved interactions. By comparing instances of each recurrent motif and applying base pair isostericity relations, one can identify neutral mutations that preserve its structure and function in the contexts in which it occurs.</td>
</tr>
<tr id="bib_nasalean2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{nasalean2009,
  author = {Lorena Nasalean and Jesse Stombaugh and Craig L. Zirbel and Neocles B. Leontis},
  title = {RNA 3D Structural Motifs: Definition, Identification, Annotation, and Database Searching},
  publisher = {Springer-Verlag Berlin Heidelberg},
  year = {2009},
  volume = {13},
  pages = {1-26},
  doi = {http://dx.doi.org/10.1007/978-3-540-70840-7}
}
</pre></td>
</tr>
<tr id="nino2001" class="entry">
	<td>Nino, M., Daza, E. E. &amp; Tello, M.</td>
	<td>A Criteria to Classify Biological Activity of Benzimidazoles from a Model of Structural Similarity <p class="infolinks">[<a href="javascript:toggleInfo('nino2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('nino2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>J. Chem. Inf. Comput. Sci. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci000072+">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_nino2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have classified a set of 250 benzimidazoles using a criterion of structural similarity. This criterion has led us to several clusters, which keep a close relationship between the molecules belonging to each one of them and their pharmacological activity. To study the structural similarity we have built a mathematical space where chemical structures are pictured as vectors. A set of well-chosen descriptors was used as variables. These descriptors arise from graph theoretical studies and quantum mechanical calculations. Principal components analysis was employed to find the suitable dimension for the space. Finally, cluster analysis was performed to classify the set of molecules by similarity. A Euclidean metric was used as a similarity coefficient.</td>
</tr>
<tr id="bib_nino2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{nino2001,
  author = {Marcela Nino and Edgar Eduardo Daza and Myriam Tello},
  title = {A Criteria to Classify Biological Activity of Benzimidazoles from a Model of Structural Similarity},
  journal = {J. Chem. Inf. Comput. Sci.},
  year = {2001},
  volume = {41},
  pages = {495-504},
  doi = {http://dx.doi.org/10.1021/ci000072+}
}
</pre></td>
</tr>
<tr id="nissen2000" class="entry">
	<td>Nissen, P., Nissen, J., Ban, N., Moore, P. B. &amp; Steitz, T. A.</td>
	<td>The Structural Basis of Ribosome Activity in Peptide Bond Synthesis <p class="infolinks">[<a href="javascript:toggleInfo('nissen2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('nissen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.289.5481.920">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_nissen2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Using the atomic structures of the large ribosomal subunit from Haloarcula marismortui and its complexes with two substrate analogs, we establish that the ribosome is a ribozyme and address the catalytic properties of its all-RNA active site. Both substrate analogs are contacted exclusively by conserved ribosomal RNA (rRNA) residues from domain V of 23S rRNA; there are no protein side-chain atoms closer than about 18 angstroms to the peptide bond being synthesized. The mechanism of peptide bond synthesis appears to resemble the reverse of the acylation step in serine proteases, with the base of A2486 (A2451 in Escherichia coli) playing the same general base role as histidine-57 in chymotrypsin. The unusual pKa (where Ka is the acid dissociation constant) required for A2486 to perform this function may derive in part from its hydrogen bonding to G2482 (G2447 in E. coli), which also interacts with a buried phosphate that could stabilize unusual tautomers of these two bases. The polypeptide exit tunnel is largely formed by RNA but has significant contributions from proteins L4, L22, and L39e, and its exit is encircled by proteins L19, L22, L23, L24, L29, and L31e</td>
</tr>
<tr id="bib_nissen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{nissen2000,
  author = {Poul Nissen and J. Nissen and Nenad Ban and Peter B. Moore and Thomas A. Steitz },
  title = {The Structural Basis of Ribosome Activity in Peptide Bond Synthesis},
  journal = {Science},
  year = {2000},
  volume = {289},
  pages = {920-930},
  doi = {http://dx.doi.org/10.1126/science.289.5481.920}
}
</pre></td>
</tr>
<tr id="noller2005" class="entry">
	<td>Noller, H. F.</td>
	<td>RNA Structure: Reading the Ribosome <p class="infolinks">[<a href="javascript:toggleInfo('noller2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('noller2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1111771">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_noller2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structures of the ribosome and its subunits have increased the amount of information about RNA structure by about two orders of magnitude. This is leading to an understanding of the principles of RNA folding and of the molecular interactions that underlie the functional capabilities of the ribosome and other RNA systems. Nearly all of the possible types of RNA tertiary interactions have been found in ribosomal RNA. One of these, an abundant tertiary structural motif called the A-minor interaction, has been shown to participate in both aminoacyl-transfer RNA selection and in peptidyl transferase; it may also play an important role in the structural dynamics of the ribosome.</td>
</tr>
<tr id="bib_noller2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{noller2005,
  author = {Harry F. Noller},
  title = {RNA Structure: Reading the Ribosome},
  journal = {Science},
  year = {2005},
  volume = {309},
  pages = {1508-1514},
  doi = {http://dx.doi.org/10.1126/science.1111771}
}
</pre></td>
</tr>
<tr id="noller1981" class="entry">
	<td>Noller, H. F., Kop, J., Wheaton, V., Brosius, J., Gutell, R. R., Kopylov, A. M., Dohme, F., Herr, W., Stahl, D. A., Gupta, R. &amp; Woese, C. R.</td>
	<td>Secondary structure model for 23S ribosomal RNA <p class="infolinks">[<a href="javascript:toggleInfo('noller1981','bibtex')">BibTeX</a>]</p></td>
	<td>1981</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/9.22.6167">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_noller1981" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{noller1981,
  author = {Harry F. Noller and JoAnn Kop and Virginia Wheaton and Jurgen Brosius and Robin R. Gutell and Alexei M. Kopylov and Ferdinand Dohme and Winship Herr and David A. Stahl and Ramesh Gupta and Carl R. Woese},
  title = {Secondary structure model for 23S ribosomal RNA},
  journal = {Nucleic Acids Research},
  year = {1981},
  volume = {22},
  pages = {6167-6189},
  doi = {http://dx.doi.org/10.1093/nar/9.22.6167}
}
</pre></td>
</tr>
<tr id="nowakowski2000" class="entry">
	<td>Nowakowski, J., Shim, P. J., Stout, C. D. &amp; Joyce, G. F.</td>
	<td>Alternative conformations of a nucleic acid four-way junction <p class="infolinks">[<a href="javascript:toggleInfo('nowakowski2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('nowakowski2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.2000.3826">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_nowakowski2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A crystal structure of a 108 nucleotide RNA-DNA complex containing a four-way junction was solved at 3.1 ï¿½ resolution. The structure of the junction differs substantially from the "stacked-X" conformation observed previously, due to a 135ï¿½ rotation of the branches. Comparison of the two conformers provides insight into the factors contributing to the flexibility of four-way junctions. The stacked-X conformation maximizes base-stacking but causes unfavorable repulsion between phosphate groups, whereas the 135ï¿½-rotated "crossed" conformation minimizes electrostatic clashes at the expense of reduced base-stacking. Despite the large rotation of the branches, both junction structures exhibit an antiparallel arrangement of the continuous strands and opposite polarity of the crossover strands.</td>
</tr>
<tr id="bib_nowakowski2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{nowakowski2000,
  author = {Jacek Nowakowski and Peter J. Shim and C. David Stout and Gerald F. Joyce},
  title = {Alternative conformations of a nucleic acid four-way junction},
  journal = {Journal of Molecular Biology},
  year = {2000},
  volume = {300},
  pages = {93-102},
  doi = {http://dx.doi.org/10.1006/jmbi.2000.3826}
}
</pre></td>
</tr>
<tr id="oliva2006" class="entry">
	<td>Oliva, R., Cavallo, L. &amp; Tramontano, A.</td>
	<td>Accurate energies of hydrogen bonded nucleic acids base pairs and triplets in tRNA tertiary interactions <p class="infolinks">[<a href="javascript:toggleInfo('oliva2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('oliva2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkj491">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_oliva2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Tertiary interactions are crucial in maintaining the tRNA structure and functionality. We used a combined sequence analysis and quantum mechanics approach to calculate accurate energies of the most frequent tRNA tertiary base pairing interactions. Our analysis indicates that six out of the nine classical tertiary interactions are held in place mainly by H-bonds between the bases. In the remaining three cases other effects have to be considered. Tertiary base pairing interaction energies range from ï¿½8 to ï¿½38 kcal/mol in yeast tRNAPhe and are estimated to contribute roughly 25% of the overall tRNA base pairing interaction energy. Six analyzed posttranslational chemical modifications were shown to have minor effect on the geometry of the tertiary interactions. Modifications that introduce a positive charge strongly stabilize the corresponding tertiary interactions. Non-additive effects contribute to the stability of base triplets.</td>
</tr>
<tr id="bib_oliva2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{oliva2006,
  author = {Romina Oliva and Luigi Cavallo and Anna Tramontano},
  title = {Accurate energies of hydrogen bonded nucleic acids base pairs and triplets in tRNA tertiary interactions},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {865-879},
  doi = {http://dx.doi.org/10.1093/nar/gkj491}
}
</pre></td>
</tr>
<tr id="olson1996" class="entry">
	<td>Olson, W. K.</td>
	<td>Simulating DNA at low resolution <p class="infolinks">[<a href="javascript:toggleInfo('olson1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(96)80082-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_olson1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The past year has witnessed the development of several new mathematical approaches to analyzing the structure of double-helical DNA and to incorporating the sequence-dependent features of the chain in computer simulations of long polymers. Of special interest in this respect are the local and global structural changes induced by the binding of various proteins to DNA, ranging from subtle bending, untwisting and sliding motions at the base-pair level to the apparent organization of supercoiled structure in chains that are thousands of residues long. The computational effort has also included both new ways to incorporate the polyelectrolyte character of DNA and other environmental forces in simulations of long chains and new methods to keep track of the multitude of configurations so generated. The collective advances are pointing to ways that will soon connect the sequences of base pairs in large genomes to folded three-dimensional structures based on natural bending, twisting and translational tendencies and in response to deformations produced by the binding of different proteins.</td>
</tr>
<tr id="bib_olson1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson1996,
  author = {Wilma K. Olson},
  title = {Simulating DNA at low resolution},
  journal = {Current Opinion in Structural Biology},
  year = {1996},
  volume = {6},
  pages = {242-256},
  doi = {http://dx.doi.org/10.1016/S0959-440X(96)80082-0}
}
</pre></td>
</tr>
<tr id="olson1976" class="entry">
	<td>Olson, W. K.</td>
	<td>The spatial configuration of ordered polynucleotide chains. I. Helix formation and base stacking. <p class="infolinks">[<a href="javascript:toggleInfo('olson1976','bibtex')">BibTeX</a>]</p></td>
	<td>1976</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.1976.360150505">DOI</a><a href="http://dx.doi.org/10.1002/bip.1976.360150505">URL</a> &nbsp;</td>
</tr>
<tr id="bib_olson1976" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson1976,
  author = {W. K. Olson},
  title = {The spatial configuration of ordered polynucleotide chains. I. Helix formation and base stacking.},
  journal = {Biopolymers},
  year = {1976},
  volume = {15},
  number = {5},
  pages = {859--878},
  url = {http://dx.doi.org/10.1002/bip.1976.360150505},
  doi = {http://dx.doi.org/10.1002/bip.1976.360150505}
}
</pre></td>
</tr>
<tr id="olson2001" class="entry">
	<td>Olson, W. K., Bansal, M., Burley, S. K., Dickerson, R. E., Gerstein, M., Harvey, S. C., Heinemann, U., Lu, X., Neidle, S., Shakked, Z., Sklenar, H., Suzuki, M., Tung, C., Westhof, E., Wolberger, C. &amp; Berman, H. M.</td>
	<td>A Standard Reference Frame for the Description of Nucleic Acid Base-pair Geometry <p class="infolinks">[<a href="javascript:toggleInfo('olson2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.2001.4987">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_olson2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A common point of reference is needed to describe the three-dimensional arrangements of bases and base-pairs in nucleic acid structures. The different standards used in computer programs created for this purpose give rise to conflicting interpretations of the same structure.[1] For example, parts of a structure that appear "normal" according to one computational scheme may be highly unusual according to another and vice versa. It is thus difficult to carry out comprehensive comparisons of nucleic acid structures and to pinpoint unique conformational features in individual structures. In order to resolve these issues, a group of researchers who create and use the different software packages have proposed the standard base reference frames outlined below for nucleic acid conformational analysis. The definitions build upon qualitative guidelines established previously to specify the arrangements of bases and base-pairs in DNA and RNA structures.[2] Base coordinates are derived from a survey of high-resolution crystal structures of nucleic acid analogs stored in the Cambridge Structural Database (CSD). [3] The coordinate frames are chosen so that complementary bases form an ideal, planar Watson-Crick base-pair in the undistorted reference state with hydrogen bond donor-acceptor distances, C1 ??C1? virtual lengths, and purine N9-C1??C1? and pyrimidine N1-C1??C1? virtual angles consistent with values observed in the crystal structures of relevant small molecules. Conformational analyses performed in this reference frame lead to interpretations of local helical structure that are essentially independent of computational scheme. A compilation of base-pair parameters from representative A-DNA, B-DNA, and protein-bound DNA structures from the Nucleic Acid Database (NDB)[4] provides useful guidelines for understanding other nucleic acid structures.</td>
</tr>
<tr id="bib_olson2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson2001,
  author = {Wilma K. Olson and Manju Bansal and Stephen K. Burley and Richard E. Dickerson and Mark Gerstein and Stephen C. Harvey and Udo Heinemann and Xiang-Jun Lu and Stephen Neidle and Zippora Shakked and Heinz Sklenar and Masashi Suzuki and Chang-Shung Tung and Eric Westhof and Cynthia Wolberger and Helen M. Berman},
  title = {A Standard Reference Frame for the Description of Nucleic Acid Base-pair Geometry},
  journal = {Journal of Molecular Biology},
  year = {2001},
  volume = {313},
  pages = {229-237},
  doi = {http://dx.doi.org/10.1006/jmbi.2001.4987}
}
</pre></td>
</tr>
<tr id="olson2006" class="entry">
	<td>Olson, W. K., Colasanti, A. V., Li, Y., Ge, W., Zheng, G. &amp; Zhurkin, V. B.</td>
	<td>DNA Simulation Benchmarks as Revealed by X-Ray Structures <p class="infolinks">[<a href="javascript:toggleInfo('olson2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_olson2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The inferences that can be drawn from known DNA structures provide new<p>stimuli for improvement of nucleic acid force fields and fresh ideas for<p>exploration of the sequence-dependent properties of DNA. The rapidly<p>growing database of high-resolution nucleic acid crystal structures reveals long<p>anticipated sequence-dependent variability in DNA backbone conformation.<p>Nucleotides in specific sequence contexts exhibit decided tendencies to<p>undergo changes of rotational state that are associated with large-scale helical<p>transitions. In particular, the sugars attached to cytosine exhibit a clear-cut<p>tendency to adopt A-like conformations. Overall, however, the large set of<p>protein-bound DNA structures remains very close to the classical B form. This<p>distinguishes the crystallographically observed DNA duplexes from computer-<p>generated atomic-level DNA models, which are characterized by a systematic<p>shift toward the A form. In addition, the base-pair steps found in different<p>protein-DNA complexes span a narrower range of conformational states than<p>those generated with state-of-the-art molecular simulations. The sequence-<p>dependent positioning of water and the build-up of amino acid residues around<p>the DNA bases point to mechanisms which may underlie the sequence-<p>dependent structure and deformability of DNA in complexes with ligands.</td>
</tr>
<tr id="bib_olson2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{olson2006,
  author = {Wilma K. Olson and Andrew V. Colasanti and Yun Li and Wei Ge and Guohui Zheng and Victor B. Zhurkin},
  title = {DNA Simulation Benchmarks as Revealed by X-Ray Structures},
  publisher = {Springer},
  year = {2006}
}
</pre></td>
</tr>
<tr id="olson2009" class="entry">
	<td>Olson, W. K., Esguerra, M., Xin, Y. &amp; Lu, X.</td>
	<td>New information content in RNA base pairing deduced from quantitative analysis of high-resolution structures. <p class="infolinks">[<a href="javascript:toggleInfo('olson2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Methods &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.ymeth.2008.12.003">DOI</a><a href="http://dx.doi.org/10.1016/j.ymeth.2008.12.003">URL</a> &nbsp;</td>
</tr>
<tr id="abs_olson2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Non-canonical base pairs play important roles in organizing the complex three-dimensional folding of RNA. Here, we outline methodology developed both to analyze the spatial patterns of interacting base pairs in known RNA structures and to reconstruct models from the collective experimental information. We focus attention on the structural context and deformability of the seven pairing patterns found in greatest abundance in the helical segments in a set of well-resolved crystal structures, including (i-ii) the canonical A.U and G.C Watson-Crick base pairs, (iii) the G.U wobble pair, (iv) the sheared G.A pair, (v) the A.U Hoogsteen pair, (vi) the U.U wobble pair, and (vii) the G.A Watson-Crick-like pair. The non-canonical pairs stand out from the canonical associations in terms of apparent deformability, spanning a broader range of conformational states as measured by the six rigid-body parameters used to describe the spatial arrangements of the interacting bases, the root-mean-square deviations of the base-pair atoms, and the fluctuations in hydrogen-bonding geometry. The deformabilties, the modes of base-pair deformation, and the preferred sites of occurrence depend on sequence. We also characterize the positioning and overlap of the base pairs with respect to the base pairs that stack immediately above and below them in double-helical fragments. We incorporate the observed positions of the bases, base pairs, and intervening phosphorus atoms in models to predict the effects of the non-canonical interactions on overall helical structure.</td>
</tr>
<tr id="bib_olson2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson2009,
  author = {Wilma K Olson and Mauricio Esguerra and Yurong Xin and Xiang-Jun Lu},
  title = {New information content in RNA base pairing deduced from quantitative analysis of high-resolution structures.},
  journal = {Methods},
  year = {2009},
  volume = {47},
  number = {3},
  pages = {177--186},
  url = {http://dx.doi.org/10.1016/j.ymeth.2008.12.003},
  doi = {http://dx.doi.org/10.1016/j.ymeth.2008.12.003}
}
</pre></td>
</tr>
<tr id="olson1_1972" class="entry">
	<td>Olson, W. K. &amp; Flory, P. J.</td>
	<td>Spatial Configurations of Polynucleotide Chains. I. Steric Interactions in Polyribonucleotides: A Virtual Bond Model <p class="infolinks">[<a href="javascript:toggleInfo('olson1_1972','bibtex')">BibTeX</a>]</p></td>
	<td>1972</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.1972.360110102">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_olson1_1972" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson1_1972,
  author = {W. K. Olson and P. J. Flory},
  title = {Spatial Configurations of Polynucleotide Chains. I. Steric Interactions in Polyribonucleotides: A Virtual Bond Model},
  journal = {Biopolymers},
  year = {1972},
  volume = {11},
  pages = {1-23},
  doi = {http://dx.doi.org/10.1002/bip.1972.360110102}
}
</pre></td>
</tr>
<tr id="olson2_1972" class="entry">
	<td>Olson, W. K. &amp; Flory, P. J.</td>
	<td>Spatial Configuration of Polynucleotide Chains. II. Conformational Energies and the Average Dimensions of Polyribonucleotides. <p class="infolinks">[<a href="javascript:toggleInfo('olson2_1972','bibtex')">BibTeX</a>]</p></td>
	<td>1972</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.1972.360110103">DOI</a><a href="http://dx.doi.org/10.1002/bip.1972.360110103">URL</a> &nbsp;</td>
</tr>
<tr id="bib_olson2_1972" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson2_1972,
  author = {W. K. Olson and P. J. Flory},
  title = {Spatial Configuration of Polynucleotide Chains. II. Conformational Energies and the Average Dimensions of Polyribonucleotides.},
  journal = {Biopolymers},
  year = {1972},
  volume = {11},
  number = {1},
  pages = {25-56},
  url = {http://dx.doi.org/10.1002/bip.1972.360110103},
  doi = {http://dx.doi.org/10.1002/bip.1972.360110103}
}
</pre></td>
</tr>
<tr id="olson3_1972" class="entry">
	<td>Olson, W. K. &amp; Flory, P. J.</td>
	<td>Spatial Configurations of Polynucleotide Chains. III. Polydeoxyribonucleotides. <p class="infolinks">[<a href="javascript:toggleInfo('olson3_1972','bibtex')">BibTeX</a>]</p></td>
	<td>1972</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.1972.360110104">DOI</a><a href="http://dx.doi.org/10.1002/bip.1972.360110104">URL</a> &nbsp;</td>
</tr>
<tr id="bib_olson3_1972" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson3_1972,
  author = {W. K. Olson and P. J. Flory},
  title = {Spatial Configurations of Polynucleotide Chains. III. Polydeoxyribonucleotides.},
  journal = {Biopolymers},
  year = {1972},
  volume = {11},
  number = {1},
  pages = {57-66},
  url = {http://dx.doi.org/10.1002/bip.1972.360110104},
  doi = {http://dx.doi.org/10.1002/bip.1972.360110104}
}
</pre></td>
</tr>
<tr id="olson1998" class="entry">
	<td>Olson, W. K., Gorin, A. A., Lu, X., Hock, L. M. &amp; Zhurkin, V. B.</td>
	<td>DNA sequence-dependent deformability deduced from protein-DNA crystal complexes <p class="infolinks">[<a href="javascript:toggleInfo('olson1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Proceedings of the National Academy of Sciences &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.pnas.org/cgi/content/abstract/95/19/1116">URL</a> &nbsp;</td>
</tr>
<tr id="abs_olson1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The deformability of double helical DNA is critical for its packaging in the cell, recognition by other molecules, and transient opening during biochemically important processes. Here, a complete set of sequence-dependent empirical energy functions suitable for describing such behavior is extracted from the fluctuations and correlations of structural parameters in DNA-protein crystal complexes. These elastic functions provide useful stereochemical measures of the local base step movements operative in sequence-specific recognition and protein-induced deformations. In particular, the pyrimidine-purine dimers stand out as the most variable steps in the DNA-protein complexes, apparently acting as flexible "hinges" fitting the duplex to the protein surface. In addition to the angular parameters widely used to describe DNA deformations (i.e., the bend and twist angles), the translational parameters describing the displacements of base pairs along and across the helical axis are analyzed. The observed correlations of base pair bending and shearing motions are important for nonplanar folding of DNA in nucleosomes and other nucleoprotein complexes. The knowledge-based energies also offer realistic three-dimensional models for the study of long DNA polymers at the global level, incorporating structural features beyond the scope of conventional elastic rod treatments and adding a new dimension to literal analyses of genomic sequences.</td>
</tr>
<tr id="bib_olson1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson1998,
  author = {Wilma K. Olson and Andrey A. Gorin and Xiang-Jun Lu and Lynette M. Hock and Victor B. Zhurkin},
  title = {DNA sequence-dependent deformability deduced from protein-DNA crystal complexes},
  journal = {Proceedings of the National Academy of Sciences},
  year = {1998},
  volume = {95},
  pages = {11163-11168},
  url = {http://www.pnas.org/cgi/content/abstract/95/19/1116}
}
</pre></td>
</tr>
<tr id="olson2000" class="entry">
	<td>Olson, W. K. &amp; Zhurkin, V. B.</td>
	<td>Modeling DNA deformations <p class="infolinks">[<a href="javascript:toggleInfo('olson2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('olson2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(00)00086-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_olson2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent developments have been made in modeling double-helical DNA at four levels of three-dimensional structure: the all-atom level, whereby an oligonucleotide duplex is surrounded by a shroud of solvent molecules; the base-pair level, with explicit backbone atoms; the mesoscopic level, that is, a few hundred base pairs, with the local duplex conformation described by knowledge-based harmonic energy functions; and the scale of several thousand nucleotides, with the duplex described as an ideal elastic rod. Predictions of the sequence-dependent bending and twisting of the double helix, as well as solvent- and force-induced B?A and over-stretching conformational transitions, are compared with experimental data. These subtle conformational changes are critical to the functioning of the double helix, including its packaging in the close confines of the cell, the mutual fit of DNA and protein in nucleoprotein complexes, and the effective recognition of base pairs in recombination and transcription.</td>
</tr>
<tr id="bib_olson2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{olson2000,
  author = {Wilma K. Olson and Victor B. Zhurkin},
  title = {Modeling DNA deformations},
  journal = {Current Opinion in Structural Biology},
  year = {2000},
  volume = {10},
  pages = {286-297},
  doi = {http://dx.doi.org/10.1016/S0959-440X(00)00086-5}
}
</pre></td>
</tr>
<tr id="onoa2004" class="entry">
	<td>Onoa, B. &amp; Jr., I. T.</td>
	<td>RNA folding and unfolding <p class="infolinks">[<a href="javascript:toggleInfo('onoa2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('onoa2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2004.04.001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_onoa2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Single-molecule studies of RNA folding and unfolding are providing impressive details of the intermediates that occur and their rates of interconversion. The folding and unfolding of RNA are controlled by varying the concentration of magnesium ions and measuring fluorescence energy transfer, or by applying force to the RNA and measuring the end-to-end distance. The hierarchical nature of RNA folding ? first secondary structure, then tertiary structure ? makes the process susceptible to analysis and prediction.</td>
</tr>
<tr id="bib_onoa2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{onoa2004,
  author = {Bibiana Onoa and Ignacio Tinoco Jr.},
  title = {RNA folding and unfolding},
  journal = {Current Opinion in Structural Biology},
  year = {2004},
  volume = {14},
  number = {3},
  pages = {374-379},
  doi = {http://dx.doi.org/10.1016/j.sbi.2004.04.001}
}
</pre></td>
</tr>
<tr id="orgel2004" class="entry">
	<td>Orgel, L. E.</td>
	<td>Prebiotic Chemistry and the Origin of the RNA World <p class="infolinks">[<a href="javascript:toggleInfo('orgel2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Critical Reviews in Biochemistry and Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=14095641">URL</a> &nbsp;</td>
</tr>
<tr id="bib_orgel2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{orgel2004,
  author = {Leslie E. Orgel},
  title = {Prebiotic Chemistry and the Origin of the RNA World},
  journal = {Critical Reviews in Biochemistry and Molecular Biology},
  year = {2004},
  volume = {39},
  pages = {99-123},
  url = {http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=14095641}
}
</pre></td>
</tr>
<tr id="orland2002" class="entry">
	<td>Orland, H. &amp; Zee, A.</td>
	<td>RNA folding and large N matrix theory <p class="infolinks">[<a href="javascript:toggleInfo('orland2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('orland2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Nuclear Physics B &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0550-3213(01)00522-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_orland2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We formulate the RNA folding problem as an Nï¿½N matrix field theory. This matrix formalism allows us to give a systematic classification of the terms in the partition function according to their topological character. The theory is set up in such a way that the limit N?? yields the so-called secondary structure (Hartree theory). Tertiary structure and pseudo-knots are obtained by calculating the 1/N2 corrections to the partition function. We propose a generalization of the Hartree recursion relation to generate the tertiary structure.</td>
</tr>
<tr id="bib_orland2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{orland2002,
  author = {Henri Orland and A. Zee},
  title = {RNA folding and large N matrix theory},
  journal = {Nuclear Physics B},
  year = {2002},
  volume = {620},
  pages = {456-476},
  doi = {http://dx.doi.org/10.1016/S0550-3213(01)00522-3}
}
</pre></td>
</tr>
<tr id="ouyang2009" class="entry">
	<td>Ouyang, D., Zhang, H., Herten, D., Parekh, H. S. &amp; Smith, S. C.</td>
	<td>Flexibility of Short-Strand RNA in Aqueous Solution as Revealed by Molecular Dynamics Simulation: Are A-RNA and A'-RNA Distinct Conformational Structures? <p class="infolinks">[<a href="javascript:toggleInfo('ouyang2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Australian Journal of Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1071/CH09090">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_ouyang2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ouyang2009,
  author = {D. Ouyang and H. Zhang and D. Herten and H. S. Parekh and S. C. Smith},
  title = {Flexibility of Short-Strand RNA in Aqueous Solution as Revealed by Molecular Dynamics Simulation: Are A-RNA and A'-RNA Distinct Conformational Structures?},
  journal = {Australian Journal of Chemistry},
  year = {2009},
  volume = {62},
  pages = {1054-1061},
  doi = {http://dx.doi.org/10.1071/CH09090}
}
</pre></td>
</tr>
<tr id="packer1998" class="entry">
	<td>Packer, M. J. &amp; Hunter, C. A.</td>
	<td>Sequence-Dependent DNA Structure: The Role of the Sugar-Phosphate Backbone <p class="infolinks">[<a href="javascript:toggleInfo('packer1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('packer1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1998.1865">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_packer1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A detailed analysis of the coupling between the conformational properties of the sugar-phosphate backbone and the base stacking interactions in dinucleotide steps of double helical DNA is described. In X-ray crystal structures of oligonucleotides, the backbone shows one major degree of freedom, consisting of the torsion angles , , and the pseudorotation phase angle, P. The remaining torsion angles (, , and ) comprise two less important degrees of freedom. The base stacking interactions show three degrees of freedom: slide-roll-twist, shift-tilt, and rise (which is more or less constant). Coupling is observed between the base and backbone degrees of freedom. The major base stacking mode, slide-roll-twist, is coupled to the major backbone mode, -P--. The secondary base stacking mode, shift-tilt, is coupled to and and to a lesser extent to the -P-- mode. We show that the length of the backbone, C, given by the same strand C1?-C1? separation, is an excellent single parameter descriptor for the conformation of the backbone and the way in which it is coupled to the base stacking geometry. The slide-roll-twist motion relates to changes in the mean backbone length, , and the shift-tilt motion to the difference between the lengths of the two backbone strands, ?C. We use this observation to develop a simple virtual bond model which describes the coupling of the backbone conformations and the base stacking geometry. A semi-flexible bond is used to connect the same strand C1?-C1? atoms. Analysis of the X-ray crystal structure database, simple geometric considerations and model building experiments all show that this bond is flexible with respect to slide, shift and propeller but rigid with respect to the other 14 local base stacking parameters. Using this simple model for the backbone in conjunction with potential energy calculations of the base stacking interactions, we show that it is possible to predict accurately the values of these 14 base step parameters, given values of slide, shift and propeller. We also show that the base step parameters fall into three distinct groups: roll, tilt and rise are determined solely by the base stacking interactions and are independent of the backbone; twist is insensitive to the base stacking interactions and is determined solely by the constraints of a relatively rigid fixed length backbone; slide and shift are the primary degrees of freedom and cannot be predicted accurately at the dinucleotide level because they are influenced by the conformations of neighbouring steps in a sequence. We have found that the context effect on slide is mediated by the torsion angles while the context effect on shift results from a BI to BII transition in the backbone. We have therefore reduced the dimensionality of the dinucleotide step problem to two parameters, slide and shift.</td>
</tr>
<tr id="bib_packer1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{packer1998,
  author = {Martin J. Packer and Christopher A. Hunter},
  title = {Sequence-Dependent DNA Structure: The Role of the Sugar-Phosphate Backbone},
  journal = {Journal of Molecular Biology},
  year = {1998},
  volume = {280},
  number = {3},
  pages = {407-420},
  doi = {http://dx.doi.org/10.1006/jmbi.1998.1865}
}
</pre></td>
</tr>
<tr id="pan1997" class="entry">
	<td>Pan, J., Thirumalai, D. &amp; Woodson, S. A.</td>
	<td>Folding of RNA Involves Parallel Pathways <p class="infolinks">[<a href="javascript:toggleInfo('pan1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_pan1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{pan1997,
  author = {Jie Pan and D. Thirumalai and Sarah A. Woodson},
  title = {Folding of RNA Involves Parallel Pathways},
  journal = {Journal of Molecular Biology},
  year = {1997},
  volume = {273},
  pages = {7-13}
}
</pre></td>
</tr>
<tr id="pan2006" class="entry">
	<td>Pan, T. &amp; Sosnick, T.</td>
	<td>RNA Folding During Transcription <p class="infolinks">[<a href="javascript:toggleInfo('pan2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('pan2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Annual Reviews of Biophysics and Biomolecular Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.biophys.35.040405.102053">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_pan2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The evolution of RNA sequence needs to satisfy three requirements: folding, structure, and function. Studies on folding during transcription are related directly to folding in the cell. Understanding RNA folding during transcription requires the elucidation of structure formation and structural changes of the RNA, and the consideration of intrinsic properties of the RNA polymerase and other proteins that interact with the RNA. This review summarizes the research progress in this area and outlines the enormous challenges facing this field. Significant advancement requires the development of new experimental methods and theoretical considerations in all aspects of transcription and RNA folding.</td>
</tr>
<tr id="bib_pan2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{pan2006,
  author = {Tao Pan and Tobin Sosnick},
  title = {RNA Folding During Transcription},
  journal = {Annual Reviews of Biophysics and Biomolecular Structure},
  year = {2006},
  volume = {35},
  pages = {161-175},
  doi = {http://dx.doi.org/10.1146/annurev.biophys.35.040405.102053}
}
</pre></td>
</tr>
<tr id="parisien2009" class="entry">
	<td>Parisien, M., Cruz, J. A., Westhof, E. &amp; Major, F.</td>
	<td>New Metrics for Comparing and Assessing Discrepancies Between RNA 3D Structures and Models. <p class="infolinks">[<a href="javascript:toggleInfo('parisien2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('parisien2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.1700409">DOI</a><a href="http://dx.doi.org/10.1261/rna.1700409">URL</a> &nbsp;</td>
</tr>
<tr id="abs_parisien2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: To benchmark progress made in RNA three-dimensional modeling and assess newly developed techniques, reliable and meaningful comparison metrics and associated tools are necessary. Generally, the average root-mean-square deviations (RMSDs) are quoted. However, RMSD can be misleading since errors are spread over the whole molecule and do not account for the specificity of RNA base interactions. Here, we introduce two new metrics that are particularly suitable to RNAs: the deformation index and deformation profile. The deformation index is calibrated by the interaction network fidelity, which considers base-base-stacking and base-base-pairing interactions within the target structure. The deformation profile highlights dissimilarities between structures at the nucleotide scale for both intradomain and interdomain interactions. Our results show that there is little correlation between RMSD and interaction network fidelity. The deformation profile is a tool that allows for rapid assessment of the origins of discrepancies.</td>
</tr>
<tr id="bib_parisien2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{parisien2009,
  author = {Marc Parisien and Jose Almeida Cruz and Eric Westhof and Francois Major},
  title = {New Metrics for Comparing and Assessing Discrepancies Between RNA 3D Structures and Models.},
  journal = {RNA},
  year = {2009},
  volume = {15},
  number = {10},
  pages = {1875-1885},
  url = {http://dx.doi.org/10.1261/rna.1700409},
  doi = {http://dx.doi.org/10.1261/rna.1700409}
}
</pre></td>
</tr>
<tr id="parisien2008" class="entry">
	<td>Parisien, M. &amp; Major, F.</td>
	<td>The MC-Fold and MC-Sym Pipeline Infers RNA Structure from Sequence Data <p class="infolinks">[<a href="javascript:toggleInfo('parisien2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_parisien2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{parisien2008,
  author = {Marc Parisien and Francois Major},
  title = {The MC-Fold and MC-Sym Pipeline Infers RNA Structure from Sequence Data},
  journal = {Nature},
  year = {2008},
  volume = {452},
  pages = {51-55}
}
</pre></td>
</tr>
<tr id="parthasarathi2004" class="entry">
	<td>Parthasarathi, R., Amutha, R., Subramanian, V., Nair, B. U. &amp; Ramasami, T.</td>
	<td>Bader's and Reactivity Descriptors' Analysis of DNA Base Pairs <p class="infolinks">[<a href="javascript:toggleInfo('parthasarathi2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Physical Chemistry A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp031285f">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_parthasarathi2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{parthasarathi2004,
  author = {R. Parthasarathi and R. Amutha and V. Subramanian and Balachandran Unni Nair and T. Ramasami},
  title = {Bader's and Reactivity Descriptors' Analysis of DNA Base Pairs},
  journal = {Journal of Physical Chemistry A},
  year = {2004},
  volume = {108},
  pages = {3817-3828},
  doi = {http://dx.doi.org/10.1021/jp031285f}
}
</pre></td>
</tr>
<tr id="pavesi2004" class="entry">
	<td>Pavesi, G., Mauri, G., Stefani, M. &amp; Pesole, G.</td>
	<td>RNAProfile: an algorithm for finding conserved secondary structure motifs in unaligned RNA sequences <p class="infolinks">[<a href="javascript:toggleInfo('pavesi2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkh650">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_pavesi2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{pavesi2004,
  author = {Giulio Pavesi and Giancarlo Mauri and Marco Stefani and Graziano Pesole},
  title = {RNAProfile: an algorithm for finding conserved secondary structure motifs in unaligned RNA sequences},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  pages = {3258-3269},
  doi = {http://dx.doi.org/10.1093/nar/gkh650}
}
</pre></td>
</tr>
<tr id="perez2007" class="entry">
	<td>Perez, A., Marchan, I., Svozil, D., Sponer, J., III, T. E. C., Laughton, C. A. &amp; Orozco, M.</td>
	<td>Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of a/g Conformers <p class="infolinks">[<a href="javascript:toggleInfo('perez2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Biphysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.106.097782">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_perez2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{perez2007,
  author = {Alberto Perez and Ivan Marchan and Daniel Svozil and Jiri Sponer and Thomas E. Cheatham III and Charles A. Laughton and Modesto Orozco},
  title = {Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of a/g Conformers},
  journal = {Biphysical Journal},
  year = {2007},
  volume = {92},
  pages = {3817-3829},
  doi = {http://dx.doi.org/10.1529/biophysj.106.097782}
}
</pre></td>
</tr>
<tr id="perez2005" class="entry">
	<td>Perez, A., Sponer, J., Jurecka, P., Hobza, P., Luque, F. J. &amp; Orozco, M.</td>
	<td>Are the Hydrogen Bonds of RNA (AU) Stronger Than those of DNA (AT)? A Quantum Mechanics Study <p class="infolinks">[<a href="javascript:toggleInfo('perez2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('perez2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Chemistry - A European Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/chem.200500255">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_perez2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The intrinsic stability of Watson-Crick d(AT) and r(AU) hydrogen bonds was analyzed by employing a variety of quantum-mechanical techniques, such as energy calculations, determination of reactivity indexes, and analysis of electron density topology. The analyses were performed not only for equilibrium gas-phase geometries, but also on hundreds of conformations derived from molecular dynamics (MD) and database analysis. None of our results support the idea that r(AU) hydrogen bonds are intrinsically more stable than those of d(AT). Instead, our data are in accordance with the traditional view that the greater stability of RNA relative to DNA is attributable to a variety of effects (e.g., stacking, sugar puckering, solvation) rather than to a significant difference in the hydrogen bonding of DNA and RNA base pairs.</td>
</tr>
<tr id="bib_perez2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{perez2005,
  author = {Alberto Perez and Jiri Sponer and Petr Jurecka and Pavel Hobza and F. Javier Luque and Modesto Orozco},
  title = {Are the Hydrogen Bonds of RNA (AU) Stronger Than those of DNA (AT)? A Quantum Mechanics Study},
  journal = {Chemistry - A European Journal},
  year = {2005},
  volume = {11},
  pages = {5062-5066},
  doi = {http://dx.doi.org/10.1002/chem.200500255}
}
</pre></td>
</tr>
<tr id="pyle1995" class="entry">
	<td>Pyle, A. M. &amp; Green, J. B.</td>
	<td>RNA folding <p class="infolinks">[<a href="javascript:toggleInfo('pyle1995','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('pyle1995','bibtex')">BibTeX</a>]</p></td>
	<td>1995</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/0959-440X(95)80091-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_pyle1995" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The number of known motifs for RNA folding and RNA tertiary organization is expanding rapidly as we learn more about the diverse biological functions of RNA. Problems in protein and RNA folding have melded in recent investigations of ribonucleoprotein folding. Theoretical and experimental models and rapidly being developed for the pathways and stabilizing forces involved in RNA folding.</td>
</tr>
<tr id="bib_pyle1995" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{pyle1995,
  author = {Anna Marie Pyle and Justin B Green},
  title = {RNA folding},
  journal = {Current Opinion in Structural Biology},
  year = {1995},
  volume = {5},
  number = {3},
  pages = {303-310},
  doi = {http://dx.doi.org/10.1016/0959-440X(95)80091-3}
}
</pre></td>
</tr>
<tr id="Rproj" class="entry">
	<td>R Development Core Team</td>
	<td>R: A Language and Environment for Statistical Computing <p class="infolinks">[<a href="javascript:toggleInfo('Rproj','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td> &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.R-project.org">URL</a> &nbsp;</td>
</tr>
<tr id="bib_Rproj" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Rproj,
  author = {R Development Core Team},
  title = {R: A Language and Environment for Statistical Computing},
  year = {2007},
  note = {ISBN 3-900051-07-0},
  url = {http://www.R-project.org}
}
</pre></td>
</tr>
<tr id="randic2004" class="entry">
	<td>Randic, M.</td>
	<td>Graphical representation of DNA as 2-D map <p class="infolinks">[<a href="javascript:toggleInfo('randic2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('randic2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Chem. Phys Lett. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.cplett.2004.01.088">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_randic2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We describe a modification of the compact representation of DNA sequences which transforms the sequence into a 2-D diagram in which the ?spots? have integer coordinates. As a result the accompanying numerical characterization of DNA is quite simple and straightforward. This is an important advantage, particularly when considering DNA sequences having thousands of nucleic bases. The approach starts with the compact representation of DNA based on zigzag spiral template used for placing ?spots? associated with binary codes of the nucleic acids and subsequent suppression of the underlying zigzag curve. As a result, a 2-D map is formed in which all ?spots? have integer coordinates. By using only distances between spots having the same x or the same y coordinate one can construct a ?map profile? using integer arithmetic. The approach is illustrated on DNA sequences of the first exon of human -globin.</td>
</tr>
<tr id="bib_randic2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic2004,
  author = {Milan Randic},
  title = {Graphical representation of DNA as 2-D map},
  journal = {Chem. Phys Lett.},
  year = {2004},
  volume = {386},
  pages = {468-471},
  doi = {http://dx.doi.org/10.1016/j.cplett.2004.01.088}
}
</pre></td>
</tr>
<tr id="randic2001" class="entry">
	<td>Randic, M.</td>
	<td>The connectivity index 25 years after <p class="infolinks">[<a href="javascript:toggleInfo('randic2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('randic2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Molecular Graphics and Modelling &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S1093-3263(01)00098-5">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_randic2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We review the developments following introduction of the connectivity indices as molecular descriptors in multiple linear regression analysis (MLRA) for structure?property?activity studies. We end the review with discussion of results obtained with applications of the variable connectivity index. A comparison is made between some results obtained with the traditional topological indices and the variable connectivity index.</td>
</tr>
<tr id="bib_randic2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic2001,
  author = {Milan Randic},
  title = {The connectivity index 25 years after},
  journal = {Journal of Molecular Graphics and Modelling},
  year = {2001},
  volume = {20},
  pages = {19-35},
  doi = {http://dx.doi.org/10.1016/S1093-3263(01)00098-5}
}
</pre></td>
</tr>
<tr id="randic1996" class="entry">
	<td>Randic, M.</td>
	<td>Mathematical Methods in Contemporary Chemistry <p class="infolinks">[<a href="javascript:toggleInfo('randic1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_randic1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{randic1996,
  author = {Milan Randic},
  title = {Mathematical Methods in Contemporary Chemistry},
  publisher = {Gordon and Breach Publishers},
  year = {1996},
  pages = {35-43}
}
</pre></td>
</tr>
<tr id="randic1975" class="entry">
	<td>Randic, M.</td>
	<td>On the Characterization of Molecular Branching <p class="infolinks">[<a href="javascript:toggleInfo('randic1975','bibtex')">BibTeX</a>]</p></td>
	<td>1975</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_randic1975" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic1975,
  author = {Milan Randic},
  title = {On the Characterization of Molecular Branching},
  journal = {Journal of the American Chemical Society},
  year = {1975},
  volume = {97},
  pages = {6609-6615}
}
</pre></td>
</tr>
<tr id="randic2003" class="entry">
	<td>Randic, M. &amp; Balaba, A. T.</td>
	<td>On A Four-Dimensional Representation of DNA Primary Sequences <p class="infolinks">[<a href="javascript:toggleInfo('randic2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Journal of Chemical Information and Computer Sciences &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci020051a">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_randic2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic2003,
  author = {Milan Randic and Alexandru T. Balaba},
  title = {On A Four-Dimensional Representation of DNA Primary Sequences},
  journal = {Journal of Chemical Information and Computer Sciences},
  year = {2003},
  volume = {43},
  pages = {532-539},
  doi = {http://dx.doi.org/10.1021/ci020051a}
}
</pre></td>
</tr>
<tr id="randic1994" class="entry">
	<td>Randic, M., Kleiner, A. F. &amp; DeAlba, L. M.</td>
	<td>Distance Distance Matrices <p class="infolinks">[<a href="javascript:toggleInfo('randic1994','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('randic1994','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td>Journal of Chemical Information and Computer Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci00018a008">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_randic1994" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We introduce novel matrices for graphs embedded on two- and three-dimensional grids. The matrices are<p>defined in terms of geometrical and topological distances in such graphs. We report on some properties of<p>these distance/distance matrices and have listed several structural invariants derived from distance/distance<p>matrices. The normalized Perron root (the first eigenvalue) of such matrices, X/n, for path graphs apparently<p>is an index of molecular folding. The ratio C#J = X/n is 1 for (geometrically) linear structures, while it approaches<p>0 as the path graph is repeatedly folded.</td>
</tr>
<tr id="bib_randic1994" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic1994,
  author = {Milan Randic and Alexander F. Kleiner and Luz M. DeAlba},
  title = {Distance Distance Matrices},
  journal = {Journal of Chemical Information and Computer Science},
  year = {1994},
  volume = {34},
  pages = {277-286},
  doi = {http://dx.doi.org/10.1021/ci00018a008}
}
</pre></td>
</tr>
<tr id="randic1999" class="entry">
	<td>Randic, M. &amp; Krilov, G.</td>
	<td>On the Characterization of the Folding of Proteins <p class="infolinks">[<a href="javascript:toggleInfo('randic1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('randic1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>International Journal of Quantum Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/(SICI)1097-461X(1999)75:6<1017::AID-QUA6>3.0.CO;2-C">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_randic1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We consider a characterization of the folding of 3-D model proteins. Using as the input the geometry of molecules, we first construct a distance/distance matrix in which element i, j is given by the quotient of the Euclidean and the graph theoretical distance between the two vertices. The leading eigenvalue of the D/D matrix gives a measure of the folding of the protein structure. Using higher powers of the elements of D/D matrices, we generate the corresponding leading eigenvalue (k) for different exponents (k=1, 2, 3,). So, derived invariants represent a characterization of the folding of a structure - here, model proteins. The derived invariants are analogous to the characterization of proteins based on the average distance, referred to as a protein profile. The folding index =/n, that is, the leading eigenvalue, is normalized to the number of points representing the structure. Structures that are more folded have a smaller folding index. We illustrate the use of the folding indices to measure the degree of similarity of molecules.</td>
</tr>
<tr id="bib_randic1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic1999,
  author = {Milan Randic and Goran Krilov},
  title = {On the Characterization of the Folding of Proteins},
  journal = {International Journal of Quantum Chemistry},
  year = {1999},
  volume = {75},
  pages = {1017-1026},
  doi = {http://dx.doi.org/10.1002/(SICI)1097-461X(1999)75:6<1017::AID-QUA6>3.0.CO;2-C}
}
</pre></td>
</tr>
<tr id="randic1997" class="entry">
	<td>Randic, M. &amp; Krilov, G.</td>
	<td>Characterization of 3-D Sequences of Proteins <p class="infolinks">[<a href="javascript:toggleInfo('randic1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('randic1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Chemical Physics Letters &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0009-2614(97)00447-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_randic1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We consider the characterization of 3-D sequences of proteins. Using as the input the geometry of the sequences from matrices (dij)k, in which dij is the Euclidean distances between points along the path of the protein (K = 1, 2, 3,? ) we derive the average values of suitably normalized matrix elements, referred to as protein profile. So derived invariants were used for characterization. The degree of similarity between differently folded models of proteins is examined using the derived protein profiles.</td>
</tr>
<tr id="bib_randic1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic1997,
  author = {Milan Randic and Goran Krilov},
  title = {Characterization of 3-D Sequences of Proteins},
  journal = {Chemical Physics Letters},
  year = {1997},
  volume = {272},
  pages = {115-119},
  doi = {http://dx.doi.org/10.1016/S0009-2614(97)00447-8}
}
</pre></td>
</tr>
<tr id="randic2001b" class="entry">
	<td>Randic, M. &amp; Zupan, J.</td>
	<td>On Interpretation of Well-Known Topological Indices <p class="infolinks">[<a href="javascript:toggleInfo('randic2001b','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Chemical and Computational Science &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_randic2001b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{randic2001b,
  author = {Milan Randic and Jure Zupan},
  title = {On Interpretation of Well-Known Topological Indices},
  journal = {Journal of Chemical and Computational Science},
  year = {2001},
  volume = {41},
  pages = {550-560}
}
</pre></td>
</tr>
<tr id="rangan2003" class="entry">
	<td>Rangan, P., Masquida, B., Westhof, E. &amp; Woodson, S. A.</td>
	<td>Assembly of core helices and rapid tertiary folding of a small bacterial group I ribozyme <p class="infolinks">[<a href="javascript:toggleInfo('rangan2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('rangan2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Proceedings of the National Academy of Sciences of the United Stated of America &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0337743100">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_rangan2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Compact but non-native intermediates have been implicated in the hierarchical folding of several large RNAs, but there is little information on their structure. In this article, ribonuclease and hydroxyl radical cleavage protection assays showed that base pairing of core helices stabilize a compact state of a small group I ribozyme from Azoarcus pre-tRNAile. Base pairing of the ribozyme core requires 10-fold less Mg2+ than stable tertiary interactions, indicating that assembly of helices in the catalytic core represents a distinct phase that precedes the formation of native tertiary structure. Tertiary folding occurs in <100 ms at 37ï¿½C. Such rapid folding is unprecedented among group I ribozymes and illustrates the association between structural complexity and folding time. A 3D model of the Azoarcus ribozyme was constructed by identifying homologous sequence motifs in rRNA. The model reveals distinct structural features, such as a large interface between the P4-P6 and P3-P9 domains, that may explain the unusual stability of the Azoarcus ribozyme and the cooperativity of folding.</td>
</tr>
<tr id="bib_rangan2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{rangan2003,
  author = {Prashanth Rangan and Benoit Masquida and Eric Westhof and Sarah A. Woodson},
  title = {Assembly of core helices and rapid tertiary folding of a small bacterial group I ribozyme},
  journal = {Proceedings of the National Academy of Sciences of the United Stated of America},
  year = {2003},
  volume = {100},
  pages = {1574-1579},
  doi = {http://dx.doi.org/10.1073/pnas.0337743100}
}
</pre></td>
</tr>
<tr id="razga2007" class="entry">
	<td>Razga, F., Koca, J., Mokdad, A. &amp; Sponer, J.</td>
	<td>Elastic properties of ribosomal RNA building blocks: molecular dynamics of the GTPase-associated center rRNA <p class="infolinks">[<a href="javascript:toggleInfo('razga2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkm245">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_razga2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{razga2007,
  author = {Filip Razga and Jaroslav Koca and Ali Mokdad and Jiri Sponer},
  title = {Elastic properties of ribosomal RNA building blocks: molecular dynamics of the GTPase-associated center rRNA},
  journal = {Nucleic Acids Research},
  year = {2007},
  volume = {35},
  pages = {4007-4017},
  doi = {http://dx.doi.org/10.1093/nar/gkm245}
}
</pre></td>
</tr>
<tr id="reeder2007" class="entry">
	<td>Reeder, J., Reeder, J. &amp; Giegerich, R.</td>
	<td>Locomotif: From Graphical Motif Description to RNA Motif Search <p class="infolinks">[<a href="javascript:toggleInfo('reeder2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('reeder2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/bioinformatics/btm179">DOI</a><a href="http://dx.doi.org/10.1093/bioinformatics/btm179">URL</a> &nbsp;</td>
</tr>
<tr id="abs_reeder2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MOTIVATION AND RESULTS: Motivated by the recent rise of interest in small regulatory RNAs, we present Locomotif--a new approach for locating RNA motifs that goes beyond the previous ones in three ways: (1) motif search is based on efficient dynamic programming algorithms, incorporating the established thermodynamic model of RNA secondary structure formation. (2) motifs are described graphically, using a Java-based editor, and search algorithms are derived from the graphics in a fully automatic way. The editor allows us to draw secondary structures, annotated with size and sequence information. They closely resemble the established, but informal way in which RNA motifs are communicated in the literature. Thus, the learning effort for Locomotif users is minimal. (3) Locomotif employs a client-server approach. Motifs are designed by the user locally. Search programs are generated and compiled on a bioinformatics server. They are made available both for execution on the server, and for download as C source code plus an appropriate makefile. AVAILABILITY: Locomotif is available at http://bibiserv.techfak.uni-bielefeld.de/locomotif.</td>
</tr>
<tr id="bib_reeder2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{reeder2007,
  author = {Janina Reeder and Jens Reeder and Robert Giegerich},
  title = {Locomotif: From Graphical Motif Description to RNA Motif Search},
  journal = {Bioinformatics},
  year = {2007},
  volume = {23},
  number = {13},
  pages = {i392-i400},
  url = {http://dx.doi.org/10.1093/bioinformatics/btm179},
  doi = {http://dx.doi.org/10.1093/bioinformatics/btm179}
}
</pre></td>
</tr>
<tr id="reenan2005" class="entry">
	<td>Reenan, R. A.</td>
	<td>Molecular determinants and guided evolution of species-specific RNA editing <p class="infolinks">[<a href="javascript:toggleInfo('reenan2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('reenan2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature03364">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_reenan2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Most RNA editing systems are mechanistically diverse, informationally restorative, and scattershot in eukaryotic lineages1. In contrast, genetic recoding by adenosine-to-inosine RNA editing seems common in animals; usually, altering highly conserved or invariant coding positions in proteins2 4. Here I report striking variation between species in the recoding of synaptotagmin I (sytI). Fruitflies, mosquitoes and butterflies possess shared and species-specific sytI editing sites, all within a single exon. Honeybees, beetles and roaches do not edit sytI. The editing machinery is usually directed to modify particular adenosines by information stored in intron-mediated RNA structures5 7. Combining comparative genomics of 34 species with mutational analysis reveals that complex, multi-domain, pre-mRNA structures solely determine species-appropriate RNA editing. One of these is a previously unreported long-range pseudoknot. I show that small changes to intronic sequences, far removed from an editing site, can transfer the species specificity of editing between RNA substrates. Taken together, these data support a phylogeny of sytI gene editing spanning more than 250 million years of hexapod evolution. The results also provide models for the genesis of RNA editing sites through the stepwise addition of structural domains, or by short walks through sequence space from ancestral structures.</td>
</tr>
<tr id="bib_reenan2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{reenan2005,
  author = {Robert A. Reenan},
  title = {Molecular determinants and guided evolution of species-specific RNA editing},
  journal = {Nature},
  year = {2005},
  volume = {434},
  pages = {409-413},
  doi = {http://dx.doi.org/10.1038/nature03364}
}
</pre></td>
</tr>
<tr id="reijmers2001" class="entry">
	<td>Reijmers, T. H., Wehrens, R. &amp; Buydens, L. M. C.</td>
	<td>The Influence of Different Structure Representations on the Clustering of an RNA Nucleotides Data Set <p class="infolinks">[<a href="javascript:toggleInfo('reijmers2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Chemical Information and Computer Science &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_reijmers2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{reijmers2001,
  author = {T. H. Reijmers and R. Wehrens and L. M. C. Buydens},
  title = {The Influence of Different Structure Representations on the Clustering of an RNA Nucleotides Data Set},
  journal = {Journal of Chemical Information and Computer Science},
  year = {2001},
  volume = {41},
  pages = {1388-1394}
}
</pre></td>
</tr>
<tr id="reiter2008" class="entry">
	<td>Reiter, N. J., Maher, L. J. &amp; Butcher, S. E.</td>
	<td>DNA mimicry by a high-affinity anti-NF-kappaB RNA aptamer. <p class="infolinks">[<a href="javascript:toggleInfo('reiter2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('reiter2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Nucleic Acids Res &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkm1141">DOI</a><a href="http://dx.doi.org/10.1093/nar/gkm1141">URL</a> &nbsp;</td>
</tr>
<tr id="abs_reiter2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The binding of RNA molecules to proteins or other ligands can require extensive RNA folding to create an induced fit. Understanding the generality of this principle involves comparing structures of RNA before and after complex formation. Here we report the NMR solution structure of a 29-nt RNA aptamer whose crystal structure had previously been determined in complex with its transcription factor target, the p50(2) form of NF-kappaB. The RNA aptamer internal loop structure has pre-organized features that are also found in the complex, including non-canonical base pairing and cross-strand base stacking. Remarkably, the free RNA aptamer structure possesses a major groove that more closely resembles B-form DNA than RNA. Upon protein binding, changes in RNA structure include the kinking of the internal loop and distortion of the terminal tetraloop. Thus, complex formation involves both pre-formed and induced fit binding interactions. The high affinity of the NF-kappaB transcription factor for this RNA aptamer may largely be due to the structural pre-organization of the RNA that results in its ability to mimic DNA.</td>
</tr>
<tr id="bib_reiter2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{reiter2008,
  author = {Nicholas J Reiter and L. James Maher and Samuel E Butcher},
  title = {DNA mimicry by a high-affinity anti-NF-kappaB RNA aptamer.},
  journal = {Nucleic Acids Res},
  year = {2008},
  volume = {36},
  number = {4},
  pages = {1227--1236},
  url = {http://dx.doi.org/10.1093/nar/gkm1141},
  doi = {http://dx.doi.org/10.1093/nar/gkm1141}
}
</pre></td>
</tr>
<tr id="restrepo2006" class="entry">
	<td>Restrepo, G., Llanos, E. J. &amp; Meza, H.</td>
	<td>Topological Space of the Chemical Elements and its Properties <p class="infolinks">[<a href="javascript:toggleInfo('restrepo2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Mathematical Chemistry &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_restrepo2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{restrepo2006,
  author = {Guillermo Restrepo and Eugenio J. Llanos and Heber Meza},
  title = {Topological Space of the Chemical Elements and its Properties},
  journal = {Journal of Mathematical Chemistry},
  year = {2006},
  volume = {39},
  pages = {401-416}
}
</pre></td>
</tr>
<tr id="restrepo2004" class="entry">
	<td>Restrepo, G., Mesa, H., Llanos, E. J. &amp; Villaveces, J. L.</td>
	<td>Topological Study of the Periodic System <p class="infolinks">[<a href="javascript:toggleInfo('restrepo2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Chemical Information and Computer Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ci034217z">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_restrepo2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{restrepo2004,
  author = {Guillermo Restrepo and Heber Mesa and Eugenio J. Llanos and Jose L. Villaveces},
  title = {Topological Study of the Periodic System},
  journal = {Journal of Chemical Information and Computer Science},
  year = {2004},
  volume = {44},
  pages = {68-75},
  doi = {http://dx.doi.org/10.1021/ci034217z}
}
</pre></td>
</tr>
<tr id="restrepo2006b" class="entry">
	<td>Restrepo, G., Mesa, H. &amp; Villaveces, J. L.</td>
	<td>On the Topological Sense of Chemical Sets <p class="infolinks">[<a href="javascript:toggleInfo('restrepo2006b','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Journal of Mathematical Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1007/s10910-005-9013-5">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_restrepo2006b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{restrepo2006b,
  author = {Gullermo Restrepo and Heber Mesa and Jose L. Villaveces},
  title = {On the Topological Sense of Chemical Sets},
  journal = {Journal of Mathematical Chemistry},
  year = {2006},
  volume = {39},
  pages = {363-376},
  doi = {http://dx.doi.org/10.1007/s10910-005-9013-5}
}
</pre></td>
</tr>
<tr id="rich1956" class="entry">
	<td>Rich, A. &amp; Davies, D. R.</td>
	<td>A New Two Stranded Helical Structure: Polyadenylic Acid and Polyuridylic Acid <p class="infolinks">[<a href="javascript:toggleInfo('rich1956','bibtex')">BibTeX</a>]</p></td>
	<td>1956</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja01595a086">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_rich1956" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{rich1956,
  author = {Alexander Rich and David R. Davies},
  title = {A New Two Stranded Helical Structure: Polyadenylic Acid and Polyuridylic Acid},
  journal = {Journal of the American Chemical Society},
  year = {1956},
  volume = {78},
  pages = {3548-3549},
  doi = {http://dx.doi.org/10.1021/ja01595a086}
}
</pre></td>
</tr>
<tr id="richardson2002" class="entry">
	<td>Richardson, D. C. &amp; Richardson, J. S.</td>
	<td>Teaching Molecular 3-D Literacy <p class="infolinks">[<a href="javascript:toggleInfo('richardson2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('richardson2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Biochemistry and Molecular Biology Education &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.bambed.org/cgi/content/full/30/1/21">URL</a> &nbsp;</td>
</tr>
<tr id="abs_richardson2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An understanding of three-dimensional structure/function relationships is increasingly important to modern biochemistry and molecular biology. Not only are the overall "folds" that illuminate evolutionary relationships uncompromisingly three-dimensional but so, too, are the critical details at active sites. Fortunately, most protein and nucleic acid structures are now readily available on the Internet from the Protein Data Bank [1, 2], as are good tools for displaying them on small computers such as Mage [3, 4], Rasmol [5, 6], Chime [7], and SwissPDBViewer [8, 9]. However, teaching these skills and concepts effectively is nevertheless a challenge, because the prior education of students has concentrated entirely on one-dimensional (verbal) and two-dimensional (static pictures) information; many students feel unfamiliar and uncomfortable with 3-D1 materials.</td>
</tr>
<tr id="bib_richardson2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{richardson2002,
  author = {David C. Richardson and Jane S. Richardson},
  title = {Teaching Molecular 3-D Literacy},
  journal = {Biochemistry and Molecular Biology Education},
  year = {2002},
  volume = {30},
  pages = {21-26},
  url = {http://www.bambed.org/cgi/content/full/30/1/21}
}
</pre></td>
</tr>
<tr id="richardson1992" class="entry">
	<td>Richardson, D. C. &amp; Richardson, J. S.</td>
	<td>The kinemage: A tool for scientific communication <p class="infolinks">[<a href="javascript:toggleInfo('richardson1992','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('richardson1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Protein Science &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_richardson1992" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A "kinemage" (kinetic image) is a scientific illustration presented as an interactive computer display. Operations on the displayed kinemage respond within a fraction of a second: the entire image can be rotated in real time, parts of the display can be turned on or off, points can be identified by selecting them, and the change between different forms can be animated. A kinemage is prepared and specified by the author(s) of a journal article, in order to better communicate ideas that depend on three-dimensional information. The kinemages are distributed as plain text files of commented display lists and accompanying explanations. They are viewed and explored in an open-ended way by the reader using a simple graphics program, such as the one described here (called MAGE), which presently runs on Macintosh computers. A utility (called PREKIN) helps authors prepare the kinemages. Kinemages are being implemented under the auspices of the Innovative Technology Fund.</td>
</tr>
<tr id="bib_richardson1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{richardson1992,
  author = {D. C. Richardson and J. S. Richardson},
  title = {The kinemage: A tool for scientific communication},
  journal = {Protein Science},
  year = {1992},
  volume = {1},
  pages = {3-9}
}
</pre></td>
</tr>
<tr id="richardson2000" class="entry">
	<td>Richardson, J. S.</td>
	<td>Early ribbon drawings of proteins <p class="infolinks">[<a href="javascript:toggleInfo('richardson2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('richardson2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nature Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/77912">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_richardson2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The ubiquitous ribbon drawings of protein structures that are commonly made these days using programs such as Molscript or Ribbons had their origin approx20 years ago in drawings made by hand. Some earlier schematic drawings had been made of individual proteins: Dick Dickerson was the first to make a protein schematic1 and Irving Geis the first to show successive peptide planes with ribbons2; ribbon drawings, using various conventions, were made by a few others3, most notably Bo Furugren4. My husband David and I had made 'worm drawings' for our own staphylococcal nuclease and superoxide dismutase structures, and for other proteins5 (Fig. 1a). However, the first attempt to illustrate the full range of known protein structures (only 75 different ones, then!) with a consistent system of representation was the 1981 article "The anatomy and taxonomy of protein structure"6 (Fig. 1b d).</td>
</tr>
<tr id="bib_richardson2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{richardson2000,
  author = {Jane S. Richardson},
  title = {Early ribbon drawings of proteins},
  journal = {Nature Structural Biology},
  year = {2000},
  volume = {7},
  pages = {624-625},
  doi = {http://dx.doi.org/10.1038/77912}
}
</pre></td>
</tr>
<tr id="richardson2008" class="entry">
	<td>Richardson, J. S., Schneider, B., Murray, L. W., Kapral, G. J., Immormino, R. M., Headd, J. J., Richardson, D. C., Ham, D., Hershkovits, E., Williams, L. D., Keating, K. S., Pyle, A. M., Micallef, D., Westbrook, J. &amp; Berman, H. M.</td>
	<td>RNA backbone: Consensus all-angle conformers and modular string nomenclature (an RNA Ontology Consortium contribution) <p class="infolinks">[<a href="javascript:toggleInfo('richardson2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('richardson2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1261/rna.657708">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_richardson2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A consensus classification and nomenclature are defined for RNA backbone structure using all of the backbone torsion angles.<p>By a consensus of several independent analysis methods, 46 discrete conformers are identified as suitably clustered in a qualityfiltered,<p>multidimensional dihedral angle distribution. Most of these conformers represent identifiable features or roles within<p>RNA structures. The conformers are given two-character names that reflect the seven-angle dezabgd combinations empirically<p>found favorable for the sugar-to-sugar ï¿½ï¿½suiteï¿½ï¿½ unit within which the angle correlations are strongest (e.g., 1a for A-form, 5z for<p>the start of S-motifs). Since the half-nucleotides are specified by a number for dez and a lowercase letter for abgd, this modular<p>system can also be parsed to describe traditional nucleotide units (e.g., a1) or the dinucleotides (e.g., a1a1) that are especially<p>useful at the level of crystallographic map fitting. This nomenclature can also be written as a string with two-character suite<p>names between the uppercase letters of the base sequence (N1aG1gN1aR1aA1cN1a for a GNRA tetraloop), facilitating<p>bioinformatic comparisons. Cluster means, standard deviations, coordinates, and examples are made available, as well as the<p>Suitename software that assigns suite conformer names and conformer match quality (suiteness) from atomic coordinates. The<p>RNA Ontology Consortium will combine this new backbone system with others that define base pairs, base-stacking, and<p>hydrogen-bond relationships to provide a full description of RNA structural motifs.</td>
</tr>
<tr id="bib_richardson2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{richardson2008,
  author = {Jane S. Richardson and Bohdan Schneider and Laura W. Murray and Gary J. Kapral and Robert M. Immormino and Jeffrey J. Headd and David C. Richardson and Daniela Ham and Eli Hershkovits and Loren Dean Williams and Kevin S. Keating and Anna Marie Pyle and David Micallef and John Westbrook and and Helen M. Berman},
  title = {RNA backbone: Consensus all-angle conformers and modular string nomenclature (an RNA Ontology Consortium contribution)},
  journal = {RNA},
  year = {2008},
  doi = {http://dx.doi.org/10.1261/rna.657708}
}
</pre></td>
</tr>
<tr id="robertus1974" class="entry">
	<td>Robertus, J. D., Ladner, J. E., Finch, J. T., Rhodes, D., Brown, R. S., Clark, B. F. C. &amp; Klug, A.</td>
	<td>Structure of Yeast Phenylalanine tRNA at 3 &Aring; Resolution <p class="infolinks">[<a href="javascript:toggleInfo('robertus1974','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('robertus1974','bibtex')">BibTeX</a>]</p></td>
	<td>1974</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.nature.com/nature/journal/v250/n5467/pdf/250546a0.pdf">URL</a> &nbsp;</td>
</tr>
<tr id="abs_robertus1974" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The structure of a tRNA has been determined by isomorphous replacement. Some of the interactions which maintain the tertiary structure are of a novel type. Our model differs significantly from one which has recently been proposed.</td>
</tr>
<tr id="bib_robertus1974" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{robertus1974,
  author = {J. D. Robertus and Jane E. Ladner and J. T. Finch and Daniela Rhodes and R. S. Brown and B. F. C. Clark and A. Klug},
  title = {Structure of Yeast Phenylalanine tRNA at 3 &Aring; Resolution},
  journal = {Nature},
  year = {1974},
  volume = {250},
  pages = {546},
  url = {http://www.nature.com/nature/journal/v250/n5467/pdf/250546a0.pdf}
}
</pre></td>
</tr>
<tr id="ruvkun2001" class="entry">
	<td>Ruvkun, G.</td>
	<td>Glimpses of a Tiny RNA World <p class="infolinks">[<a href="javascript:toggleInfo('ruvkun2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('ruvkun2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1066315">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_ruvkun2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Naturally tiny regulatory RNAs of about 22 nucleotides were first identified from Caenorhabditis elegans genetics. Artificially induced RNAs of this size are also an intermediate in RNA interference, and both such RNAs use a common biochemical processing pathway. Now three groups have shown that there is an entire world of tiny RNAs that had almost escaped detection until now. These tiny RNAs are likely to regulate the translation of other mRNAs during development and, like the intermediates in RNAi, may be subject to amplification and systemic spread.</td>
</tr>
<tr id="bib_ruvkun2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ruvkun2001,
  author = {Gary Ruvkun},
  title = {Glimpses of a Tiny RNA World},
  journal = {Science},
  year = {2001},
  volume = {294},
  pages = {797-799},
  doi = {http://dx.doi.org/10.1126/science.1066315}
}
</pre></td>
</tr>
<tr id="rypniewski2006" class="entry">
	<td>Rypniewski, W., Vallazza, M., Perbandt, M., Klussmann, S., DeLucas, L. J., Betzel, C. &amp; Erdmann, V. A.</td>
	<td>The first crystal structure of an RNA racemate <p class="infolinks">[<a href="javascript:toggleInfo('rypniewski2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('rypniewski2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Acta Crystallographica Section D &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1107/S090744490601359X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_rypniewski2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The racemate of the RNA duplex r(CUGGGCGG). r(CCGCCUGG) from Thermus flavus 5S rRNA has been<p>crystallized and examined by X-ray crystallography. The space<p>group is P1 with approximate unit-cell parameters a = 26.5,<p>b = 38.0, c = 45.4 A ï¿½ , alpha = 113.1, beta = 100.5, gamma = 93.3degrees. The<p>structure was solved by molecular replacement. There are four<p>RNA duplexes in the unit cell. The crystal lattice consists of<p>columns of RNA duplexes. The duplexes are stacked end-toend<p>and are stabilized by intermolecular base-stacking<p>interactions. Within each column the l-duplexes and<p>d-duplexes are stacked alternately. Every other duplex in<p>each stack has two alternative conformations, approximately<p>equally occupied, corresponding to molecules oriented in<p>opposite directions. Neighbouring columns are related by the<p>crystallographic centre of symmetry. The unit cell also<p>contains approximately 250 ordered water molecules and six<p>ordered calcium ions. A glycerol molecule is visible in the<p>minor groove interacting with a guanosine residue.</td>
</tr>
<tr id="bib_rypniewski2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{rypniewski2006,
  author = {W. Rypniewski and M. Vallazza and M. Perbandt and S. Klussmann and L. J. DeLucas and C. Betzel and V. A. Erdmann},
  title = {The first crystal structure of an RNA racemate},
  journal = {Acta Crystallographica Section D},
  year = {2006},
  volume = {D62},
  pages = {659-664},
  doi = {http://dx.doi.org/10.1107/S090744490601359X}
}
</pre></td>
</tr>
<tr id="reblova2003" class="entry">
	<td>Réblová, K., Spacková, N., Stefl, R., Csaszar, K., Koca, J., Leontis, N. B. &amp; Sponer, J.</td>
	<td>Non-Watson-Crick Basepairing and Hydration in RNA Motifs: Molecular Dynamics of 5S rRNA Loop E <p class="infolinks">[<a href="javascript:toggleInfo('reblova2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('reblova2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0006-3495(03)75089-9">DOI</a><a href="http://dx.doi.org/10.1016/S0006-3495(03)75089-9">URL</a> &nbsp;</td>
</tr>
<tr id="abs_reblova2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Explicit solvent and counterion molecular dynamics simulations have been carried out for a total of >80 ns on the bacterial and spinach chloroplast 5S rRNA Loop E motifs. The Loop E sequences form unique duplex architectures composed of seven consecutive non-Watson-Crick basepairs. The starting structure of spinach chloroplast Loop E was modeled using isostericity principles, and the simulations refined the geometries of the three non-Watson-Crick basepairs that differ from the consensus bacterial sequence. The deep groove of Loop E motifs provides unique sites for cation binding. Binding of Mg(2+) rigidifies Loop E and stabilizes its major groove at an intermediate width. In the absence of Mg(2+), the Loop E motifs show an unprecedented degree of inner-shell binding of monovalent cations that, in contrast to Mg(2+), penetrate into the most negative regions inside the deep groove. The spinach chloroplast Loop E shows a marked tendency to compress its deep groove compared with the bacterial consensus. Structures with a narrow deep groove essentially collapse around a string of Na(+) cations with long coordination times. The Loop E non-Watson-Crick basepairing is complemented by highly specific hydration sites ranging from water bridges to hydration pockets hosting 2 to 3 long-residing waters. The ordered hydration is intimately connected with RNA local conformational variations.</td>
</tr>
<tr id="bib_reblova2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{reblova2003,
  author = {Kamila Réblová and Nad'a Spacková and Richard Stefl and Kristina Csaszar and Jaroslav Koca and Neocles B Leontis and Jirí Sponer},
  title = {Non-Watson-Crick Basepairing and Hydration in RNA Motifs: Molecular Dynamics of 5S rRNA Loop E},
  journal = {Biophysical Journal},
  year = {2003},
  volume = {84},
  number = {6},
  pages = {3564-3582},
  url = {http://dx.doi.org/10.1016/S0006-3495(03)75089-9},
  doi = {http://dx.doi.org/10.1016/S0006-3495(03)75089-9}
}
</pre></td>
</tr>
<tr id="saenger1984" class="entry">
	<td>Saenger, W.</td>
	<td>Principles of Nucleic Acid Structure <p class="infolinks">[<a href="javascript:toggleInfo('saenger1984','bibtex')">BibTeX</a>]</p></td>
	<td>1984</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_saenger1984" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{saenger1984,
  author = {Wolfram Saenger},
  title = {Principles of Nucleic Acid Structure},
  publisher = {Springer-Verlag},
  year = {1984}
}
</pre></td>
</tr>
<tr id="sanbonmatsu2005" class="entry">
	<td>Sanbonmatsu, K. Y., Simpson, J. &amp; Tung, C.</td>
	<td>Simulating movement of tRNA into the ribosome during decoding <p class="infolinks">[<a href="javascript:toggleInfo('sanbonmatsu2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sanbonmatsu2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Proceedings of the National Academy of Sciences &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0503456102">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sanbonmatsu2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Decoding is the key step during protein synthesis that enables information transfer from RNA to protein, a process critical for the survival of all organisms. We have used large-scale (2.64 x 106 atoms) all-atom simulations of the entire ribosome to understand a critical step of decoding. Although the decoding problem has been studied for more than four decades, the rate-limiting step of cognate tRNA selection has only recently been identified. This step, known as accommodation, involves the movement inside the ribosome of the aminoacyl-tRNA from the partially bound "A/T" state to the fully bound "A/A" state. Here, we show that a corridor of 20 universally conserved ribosomal RNA bases interacts with the tRNA during the accommodation movement. Surprisingly, the tRNA is impeded by the A-loop (23S helix 92), instead of enjoying a smooth transition to the A/A state. In particular, universally conserved 23S ribosomal RNA bases U2492, C2556, and C2573 act as a 3D gate, causing the acceptor stem to pause before allowing entrance into the peptidyl transferase center. Our simulations demonstrate that the flexibility of the acceptor stem of the tRNA, in addition to flexibility of the anticodon arm, is essential for tRNA selection. This study serves as a template for simulating conformational changes in large (>106 atoms) biological and artificial molecular machines.</td>
</tr>
<tr id="bib_sanbonmatsu2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sanbonmatsu2005,
  author = {Kevin Y. Sanbonmatsu and Joseph Simpson and Chang-Shung Tung},
  title = {Simulating movement of tRNA into the ribosome during decoding},
  journal = {Proceedings of the National Academy of Sciences},
  year = {2005},
  volume = {102},
  pages = {15854-15859},
  doi = {http://dx.doi.org/10.1073/pnas.0503456102}
}
</pre></td>
</tr>
<tr id="sarver2008" class="entry">
	<td>Sarver, M., Zirbel, C. L., Stombaugh, J., Mokdad, A. &amp; Leontis, N. B.</td>
	<td>FR3D: Finding Local and Composite Recurrent Structural Motifs in RNA 3D Structures <p class="infolinks">[<a href="javascript:toggleInfo('sarver2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sarver2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Journal of Mathematical Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1007/s00285-007-0110-x">DOI</a><a href="http://dx.doi.org/10.1007/s00285-007-0110-x">URL</a> &nbsp;</td>
</tr>
<tr id="abs_sarver2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: New methods are described for finding recurrent three-dimensional (3D) motifs in RNA atomic-resolution structures. Recurrent RNA 3D motifs are sets of RNA nucleotides with similar spatial arrangements. They can be local or composite. Local motifs comprise nucleotides that occur in the same hairpin or internal loop. Composite motifs comprise nucleotides belonging to three or more different RNA strand segments or molecules. We use a base-centered approach to construct efficient, yet exhaustive search procedures using geometric, symbolic, or mixed representations of RNA structure that we implement in a suite of MATLAB programs, "Find RNA 3D" (FR3D). The first modules of FR3D preprocess structure files to classify base-pair and -stacking interactions. Each base is represented geometrically by the position of its glycosidic nitrogen in 3D space and by the rotation matrix that describes its orientation with respect to a common frame. Base-pairing and base-stacking interactions are calculated from the base geometries and are represented symbolically according to the Leontis/Westhof basepairing classification, extended to include base-stacking. These data are stored and used to organize motif searches. For geometric searches, the user supplies the 3D structure of a query motif which FR3D uses to find and score geometrically similar candidate motifs, without regard to the sequential position of their nucleotides in the RNA chain or the identity of their bases. To score and rank candidate motifs, FR3D calculates a geometric discrepancy by rigidly rotating candidates to align optimally with the query motif and then comparing the relative orientations of the corresponding bases in the query and candidate motifs. Given the growing size of the RNA structure database, it is impossible to explicitly compute the discrepancy for all conceivable candidate motifs, even for motifs with less than ten nucleotides. The screening algorithm that we describe finds all candidate motifs whose geometric discrepancy with respect to the query motif falls below a user-specified cutoff discrepancy. This technique can be applied to RMSD searches. Candidate motifs identified geometrically may be further screened symbolically to identify those that contain particular basepair types or base-stacking arrangements or that conform to sequence continuity or nucleotide identity constraints. Purely symbolic searches for motifs containing user-defined sequence, continuity and interaction constraints have also been implemented. We demonstrate that FR3D finds all occurrences, both local and composite and with nucleotide substitutions, of sarcin/ricin and kink-turn motifs in the 23S and 5S ribosomal RNA 3D structures of the H. marismortui 50S ribosomal subunit and assigns the lowest discrepancy scores to bona fide examples of these motifs. The search algorithms have been optimized for speed to allow users to search the non-redundant RNA 3D structure database on a personal computer in a matter of minutes.</td>
</tr>
<tr id="bib_sarver2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sarver2008,
  author = {Michael Sarver and Craig L Zirbel and Jesse Stombaugh and Ali Mokdad and Neocles B Leontis},
  title = {FR3D: Finding Local and Composite Recurrent Structural Motifs in RNA 3D Structures},
  journal = {Journal of Mathematical Biology},
  year = {2008},
  volume = {56},
  number = {1-2},
  pages = {215-252},
  url = {http://dx.doi.org/10.1007/s00285-007-0110-x},
  doi = {http://dx.doi.org/10.1007/s00285-007-0110-x}
}
</pre></td>
</tr>
<tr id="schluenzen2000" class="entry">
	<td>Schluenzen, F., Tocilj, A., Zarivach, R., Harms, J., Gluehmann, M., Janell, D., Bashan, A., Bartels, H., Agmon, I., Franceschi, F. &amp; Yonath, A.</td>
	<td>Structure of Functionally Activated Small Ribosomal Subunit at 3.3 &Aring; Resolution <p class="infolinks">[<a href="javascript:toggleInfo('schluenzen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Cell &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0092-8674(00)00084-2">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_schluenzen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{schluenzen2000,
  author = {Frank Schluenzen and Ante Tocilj and Raz Zarivach and Joerg Harms and Marco Gluehmann and Daniela Janell and Anat Bashan and Heike Bartels and Ilana Agmon and Francois Franceschi and Ada Yonath},
  title = {Structure of Functionally Activated Small Ribosomal Subunit at 3.3 &Aring; Resolution},
  journal = {Cell},
  year = {2000},
  volume = {102},
  pages = {615-623},
  doi = {http://dx.doi.org/10.1016/S0092-8674(00)00084-2}
}
</pre></td>
</tr>
<tr id="schneider2004" class="entry">
	<td>Schneider, B., Moravek, Z. &amp; Berman, H.</td>
	<td>RNA Conformational Classes <p class="infolinks">[<a href="javascript:toggleInfo('schneider2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkh333">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_schneider2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{schneider2004,
  author = {Bohdan Schneider and Zdenek Moravek and Helen Berman},
  title = {RNA Conformational Classes},
  journal = {Nucleic Acids Research},
  year = {2004},
  volume = {32},
  pages = {1666-1677},
  doi = {http://dx.doi.org/10.1093/nar/gkh333}
}
</pre></td>
</tr>
<tr id="schroeder2002" class="entry">
	<td>Schroeder, R., Grossberger, R., Pichler, A. &amp; Waldsich, C.</td>
	<td>RNA folding in vivo <p class="infolinks">[<a href="javascript:toggleInfo('schroeder2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('schroeder2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(02)00325-1">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_schroeder2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA folding in vivo is influenced by the cellular environment, the vectorial nature of transcription and translation, trans-acting factors and ion homeostasis. Specific RNA-binding proteins promote RNA folding by stabilizing the native structure or by guiding folding. In contrast, RNA chaperones, which are believed to interact nonspecifically with RNA, were proposed to resolve misfolded RNA structures and to promote intermolecular RNA?RNA annealing. Small trans-acting noncoding RNAs are thought to modulate mRNA structures, thereby regulating gene expression. So far, there is some evidence that in vitro and invivo RNA folding pathways share basic features. However, it is unclear whether the rules deduced from in vitro folding experiments generally apply to invivo conditions.</td>
</tr>
<tr id="bib_schroeder2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{schroeder2002,
  author = {Renee Schroeder and Rupert Grossberger and Andrea Pichler and Christina Waldsich},
  title = {RNA folding in vivo},
  journal = {Current Opinion in Structural Biology},
  year = {2002},
  volume = {12},
  pages = {296-300},
  doi = {http://dx.doi.org/10.1016/S0959-440X(02)00325-1}
}
</pre></td>
</tr>
<tr id="schroeder2009" class="entry">
	<td>Schroeder, S. J.</td>
	<td>Advances in RNA Structure Prediction from Sequence: New Tools for Generating Hypotheses About Viral RNA Structure-Function Relationships <p class="infolinks">[<a href="javascript:toggleInfo('schroeder2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Virology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1128/JVI.00251-09">DOI</a><a href="http://dx.doi.org/10.1128/JVI.00251-09">URL</a> &nbsp;</td>
</tr>
<tr id="bib_schroeder2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{schroeder2009,
  author = {Susan J Schroeder},
  title = {Advances in RNA Structure Prediction from Sequence: New Tools for Generating Hypotheses About Viral RNA Structure-Function Relationships},
  journal = {Journal of Virology},
  year = {2009},
  volume = {83},
  number = {13},
  pages = {6326-6334},
  url = {http://dx.doi.org/10.1128/JVI.00251-09},
  doi = {http://dx.doi.org/10.1128/JVI.00251-09}
}
</pre></td>
</tr>
<tr id="schwans2004" class="entry">
	<td>Schwans, J. P., Li, N. &amp; Piccirilli, J. A.</td>
	<td>A Packing-Density Metric for Exploring the Interior of Folded RNA Molecules <p class="infolinks">[<a href="javascript:toggleInfo('schwans2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('schwans2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Angewandte Chemie International Edition &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/anie.200353575">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_schwans2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RNA molecules fold into intricate three-dimensional architectures that perform important biological functions. These complex architectures often contain coaxially stacked helices that adopt specific spatial arrangements through tertiary interactions between distinctly structured, complementary surfaces. These interactions result in the formation of expansive close-packed interfaces.[1] Within these interfaces, atoms acquire distinct packing environments that together uniquely specify an RNA fold. To define these packing environments biochemically, we exploited a series of nucleotide analogues with 2-substituents that span a range of molecular volumes (Figure 1). We used nucleotide analogue interference mapping (NAIM)[2] to examine the effect of these volume changes at every residue within the structurally well-defined, independently folding P4-P6 RNA domain.[3] Our results show that these analogues together provide a sensitive metric for the packing density of the atoms surrounding the 2-hydroxy groups in an RNA molecule.</td>
</tr>
<tr id="bib_schwans2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{schwans2004,
  author = {Jason P. Schwans and Nan-Sheng Li and Joseph A. Piccirilli},
  title = {A Packing-Density Metric for Exploring the Interior of Folded RNA Molecules},
  journal = {Angewandte Chemie International Edition},
  year = {2004},
  volume = {43},
  pages = {3033-3037},
  doi = {http://dx.doi.org/10.1002/anie.200353575}
}
</pre></td>
</tr>
<tr id="sengupta2004" class="entry">
	<td>Sengupta, J., Nilsson, J., Gursky, R., Spahn, C. M. T., Nissen, P. &amp; Frank, J.</td>
	<td>Identification of the versatile scaffold protein RACK1 on the eukaryotic ribosome by cryo-EM. <p class="infolinks">[<a href="javascript:toggleInfo('sengupta2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sengupta2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Nat Struct Mol Biol &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nsmb822">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sengupta2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: RACK1 serves as a scaffold protein for a wide range of kinases and membrane-bound receptors. It is a WD-repeat family protein and is predicted to have a beta-propeller architecture with seven blades like a Gbeta protein. Mass spectrometry studies have identified its association with the small subunit of eukaryotic ribosomes and, most recently, it has been shown to regulate initiation by recruiting protein kinase C to the 40S subunit. Here we present the results of a cryo-EM study of the 80S ribosome that positively locate RACK1 on the head region of the 40S subunit, in the immediate vicinity of the mRNA exit channel. One face of RACK1 exposes the WD-repeats as a platform for interactions with kinases and receptors. Using this platform, RACK1 can recruit other proteins to the ribosome.</td>
</tr>
<tr id="bib_sengupta2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sengupta2004,
  author = {Jayati Sengupta and Jakob Nilsson and Richard Gursky and Christian M T Spahn and Poul Nissen and Joachim Frank},
  title = {Identification of the versatile scaffold protein RACK1 on the eukaryotic ribosome by cryo-EM.},
  journal = {Nat Struct Mol Biol},
  year = {2004},
  volume = {11},
  number = {10},
  pages = {957--962},
  doi = {http://dx.doi.org/10.1038/nsmb822}
}
</pre></td>
</tr>
<tr id="shi2000" class="entry">
	<td>Shi, H. &amp; Moore, P. B.</td>
	<td>The crystal structure of yeast phenylalanine tRNA at 1.93 A resolution: a classic structure revisited. <p class="infolinks">[<a href="javascript:toggleInfo('shi2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('shi2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>RNA &nbsp;</td>
	<td>article</td>
	<td><a href="http://rnajournal.cshlp.org/content/6/8/1091.abstract">URL</a> &nbsp;</td>
</tr>
<tr id="abs_shi2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structure of the monoclinic form of yeast phenylalanine tRNA has been redetermined at a resolution of 1.93 A. The structure of yeast tRNAphe described here is more accurate than its predecessors not only because it incorporates higher resolution data, but also because it has been refined using techniques that had not been developed when its predecessors were determined more than 20 years ago. The 1.93 A resolution version of this structure differs interestingly from its predecessors in its details. In loop regions particularly, the backbone torsion angles in the new structure are not the same as those reported earlier. Several new divalent cation binding sites have been identified, and the water structure that has emerged is also different.</td>
</tr>
<tr id="bib_shi2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{shi2000,
  author = {H. Shi and P. B. Moore},
  title = {The crystal structure of yeast phenylalanine tRNA at 1.93 A resolution: a classic structure revisited.},
  journal = {RNA},
  year = {2000},
  volume = {6},
  number = {8},
  pages = {1091--1105},
  url = {http://rnajournal.cshlp.org/content/6/8/1091.abstract}
}
</pre></td>
</tr>
<tr id="silvian1999" class="entry">
	<td>Silvian, L. F., Wang, J. &amp; Steitz, T. A.</td>
	<td>Insights into Editing from an Ile-tRNA Synthetase Structure with tRNAIle and Mupirocin <p class="infolinks">[<a href="javascript:toggleInfo('silvian1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('silvian1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.285.5430.1074">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_silvian1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Isoleucyl-transfer RNA (tRNA) synthetase (IleRS) joins Ile to tRNAIle at its synthetic active site and hydrolyzes incorrectly acylated amino acids at its editing active site. The 2.2 angstrom resolution crystal structure of Staphylococcus aureus IleRS complexed with tRNAIle and Mupirocin shows the acceptor strand of the tRNAIle in the continuously stacked, A-form conformation with the 3' terminal nucleotide in the editing active site. To position the 3' terminus in the synthetic active site, the acceptor strand must adopt the hairpinned conformation seen in tRNAGln complexed with its synthetase. The amino acid editing activity of the IleRS may result from the incorrect products shuttling between the synthetic and editing active sites, which is reminiscent of the editing mechanism of DNA polymerases.</td>
</tr>
<tr id="bib_silvian1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{silvian1999,
  author = {Laura F. Silvian and Jimin Wang and Thomas A. Steitz},
  title = {Insights into Editing from an Ile-tRNA Synthetase Structure with tRNAIle and Mupirocin},
  journal = {Science},
  year = {1999},
  volume = {285},
  pages = {1074-1077},
  doi = {http://dx.doi.org/10.1126/science.285.5430.1074}
}
</pre></td>
</tr>
<tr id="sorin2002" class="entry">
	<td>Sorin, E. J., Engelhardt, M. A. &amp; Herschlag, D.</td>
	<td>RNA Simulations: Probing Hairpin Unfolding and the Dynamics of a GNRA Tetraloop <p class="infolinks">[<a href="javascript:toggleInfo('sorin2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.2002.5447">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_sorin2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sorin2002,
  author = {Eric J. Sorin and Mark A. Engelhardt and Daniel Herschlag},
  title = {RNA Simulations: Probing Hairpin Unfolding and the Dynamics of a GNRA Tetraloop},
  journal = {Journal of Molecular Biology},
  year = {2002},
  volume = {317},
  pages = {493-506},
  doi = {http://dx.doi.org/10.1006/jmbi.2002.5447}
}
</pre></td>
</tr>
<tr id="sorin2004" class="entry">
	<td>Sorin, E. J., Nakatani, B. J., Rhee, Y. M., Jayachandran, G., Vishal, V. &amp; Pande, V. S.</td>
	<td>Does Native State Topology Determine the RNA Folding Mechanism? <p class="infolinks">[<a href="javascript:toggleInfo('sorin2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sorin2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2004.02.024">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sorin2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent studies in protein folding suggest that native state topology plays a dominant role in determining the folding mechanism, yet an analogous statement has not been made for RNA, most likely due to the strong coupling between the ionic environment and conformational energetics that make RNA folding more complex than protein folding. Applying a distributed computing architecture to sample nearly 5000 complete tRNA folding events using a minimalist, atomistic model, we have characterized the role of native topology in tRNA folding dynamics: the simulated bulk folding behavior predicts well the experimentally observed folding mechanism. In contrast, single-molecule folding events display multiple discrete folding transitions and compose a largely diverse, heterogeneous dynamic ensemble. This both supports an emerging view of heterogeneous folding dynamics at the microscopic level and highlights the need for single-molecule experiments and both single-molecule and bulk simulations in interpreting bulk experimental measurements.</td>
</tr>
<tr id="bib_sorin2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sorin2004,
  author = {Eric J. Sorin and Bradley J. Nakatani and Young Min Rhee and Guha Jayachandran and V Vishal and Vijay S. Pande},
  title = {Does Native State Topology Determine the RNA Folding Mechanism?},
  journal = {Journal of Molecular Biology},
  year = {2004},
  volume = {337},
  pages = {789-797},
  doi = {http://dx.doi.org/10.1016/j.jmb.2004.02.024}
}
</pre></td>
</tr>
<tr id="sorin2003" class="entry">
	<td>Sorin, E. J., Rhee, Y. M., Nakatani, B. J. &amp; Pande, V. S.</td>
	<td>Insights into Nucleic Acid Conformational Dynamics from Massively Parallel Stochastic Simulations <p class="infolinks">[<a href="javascript:toggleInfo('sorin2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sorin2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_sorin2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The helical hairpin is one of the most ubiquitous and elementary secondary structural motifs in nucleic acids,<p>capable of serving functional roles and participating in long-range tertiary contacts. Yet the self-assembly of these structures<p>has not been well-characterized at the atomic level. With this in mind, the dynamics of nucleic acid hairpin formation and<p>disruption have been studied using a novel computational tool: large-scale, parallel, atomistic molecular dynamics simulation<p>employing an inhomogeneous distributed computer consisting of more than 40,000 processors. Using multiple methodologies,<p>over 500 ms of atomistic simulation time has been collected for a large ensemble of hairpins (sequence 59-<p>GGGC[GCAA]GCCU-39), allowing characterization of rare events not previously observable in simulation. From uncoupled<p>ensemble dynamics simulations in unperturbed folding conditions, we report on 1), competing pathways between the folded and<p>unfolded regions of the conformational space; 2), observed nonnative stacking and basepairing traps; and 3), a helix unwindingrewindingmode that is differentiated from the unfolding and folding dynamics. A heterogeneous transition state ensemble is characterized structurally through calculations of conformer-specific folding probabilities and a multiplexed replica exchange stochastic dynamics algorithm is used to derive an approximate folding landscape. A comparison between the observed folding mechanism and that of a peptide b-hairpin analog suggests that although native topology defines the character of the folding landscape, the statistical weighting of potential folding pathways is determined by the chemical nature of the polymer.</td>
</tr>
<tr id="bib_sorin2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sorin2003,
  author = {Eric J. Sorin and Young Min Rhee and Bradley J. Nakatani and Vijay S. Pande},
  title = {Insights into Nucleic Acid Conformational Dynamics from Massively Parallel Stochastic Simulations},
  journal = {Biophysical Journal},
  year = {2003},
  volume = {85},
  pages = {790-803}
}
</pre></td>
</tr>
<tr id="sorin2005" class="entry">
	<td>Sorin, E. J., Rhee, Y. M. &amp; Pande, V. S.</td>
	<td>Does Water Play a Structural Role in the Folding of Small Nucleic Acids? <p class="infolinks">[<a href="javascript:toggleInfo('sorin2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sorin2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.104.055087">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sorin2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Nucleic acid structure and dynamics are known to be closely coupled to local environmental conditions and, in<p>particular, to the ionic character of the solvent. Here we consider what role the discrete properties of water and ions play in the<p>collapse and folding of small nucleic acids. We study the folding of an experimentally well-characterized RNA hairpin-loop motif<p>(sequence 59-GGGC[GCAA]GCCU-39) via ensemble molecular dynamics simulation and, with nearly 500 ms of aggregate<p>simulation time using an explicit representation of the ionic solvent, report successful ensemble folding simulations with<p>a predicted folding time of 8.8(62.0) ms, in agreement with experimental measurements of ;10 ms. Comparing our results to<p>previous folding simulations using the GB/SA continuum solvent model shows that accounting for water-mediated interactions is<p>necessary to accurately characterize the free energy surface and stochastic nature of folding. The formation of the secondary<p>structure appears to be more rapid than the fastest ionic degrees of freedom, and counterions do not participate discretely in<p>observed folding events. We find that hydrophobic collapse follows a predominantly expulsive mechanism in which a diffusionsearch<p>of early structural compaction is followed by the final formation of native structure that occurs in tandem with solvent<p>evacuation.</td>
</tr>
<tr id="bib_sorin2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sorin2005,
  author = {Eric J. Sorin and Young Min Rhee and Vijay S. Pande},
  title = {Does Water Play a Structural Role in the Folding of Small Nucleic Acids?},
  journal = {Biophysical Journal},
  year = {2005},
  volume = {88},
  pages = {2516-2524},
  doi = {http://dx.doi.org/10.1529/biophysj.104.055087}
}
</pre></td>
</tr>
<tr id="sosnick2003" class="entry">
	<td>Sosnick, T. R. &amp; Pan, T.</td>
	<td>RNA folding: models and perspectives <p class="infolinks">[<a href="javascript:toggleInfo('sosnick2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sosnick2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(03)00066-6">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sosnick2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Intrinsic events during RNA folding include conformational search and metal ion binding. Several experimentally testable models have been proposed to explain how large ribozymes accomplish folding. Future challenges include the validation of these models, and the correlation of experimental results and theoretical simulations.</td>
</tr>
<tr id="bib_sosnick2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sosnick2003,
  author = {Tobin R. Sosnick and Tao Pan},
  title = {RNA folding: models and perspectives},
  journal = {Current Opinion in Structural Biology},
  year = {2003},
  volume = {13},
  number = {3},
  pages = {309-316},
  doi = {http://dx.doi.org/10.1016/S0959-440X(03)00066-6}
}
</pre></td>
</tr>
<tr id="spackova2006" class="entry">
	<td>Spacková, N. &amp; Sponer, J.</td>
	<td>Molecular Dynamics Simulations of Sarcin-Ricin rRNA Motif <p class="infolinks">[<a href="javascript:toggleInfo('spackova2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('spackova2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkj470">DOI</a><a href="http://dx.doi.org/10.1093/nar/gkj470">URL</a> &nbsp;</td>
</tr>
<tr id="abs_spackova2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Explicit solvent molecular dynamics (MD) simulations were carried out for sarcin-ricin domain (SRD) motifs from 23S (Escherichia coli) and 28S (rat) rRNAs. The SRD motif consists of GAGA tetraloop, G-bulged cross-strand A-stack, flexible region and duplex part. Detailed analysis of the overall dynamics, base pairing, hydration, cation binding and other SRD features is presented. The SRD is surprisingly static in multiple 25 ns long simulations and lacks any non-local motions, with root mean square deviation (r.m.s.d.) values between averaged MD and high-resolution X-ray structures of 1-1.4 A. Modest dynamics is observed in the tetraloop, namely, rotation of adenine in its apex and subtle reversible shift of the tetraloop with respect to the adjacent base pair. The deformed flexible region in low-resolution rat X-ray structure is repaired by simulations. The simulations reveal few backbone flips, which do not affect positions of bases and do not indicate a force field imbalance. Non-Watson-Crick base pairs are rigid and mediated by long-residency water molecules while there are several modest cation-binding sites around SRD. In summary, SRD is an unusually stiff rRNA building block. Its intrinsic structural and dynamical signatures seen in simulations are strikingly distinct from other rRNA motifs such as Loop E and Kink-turns.</td>
</tr>
<tr id="bib_spackova2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{spackova2006,
  author = {Nad'a Spacková and Jirí Sponer},
  title = {Molecular Dynamics Simulations of Sarcin-Ricin rRNA Motif},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  number = {2},
  pages = {697-708},
  url = {http://dx.doi.org/10.1093/nar/gkj470},
  doi = {http://dx.doi.org/10.1093/nar/gkj470}
}
</pre></td>
</tr>
<tr id="spahn2001" class="entry">
	<td>Spahn, C. M. T., Beckmann, R., Eswar, N., Penczek, P., Sali, A., Blobel, G. &amp; Frank, J.</td>
	<td>Structure of the 80S Ribosome from Saccharomyces cerevisiae?tRNA-Ribosome and Subunit-Subunit Interactions <p class="infolinks">[<a href="javascript:toggleInfo('spahn2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('spahn2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Cell &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0092-8674(01)00539-6">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_spahn2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A cryo-EM reconstruction of the translating yeast 80S ribosome was analyzed. Computationally separated rRNA and protein densities were used for docking of appropriately modified rRNA models and homology models of yeast ribosomal proteins. The core of the ribosome shows a remarkable degree of conservation. However, some significant differences in functionally important regions and dramatic changes in the periphery due to expansion segments and additional ribosomal proteins are evident. As in the bacterial ribosome, bridges between the subunits are mainly formed by RNA contacts. Four new bridges are present at the periphery. The position of the P site tRNA coincides precisely with its prokaryotic counterpart, with mainly rRNA contributing to its molecular environment. This analysis presents an exhaustive inventory of an eukaryotic ribosome at the molecular level.</td>
</tr>
<tr id="bib_spahn2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{spahn2001,
  author = {C. M. T. Spahn and R. Beckmann and N. Eswar and P.A. Penczek and A. Sali and G. Blobel and Joachim Frank},
  title = {Structure of the 80S Ribosome from Saccharomyces cerevisiae?tRNA-Ribosome and Subunit-Subunit Interactions},
  journal = {Cell},
  year = {2001},
  volume = {107},
  pages = {373-386},
  doi = {http://dx.doi.org/10.1016/S0092-8674(01)00539-6}
}
</pre></td>
</tr>
<tr id="spirin2002" class="entry">
	<td>Spirin, A.</td>
	<td>Omnipotent RNA <p class="infolinks">[<a href="javascript:toggleInfo('spirin2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('spirin2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>FEBS &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0014-5793(02)03434-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_spirin2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The capability of polyribonucleotide chains to form unique, compactly folded structures is considered the basis for diverse non-genetic functions of RNA, including the function of recognition of various ligands and the catalytic function. Together with well-known genetic functions of RNA ? coding and complementary replication ? this has led to the concept of the functional omnipotence of RNA and the hypothesis that an ancient RNA world supposedly preceded the contemporary DNA?RNA?protein life. It is proposed that the Woese universal precursor in the ancient RNA world could be a cell-free community of mixed RNA colonies growing and multiplying on solid surfaces.</td>
</tr>
<tr id="bib_spirin2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{spirin2002,
  author = {Alexander Spirin},
  title = {Omnipotent RNA},
  journal = {FEBS},
  year = {2002},
  volume = {530},
  pages = {4-8},
  doi = {http://dx.doi.org/10.1016/S0014-5793(02)03434-8}
}
</pre></td>
</tr>
<tr id="sponer2000" class="entry">
	<td>Sponer, J., Berger, I., Spackova, N., Leszczynski, J. &amp; Hobza, P.</td>
	<td>Aromatic Base Stacking in DNA: From ab initio Calculations to Molecular Dynamics Simulations <p class="infolinks">[<a href="javascript:toggleInfo('sponer2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Journal of Biomolecular Structure and Dynamics &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_sponer2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer2000,
  author = {Jiri Sponer and Imre Berger and Nada Spackova and Jerzy Leszczynski and Pavel Hobza},
  title = {Aromatic Base Stacking in DNA: From ab initio Calculations to Molecular Dynamics Simulations},
  journal = {Journal of Biomolecular Structure and Dynamics},
  year = {2000},
  volume = {11},
  pages = {1-24}
}
</pre></td>
</tr>
<tr id="sponer2000nar" class="entry">
	<td>Sponer, J., Florian, J., Ng, H., Sponer, J. E. &amp; Spackova, N.</td>
	<td>Local conformational variations observed in B-DNA crystals do not improve base stacking: computational analysis of base stacking in a d(CATGGGCCCATG)2 B<->A intermediate crystal structure. <p class="infolinks">[<a href="javascript:toggleInfo('sponer2000nar','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sponer2000nar','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/28.24.4893">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sponer2000nar" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The crystal structure of d(CATGGGCCCATG)2 shows unique stacking patterns of a stable B<--->A-DNA intermediate. We evaluated intrinsic base stacking energies in this crystal structure using an ab initio quantum mechanical method. We found that all crystal base pair steps have stacking energies close to their values in the standard and crystal B-DNA geometries. Thus, naturally occurring stacking geometries were essentially isoenergetic while individual base pair steps differed substantially in the balance of intra-strand and inter-strand stacking terms. Also, relative dispersion, electrostatic and polarization contributions to the stability of different base pair steps were very sensitive to base composition and sequence context. A large stacking flexibility is most apparent for the CpA step, while the GpG step is characterized by weak intra-strand stacking. Hydration effects were estimated using the Langevin dipoles solvation model. These calculations showed that an aqueous environment efficiently compensates for electrostatic stacking contributions. Finally, we have carried out explicit solvent molecular dynamics simulation of the d(CATGGGCCCATG)2 duplex in water. Here the DNA conformation did not retain the initial crystal geometry, but moved from the B<--->A intermediate towards the B-DNA structure. The base stacking energy improved in the course of this simulation. Our findings indicate that intrinsic base stacking interactions are not sufficient to stabilize the local conformational variations in crystals.</td>
</tr>
<tr id="bib_sponer2000nar" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer2000nar,
  author = {Jiri Sponer and Jan Florian and Ho-Leung Ng and Judit E. Sponer and Nadia Spackova},
  title = {Local conformational variations observed in B-DNA crystals do not improve base stacking: computational analysis of base stacking in a d(CATGGGCCCATG)2 B<->A intermediate crystal structure.},
  journal = {Nucleic Acids Research},
  year = {2000},
  volume = {28},
  pages = {4893-4902},
  doi = {http://dx.doi.org/10.1093/nar/28.24.4893}
}
</pre></td>
</tr>
<tr id="sponer2006" class="entry">
	<td>Sponer, J., Jureka, P., Marchan, I., Luque, F. J., Orozco, M. &amp; Hobza, P.</td>
	<td>Nature of Base Stacking: Reference Quantum-Chemical Stacking Energies in Ten Unique B-DNA Base-Pair Steps. <p class="infolinks">[<a href="javascript:toggleInfo('sponer2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sponer2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Chemistry - A European Journal &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_sponer2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Base-stacking energies in ten unique B-DNA base-pair steps and some other arrangements were evaluated by the second-order Moller-Plesset (MP2) method, complete basis set (CBS) extrapolation, and correction for triple (T) electron-correlation contributions. The CBS(T) calculations were compared with decade-old MP2/6-31G*(0.25) reference data and AMBER force field. The new calculations show modest increases in stacking stabilization compared to the MP2/6-31G*(0.25) data and surprisingly large sequence-dependent variation of stacking energies. The absolute force-field values are in better agreement with the new reference data, while relative discrepancies between quantum-chemical (QM) and force-field values increase modestly. Nevertheless, the force field provides good qualitative description of stacking, and there is no need to introduce additional pair-additive electrostatic terms, such as distributed multipoles or out-of-plane charges. There is a rather surprising difference of about 0.1 A between the vertical separation of base pairs predicted by quantum chemistry and derived from crystal structures. Evaluations of different local arrangements of the 5'-CG-3' step indicate a sensitivity of the relative stacking energies to the level of calculation. Thus, describing quantitative relations between local DNA geometrical variations and stacking may be more complicated than usually assumed. The reference calculations are complemented by continuum-solvent assessment of solvent-screening effects.</td>
</tr>
<tr id="bib_sponer2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer2006,
  author = {Jiri Sponer and Petr Jureka and Ivan Marchan and F. Javier Luque and Modesto Orozco and Pavel Hobza},
  title = {Nature of Base Stacking: Reference Quantum-Chemical Stacking Energies in Ten Unique B-DNA Base-Pair Steps.},
  journal = {Chemistry - A European Journal},
  year = {2006}
}
</pre></td>
</tr>
<tr id="sponer2002" class="entry">
	<td>Sponer, J., Leszczynski, J. &amp; Hobza, P.</td>
	<td>Electronic properties, hydrogen bonding, stacking, and cation binding of DNA and RNA bases <p class="infolinks">[<a href="javascript:toggleInfo('sponer2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sponer2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Biopolymers &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/bip.10048">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sponer2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: This review summarizes results concerning molecular interactions of nucleic acid bases as revealed by advanced ab initio quantum chemical (QM) calculations published in last few years. We first explain advantages and limitations of modern QM calculations of nucleobases and provide a brief history of this still rather new field. Then we provide an overview of key electronic properties of standard and selected modified nucleobases, such as their charge distributions, dipole moments, polarizabilities, proton affinities, tautomeric equilibria, and amino group hybridization. Then we continue with hydrogen bonding of nucleobases, by analyzing energetics of standard base pairs, mismatched base pairs, thio-base pairs, and others. After this, the nature of aromatic stacking interactions is explained. Also, nonclassical interactions in nucleic acids such as interstrand bifurcated hydrogen bonds, interstrand close amino group contacts, C [bond] H...O interbase contacts, sugar-base stacking, intrinsically nonplanar base pairs, out-of-plane hydrogen bonds, and amino-acceptor interactions are commented on. Finally, we overview recent calculations on interactions between nucleic acid bases and metal cations. These studies deal with effects of cation binding on the strength of base pairs, analysis of specific differences among cations, such as the difference between zinc and magnesium, the influence of metalation on protonation and tautomeric equlibria of bases, and cation-pi interactions involving nucleobases. In this review, we do not provide methodological details, as these can be found in our preceding reviews. The interrelation between advanced QM approaches and classical molecular dynamics simulations is briefly discussed.</td>
</tr>
<tr id="bib_sponer2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer2002,
  author = {Jiri Sponer and Jerzy Leszczynski and Pavel Hobza},
  title = {Electronic properties, hydrogen bonding, stacking, and cation binding of DNA and RNA bases},
  journal = {Biopolymers},
  year = {2002},
  volume = {61},
  pages = {3-31},
  doi = {http://dx.doi.org/10.1002/bip.10048}
}
</pre></td>
</tr>
<tr id="sponer1997" class="entry">
	<td>Sponer, J., Leszczynski, J. &amp; Hobza, P.</td>
	<td>Thioguanine and Thiouracil: Hydrogen-Bonding and Stacking Properties <p class="infolinks">[<a href="javascript:toggleInfo('sponer1997','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sponer1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Journal of Physical Chemistry A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp9720404">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sponer1997" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Base stacking and H-bonding properties of thioguanine and thiouracils were studied using an ab initio quantum chemical method with inclusion of electron correlation (second-order Mller-Plesset perturbational method). Hydrogen-bonded base pairs containing thiobases are only slightly less stable (up to 2 kcal/mol) than the unmodified base pairs. The Nï¿½ï¿½ï¿½S distances are larger by 0.4-0.7 ï¿½ compared to the Nï¿½ï¿½ï¿½O distances in the standard base pairs. The thio group enhances polarizability of the monomers and their dipole moments. Thus, in stacked complexes of thiobases, both dispersion attraction and electrostatic interactions are enhanced. Mutual contact of the sulfur atoms and their interaction with second-row elements lead to steric clashes destabilizing the stacking, though, in DNA, such clashes should be eliminated by rather small adjustments of the local DNA conformation. The thio group significantly destabilizes the hydration of the 6-position of thioguanine with respect to guanine. The first hydration shell in the major groove might be significantly altered by thioguanine.</td>
</tr>
<tr id="bib_sponer1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer1997,
  author = {J. Sponer and J. Leszczynski and P. Hobza},
  title = {Thioguanine and Thiouracil: Hydrogen-Bonding and Stacking Properties},
  journal = {Journal of Physical Chemistry A},
  year = {1997},
  volume = {101},
  pages = {9489-9495},
  doi = {http://dx.doi.org/10.1021/jp9720404}
}
</pre></td>
</tr>
<tr id="sponer1996" class="entry">
	<td>Sponer, J., Leszczynski, J. &amp; Hobza, P.</td>
	<td>Nature of Nucleic Acid-Base Stacking: Nonempirical ab Initio and Empirical Potential Characterization of 10 Stacked Base Dimers. Comparison of Stacked and H-Bonded Base Pairs <p class="infolinks">[<a href="javascript:toggleInfo('sponer1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sponer1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Journal of Physical Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp953306e">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sponer1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Ab initio (MP2/6-31G*(0.25)) interaction energies were calculated for almost 240 geometries of 10 stacked nucleic acid-base pairs: Aï¿½ï¿½ï¿½A, Cï¿½ï¿½ï¿½C, Gï¿½ï¿½ï¿½G, Uï¿½ï¿½ï¿½U, Aï¿½ï¿½ï¿½C, Gï¿½ï¿½ï¿½A, Aï¿½ï¿½ï¿½U, Gï¿½ï¿½ï¿½C, Cï¿½ï¿½ï¿½U, and Gï¿½ï¿½ï¿½U; in some cases uracil was replaced by thymine. The most stable stacked pair is the Gï¿½ï¿½ï¿½G dimer (-11.3 kcal/mol), and the least stable is the uracil dimer (-6.5 kcal/mol). The interaction energies of H-bonded base pairs range from -25.8 kcal/mol (Gï¿½ï¿½ï¿½C) to -10.6 kcal/mol (Tï¿½ï¿½ï¿½T). The stability of stacked pairs originates in the electron correlation, while all the H-bonded pairs are dominated by the HF energy. The mutual orientation of the stacked bases is, however, primarily determined by the HF interaction energy. The ab initio base stacking energies are well reproduced by the empirical potential calculations, provided the atomic charges are derived by the same method as used in the ab initio calculations. Some contributions previously postulated to significantly influence base stacking (induction interactions, - interactions) were not found. Base stacking was also investigated in six B-DNA and two Z-DNA base pair steps; their geometries were taken from the oligonucleotide crystal data. The many-body correction was estimated at the HF/MINI-1 level. The sequence-dependent variations of the total base pair step stacking energies range from -9.9 to -14.7 kcal/mol. The range of the calculated many-body corrections to the stacking energy is 2 kcal/mol. The ab initio calculations exclude the consideration that the unusual conformational properties of the CpA(TpG) steps might be associated with attractive induction interactions of the exocyclic groups of DNA bases and the aromatic rings of bases.</td>
</tr>
<tr id="bib_sponer1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer1996,
  author = {J. Sponer and J. Leszczynski and P. Hobza},
  title = {Nature of Nucleic Acid-Base Stacking: Nonempirical ab Initio and Empirical Potential Characterization of 10 Stacked Base Dimers. Comparison of Stacked and H-Bonded Base Pairs},
  journal = {Journal of Physical Chemistry},
  year = {1996},
  volume = {100},
  pages = {5590-5596},
  doi = {http://dx.doi.org/10.1021/jp953306e}
}
</pre></td>
</tr>
<tr id="sponer2007" class="entry">
	<td>Sponer, J. &amp; Spackova, N.</td>
	<td>Molecular dynamics simulations and their application to four-stranded DNA <p class="infolinks">[<a href="javascript:toggleInfo('sponer2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Methods &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.ymeth.2007.02.004">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_sponer2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sponer2007,
  author = {Jiri Sponer and Nadia Spackova},
  title = {Molecular dynamics simulations and their application to four-stranded DNA},
  journal = {Methods},
  year = {2007},
  volume = {43},
  pages = {278-290},
  doi = {http://dx.doi.org/10.1016/j.ymeth.2007.02.004}
}
</pre></td>
</tr>
<tr id="stagg2003" class="entry">
	<td>Stagg, S. M., Mears, J. A. &amp; Harvey, S. C.</td>
	<td>A Structural Model for the Assembly of the 30 S Subunit of the Ribosome <p class="infolinks">[<a href="javascript:toggleInfo('stagg2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('stagg2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0022-2836(03)00174-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_stagg2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The order in which proteins bind to 16 S rRNA, the assembly map, was determined by Nomura and co-workers in the early 1970s. The assembly map shows the dependencies of binding of successive proteins but fails to address the relationship of these dependencies to the three-dimensional folding of the ribosome. Here, using molecular mechanics techniques, we rationalize the order of protein binding in terms of ribosomal folding. We determined the specific contacts between the ribosomal proteins and 16 S rRNA from a crystal structure of the 30 S subunit (1FJG). We then used these contacts as restraints in a rigid body Monte?Carlo simulation with reduced-representation models of the RNA and proteins. Proteins were added sequentially to the RNA in the order that they appear in the assembly map. Our results show that proteins nucleate the folding of the head, platform, and body domains, but they do not strongly restrict the orientations of the domains relative to one another. We also examined the contributions of individual proteins to the formation of binding sites for sequential proteins in the assembly process. Binding sites for the primary binding proteins are generally more ordered in the naked RNA than those for other proteins. Furthermore, we examined one pathway in the assembly map and found that the addition of early binding proteins helps to organize the RNA around the binding sites of proteins that bind later. It appears that the order of assembly depends on the degree of pre-organization of each protein's binding site at a given stage of assembly, and the impact that the binding of each protein has on the organization of the remaining unoccupied binding sites.</td>
</tr>
<tr id="bib_stagg2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{stagg2003,
  author = {Scott M. Stagg and Jason A. Mears and Stephen C. Harvey},
  title = {A Structural Model for the Assembly of the 30 S Subunit of the Ribosome},
  journal = {Journal of Molecular Biology},
  year = {2003},
  volume = {328},
  pages = {49-61},
  doi = {http://dx.doi.org/10.1016/S0022-2836(03)00174-8}
}
</pre></td>
</tr>
<tr id="stonge2007" class="entry">
	<td>Stonge, K., Thibault, P., Hamel, S. &amp; Major, F.</td>
	<td>Modeling RNA Tertiary Structure Motifs by Graph-Grammars <p class="infolinks">[<a href="javascript:toggleInfo('stonge2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('stonge2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Nucleic Acids Research, 2007, 1ï¿½11 &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkm069">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_stonge2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A new approach, graph-grammars, to encode<p>RNA tertiary structure patterns is introduced and<p>exemplified with the classical sarcinï¿½ricin motif.<p>The sarcinï¿½ricin motif is found in the stem of the<p>crucial ribosomal loop E (also referred to as the<p>sarcinï¿½ricin loop), which is sensitive to the a-sarcin<p>and ricin toxins. Here, we generate a graphgrammar<p>for the sarcin-ricin motif and apply it to<p>derive putative sequences that would fold in this<p>motif. The biological relevance of the derived<p>sequences is confirmed by a comparison with<p>those found in known sarcinï¿½ricin sites in an<p>alignment of over 800 bacterial 23S ribosomal<p>RNAs. The comparison raised alternative alignments<p>in few sarcinï¿½ricin sites, which were<p>assessed using tertiary structure predictions and<p>3D modeling. The sarcinï¿½ricin motif graph-grammar<p>was built with indivisible nucleotide interaction<p>cycles that were recently observed in structured<p>RNAs. A comparison of the sequences and 3D<p>structures of each cycle that constitute the sarcinï¿½<p>ricin motif gave us additional insights about RNA<p>sequenceï¿½structure relationships. In particular,<p>this analysis revealed the sequence space of an<p>RNA motif depends on a structural context that<p>goes beyond the single base pairing and basestacking<p>interactions.</td>
</tr>
<tr id="bib_stonge2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{stonge2007,
  author = {Karine Stonge and Philippe Thibault and Sylvie Hamel and Francois Major},
  title = {Modeling RNA Tertiary Structure Motifs by Graph-Grammars},
  journal = {Nucleic Acids Research, 2007, 1ï¿½11},
  year = {2007},
  pages = {1-11},
  doi = {http://dx.doi.org/10.1093/nar/gkm069}
}
</pre></td>
</tr>
<tr id="strobel1997" class="entry">
	<td>Strobel, S. A. &amp; Doudna, J. A.</td>
	<td>RNA Seeing Double: Close-Packing of Helices in RNA Tertiary Structure <p class="infolinks">[<a href="javascript:toggleInfo('strobel1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>Trends in Biochemical Sciences &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_strobel1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{strobel1997,
  author = {Scott A. Strobel and Jennifer A. Doudna},
  title = {RNA Seeing Double: Close-Packing of Helices in RNA Tertiary Structure},
  journal = {Trends in Biochemical Sciences},
  year = {1997},
  volume = {22},
  pages = {262-266}
}
</pre></td>
</tr>
<tr id="su2003" class="entry">
	<td>Su, L. J., Brenowitz, M. &amp; Pyle, A. M.</td>
	<td>An Alternative Route for the Folding of Large RNAs: Apparent Two-state Folding by a Group II Intron Ribozyme <p class="infolinks">[<a href="javascript:toggleInfo('su2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('su2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2003.09.071">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_su2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Despite a growing literature on the folding of RNA, our understanding of tertiary folding in large RNAs derives from studies on a small set of molecular examples, with primary focus on group I introns and RNase P RNA. To broaden the scope of RNA folding models and to better understand group II intron function, we have examined the tertiary folding of a ribozyme (D135) that is derived from the self-splicing ai5 intron from yeast mitochondria. The D135 ribozyme folds homogeneously and cooperatively into a compact, well-defined tertiary structure that includes all regions critical for active-site organization and substrate recognition. When D135 was treated with increasing concentrations of Mg2+ and then subjected to hydroxyl radical footprinting, similar Mg2+ dependencies were seen for internalization of all regions of the molecule, suggesting a highly cooperative folding behavior. In this work, we show that global folding and compaction of the molecule have the same magnesium dependence as the local folding previously observed. Furthermore, urea denaturation studies indicate highly cooperative unfolding of the ribozyme that is governed by thermodynamic parameters similar to those for forward folding. In fact, D135 folds homogeneously and cooperatively from the unfolded state to its native, active structure, thereby demonstrating functional reversibility in RNA folding. Taken together, the data are consistent with two-state folding of the D135 ribozyme, which is surprising given the size and multi-domain structure of the RNA. The findings establish that the accumulation of stable intermediates prior to formation of the native state is not a universal feature of RNA folding and that there is an alternative paradigm in which the folding landscape is relatively smooth, lacking rugged features that obstruct folding to the native state.</td>
</tr>
<tr id="bib_su2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{su2003,
  author = {Linhui Julie Su and Michael Brenowitz and Anna Marie Pyle},
  title = {An Alternative Route for the Folding of Large RNAs: Apparent Two-state Folding by a Group II Intron Ribozyme},
  journal = {Journal of Molecular Biology},
  year = {2003},
  volume = {334},
  number = {4},
  pages = {639-652},
  doi = {http://dx.doi.org/10.1016/j.jmb.2003.09.071}
}
</pre></td>
</tr>
<tr id="su2005" class="entry">
	<td>Su, L. J., Waldsich, C. &amp; Pyle, A. M.</td>
	<td>An obligate intermediate along the slow folding pathway of a group II intron ribozyme <p class="infolinks">[<a href="javascript:toggleInfo('su2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('su2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gki973">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_su2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Most RNA molecules collapse rapidly and reach the native state through a pathway that contains numerous traps and unproductive intermediates. The D135 group II intron ribozyme is unusual in that it can fold slowly and directly to the native state, despite its large size and structural complexity. Here we use hydroxyl radical footprinting and native gel analysis to monitor the timescale of tertiary structure collapse and to detect the presence of obligate intermediates along the folding pathway of D135. We find that structural collapse and native folding of Domain 1 precede assembly of the entire ribozyme, indicating that D1 contains an on-pathway intermediate to folding of the D135 ribozyme. Subsequent docking of Domains 3 and 5, for which D1 provides a preorganized scaffold, appears to be very fast and independent of one another. In contrast to other RNAs, the D135 ribozyme undergoes slow tertiary collapse to a compacted state, with a rate constant that is also limited by the formation D1. These findings provide a new paradigm for RNA folding and they underscore the diversity of RNA biophysical behaviors.</td>
</tr>
<tr id="bib_su2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{su2005,
  author = {Linhui Julie Su and Christina Waldsich and Anna Marie Pyle},
  title = {An obligate intermediate along the slow folding pathway of a group II intron ribozyme},
  journal = {Nucleic Acids Research},
  year = {2005},
  volume = {33},
  pages = {6674-6687},
  doi = {http://dx.doi.org/10.1093/nar/gki973}
}
</pre></td>
</tr>
<tr id="sykes2005" class="entry">
	<td>Sykes, M. T. &amp; Levitt, M.</td>
	<td>Describing RNA Structure by Libraries of Clustered Nucleotide Doublets <p class="infolinks">[<a href="javascript:toggleInfo('sykes2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('sykes2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2005.06.024">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_sykes2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The rapidly increasing wealth of structural information on RNA and knowledge of its varying roles in biology have facilitated the study of RNA structure using computational methods. Here, we present a new method to describe RNA structure based on nucleotide doublets, where a doublet is any two nucleotides in a structure. We restrict our search to doublets that are close together in space, but not necessarily in sequence, and obtain doublet libraries of various sizes by clustering a large set of doublets taken from a data set of high-resolution RNA structures. We demonstrate that these libraries are able to both capture structural features present in RNA and fit local RNA structure with a high level of accuracy. Libraries ranging in size from ten to 100 doublets are examined, and a detailed analysis shows that a library with as few as 30 doublets is sufficient to capture the most common structural features, while larger libraries would be more appropriate for accurate modeling. We anticipate many uses for these libraries, from annotation to structure refinement and prediction.</td>
</tr>
<tr id="bib_sykes2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sykes2005,
  author = {Michael T. Sykes and Michael Levitt},
  title = {Describing RNA Structure by Libraries of Clustered Nucleotide Doublets},
  journal = {Journal of Molecular Biology},
  year = {2005},
  volume = {351},
  pages = {26-38},
  doi = {http://dx.doi.org/10.1016/j.jmb.2005.06.024}
}
</pre></td>
</tr>
<tr id="tan2006" class="entry">
	<td>Tan, Z. &amp; Chen, S.</td>
	<td>Electrostatic Free Energy Landscapes for Nucleic Acid Helix Assembly <p class="infolinks">[<a href="javascript:toggleInfo('tan2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('tan2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkl810">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_tan2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Metal ions are crucial for nucleic acid folding. From the free energy landscapes, we investigate the detailed mechanism for ion-induced collapse for a paradigm system: loop-tethered short DNA helices. We find that Na+ and Mg2+ play distinctive roles in helixï¿½helix assembly. High [Na+] (>0.3 M) causes a reduced helixï¿½helix electrostatic repulsion and a subsequent disordered packing of helices. In contrast, Mg2+ of concentration >1 mM is predicted to induce helixï¿½helix attraction and results in a more compact and ordered helixï¿½helix packing. Mg2+ is much more efficient in causing nucleic acid compaction. In addition, the free energy landscape shows that the tethering loops between the helices also play a significant role. A flexible loop, such as a neutral loop or a polynucleotide loop in high salt concentration, enhances the close approach of the helices in order to gain the loop entropy. On the other hand, a rigid loop, such as a polynucleotide loop in low salt concentration, tends to de-compact the helices. Therefore, a polynucleotide loop significantly enhances the sharpness of the ion-induced compaction transition. Moreover, we find that a larger number of helices in the system or a smaller radius of the divalent ions can cause a more abrupt compaction transition and a more compact state at high ion concentration, and the ion size effect becomes more pronounced as the number of helices is increased.</td>
</tr>
<tr id="bib_tan2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tan2006,
  author = {Zhi-Jie Tan and Shi-Jie Chen},
  title = {Electrostatic Free Energy Landscapes for Nucleic Acid Helix Assembly},
  journal = {Nucleic Acids Research},
  year = {2006},
  volume = {34},
  pages = {6629-6639},
  doi = {http://dx.doi.org/10.1093/nar/gkl810}
}
</pre></td>
</tr>
<tr id="theobald2005b" class="entry">
	<td>Theobald, D. L.</td>
	<td>Rapid calculation of RMSDs using a quaternion-based characteristic polynomial. <p class="infolinks">[<a href="javascript:toggleInfo('theobald2005b','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('theobald2005b','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Acta Crystallogr A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1107/S0108767305015266">DOI</a><a href="http://dx.doi.org/10.1107/S0108767305015266">URL</a> &nbsp;</td>
</tr>
<tr id="abs_theobald2005b" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: A common measure of conformational similarity in structural bioinformatics is the minimum root mean square deviation (RMSD) between the coordinates of two macromolecules. In many applications, the rotations relating the structures are not needed. Several common algorithms for calculating RMSDs require the computationally costly procedures of determining either the eigen decomposition or matrix inversion of a 3x3 or 4x4 matrix. Using a quaternion-based method, here a simple algorithm is developed that rapidly and stably determines RMSDs by circumventing the decomposition and inversion problems.</td>
</tr>
<tr id="bib_theobald2005b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{theobald2005b,
  author = {Douglas L Theobald},
  title = {Rapid calculation of RMSDs using a quaternion-based characteristic polynomial.},
  journal = {Acta Crystallogr A},
  year = {2005},
  volume = {61},
  number = {Pt 4},
  pages = {478--480},
  url = {http://dx.doi.org/10.1107/S0108767305015266},
  doi = {http://dx.doi.org/10.1107/S0108767305015266}
}
</pre></td>
</tr>
<tr id="theobald2008" class="entry">
	<td>Theobald, D. L. &amp; Wuttke, D. S.</td>
	<td>Accurate structural correlations from maximum likelihood superpositions. <p class="infolinks">[<a href="javascript:toggleInfo('theobald2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('theobald2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>PLoS Comput Biol &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1371/journal.pcbi.0040043">DOI</a><a href="http://dx.doi.org/10.1371/journal.pcbi.0040043">URL</a> &nbsp;</td>
</tr>
<tr id="abs_theobald2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The cores of globular proteins are densely packed, resulting in complicated networks of structural interactions. These interactions in turn give rise to dynamic structural correlations over a wide range of time scales. Accurate analysis of these complex correlations is crucial for understanding biomolecular mechanisms and for relating structure to function. Here we report a highly accurate technique for inferring the major modes of structural correlation in macromolecules using likelihood-based statistical analysis of sets of structures. This method is generally applicable to any ensemble of related molecules, including families of nuclear magnetic resonance (NMR) models, different crystal forms of a protein, and structural alignments of homologous proteins, as well as molecular dynamics trajectories. Dominant modes of structural correlation are determined using principal components analysis (PCA) of the maximum likelihood estimate of the correlation matrix. The correlations we identify are inherently independent of the statistical uncertainty and dynamic heterogeneity associated with the structural coordinates. We additionally present an easily interpretable method ("PCA plots") for displaying these positional correlations by color-coding them onto a macromolecular structure. Maximum likelihood PCA of structural superpositions, and the structural PCA plots that illustrate the results, will facilitate the accurate determination of dynamic structural correlations analyzed in diverse fields of structural biology.</td>
</tr>
<tr id="bib_theobald2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{theobald2008,
  author = {Douglas L Theobald and Deborah S Wuttke},
  title = {Accurate structural correlations from maximum likelihood superpositions.},
  journal = {PLoS Comput Biol},
  year = {2008},
  volume = {4},
  number = {2},
  pages = {e43},
  url = {http://dx.doi.org/10.1371/journal.pcbi.0040043},
  doi = {http://dx.doi.org/10.1371/journal.pcbi.0040043}
}
</pre></td>
</tr>
<tr id="theobald2006" class="entry">
	<td>Theobald, D. L. &amp; Wuttke, D. S.</td>
	<td>Empirical Bayes hierarchical models for regularizing maximum likelihood estimation in the matrix Gaussian Procrustes problem. <p class="infolinks">[<a href="javascript:toggleInfo('theobald2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('theobald2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Proc Natl Acad Sci U S A &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.0508445103">DOI</a><a href="http://dx.doi.org/10.1073/pnas.0508445103">URL</a> &nbsp;</td>
</tr>
<tr id="abs_theobald2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Procrustes analysis involves finding the optimal superposition of two or more "forms" via rotations, translations, and scalings. Procrustes problems arise in a wide range of scientific disciplines, especially when the geometrical shapes of objects are compared, contrasted, and analyzed. Classically, the optimal transformations are found by minimizing the sum of the squared distances between corresponding points in the forms. Despite its widespread use, the ordinary unweighted least-squares (LS) criterion can give erroneous solutions when the errors have heterogeneous variances (heteroscedasticity) or the errors are correlated, both common occurrences with real data. In contrast, maximum likelihood (ML) estimation can provide accurate and consistent statistical estimates in the presence of both heteroscedasticity and correlation. Here we provide a complete solution to the nonisotropic ML Procrustes problem assuming a matrix Gaussian distribution with factored covariances. Our analysis generalizes, simplifies, and extends results from previous discussions of the ML Procrustes problem. An iterative algorithm is presented for the simultaneous, numerical determination of the ML solutions.</td>
</tr>
<tr id="bib_theobald2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{theobald2006,
  author = {Douglas L Theobald and Deborah S Wuttke},
  title = {Empirical Bayes hierarchical models for regularizing maximum likelihood estimation in the matrix Gaussian Procrustes problem.},
  journal = {Proc Natl Acad Sci U S A},
  year = {2006},
  volume = {103},
  number = {49},
  pages = {18521--18527},
  url = {http://dx.doi.org/10.1073/pnas.0508445103},
  doi = {http://dx.doi.org/10.1073/pnas.0508445103}
}
</pre></td>
</tr>
<tr id="thirumalai2005" class="entry">
	<td>Thirumalai, D. &amp; Hyeon, C.</td>
	<td>RNA and Protein Folding: Common Themes and Variations <p class="infolinks">[<a href="javascript:toggleInfo('thirumalai2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('thirumalai2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biochemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/bi047314+ S0006-2960(04)07314-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_thirumalai2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Visualizing the navigation of an ensemble of unfolded molecules through the bumpy energy landscape in search of the native state gives a pictorial view of biomolecular folding. This picture, when combined with concepts in polymer theory, provides a unified theory of RNA and protein folding. Just as for proteins, the major folding free energy barrier for RNA scales sublinearly with the number of nucleotides, which allows us to extract the elusive prefactor for RNA folding. Several folding scenarios can be anticipated by considering variations in the energy landscape that depend on sequence, native topology, and external conditions. RNA and protein folding mechanism can be described by the kinetic partitioning mechanism (KPM) according to which a fraction () of molecules reaches the native state directly, whereas the remaining fraction gets kinetically trapped in metastable conformations. For two-state folders 1. Molecular chaperones are recruited to assist protein folding whenever is small. We show that the iterative annealing mechanism, introduced to describe chaperonin-mediated folding, can be generalized to understand protein-assisted RNA folding. The major differences between the folding of proteins and RNA arise in the early stages of folding. For RNA, folding can only begin after the polyelectrolyte problem is solved, whereas protein collapse requires burial of hydrophobic residues. Cross-fertilization of ideas between the two fields should lead to an understanding of how RNA and proteins solve their folding problems.</td>
</tr>
<tr id="bib_thirumalai2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{thirumalai2005,
  author = {D. Thirumalai and Changbong Hyeon },
  title = {RNA and Protein Folding: Common Themes and Variations},
  journal = {Biochemistry},
  year = {2005},
  volume = {44},
  pages = {4957-4970},
  doi = {http://dx.doi.org/10.1021/bi047314+ S0006-2960(04)07314-3}
}
</pre></td>
</tr>
<tr id="thirumalai2001" class="entry">
	<td>Thirumalai, D., Lee, N., Woodson, S. A. &amp; Klimov, D.</td>
	<td>Early Events in RNA Folding <p class="infolinks">[<a href="javascript:toggleInfo('thirumalai2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('thirumalai2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Annu. Rev. Phys. Chem. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.physchem.52.1.751">DOI</a><a href="http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=5367085">URL</a> &nbsp;</td>
</tr>
<tr id="abs_thirumalai2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We describe a conceptual framework for understanding the way large RNA molecules fold based on the notion that their free-energy landscape is rugged. A key prediction of our theory is that RNA folding can be described by the kinetic partitioning mechanism (KPM). According to KPM a small fraction of molecules folds rapidly to the native state whereas the remaining fraction is kinetically trapped in a low free-energy non-native state. This model provides a unified description of the way RNA and proteins fold. Single-molecule experiments on Tetrahymena ribozyme, which directly validate our theory, are analyzed using KPM.We also describe the earliest events that occur on microsecond time scales in RNA folding. These must involve collapse of RNA molecules that are mediated by counterion-condensation. Estimates of time scales for the initial events in RNA folding are provided for the Tetrahymena ribozyme.</td>
</tr>
<tr id="bib_thirumalai2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{thirumalai2001,
  author = {D. Thirumalai and Namkyung Lee and Sarah A. Woodson and D.K. Klimov},
  title = {Early Events in RNA Folding},
  journal = {Annu. Rev. Phys. Chem.},
  year = {2001},
  volume = {52},
  pages = {751-761},
  url = {http://search.epnet.com/login.aspx?direct=true&amp;db=aph&amp;an=5367085},
  doi = {http://dx.doi.org/10.1146/annurev.physchem.52.1.751}
}
</pre></td>
</tr>
<tr id="thirumalai1996" class="entry">
	<td>Thirumalai, D. &amp; Woodson, S. A.</td>
	<td>Kinetics of Folding of Proteins and RNA <p class="infolinks">[<a href="javascript:toggleInfo('thirumalai1996','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('thirumalai1996','bibtex')">BibTeX</a>]</p></td>
	<td>1996</td>
	<td>Accounts in Chemical Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ar9500933">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_thirumalai1996" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The assembly of biological molecules, most notably globular proteins1 and RNA,2,3 into unique three-dimensional structures with well-defined topology is a complex and fascinating phenomenon in molecular biology. There are two aspects to the problem of folding of proteins and RNA. The first is the prediction of the three-dimensional structure of the folded state from the one-dimensional primary sequence of amino acids (for proteins) and nucleotides (for RNA). The second question is concerned with the kinetics of approach to the essentially unique folded state (also referred to as the native state) starting from an initial ensemble of disordered structures. In this Account we describe recent advances in our understanding of the kinetics of in vitro folding of globular proteins in terms of the underlying energy landscape. We further show that similar considerations can be usefully applied to describe the general features of the folding of RNA molecules.</td>
</tr>
<tr id="bib_thirumalai1996" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{thirumalai1996,
  author = {D. Thirumalai and S. A. Woodson},
  title = {Kinetics of Folding of Proteins and RNA},
  journal = {Accounts in Chemical Research},
  year = {1996},
  volume = {29},
  pages = {433-439},
  doi = {http://dx.doi.org/10.1021/ar9500933}
}
</pre></td>
</tr>
<tr id="tinoco2004" class="entry">
	<td>Tinoco, I.</td>
	<td>FORCE AS A USEFUL VARIABLE IN REACTIONS: Unfolding RNA. <p class="infolinks">[<a href="javascript:toggleInfo('tinoco2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('tinoco2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Annual Review of Biophysics &amp; Biomolecular Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.biophys.33.110502.140418">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_tinoco2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The effect of force on the thermodynamics and kinetics of reactions is described. The key parameters are the difference in end-to-end distance between reactant and product for thermodynamics, and the distance to the transition state for kinetics. I focus the review on experimental results on force unfolding of RNA. Methods to measure Gibbs free energies and kinetics for reversible and irreversible reactions are described. The use of the worm-like-chain model to calculate the effects of force on thermodynamics and kinetics is illustrated with simple models. The main purpose of the review is to describe the simple experiments that have been done so far, and to encourage more people to enter a field that is new and full of opportunities.</td>
</tr>
<tr id="bib_tinoco2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tinoco2004,
  author = {Ignacio Tinoco},
  title = {FORCE AS A USEFUL VARIABLE IN REACTIONS: Unfolding RNA.},
  journal = {Annual Review of Biophysics &amp; Biomolecular Structure},
  year = {2004},
  volume = {33},
  pages = {363-385},
  doi = {http://dx.doi.org/10.1146/annurev.biophys.33.110502.140418}
}
</pre></td>
</tr>
<tr id="tinoco2002" class="entry">
	<td>Tinoco, I.</td>
	<td>Physical Chemistry of Nucleic Acids <p class="infolinks">[<a href="javascript:toggleInfo('tinoco2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Annu. Rev. Phys. Chem. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1146/annurev.physchem.53.082001.144341">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_tinoco2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tinoco2002,
  author = {Ignacio Tinoco},
  title = {Physical Chemistry of Nucleic Acids},
  journal = {Annu. Rev. Phys. Chem.},
  year = {2002},
  volume = {53},
  pages = {1-15},
  doi = {http://dx.doi.org/10.1146/annurev.physchem.53.082001.144341}
}
</pre></td>
</tr>
<tr id="tinoco1999" class="entry">
	<td>Tinoco, I. &amp; Bustamante, C.</td>
	<td>How RNA folds <p class="infolinks">[<a href="javascript:toggleInfo('tinoco1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('tinoco1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.1999.3001">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_tinoco1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We describe the RNA folding problem and contrast it with the much more difficult protein folding problem. RNA has four similar monomer units, whereas proteins have 20 very different residues. The folding of RNA is hierarchical in that secondary structure is much more stable than tertiary folding. In RNA the two levels of folding (secondary and tertiary) can be experimentally separated by the presence or absence of Mg2+. Secondary structure can be predicted successfully from experimental thermodynamic data on secondary structure elements: helices, loops, and bulges. Tertiary interactions can then be added without much distortion of the secondary structure. These observations suggest a folding algorithm to predict the structure of an RNA from its sequence. However, to solve the RNA folding problem one needs thermodynamic data on tertiary structure interactions, and identification and characterization of metal-ion binding sites. These data, together with force versus extension measurements on single RNA molecules, should provide the information necessary to test and refine the proposed algorithm.</td>
</tr>
<tr id="bib_tinoco1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tinoco1999,
  author = {Ignacio Tinoco and Carlos Bustamante},
  title = {How RNA folds},
  journal = {Journal of Molecular Biology},
  year = {1999},
  volume = {293},
  number = {2},
  pages = {271-281},
  doi = {http://dx.doi.org/10.1006/jmbi.1999.3001}
}
</pre></td>
</tr>
<tr id="consonni2000" class="entry">
	<td>Todeschini, R. &amp; Consonni, V.</td>
	<td>Handbook of Molecular Descriptors <p class="infolinks">[<a href="javascript:toggleInfo('consonni2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_consonni2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{consonni2000,
  author = {Roberto Todeschini and Viviana Consonni},
  title = {Handbook of Molecular Descriptors},
  publisher = {Wiley - VCH},
  year = {2000}
}
</pre></td>
</tr>
<tr id="tostesen2001" class="entry">
	<td>Tostesen, E., Chen, S. &amp; Dill, K. A.</td>
	<td>RNA Folding Transitions and Cooperativity <p class="infolinks">[<a href="javascript:toggleInfo('tostesen2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('tostesen2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>J. Phys. Chem. B &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/jp002877q">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_tostesen2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Using a new theoretical model, we explore the conformational energy landscapes and the statistical thermodynamics of RNA secondary structure folding. The model is physical, treats the conformational entropies and excluded volume explicitly, and has previously been shown to give reasonable agreement with experimental melting curves. Here we use it to design energy landscapes that can be tested by experiments. The model predicts that even simple RNA's, less than 60 nucleotides long and having only secondary structures, can have remarkably complex and bumpy landscapes that can be altered substantially by small mutations. Moving a GC block around in a sequence of fixed composition can lead from one melting peak to two. A mutation can switch it back to one. Two melting peaks do not imply two simple stem melting events. In one system, we find 5 transitions, each of which can be either 2-state or 1-state. In some sequences we find triple points, where 3 macroscopically identifiable species have equal populations at the same temperature. We show designs of RNA's with multiple native states, and conformational switching between them, as functions of mutations or temperature.</td>
</tr>
<tr id="bib_tostesen2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tostesen2001,
  author = {Eivind Tostesen and Shi-Jie Chen and Ken A. Dill },
  title = {RNA Folding Transitions and Cooperativity},
  journal = {J. Phys. Chem. B},
  year = {2001},
  volume = {105},
  number = {8},
  pages = {1618-1630},
  doi = {http://dx.doi.org/10.1021/jp002877q}
}
</pre></td>
</tr>
<tr id="trinajstic1983a" class="entry">
	<td>Trinajstic, N.</td>
	<td>Chemical graph Theory Volume I <p class="infolinks">[<a href="javascript:toggleInfo('trinajstic1983a','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_trinajstic1983a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{trinajstic1983a,
  author = {Nenad Trinajstic},
  title = {Chemical graph Theory Volume I},
  publisher = {CRC Press},
  year = {1983}
}
</pre></td>
</tr>
<tr id="trinajstic1983b" class="entry">
	<td>Trinajstic, N.</td>
	<td>Chemical graph Theory Volume II <p class="infolinks">[<a href="javascript:toggleInfo('trinajstic1983b','bibtex')">BibTeX</a>]</p></td>
	<td>1983</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_trinajstic1983b" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{trinajstic1983b,
  author = {Nenad Trinajstic},
  title = {Chemical graph Theory Volume II},
  publisher = {CRC Press},
  year = {1983}
}
</pre></td>
</tr>
<tr id="trinajstic1991" class="entry">
	<td>Trinajstic, N., Nikolic, S., Knop, J. V., Muller, W. R. &amp; Szymanski, K.</td>
	<td>Computational Chemical Graph Theory <p class="infolinks">[<a href="javascript:toggleInfo('trinajstic1991','bibtex')">BibTeX</a>]</p></td>
	<td>1991</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_trinajstic1991" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{trinajstic1991,
  author = {N. Trinajstic and S. Nikolic and J. V. Knop and W. R. Muller and K. Szymanski},
  title = {Computational Chemical Graph Theory},
  publisher = {Ellis Horwood},
  year = {1991}
}
</pre></td>
</tr>
<tr id="tung2004" class="entry">
	<td>Tung, C. &amp; Sanbonmatsu, K. Y.</td>
	<td>Atomic Model of the Thermus thermophilus 70S Ribosome Developed in Silico <p class="infolinks">[<a href="javascript:toggleInfo('tung2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.104.040162">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_tung2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{tung2004,
  author = {Chang-Shung Tung and Kevin Y. Sanbonmatsu},
  title = {Atomic Model of the Thermus thermophilus 70S Ribosome Developed in Silico},
  journal = {Biophysical Journal},
  year = {2004},
  volume = {87},
  pages = {2714-2722},
  doi = {http://dx.doi.org/10.1529/biophysj.104.040162}
}
</pre></td>
</tr>
<tr id="unrau1998" class="entry">
	<td>Unrau, P. J. &amp; Bartel, D. P.</td>
	<td>RNA-catalysed Nucleotide Synthesis <p class="infolinks">[<a href="javascript:toggleInfo('unrau1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/26193">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_unrau1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{unrau1998,
  author = {Peter J. Unrau and David P. Bartel},
  title = {RNA-catalysed Nucleotide Synthesis},
  journal = {Nature},
  year = {1998},
  volume = {395},
  pages = {260-263},
  doi = {http://dx.doi.org/10.1038/26193}
}
</pre></td>
</tr>
<tr id="vernizzi2005" class="entry">
	<td>Vernizzi, G., Orland, H. &amp; Zee, A.</td>
	<td>Enumeration of RNA Structures by Matrix Models <p class="infolinks">[<a href="javascript:toggleInfo('vernizzi2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('vernizzi2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Physical Review Letters &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1103/PhysRevLett.94.168103">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_vernizzi2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We enumerate the number of RNA contact structures according to their genus, i.e., the topological character of their pseudoknots. By using a recently proposed matrix model formulation for the RNA folding problem, we obtain exact results for the simple case of an RNA molecule with an infinitely flexible backbone, in which any arbitrary pair of bases is allowed. We analyze the distribution of the genus of pseudoknots as a function of the total number of nucleotides along the phosphate-sugar backbone.</td>
</tr>
<tr id="bib_vernizzi2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{vernizzi2005,
  author = {Graziano Vernizzi and Henri Orland and A. Zee},
  title = {Enumeration of RNA Structures by Matrix Models},
  journal = {Physical Review Letters},
  year = {2005},
  volume = {94},
  pages = {1-4},
  doi = {http://dx.doi.org/10.1103/PhysRevLett.94.168103}
}
</pre></td>
</tr>
<tr id="vila-sanjurjo2003" class="entry">
	<td>Vila-Sanjurjo, A., Ridgeway, W. K., Seymaner, V., Zhang, W., Santoso, S., Yu, K. &amp; Cate, J. H. D.</td>
	<td>X-ray crystal structures of the WT and o hyper-accurate ribosome from Eschericia coli <p class="infolinks">[<a href="javascript:toggleInfo('vila-sanjurjo2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('vila-sanjurjo2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>PNAS &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.1133380100">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_vila-sanjurjo2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Protein biosynthesis on the ribosome requires accurate reading of the genetic code in mRNA. Two conformational rearrangements in the small ribosomal subunit, a closing of the head and body around the incoming tRNA and an RNA helical switch near the mRNA decoding site, have been proposed to select for complementary base-pairing between mRNA codons and tRNA anticodons. We determined x-ray crystal structures of the WT and a hyper-accurate variant of the Escherichia coli ribosome at resolutions of 10 and 9 ï¿½, respectively, revealing that formation of the intact 70S ribosome from its two subunits closes the conformation of the head of the small subunit independent of mRNA decoding. Moreover, no change in the conformation of the switch helix is observed in two steps of tRNA discrimination. These 70S ribosome structures indicate that mRNA decoding is coupled primarily to movement of the small subunit body, consistent with previous proposals, whereas closing of the head and the helical switch may function in other steps of protein synthesis. </td>
</tr>
<tr id="bib_vila-sanjurjo2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{vila-sanjurjo2003,
  author = {Anton Vila-Sanjurjo and William K. Ridgeway and Veysel Seymaner and Wen Zhang and Steve Santoso and Kexin Yu and Jamie H. Doudna Cate},
  title = {X-ray crystal structures of the WT and o hyper-accurate ribosome from Eschericia coli},
  journal = {PNAS},
  year = {2003},
  volume = {100},
  number = {15},
  pages = {8682-8687},
  doi = {http://dx.doi.org/10.1073/pnas.1133380100}
}
</pre></td>
</tr>
<tr id="wadley2007" class="entry">
	<td>Wadley, L. M., Keating, K. S., Duarte, C. M. &amp; Pyle, A. M.</td>
	<td>Evaluating and Learning from RNA Pseudotorsional Space: Quantitative Validation of a Reduced Representation for RNA Structure <p class="infolinks">[<a href="javascript:toggleInfo('wadley2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('wadley2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jmb.2007.06.058">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_wadley2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Quantitatively describing RNA structure and conformational elements<p>remains a formidable problem. Seven standard torsion angles and the sugar<p>pucker are necessary to characterize the conformation of an RNA nucleotide<p>completely. Progress has been made toward understanding the discrete<p>nature of RNA structure, but classifying simple and ubiquitous structural<p>elements such as helices and motifs remains a difficult task. One approach<p>for describing RNA structure in a simple, mathematically consistent, and<p>computationally accessible manner involves the invocation of two pseudotorsions,<p>n (C4'n-1, Pn, C4'n, Pn+1) and t (Pn, C4'n, Pn+1, C4'n+1), which can be<p>used to describe RNA conformation in much the same way that phi and psi are<p>used to describe backbone configuration of proteins. Here, we conduct an<p>exploration and statistical evaluation of pseudotorsional space and of the<p>Ramachandran-like nablaï¿½theta plot. We show that, through the rigorous<p>quantitative analysis of the nablaï¿½theta plot, the pseudotorsional descriptors nabla<p>and theta, together with sugar pucker, are sufficient to describe RNA backbone<p>conformation fully in most cases. These descriptors are also shown to<p>contain considerable information about nucleotide base conformation,<p>revealing a previously uncharacterized interplay between backbone and<p>base orientation. A window function analysis is used to discern statistically<p>relevant regions of density in the nablaï¿½theta scatter plot and then nucleotides in<p>colocalized clusters in the nablaï¿½theta plane are shown to have similar 3-D<p>structures through RMSD analysis of the RNA structural constituents. We<p>find that major clusters in the nablaï¿½theta plot are few, underscoring the discrete<p>nature of RNA backbone conformation. Like the Ramachandran plot, the<p>nablaï¿½theta plot is a valuable system for conceptualizing biomolecular conformation,<p>it is a useful tool for analyzing RNA tertiary structures, and it is a vital<p>component of new approaches for solving the 3-D structures of large RNA<p>molecules and RNA assemblies.</td>
</tr>
<tr id="bib_wadley2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{wadley2007,
  author = {Leven M. Wadley and Kevin S. Keating and Carlos M. Duarte and Anna Marie Pyle},
  title = {Evaluating and Learning from RNA Pseudotorsional Space: Quantitative Validation of a Reduced Representation for RNA Structure},
  journal = {Journal of Molecular Biology},
  year = {2007},
  volume = {372},
  pages = {942-957},
  doi = {http://dx.doi.org/10.1016/j.jmb.2007.06.058}
}
</pre></td>
</tr>
<tr id="waldrop1992" class="entry">
	<td>Waldrop, M.</td>
	<td>Finding RNA Makes Proteins Gives 'RNA World' a Big Boost <p class="infolinks">[<a href="javascript:toggleInfo('waldrop1992','bibtex')">BibTeX</a>]</p></td>
	<td>1992</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://links.jstor.org/sici?sici=0036-8075%2819920605%293%3A256%3A5062%3C1396%3AFRMPG%60%3E2.0.CO%3B2-H">URL</a> &nbsp;</td>
</tr>
<tr id="bib_waldrop1992" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{waldrop1992,
  author = {Michael Waldrop},
  title = {Finding RNA Makes Proteins Gives 'RNA World' a Big Boost},
  journal = {Science},
  year = {1992},
  volume = {256},
  pages = {1396-1397},
  url = {http://links.jstor.org/sici?sici=0036-8075%2819920605%293%3A256%3A5062%3C1396%3AFRMPG%60%3E2.0.CO%3B2-H}
}
</pre></td>
</tr>
<tr id="hobza2007c" class="entry">
	<td>Wang, W., Pitonak, M. &amp; Hobza, P.</td>
	<td>CH Stretching Vibrational Shift of Benzene Dimer: Consistency of Experiment and Calculation <p class="infolinks">[<a href="javascript:toggleInfo('hobza2007c','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('hobza2007c','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>ChemPhysChem &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/cphc.200700415">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_hobza2007c" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Three low-energy structures of the benzene dimer are investigated by several theoretical procedures (RI-MP2, CCSD(T), RI-DFT-D, DFT/BH&amp;H) covering London dispersion energy. The RI-DFT-D and CCSD(T) calculations are used to verify the DFT/BH&amp;H dimer characteristics, as only at this level can anharmonic calculations be performed. It is ascertained that the T-shaped (C2v) structure, in which the CH stretching frequency of the proton donor shows a significant blue shift, is not stable at any level of theory. It is either a transition structure or a minimum which is easily transformed into a parallel-displaced structure or a T-shaped (Cs) structure, even at low temperature. Consequently, no blue shift can be detected. On the other hand, the calculated anharmonic IR spectra of the two most stable structures of benzene dimer, namely, the T-shaped (Cs) and the parallel-displaced ones, give rise to a small red (and no blue) shift of the CH stretching vibration. This finding is fully consistent with the experimental results.</td>
</tr>
<tr id="bib_hobza2007c" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{hobza2007c,
  author = {Weizhou Wang and Michal Pitonak and Pavel Hobza},
  title = {CH Stretching Vibrational Shift of Benzene Dimer: Consistency of Experiment and Calculation},
  journal = {ChemPhysChem},
  year = {2007},
  volume = {8},
  pages = {1-6},
  doi = {http://dx.doi.org/10.1002/cphc.200700415}
}
</pre></td>
</tr>
<tr id="wang2005" class="entry">
	<td>Wang, Y. &amp; Jernigan, R. L.</td>
	<td>Comparison of tRNA Motions in the Free and Ribosomal Bound Structures <p class="infolinks">[<a href="javascript:toggleInfo('wang2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1529/biophysj.105.064840">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_wang2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{wang2005,
  author = {Yongmei Wang and Robert L. Jernigan},
  title = {Comparison of tRNA Motions in the Free and Ribosomal Bound Structures},
  journal = {Biophysical Journal},
  year = {2005},
  volume = {89},
  pages = {3399-3409},
  doi = {http://dx.doi.org/10.1529/biophysj.105.064840}
}
</pre></td>
</tr>
<tr id="y_wang2004" class="entry">
	<td>Wang, Y., Rader, A. J., Bahar, I. &amp; Jernigan, R. L.</td>
	<td>Global ribosome motions revealed with elastic network model <p class="infolinks">[<a href="javascript:toggleInfo('y_wang2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('y_wang2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Journal of Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jsb.2004.01.005">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_y_wang2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The motions of large systems such as the ribosome are not fully accessible with conventional molecular simulations. A coarse-grained, less-than-atomic-detail model such as the anisotropic network model (ANM) is a convenient informative tool to study the cooperative motions of the ribosome. The motions of the small 30S subunit, the larger 50S subunit, and the entire 70S assembly of the two subunits have been analyzed using ANM. The lowest frequency collective modes predicted by ANM show that the 50S subunit and 30S subunit are strongly anti-correlated in the motion of the 70S assembly. A ratchet-like motion is observed that corresponds well to the experimentally reported ratchet motion. Other slow modes are also examined because of their potential links to the translocation steps in the ribosome. We identify several modes that may facilitate the E-tRNA exiting from the assembly. The A-site t-RNA and P-site t-RNA are found to be strongly coupled and positively correlated in these slow modes, suggesting that the translocations of these two t-RNAs occur simultaneously, while the motions of the E-site t-RNA are less correlated, and thus less likely to occur simultaneously. Overall the t-RNAs exhibit relatively large deformations. Animations of these slow modes of motion can be viewed at http://ribosome.bb.iastate.edu/70SnKmode.</td>
</tr>
<tr id="bib_y_wang2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{y_wang2004,
  author = {Yongmei Wang and A. J. Rader and Ivet Bahar and Robert L. Jernigan},
  title = {Global ribosome motions revealed with elastic network model},
  journal = {Journal of Structural Biology},
  year = {2004},
  volume = {147},
  pages = {302-314},
  doi = {http://dx.doi.org/10.1016/j.jsb.2004.01.005}
}
</pre></td>
</tr>
<tr id="weikl2003" class="entry">
	<td>Weikl, T. R. &amp; Dill, K. A.</td>
	<td>Folding Rates and Low-entropy-loss Routes of Two-state Proteins <p class="infolinks">[<a href="javascript:toggleInfo('weikl2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('weikl2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>J. Mol. Biol. &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0022-2836(03)00436-4">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_weikl2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We develop a simple model for computing the rates and routes of folding of two-state proteins from the contact maps of their native structures. The model is based on the graph-theoretical concept of effective contact order (ECO). The model predicts that proteins fold by "zipping up" in a sequence of small-loop-closure events, depending on the native chain fold. Using a simple equation, with a few physical rate parameters, we obtain a good correlation with the folding rates of 24 two-state folding proteins. The model rationalizes data from phi-value analysis that have been interpreted in terms of delocalized or polarized transition states. This model indicates how much of protein folding may take place in parallel, not along a single reaction coordinate or with a single transition state.</td>
</tr>
<tr id="bib_weikl2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{weikl2003,
  author = {Thomas R. Weikl and Ken A. Dill},
  title = {Folding Rates and Low-entropy-loss Routes of Two-state Proteins},
  journal = {J. Mol. Biol.},
  year = {2003},
  volume = {329},
  pages = {585-598},
  doi = {http://dx.doi.org/10.1016/S0022-2836(03)00436-4}
}
</pre></td>
</tr>
<tr id="weinberg2009" class="entry">
	<td>Weinberg, Z., Perreault, J., Meyer, M. M. &amp; Breaker, R. R.</td>
	<td>Exceptional structured noncoding RNAs revealed by bacterial metagenome analysis <p class="infolinks">[<a href="javascript:toggleInfo('weinberg2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('weinberg2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/nature08586">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_weinberg2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Estimates of the total number of bacterial species1–3 indicate that existing DNA sequence databases carry only a tiny fraction of the total amount of DNA sequence space represented by this division of life. Indeed, environmental DNA samples have been shown to encode many previously unknown classes of proteins4 and RNAs5. Bioinformatics searches6–10 of genomic DNA from bacteria commonly identify new noncoding RNAs (ncRNAs)10–12 such as riboswitches13,14. In rare instances, RNAs that exhibit more extensive sequence and structural conservation across a wide range of bacteria are encountered15,16. Given that large structured RNAs are known to carry out complex biochemical functions such as protein synthesis and RNA processing reactions, identifying more RNAs of great size and intricate structure is likely to reveal additional biochemical functions that can be achieved by RNA. We applied an updated computational pipeline17 to discover ncRNAs that rival the known large ribozymes in size and structural complexity or that are among the most abundant RNAs in bacteria that encode them. These RNAs would have been difficult or impossible to detect without examining environmental DNA sequences, indicating that numerous RNAs with extraordinary size, structural complexity, or other exceptional characteristics remain to be discovered in unexplored sequence space.</td>
</tr>
<tr id="bib_weinberg2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{weinberg2009,
  author = {Zasha Weinberg and Jonathan Perreault and Michelle M. Meyer and Ronald R. Breaker},
  title = {Exceptional structured noncoding RNAs revealed by bacterial metagenome analysis},
  journal = {Nature},
  year = {2009},
  volume = {462},
  pages = {656-659},
  doi = {http://dx.doi.org/10.1038/nature08586}
}
</pre></td>
</tr>
<tr id="westhead1999" class="entry">
	<td>Westhead, D., Slidel, T., Flores, T. &amp; Thornton, J.</td>
	<td>Protein structural topology: Automated analysis and diagrammatic representation <p class="infolinks">[<a href="javascript:toggleInfo('westhead1999','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('westhead1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Protein Science &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="abs_westhead1999" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The topology of a protein structure is a highly simplified description of its fold including only the sequence of secondary structure elements, and their relative spatial positions and approximate orientations. This information can be embodied in a two-dimensional diagram of protein topology, called a TOPS cartoon. These cartoons are useful for the understanding of particular folds and making comparisons between folds. Here we describe a new algorithm for the production of TOPS cartoons, which is more robust than those previously available, and has a much higher success rate. This algorithm has been used to produce a database of protein topology cartoons that covers most of the data bank of known protein structures.</td>
</tr>
<tr id="bib_westhead1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{westhead1999,
  author = {D.R. Westhead and T.W. Slidel and T.P. Flores and J.M. Thornton},
  title = {Protein structural topology: Automated analysis and diagrammatic representation},
  journal = {Protein Science},
  year = {1999},
  volume = {8},
  pages = {897-904}
}
</pre></td>
</tr>
<tr id="westhof2000" class="entry">
	<td>Westhof, E. &amp; Fritsch, V.</td>
	<td>RNA folding: beyond Watson-Crick pairs <p class="infolinks">[<a href="javascript:toggleInfo('westhof2000','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('westhof2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Structure &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0969-2126(00)00112-X">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_westhof2000" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Several crystal structures of RNA fragments, alone or in complex with a specific protein, have been recently solved. In addition, the structures of an artificial ribozyme, the leadzyme, and the cleavage product of a human pathogen ribozyme, have extended the structural diversity of ribozyme architectures. The attained set of folding rules and motifs expand the repertoire seen previously in tRNA structures.</td>
</tr>
<tr id="bib_westhof2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{westhof2000,
  author = {Eric Westhof and Valerie Fritsch},
  title = {RNA folding: beyond Watson-Crick pairs},
  journal = {Structure},
  year = {2000},
  volume = {8},
  pages = {R55-R65},
  doi = {http://dx.doi.org/10.1016/S0969-2126(00)00112-X}
}
</pre></td>
</tr>
<tr id="westhof2005" class="entry">
	<td>Westhof, E. &amp; Patel, D. J.</td>
	<td>Nucleic acids RNA: from architecture to recognition and catalysis <p class="infolinks">[<a href="javascript:toggleInfo('westhof2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('westhof2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.sbi.2005.05.012">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_westhof2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The emphasis of this section of Current Opinion in Structural Biology is on RNA architecture, recognition and catalysis. The topics covered range from ribozymes to riboswitches, as well as molecular machines such as the ribosome and the machinery for RNA-silencing-mediated cleavage of mRNA. We now realize that small 21?25-nucleotide RNAs control gene regulation and are central to eukaryotic complexity. Such evolving intricate regulatory networks are fundamentally controlled by the dynamics and the energetics of molecular recognition between RNA molecules, as well as between RNA molecules and their various partners. Molecular recognition is therefore a recurring theme in this overview, with the emphasis on complexes of RNA with metabolites, antibiotics and proteins.</td>
</tr>
<tr id="bib_westhof2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{westhof2005,
  author = {Eric Westhof and Dinshaw J. Patel},
  title = {Nucleic acids RNA: from architecture to recognition and catalysis},
  journal = {Current Opinion in Structural Biology},
  year = {2005},
  volume = {15},
  pages = {309-312},
  doi = {http://dx.doi.org/10.1016/j.sbi.2005.05.012}
}
</pre></td>
</tr>
<tr id="widom2001" class="entry">
	<td>Widom, J.</td>
	<td>Role of DNA Sequence in Nucleosome Stability and Dynamics <p class="infolinks">[<a href="javascript:toggleInfo('widom2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Quarterly Reviews of Biophysics &nbsp;</td>
	<td>article</td>
	<td><a href="http://journals.cambridge.org/bin/bladerunner?REQUNIQ=1106169319&amp;REQSESS=979243&amp;117000REQEVENT=&amp;REQINT1=94950&amp;REQAUTH=0">URL</a> &nbsp;</td>
</tr>
<tr id="bib_widom2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{widom2001,
  author = {J. Widom},
  title = {Role of DNA Sequence in Nucleosome Stability and Dynamics},
  journal = {Quarterly Reviews of Biophysics},
  year = {2001},
  volume = {34},
  number = {3},
  pages = {269-324},
  url = {http://journals.cambridge.org/bin/bladerunner?REQUNIQ=1106169319&amp;REQSESS=979243&amp;117000REQEVENT=&amp;REQINT1=94950&amp;REQAUTH=0}
}
</pre></td>
</tr>
<tr id="wiener1947" class="entry">
	<td>Wiener, H.</td>
	<td>Structural Determination of Paraffin Boiling Points <p class="infolinks">[<a href="javascript:toggleInfo('wiener1947','bibtex')">BibTeX</a>]</p></td>
	<td>1947</td>
	<td>Journal of the American Chemical Society &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1021/ja01193a005">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_wiener1947" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{wiener1947,
  author = {Harry Wiener},
  title = {Structural Determination of Paraffin Boiling Points},
  journal = {Journal of the American Chemical Society},
  year = {1947},
  volume = {69},
  pages = {17-20},
  doi = {http://dx.doi.org/10.1021/ja01193a005}
}
</pre></td>
</tr>
<tr id="wimberly2000" class="entry">
	<td>Wimberly, B. T., Brodersen, D. E., Clemons, W. M., Morgan-Warren, R. J., Carter, A. P., Vonrhein, C., Hartschk, T. &amp; Ramakrishnan, V.</td>
	<td>Structure of the 30S Ribosomal Subunit <p class="infolinks">[<a href="javascript:toggleInfo('wimberly2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Nature &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1038/35030006">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_wimberly2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{wimberly2000,
  author = {Brian T. Wimberly and Ditlev E. Brodersen and William M. Clemons and Robert J. Morgan-Warren and Andrew P. Carter and Clemens Vonrhein and Thomas Hartschk and V. Ramakrishnan},
  title = {Structure of the 30S Ribosomal Subunit},
  journal = {Nature},
  year = {2000},
  volume = {407},
  pages = {327-339},
  doi = {http://dx.doi.org/10.1038/35030006}
}
</pre></td>
</tr>
<tr id="woodson2008" class="entry">
	<td>Woodson, S. A.</td>
	<td>RNA folding and ribosome assembly. <p class="infolinks">[<a href="javascript:toggleInfo('woodson2008','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('woodson2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>Curr Opin Chem Biol &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.cbpa.2008.09.024">DOI</a><a href="http://dx.doi.org/10.1016/j.cbpa.2008.09.024">URL</a> &nbsp;</td>
</tr>
<tr id="abs_woodson2008" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Ribosome synthesis is a tightly regulated process that is crucial for cell survival. Chemical footprinting, mass spectrometry, and cryo-electron microscopy are revealing how these complex cellular machines are assembled. Rapid folding of the rRNA provides a platform for protein-induced assembly of the bacterial 30S ribosome. Multiple assembly pathways increase the flexibility of the assembly process, while accessory factors and modification enzymes chaperone the late stages of assembly and control the quality of the mature subunits.</td>
</tr>
<tr id="bib_woodson2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{woodson2008,
  author = {Sarah A Woodson},
  title = {RNA folding and ribosome assembly.},
  journal = {Curr Opin Chem Biol},
  year = {2008},
  volume = {12},
  number = {6},
  pages = {667--673},
  url = {http://dx.doi.org/10.1016/j.cbpa.2008.09.024},
  doi = {http://dx.doi.org/10.1016/j.cbpa.2008.09.024}
}
</pre></td>
</tr>
<tr id="woolley1998" class="entry">
	<td>Woolley, R.</td>
	<td>Is there a quantum definition of a molecule? <p class="infolinks">[<a href="javascript:toggleInfo('woolley1998','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('woolley1998','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Journal of Mathematical Chemistry &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1023/A:1019144518901">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_woolley1998" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The paper surveys how chemistry has developed over the past two centuries starting from Lavoisierrsquos classification of the chemical elements at the end of the eighteenth century; the subsequent development of the atomicï¿½molecular model of matter preoccupied chemists throughout the nineteenth century, while the results of the application of quantum theory to the molecular model has been the story of this century. Whereas physical chemistry originated in the nineteenth century with the measurement of the physical properties of groups of chemical compounds that chemists identified as families, the goal of chemical physics is the explanation of the facts of chemistry in terms of the principles and theories of physics. Chemical physics as such was only possible after the discovery of the quantum theory in the 1920rsquos. By then the first of the subdashatomic particles had been discovered and seemingly it is no longer possible to discuss chemical facts purely in terms of atoms and molecules ï¿½ one has to recognize the electron and the nucleus, the parts of atoms. The combination of classical molecular structure with the quantum properties of the electron has given us a tremendously successful account of chemistry called lsquoquantum chemistryrsquo. Yet from the perspective of the quantum theory the deepest part of chemistry, the existence of chemical isomers and the very idea of molecular structure that rationalizes it, remains a central problem for chemical physics.</td>
</tr>
<tr id="bib_woolley1998" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{woolley1998,
  author = {R.G. Woolley},
  title = {Is there a quantum definition of a molecule?},
  journal = {Journal of Mathematical Chemistry},
  year = {1998},
  volume = {23},
  pages = {3-12},
  doi = {http://dx.doi.org/10.1023/A:1019144518901}
}
</pre></td>
</tr>
<tr id="wyatt1989" class="entry">
	<td>Wyatt, J. R., Puglisi, J. D. &amp; Tinoco, I.</td>
	<td>RNA Folding: Pseudoknots, Loops and Bulges <p class="infolinks">[<a href="javascript:toggleInfo('wyatt1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Bioessays &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_wyatt1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{wyatt1989,
  author = {Jaqueline R. Wyatt and Joseph D. Puglisi and Ignacio Tinoco},
  title = {RNA Folding: Pseudoknots, Loops and Bulges},
  journal = {Bioessays},
  year = {1989},
  volume = {11},
  pages = {100-106}
}
</pre></td>
</tr>
<tr id="yurongthesis" class="entry">
	<td>Xin, Y.</td>
	<td>Non-Canonical Base Pairs in RNA Structures and Folding, Ph.D. thesis <p class="infolinks">[<a href="javascript:toggleInfo('yurongthesis','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Rutgers, The State University of New Jersey &nbsp;</td>
	<td>article</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_yurongthesis" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{yurongthesis,
  author = {Yurong Xin},
  title = {Non-Canonical Base Pairs in RNA Structures and Folding, Ph.D. thesis},
  journal = {Rutgers, The State University of New Jersey},
  year = {2005}
}
</pre></td>
</tr>
<tr id="xu2007" class="entry">
	<td>Xu, D., Landon, T., Greenbaum, N. L. &amp; Fenley, M. O.</td>
	<td>The electrostatic characteristics of G.U wobble base pairs. <p class="infolinks">[<a href="javascript:toggleInfo('xu2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('xu2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Nucleic Acids Res &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkm274">DOI</a><a href="http://dx.doi.org/10.1093/nar/gkm274">URL</a> &nbsp;</td>
</tr>
<tr id="abs_xu2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: G.U wobble base pairs are the most common and highly conserved non-Watson-Crick base pairs in RNA. Previous surface maps imply uniformly negative electrostatic potential at the major groove of G.U wobble base pairs embedded in RNA helices, suitable for entrapment of cationic ligands. In this work, we have used a Poisson-Boltzmann approach to gain a more detailed and accurate characterization of the electrostatic profile. We found that the major groove edge of an isolated G.U wobble displays distinctly enhanced negativity compared with standard GC or AU base pairs; however, in the context of different helical motifs, the electrostatic pattern varies. G.U wobbles with distinct widening have similar major groove electrostatic potentials to their canonical counterparts, whereas those with minimal widening exhibit significantly enhanced electronegativity, ranging from 0.8 to 2.5 kT/e, depending upon structural features. We propose that the negativity at the major groove of G.U wobble base pairs is determined by the combined effect of the base atoms and the sugar-phosphate backbone, which is impacted by stacking pattern and groove width as a result of base sequence. These findings are significant in that they provide predictive power with respect to which G.U sites in RNA are most likely to bind cationic ligands.</td>
</tr>
<tr id="bib_xu2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{xu2007,
  author = {Darui Xu and Theresa Landon and Nancy L Greenbaum and Marcia O Fenley},
  title = {The electrostatic characteristics of G.U wobble base pairs.},
  journal = {Nucleic Acids Res},
  year = {2007},
  volume = {35},
  number = {11},
  pages = {3836--3847},
  url = {http://dx.doi.org/10.1093/nar/gkm274},
  doi = {http://dx.doi.org/10.1093/nar/gkm274}
}
</pre></td>
</tr>
<tr id="yang2003" class="entry">
	<td>Yang, H. &amp; Jossinet, F.</td>
	<td>Tools for the Automatic Identification and Classification of RNA base pairs <p class="infolinks">[<a href="javascript:toggleInfo('yang2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('yang2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg529">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_yang2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Three programs have been developed to aid in the classification and visualization of RNA structure. BPViewer provides a web interface for displaying three-dimensional (3D) coordinates of individual base pairs or base pair collections. A web server, RNAview, automatically identifies and classifies the types of base pairs that are formed in nucleic acid structures by various combinations of the three edges, Watson?Crick, Hoogsteen and the Sugar edge. RNAView produces two-dimensional (2D) diagrams of secondary and tertiary structure in either Postscript, VRML or RNAML formats. The application RNAMLview can be used to rearrange various parts of the RNAView 2D diagram to generate a standard representation (like the cloverleaf structure of tRNAs) or any layout desired by the user. A 2D diagram can be rapidly reformatted using RNAMLview since all the parts of RNA (like helices and single strands) are dynamically linked while moving the selected parts. With the base pair annotation and the 2D graphic display, RNA motifs are rapidly identified and classified. A survey has been carried out for 41 unique structures selected from the NDB database. The statistics for the occurrence of each edge and of each of the 12 bp families are given for the combinations of the four bases: A, G, U and C. The program also allows for visualization of the base pair interactions by using a symbolic convention previously proposed for base pairs. The web servers for BPViewer and RNAview are available at http://ndbserver.rutgers.edu/services/. The application RNAMLview can also be downloaded from this site. The 2D diagrams produced by RNAview are available for RNA structures in the Nucleic Acid Database (NDB) at http://ndbserver.rutgers.edu/atlas/.</td>
</tr>
<tr id="bib_yang2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{yang2003,
  author = {H. Yang and F. Jossinet},
  title = {Tools for the Automatic Identification and Classification of RNA base pairs},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  number = {13},
  pages = {3450-3460},
  doi = {http://dx.doi.org/10.1093/nar/gkg529}
}
</pre></td>
</tr>
<tr id="yao2006" class="entry">
	<td>Yao, Z., Weinberg, Z. &amp; Ruzzo, W. L.</td>
	<td>CMfinder--A Covariance Model Based RNA Motif Finding Algorithm <p class="infolinks">[<a href="javascript:toggleInfo('yao2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('yao2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Bioinformatics &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/bioinformatics/btk008">DOI</a><a href="http://dx.doi.org/10.1093/bioinformatics/btk008">URL</a> &nbsp;</td>
</tr>
<tr id="abs_yao2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MOTIVATION: The recent discoveries of large numbers of non-coding RNAs and computational advances in genome-scale RNA search create a need for tools for automatic, high quality identification and characterization of conserved RNA motifs that can be readily used for database search. Previous tools fall short of this goal. RESULTS: CMfinder is a new tool to predict RNA motifs in unaligned sequences. It is an expectation maximization algorithm using covariance models for motif description, featuring novel integration of multiple techniques for effective search of motif space, and a Bayesian framework that blends mutual information-based and folding energy-based approaches to predict structure in a principled way. Extensive tests show that our method works well on datasets with either low or high sequence similarity, is robust to inclusion of lengthy extraneous flanking sequence and/or completely unrelated sequences, and is reasonably fast and scalable. In testing on 19 known ncRNA families, including some difficult cases with poor sequence conservation and large indels, our method demonstrates excellent average per-base-pair accuracy--79% compared with at most 60% for alternative methods. More importantly, the resulting probabilistic model can be directly used for homology search, allowing iterative refinement of structural models based on additional homologs. We have used this approach to obtain highly accurate covariance models of known RNA motifs based on small numbers of related sequences, which identified homologs in deeply-diverged species.</td>
</tr>
<tr id="bib_yao2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{yao2006,
  author = {Zizhen Yao and Zasha Weinberg and Walter L Ruzzo},
  title = {CMfinder--A Covariance Model Based RNA Motif Finding Algorithm},
  journal = {Bioinformatics},
  year = {2006},
  volume = {22},
  number = {4},
  pages = {445-452},
  url = {http://dx.doi.org/10.1093/bioinformatics/btk008},
  doi = {http://dx.doi.org/10.1093/bioinformatics/btk008}
}
</pre></td>
</tr>
<tr id="younger2009" class="entry">
	<td>Younger, S. T. &amp; Corey, D. R.</td>
	<td>The puzzle of RNAs that target gene promoters. <p class="infolinks">[<a href="javascript:toggleInfo('younger2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('younger2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Chembiochem &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1002/cbic.200900015">DOI</a><a href="http://dx.doi.org/10.1002/cbic.200900015">URL</a> &nbsp;</td>
</tr>
<tr id="abs_younger2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Setting the right target: Most researchers who use small RNAs in mammalian cells assume that mRNA will be the target. Recent studies suggest that small RNAs can also target chromosomal DNA. Few discoveries have had the impact of RNAi. Most researchers who use small RNAs to control gene expression in mammalian cells assume that mRNA will be the target. Recent studies, however, have suggested that small RNAs can also target chromosomal DNA.</td>
</tr>
<tr id="bib_younger2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{younger2009,
  author = {Scott T Younger and David R Corey},
  title = {The puzzle of RNAs that target gene promoters.},
  journal = {Chembiochem},
  year = {2009},
  volume = {10},
  number = {7},
  pages = {1135--1139},
  url = {http://dx.doi.org/10.1002/cbic.200900015},
  doi = {http://dx.doi.org/10.1002/cbic.200900015}
}
</pre></td>
</tr>
<tr id="yusupov2001" class="entry">
	<td>Yusupov, M. M., Yusupova, G. Z., Baucom, A., Lieberman, K., Earnest, T. N., J., H. D. C. &amp; Noller, H. F.</td>
	<td>Crystal Structure of the Ribosome at 5.5 &Aring; Resolution <p class="infolinks">[<a href="javascript:toggleInfo('yusupov2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('yusupov2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1126/science.1060089">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_yusupov2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We describe the crystal structure of the complete Thermus thermophilus 70S ribosome containing bound messenger RNA and transfer RNAs (tRNAs) at 5.5 angstrom resolution. All of the 16S, 23S, and 5S ribosomal RNA (rRNA) chains, the A-, P-, and E-site tRNAs, and most of the ribosomal proteins can be fitted to the electron density map. The core of the interface between the 30S small subunit and the 50S large subunit, where the tRNA substrates are bound, is dominated by RNA, with proteins located mainly at the periphery, consistent with ribosomal function being based on rRNA. In each of the three tRNA binding sites, the ribosome contacts all of the major elements of tRNA, providing an explanation for the conservation of tRNA structure. The tRNAs are closely juxtaposed with the intersubunit bridges, in a way that suggests coupling of the 20 to 50 angstrom movements associated with tRNA translocation with intersubunit movement.</td>
</tr>
<tr id="bib_yusupov2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{yusupov2001,
  author = {Marat M. Yusupov and Gulnara Zh. Yusupova and Albion Baucom and Kate Lieberman and Thomas N. Earnest and H. D. Cate J. and Harry F. Noller},
  title = {Crystal Structure of the Ribosome at 5.5 &Aring; Resolution},
  journal = {Science},
  year = {2001},
  volume = {292},
  pages = {883-896},
  doi = {http://dx.doi.org/10.1126/science.1060089}
}
</pre></td>
</tr>
<tr id="zacharias2000a" class="entry">
	<td>Zacharias, M.</td>
	<td>Simulation of the Structure and Dynamics of Nonhelical RNA Motifs <p class="infolinks">[<a href="javascript:toggleInfo('zacharias2000a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zacharias2000a','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(00)00089-0">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_zacharias2000a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Computer simulation methods are increasingly being used to study possible conformations and dynamics of structural motifs in RNA. Recent results of molecular dynamics simulations and continum solvent studies of RNA structures and RNA-ligand complexes show promising agreement with experimental data. Combined with the ongoing progress in the experimental characterization of RNA structure and thermodynamics, these computational approaches can help to better understand the mechanism of RNA structure formation and the binding of ligands.</td>
</tr>
<tr id="bib_zacharias2000a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zacharias2000a,
  author = {M. Zacharias},
  title = {Simulation of the Structure and Dynamics of Nonhelical RNA Motifs},
  journal = {Current Opinion in Structural Biology},
  year = {2000},
  volume = {10},
  number = {3},
  pages = {311-317},
  doi = {http://dx.doi.org/10.1016/S0959-440X(00)00089-0}
}
</pre></td>
</tr>
<tr id="zagrovic2001" class="entry">
	<td>Zagrovic, B., Sorin, E. J. &amp; Pande, V.</td>
	<td>B-Hairpin Folding Simulations in Atomistic Detail Using an Implicit Solvent Model <p class="infolinks">[<a href="javascript:toggleInfo('zagrovic2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zagrovic2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Journal of Molecular Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jmbi.2001.5033">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_zagrovic2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We have used distributed computing techniques and a supercluster of<p>thousands of computer processors to study folding of the C-terminal<p>b-hairpin from protein G in atomistic detail using the GB/SA implicit<p>solvent model at 300 K. We have simulated a total of nearly 38 ms of<p>folding time and obtained eight complete and independent folding trajectories.<p>Starting from an extended state, we observe relaxation to an<p>unfolded state characterized by non-speciï¿½c, temporary hydrogen<p>bonding. This is followed by the appearance of interactions between<p>hydrophobic residues that stabilize a bent intermediate. Final formation<p>of the complete hydrophobic core occurs cooperatively at the same time<p>that the ï¿½nal hydrogen bonding pattern appears. The folded hairpin<p>structures we observe all contain a closely packed hydrophobic core and<p>proper b-sheet backbone dihedral angles, but they differ in backbone<p>hydrogen bonding pattern. We show that this is consistent with the existing<p>experimental data on the hairpin alone in solution. Our analysis also<p>reveals short-lived semi-helical intermediates which deï¿½ne a thermodynamic<p>trap. Our results are consistent with a three-state mechanism with a<p>single rate-limiting step in which a varying ï¿½nal hydrogen bond pattern<p>is apparent, and semi-helical off-pathway intermediates may appear early<p>in the folding process. We include details of the ensemble dynamics<p>methodology and a discussion of our achievements using this new computational<p>device for studying dynamics at the atomic level.</td>
</tr>
<tr id="bib_zagrovic2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zagrovic2001,
  author = {Bojan Zagrovic and Eric J. Sorin and Vijay Pande},
  title = {B-Hairpin Folding Simulations in Atomistic Detail Using an Implicit Solvent Model},
  journal = {Journal of Molecular Biology},
  year = {2001},
  volume = {313},
  pages = {151-169},
  doi = {http://dx.doi.org/10.1006/jmbi.2001.5033}
}
</pre></td>
</tr>
<tr id="zhang2003" class="entry">
	<td>Zhang, Y. &amp; Crothers, D. M.</td>
	<td>Statistical mechanics of sequence-dependent circular DNA and its application for DNA cyclization. <p class="infolinks">[<a href="javascript:toggleInfo('zhang2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zhang2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Biophysical Journal &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.biophysj.org/cgi/content/full/84/1/136">URL</a> &nbsp;</td>
</tr>
<tr id="abs_zhang2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: DNA cyclization is potentially the most powerful approach for systematic quantitation of sequence-dependent DNA bending and flexibility. We extend the statistical mechanics of the homogeneous DNA circle to a model that considers discrete basepairs, thus allowing for inhomogeneity, and apply the model to analysis of DNA cyclization. The theory starts from an iterative search for the minimum energy configuration of circular DNA. Thermodynamic quantities such as the J factor, which is essentially the ratio of the partition functions of circular and linear forms, are evaluated by integrating the thermal fluctuations around the configuration under harmonic approximation. Accurate analytic expressions are obtained for equilibrium configurations of homogeneous circular DNA with and without bending anisotropy. J factors for both homogeneous and inhomogeneous DNA are evaluated. Effects of curvature, helical repeat, and bending and torsional flexibility in DNA cyclization are analyzed in detail, revealing that DNA cyclization can detect as little as one degree of curvature and a few percent change in flexibility. J factors calculated by our new approach are well consistent with Monte Carlo simulations, whereas the new theory has much greater efficiency in computations. Simulation of experimental results has been demonstrated.</td>
</tr>
<tr id="bib_zhang2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zhang2003,
  author = {Y. Zhang and Donald M. Crothers},
  title = {Statistical mechanics of sequence-dependent circular DNA and its application for DNA cyclization.},
  journal = {Biophysical Journal},
  year = {2003},
  volume = {84},
  pages = {136-153},
  url = {http://www.biophysj.org/cgi/content/full/84/1/136}
}
</pre></td>
</tr>
<tr id="zhuang2003" class="entry">
	<td>Zhuang, X. &amp; Rief, M.</td>
	<td>Single-Molecule Folding <p class="infolinks">[<a href="javascript:toggleInfo('zhuang2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zhuang2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Current Opinion in Structural Biology &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0959-440X(03)00011-3">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_zhuang2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Recent developments in fluorescence and force spectroscopy enable us to go beyond the ensemble average and measure the behavior of individual biomacromolecules. These single-molecule approaches can directly resolve transient intermediate states and multiple reaction pathways, and thus are uniquely powerful in characterizing the complex dynamics of biological processes. Recent applications of these two techniques to the protein and RNA folding problems have led to exciting new results.</td>
</tr>
<tr id="bib_zhuang2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zhuang2003,
  author = {Xiaowei Zhuang and Matthias Rief},
  title = {Single-Molecule Folding},
  journal = {Current Opinion in Structural Biology},
  year = {2003},
  volume = {13},
  pages = {88-97},
  doi = {http://dx.doi.org/10.1016/S0959-440X(03)00011-3}
}
</pre></td>
</tr>
<tr id="zhurkin2005" class="entry">
	<td>Zhurkin, V. B., Tolstorukov, M. Y., Xu, F., Colasanti, A. V. &amp; Olson, W. K.</td>
	<td>Sequence-Dependent Variability of B-DNA: <p class="infolinks">[<a href="javascript:toggleInfo('zhurkin2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td> &nbsp;</td>
	<td>book</td>
	<td> &nbsp;</td>
</tr>
<tr id="bib_zhurkin2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@book{zhurkin2005,
  author = {V. B. Zhurkin and M. Y. Tolstorukov and F. Xu and A. V. Colasanti and W. K. Olson},
  title = {Sequence-Dependent Variability of B-DNA:},
  year = {2005}
}
</pre></td>
</tr>
<tr id="zuker2003" class="entry">
	<td>Zuker, M.</td>
	<td>Mfold web server for nucleic acid folding and hybridization prediction <p class="infolinks">[<a href="javascript:toggleInfo('zuker2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zuker2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Nucleic Acids Research &nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/nar/gkg595">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_zuker2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: The abbreviated name, 'mfold web server', describes a number of closely related software applications available on the World Wide Web (WWW) for the prediction of the secondary structure of single stranded nucleic acids. The objective of this web server is to provide easy access to RNA and DNA folding and hybridization software to the scientific community at large. By making use of universally available web GUIs (Graphical User Interfaces), the server circumvents the problem of portability of this software. Detailed output, in the form of structure plots with or without reliability information, single strand frequency plots and 'energy dot plots', are available for the folding of single sequences. A variety of 'bulk' servers give less information, but in a shorter time and for up to hundreds of sequences at once. The portal for the mfold web server is http://www.bioinfo.rpi.edu/applications/mfold. This URL will be referred to as 'MFOLDROOT'.</td>
</tr>
<tr id="bib_zuker2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zuker2003,
  author = {Michael Zuker},
  title = {Mfold web server for nucleic acid folding and hybridization prediction},
  journal = {Nucleic Acids Research},
  year = {2003},
  volume = {31},
  number = {13},
  pages = {3406-3415},
  doi = {http://dx.doi.org/10.1093/nar/gkg595}
}
</pre></td>
</tr>
<tr id="zuker1989" class="entry">
	<td>Zuker, M.</td>
	<td>On Finding all Suboptimal Foldings of an RNA Molecule <p class="infolinks">[<a href="javascript:toggleInfo('zuker1989','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('zuker1989','bibtex')">BibTeX</a>]</p></td>
	<td>1989</td>
	<td>Science &nbsp;</td>
	<td>article</td>
	<td><a href="http://www.jstor.org/view/00368075/ap003631/00a00210/0">URL</a> &nbsp;</td>
</tr>
<tr id="abs_zuker1989" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: An algorithm and a computer program have been prepared for determining RNA secondary structures within any prescribed increment of the computed global minimum free energy. The mathematical problem of determining how well defined a minimum energy folding is can now be solved. All predicted base pairs that can participate in suboptimal structures may be displayed and analyzed graphically. Representative suboptimal foldings are generated by selecting these base pairs one at a time and computing the best foldings that contain them. A distance criterion that ensures that no two structures are "too close" is used to avoid multiple generation of similar structures. Thermodynamic parameters, including free-energy increments for single-base stacking at the ends of helices and for terminal mismatched pairs in interior and hairpin loops, are incorporated into the underlying folding model of the above algorithm.</td>
</tr>
<tr id="bib_zuker1989" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{zuker1989,
  author = {Michael Zuker},
  title = {On Finding all Suboptimal Foldings of an RNA Molecule},
  journal = {Science},
  year = {1989},
  volume = {244},
  pages = {48-52},
  url = {http://www.jstor.org/view/00368075/ap003631/00a00210/0}
}
</pre></td>
</tr>
</tbody>
</table>

<p>
 <small>Created by <a href="http://jabref.sourceforge.net">JabRef</a> on 11/02/2010.</small>
</p>

</body>
</html>

<!-- File generated by JabRef ; Export Filter written by Mark Schenk -->

